{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "import pandas as pd\n",
    "from bs4 import BeautifulSoup"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "nature_api_key = 'f7423b1e5e5cf7154a90dfecbaa9a66b'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "url = 'https://www.biorxiv.org/search/jcode%3Abiorxiv%7C%7Cmedrxiv%20limit_from%3A2023-09-26%20limit_to%3A2023-09-28%20numresults%3A10000%20sort%3Apublication-date%20direction%3Adescending%20format_result%3Astandard'\n",
    "response = requests.get(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nSearch Results | bioRxiv\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nSkip to main content\\n\\n\\n\\n\\n\\n\\n\\n\\n \\n\\n\\n\\n\\n\\n\\n\\nHome\\nAbout\\nSubmit\\nALERTS / RSS\\n \\n\\n\\n\\n\\n\\n\\n\\n\\n\\nSearch for this keyword \\n\\n\\n\\n\\n \\n\\n\\n\\nAdvanced Search \\n\\n\\n\\n\\n\\n\\n \\n \\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\t629 Results\\n \\n\\n\\n\\nfor published between \"26 Sep, 2023  and  28 Sep, 2023\"\\n \\n\\n\\n\\n\\n \\n\\n\\n\\n\\n\\n\\n\\n\\nItems/Page \\n10255075\\n\\n\\nOrder by \\nBest MatchOldest FirstNewest First\\n\\n\\n\\n \\n\\n\\n\\n\\nSearch Terms & Keywords \\n\\nType a term to search within all articles in this journal: e.g., stem cell\\n\\nLimit ResultsDate of Publication\\nFrom \\n\\nDate \\n\\n e.g., 2023-09-28\\n\\n\\n\\n — \\nThrough \\n\\nDate \\n\\n e.g., 2023-09-28\\n\\n\\n\\n\\nInclude articles in bioRxiv and/or medRxiv: \\nbioRxivbioRxiv and medRxivmedRxiv\\n\\n\\nInclude articles in subject area(s) \\n-- All Collections --Animal Behavior and CognitionBiochemistryBioengineeringBioinformaticsBiophysicsCancer BiologyCell BiologyClinical TrialsDevelopmental BiologyEcologyEpidemiologyEvolutionary BiologyGeneticsGenomicsImmunologyMicrobiologyMolecular BiologyNeurosciencePaleontologyPathologyPharmacology and ToxicologyPhysiologyPlant BiologyScientific Communication and EducationSynthetic BiologySystems BiologyZoology\\n\\n\\nInclude articles in Table of Contents section: \\n-- All Sections --New ResultsContradictory ResultsConfirmatory Results\\n\\n\\nInclude articles in Series \\n-- All Series --\\n\\n\\nInclude Only: \\n\\n  Open Access Articles \\n\\n\\n  Review Articles \\n\\n\\n\\n\\nAuthors, KeywordsSearch for specific authors and/or words and phrases.\\nAuthor \\n\\ne.g., Smith, JS\\n\\n\\nAuthor \\n\\ne.g., Smith, JS\\n\\n\\nTitle \\n\\nType any phrase that appears in the article title\\n\\n\\nwords \\n\\n any \\n\\n\\n all \\n\\n\\n phrase \\n\\n\\n\\n\\nAbstract or Title \\n\\nType any phrase that appears within article title or abstract\\n\\n\\nwords \\n\\n any \\n\\n\\n all \\n\\n\\n phrase \\n\\n\\n\\n\\nFull Text or Abstract or Title \\n\\nType any phrase that appears within article body, title or abstract\\n\\n\\nwords \\n\\n any \\n\\n\\n all \\n\\n\\n phrase \\n\\n\\n\\n\\nBibliography \\n\\ne.g., Smith, JS\\n\\n\\nwords \\n\\n any \\n\\n\\n all \\n\\n\\n phrase \\n\\n\\n\\n\\nPublisher \\n - All Publishers - \\nBook publisher name\\n\\n\\nCitationCitation-specific search information\\nYear \\n\\ne.g., 2009\\n\\n\\nVolume \\n\\ne.g., 20\\n\\n\\nIssue \\n\\ne.g., 3\\n\\n\\nFirst page \\n\\ne.g., 29\\n\\n\\nDOI \\n\\ne.g., 10.9999/123XYZ456\\n\\n\\nISBN \\n\\n\\n\\nFormat Results\\nNumber of results per page \\n10255075\\n\\n\\nSort \\n\\n Best Match \\n\\n\\n Oldest First \\n\\n\\n Newest First \\n\\n\\n\\n\\nFormat Results \\n\\n Standard Format \\n\\n\\n Condensed Format \\n\\n\\n\\n\\n\\n Exclude meeting abstracts \\n\\n\\n\\n\\n \\n\\n\\n\\n\\n\\nFrass fertilizers from mass-reared insects: species variation, heat treatment effects, and implications for soil application \\nNadine Praeg, Thomas Klammsteiner\\nbioRxiv 2023.09.28.559882; doi: https://doi.org/10.1101/2023.09.28.559882 \\n Add to Selected Citations\\n\\n\\n\\nAnalysis of nasopharyngeal microbiome patterns in Zambian infants with fatal acute febrile illness \\nAubrey Rose Odom, Jessica McClintock, Christopher J. Gill, Rachel Pieciak, Arshad Ismail, William B. MacLeod, W. Evan Johnson, Rotem Lapidot\\nbioRxiv 2023.09.27.559805; doi: https://doi.org/10.1101/2023.09.27.559805 \\n Add to Selected Citations\\n\\n\\n\\nAltered motility in response to iron-limitation is regulated by lpdA in uropathogenic E. coli CFT073 \\nArwen E Frick-Cheng, Allyson Shea, Jacquelyn R Roberts, Sara N Smith, Melanie Ohi, Harry L.T. Mobley\\nbioRxiv 2023.09.27.559868; doi: https://doi.org/10.1101/2023.09.27.559868 \\n Add to Selected Citations\\n\\n\\n\\nMapping the molecular motions of 5-HT3 serotonin-gated channel by Voltage-Clamp Fluorometry \\nLaurie Peverini, Sophie Shi, Karima Medjebeur, Pierre-Jean Corringer\\nbioRxiv 2023.09.28.559960; doi: https://doi.org/10.1101/2023.09.28.559960 \\n Add to Selected Citations\\n\\n\\n\\nRequirement of microtubules for secretion of a micronemal protein CpTSP4 in the invasive stage of the apicomplexan Cryptosporidium parvum \\nDongqiang Wang, Peng Jiang, Xiaodong Wu, Ying Zhang, Chenchen Wang, Meng Li, Mingxiao Liu, Jigang Yin, Guan Zhu\\nbioRxiv 2023.09.27.559857; doi: https://doi.org/10.1101/2023.09.27.559857 \\n Add to Selected Citations\\n\\n\\n\\nVerbenone and L-arginine from Daucus carota Seeds Attenuate Oxidative Stress in Streptozotocin-Nicotinamide (STZ-NAD)-Induced Diabetic Mice: Evidence from In vitro and In vivo Studies \\nHabibu Tijjani, Umar Ibrahim, Sadiq Tijani, Sadiya Alka, Ahmed Olatunde, Aminu Umar Kura, Haladu Ali Gagman, Oluremi A. Saliu, Oluwafemi A. Idowu, Frank Eromosele Ebhodaghe\\nbioRxiv 2023.09.28.559552; doi: https://doi.org/10.1101/2023.09.28.559552 \\n Add to Selected Citations\\n\\n\\n\\nInefficient transmission of African swine fever virus to sentinel pigs from environmental contamination under experimental conditions \\nAnn Sofie Olesen, Louise Lohse, Francesc Accensi, Hannah Goldswain, Graham J. Belsham, Anette Bøtner, Christopher Netherton, Linda Kathleen Dixon, Raquel Portugal\\nbioRxiv 2023.09.28.559902; doi: https://doi.org/10.1101/2023.09.28.559902 \\n Add to Selected Citations\\n\\n\\n\\nComparative reservoir competence of Peromyscus leucopus and mouse models for Borrelia burgdorferi B31 \\nJeffrey Bourgeois Jr., Stephanie S You, Luke H Clendenen, Muskan Shrestha, Tanja Petnicki-Ocwieja, Sam Telford III, Linden Hu\\nbioRxiv 2023.09.28.559638; doi: https://doi.org/10.1101/2023.09.28.559638 \\n Add to Selected Citations\\n\\n\\n\\nPolysaccharides from Balanophora harlandii Hook: Isolation, Characterization, and Anti-Inflammation Activities \\nYuanyang Li, Xueqing Li, Qi Yuan, Leiqi Zhu, Fangqi Xia, Yaqi Wang, Mengzhen Xue, Yumin He, Chengfu Yuan\\nbioRxiv 2023.09.27.559774; doi: https://doi.org/10.1101/2023.09.27.559774 \\n Add to Selected Citations\\n\\n\\n\\nPan-tissue scaling of stiffness versus fibrillar collagen reflects contractility-driven strain that inhibits fibril degradation \\nKaranvir Saini, Sangkyun Cho, Manu Tewari, Abdalziz Jalil, Mai Wang, Alexander Kasznel, Kazuhiro Yamamoto, David M. Chenoweth, dennis discher\\nbioRxiv 2023.09.27.559759; doi: https://doi.org/10.1101/2023.09.27.559759 \\n Add to Selected Citations\\n\\n\\n\\nThe impact of HTLV-1 expression on the 3D structure and expression of host chromatin \\nCharles  R M Bangham, Hiroko Yaguchi, Anat Melamed, Saumya Ramanayake, Helen Kiik, Aviva Witkover\\nbioRxiv 2023.09.28.559884; doi: https://doi.org/10.1101/2023.09.28.559884 \\n Add to Selected Citations\\n\\n\\n\\nCommon food preservatives impose distinct selective pressures on Salmonella Typhimurium planktonic and biofilm populations \\nJustin Abi-Assaf, Emma R Holden, Eleftheria Trampari, Mark A Webber\\nbioRxiv 2023.09.28.559892; doi: https://doi.org/10.1101/2023.09.28.559892 \\n Add to Selected Citations\\n\\n\\n\\nControl of iron homeostasis by a regulatory protein-protein interaction in Bacillus subtilis: The FurA (YlaN) acts as an antirepressor to the ferric uptake regulator Fur \\nLorenz Demann, Rica Bremenkamp, Kolja Stahl, Bjoern Hormes, Robert Warneke, Juri Rappsilber, Joerg Stuelke\\nbioRxiv 2023.09.28.559918; doi: https://doi.org/10.1101/2023.09.28.559918 \\n Add to Selected Citations\\n\\n\\n\\nExposure of Mycobacterium tuberculosis to human alveolar lining fluid shows temporal and strain-specific adaptation to the lung environment \\nAnna Allue Guardia, Andreu Garcia Vilanova, Alyssa M Schami, Angelica M Olmo-Fontanez, Amberlee Hicks, Jay Peters, Diego J Maselli, Mark D Wewers, Yufeng Wang, Jordi B Torrelles\\nbioRxiv 2023.09.27.559381; doi: https://doi.org/10.1101/2023.09.27.559381 \\n Add to Selected Citations\\n\\n\\n\\nEvidence of antigenic drift in the fusion machinery core of SARS-CoV-2 spike \\nTimothy J.C. Tan, Abby Odle, Ruipeng Lei, Kenneth A Matreyek, Stanley Perlman, Lok-Yin Roy Wong, Nicholas C. Wu\\nbioRxiv 2023.09.27.559757; doi: https://doi.org/10.1101/2023.09.27.559757 \\n Add to Selected Citations\\n\\n\\n\\nA Type VI Secretion System in Burkholderia Species cenocepacia and orbicola Triggers Distinct Macrophage Death Pathways Independent of the Pyrin Inflammasome \\nNicole A. Loeven, Arianna D. Reuven, Abigail P. McGee, Clarrisa Dabi, Bethany W. Mwaura, James B. Bliska\\nbioRxiv 2023.09.28.559184; doi: https://doi.org/10.1101/2023.09.28.559184 \\n Add to Selected Citations\\n\\n\\n\\nEfficient inhibition of fusion inhibitor HY3000 peptide to SARS-CoV-2 emerging EG.5, EG.5.1 and BA.2.86 variants \\nLili Wu, Anqi Zheng, Yangming Tang, Xiaoyun Wang, Yue Gao, Wenwen Lei, Guizhen Wu, Qihui Wang, George F. Gao\\nbioRxiv 2023.09.28.559747; doi: https://doi.org/10.1101/2023.09.28.559747 \\n Add to Selected Citations\\n\\n\\n\\nThe effects of warming on the stability of consumer-resource interactions \\nAlexis D. Synodinos, Arnaud Sentis, Jose M Montoya, Bart Haegeman\\nbioRxiv 2023.09.27.559670; doi: https://doi.org/10.1101/2023.09.27.559670 \\n Add to Selected Citations\\n\\n\\n\\nDeep Variational Autoencoders for Population Genetics \\nMargarita Geleta, Daniel Mas Montserrat, Xavier Giro-i-Nieto, Alexander G Ioannidis\\nbioRxiv 2023.09.27.558320; doi: https://doi.org/10.1101/2023.09.27.558320 \\n Add to Selected Citations\\n\\n\\n\\nEngineered retrons generate genome-independent protein-binding DNA for cellular control \\nGeonhu Lee, Jongmin Kim\\nbioRxiv 2023.09.27.556556; doi: https://doi.org/10.1101/2023.09.27.556556 \\n Add to Selected Citations\\n\\n\\n\\nInhibiting Glutamine Metabolism Blocks Coronavirus Replication in Mammalian Cells \\nKai Su Greene, Annette Choi, Matthew Chen, Nianhui Yang, Ruizhi Li, Yijian Qiu, Michael J Lukey, Katherine S Rojas, Jonathan Shen, Kristin F Wilson, William P Katt, Gary R Whittaker, Richard A Cerione\\nbioRxiv 2023.09.27.559756; doi: https://doi.org/10.1101/2023.09.27.559756 \\n Add to Selected Citations\\n\\n\\n\\nInhibition of SARS-CoV-2 Infection in Human Airway Epithelium with a Xeno-Nucleic Acid Aptamer \\nNiayesh Razi, Weizhong Li, Maxinne A. Ignacio, Jeffrey M. Loube, Eva L. Agostino, Xiaoping Zhu, Margaret A. Scull, Jeffrey J. DeStefano\\nbioRxiv 2023.09.27.559799; doi: https://doi.org/10.1101/2023.09.27.559799 \\n Add to Selected Citations\\n\\n\\n\\nCharacterizing patterns of selection pressure on mammalian antiviral immune response \\nMohamed Hawash\\nbioRxiv 2023.09.26.559440; doi: https://doi.org/10.1101/2023.09.26.559440 \\n Add to Selected Citations\\n\\n\\n\\nIntrinsic disorder in flaviviral capsid proteins and their role in pathogenesis \\nAnirudh Sundar, Pavithra Umashankar, Priyanka Sankar, Kavitha Ramasamy, sangita venkataraman\\nbioRxiv 2023.09.26.559664; doi: https://doi.org/10.1101/2023.09.26.559664 \\n Add to Selected Citations\\n\\n\\n\\nUnveiling the enigma of Brain-resident immune cells \\nSunsook Hwang, Juneill Jang, Kyungsoo Park, Yeong Shin Yim\\nbioRxiv 2023.09.26.559602; doi: https://doi.org/10.1101/2023.09.26.559602 \\n Add to Selected Citations\\n\\n\\n\\nPhased assembly of neo-sex chromosomes reveals extensive Y degeneration and rapid genome evolution in Rumex hastatulus \\nBianca M. Sacchi, Zoë Humphries, Janka Kružlicová, Markéta Bodláková, Cassandre Pyne, Baharul Choudhury, Yunchen Gong, Václav Bačovský, Roman Hobza, Spencer C.H. Barrett, Stephen I. Wright\\nbioRxiv 2023.09.26.559509; doi: https://doi.org/10.1101/2023.09.26.559509 \\n Add to Selected Citations\\n\\n\\n\\nTesting for a role of postzygotic incompatibilities in rapidly speciated Lake Victoria cichlids \\nAnna F Feller, Catherine L Peichel, Ole Seehausen\\nbioRxiv 2023.09.26.559612; doi: https://doi.org/10.1101/2023.09.26.559612 \\n Add to Selected Citations\\n\\n\\n\\nMolecular Dynamics Activation of γ-Secretase for Cleavage of Notch1 Substrate \\nHung N Do, Shweta R Malvankar, Michael S Wolfe, Yinglong Miao\\nbioRxiv 2023.09.26.559539; doi: https://doi.org/10.1101/2023.09.26.559539 \\n Add to Selected Citations\\n\\n\\n\\nLoss of embryonically-derived Kupffer cells during hypercholesterolemia accelerates atherosclerosis development \\nRebecca Fima, Sebastien Dussaud, Cheima Benbida, Margault Blanchet, Francois Lanthiez, Lucie Poupel, Claudia Brambilla, Martine Moreau, Alexandre Boissonnas, Emmanuel Gautier, Thierry Huby\\nbioRxiv 2023.09.26.559586; doi: https://doi.org/10.1101/2023.09.26.559586 \\n Add to Selected Citations\\n\\n\\n\\nMutation rate heterogeneity at the sub-gene scale due to local DNA hypomethylation \\nDavid Mas-Ponte, Fran Supek\\nbioRxiv 2023.09.26.559585; doi: https://doi.org/10.1101/2023.09.26.559585 \\n Add to Selected Citations\\n\\n\\n\\nLymphoma B cells remodel bone marrow stromal cell organization and function to induce a supportive cancer-associated fibroblast network \\nElise Dessauge, Baptiste Brauge, Simon Leonard, David Roulois, Celine Monvoisin, Thomas Lejeune, Jerome Destin, Florence Jouan, Francisco Llamas-Gutierrez, Frederic Mourcin, Karin TARTE\\nbioRxiv 2023.09.26.559605; doi: https://doi.org/10.1101/2023.09.26.559605 \\n Add to Selected Citations\\n\\n\\n\\nMultiepitope-based vaccine design against DiiA in Streptococcus pneumoniae, An immunoinformatics approach. \\nMariam Soliman, Jihan Nassar, Dania Sheikhani, Sherif Elsabbagh, Tamer M. Ibrahim\\nbioRxiv 2023.09.26.559565; doi: https://doi.org/10.1101/2023.09.26.559565 \\n Add to Selected Citations\\n\\n\\n\\nHuman enteric glia diversity in health and disease: new avenues for the treatment of Hirschsprung disease \\nJonathan D. Windster, Laura E. Kuil, Naomi J.M. Kakiailatu, Agne Antanaviciute, Andrea Sacchetti, Katherine C. MacKenzie, Joke Peulen-Zink, Tsung Wai Kan, Eric Bindels, Emma de Pater, Michael Doukas, Soheil Yousefi, Tahsin Stefan Barakat, Conny J.H.M. Meeussen, Pim C.E.J. Sloots, Renee M.H. Wijnen, Kaushal B. Parikh, Werend Boesmans, Veerle Melotte, Robert M.W. Hofstra, Alison Simmons, Maria Alves\\nbioRxiv 2023.09.26.559481; doi: https://doi.org/10.1101/2023.09.26.559481 \\n Add to Selected Citations\\n\\n\\n\\nThe invasive lobular cancer cell line encyclopedia (ICLE) for studying potential biomarkers and exploring therapeutic opportunities \\nOsama Shiraz Shah, Fangyuan Chen, Abdalla Webn, Anokhi Kashiparekh, Laura Savariau, Benjamin Clifford, Jagmohan Hooda, Jennifer M Atkinson, Steffi Oesterreich, Adrian V Lee\\nbioRxiv 2023.09.26.559548; doi: https://doi.org/10.1101/2023.09.26.559548 \\n Add to Selected Citations\\n\\n\\n\\nMapping the anatomical and transcriptional landscape of early human fetal ovary development \\nSinead M McGlacken-Byrne, Ignacio Del Valle, Theodoros Xenakis, Ian C Simcock, Jenifer P Suntharalingham, Federica Buonocore, Berta Crespo, Nadjeda Moreno, Daneille Liptrot, Paola Niola, Tony Brooks, Gerard S Conway, Mehul T Dattani, Owen Arthurs, Nita Solanky, John Achermann\\nbioRxiv 2023.09.26.558771; doi: https://doi.org/10.1101/2023.09.26.558771 \\n Add to Selected Citations\\n\\n\\n\\nTuning electrical spiking of Schizophyllum commune with light \\nDavin Browner, Andrew Adamatzky, Ella Schunselaar, Han Wösten, Phil Ayres\\nbioRxiv 2023.09.26.559269; doi: https://doi.org/10.1101/2023.09.26.559269 \\n Add to Selected Citations\\n\\n\\n\\nPan tissue analysis of human NUMB alternative splicing. \\nYangjing Zhang, Catherine J McGlade\\nbioRxiv 2023.09.26.559629; doi: https://doi.org/10.1101/2023.09.26.559629 \\n Add to Selected Citations\\n\\n\\n\\nATF2 phosphorylation is a key event in neuronal apoptosis, linking the DLK/LZK kinase cascade to JUN upregulation \\nJorge Gomez Deza, Matthew Nebiyou, Mor R. Alkaslasi, Lara H. El Touny, Anastasia L. Slavutsky, Nathan Ashby, Ken Cheng, Claire E. Le Pichon\\nbioRxiv 2023.09.27.559856; doi: https://doi.org/10.1101/2023.09.27.559856 \\n Add to Selected Citations\\n\\n\\n\\nGeneCompass: Deciphering Universal Gene Regulatory Mechanisms with Knowledge-Informed Cross-Species Foundation Model \\nXiaodong Yang, Guole Liu, Guihai Feng, Dechao Bu, Pengfei Wang, Jie Jiang, Shubai Chen, Qinmeng Yang, Yiyang Zhang, Zhenpeng Man, Zhongming Liang, Zichen Wang, Yaning Li, Zheng Li, Yana Liu, Yao Tian, Ao Li, Jingxi Dong, Zhilong Hu, Chen Fang, Hefan Miao, Lina Cui, Zixu Deng, Haiping Jiang, Wentao Cui, Jiahao Zhang, Zhaohui Yang, Handong Li, Xingjian He, Liqun Zhong, Jiaheng Zhou, Zijian Wang, Qingqing Long, Ping Xu, Hongmei Wang, Zhen Meng, Xuezhi Wang, Yangang Wang, Yong Wang, Shihua Zhang, Jingtao Guo, Yi Zhao, Yuanchun Zhou, Fei Li, Jing Liu, Yiqiang Chen, Ge Yang, Xin Li\\nbioRxiv 2023.09.26.559542; doi: https://doi.org/10.1101/2023.09.26.559542 \\n Add to Selected Citations\\n\\n\\n\\nThreshold public goods game in well mixed populations: Cooperation in density independent and density dependent cases \\nAdel Karolyi, Istvan Scheuring\\nbioRxiv 2023.09.26.559593; doi: https://doi.org/10.1101/2023.09.26.559593 \\n Add to Selected Citations\\n\\n\\n\\nEvolution of gene expression across brain regions in behaviorally divergent deer mice \\nAndreas Kautt, Jenny Chen, Caitlin L. Lewarch, Caroline Hu, Kyle Turner, Jean-Marc Lassance, Felix Baier, Nicole L. Bedford, Andres Bendesky, Hopi E. Hoekstra\\nbioRxiv 2023.09.28.559844; doi: https://doi.org/10.1101/2023.09.28.559844 \\n Add to Selected Citations\\n\\n\\n\\nEstimating limit of detection of M. tuberculosis DNA as an analyte released from a chemically coated solid matrix and using a WHO-approved CB-NAAT platform. \\nKrishna Himmatbhai Goyani, Pratap Narayan Mukhopadhyaya\\nbioRxiv 2023.09.26.558062; doi: https://doi.org/10.1101/2023.09.26.558062 \\n Add to Selected Citations\\n\\n\\n\\nPan-viral ORFs discovery using Massively Parallel Ribosome Profiling \\nShira Weingarten-Gabbay, Matthew R Bauer, Alexandra C Stanton, Susan Klaeger, Eva K Verzani, Daniel Lopez, Karl R Clauser, Steven A Carr, Jennifer G Abelin, Charles M Rice, Pardis C Sabeti\\nbioRxiv 2023.09.26.559641; doi: https://doi.org/10.1101/2023.09.26.559641 \\n Add to Selected Citations\\n\\n\\n\\nMARCO in alveolar macrophages negatively regulates Ace expression and aldosterone production \\nConan J O O\\'Brien, Giorgio Ratti, Emma R Haberman, Charles A P Sweeney, Siamon Gordon, Ana I Domingos\\nbioRxiv 2023.09.26.559578; doi: https://doi.org/10.1101/2023.09.26.559578 \\n Add to Selected Citations\\n\\n\\n\\nIntegrating Shear Flow and Trypsin Treatment to Assess Cell Adhesion Strength \\nAntra Patel, Bhavana Bhavanam, Trevor Keenan, Venkat Maruthamuthu\\nbioRxiv 2023.09.26.559598; doi: https://doi.org/10.1101/2023.09.26.559598 \\n Add to Selected Citations\\n\\n\\n\\nExpression of the CIC-DUX4 fusion oncoprotein mimics human CIC-rearranged sarcoma in genetically engineered mouse models \\nPeter G Hendrickson, Kristianne M Oristian, MaKenna R Browne, Lixia Lou, Yan Ma, Dianna M Cardona, Corinne M Linardic, David G Kirsch\\nbioRxiv 2023.09.26.559519; doi: https://doi.org/10.1101/2023.09.26.559519 \\n Add to Selected Citations\\n\\n\\n\\nCardiac Specific Overexpression of Adenylyl Cyclase 8 Reprograms the Mouse Sinoatrial Node Transcriptome \\nJiahua Qu, Kirill V. Tarasov, Khalid Chakir, Edward G. Lakatta\\nbioRxiv 2023.09.26.559627; doi: https://doi.org/10.1101/2023.09.26.559627 \\n Add to Selected Citations\\n\\n\\n\\nTEM Experiment and Complete Genome Sequencing with Comparative Genomic Analysis to Uncover Potential Probiotic Features of Isolated Lactobacillus delbrueckii subsp. indicus TY-11 \\nSheik Mohammad Jakaria Al Mujahidy, Kirill Kryukov, Kazuho Ikeo, Kei Saito, Md. Ekhlas Uddin, Abu Ali Ibn Sina\\nbioRxiv 2023.09.26.559573; doi: https://doi.org/10.1101/2023.09.26.559573 \\n Add to Selected Citations\\n\\n\\n\\nBiallelic non-productive enhancer-promoter interaction precedes imprinted expression of Kcnk9 during mouse neural commitment \\nCecilia Rengifo Rojas, Jil Cercy, sophie Perillous, Celine Gonthier-Gueret, Bertille Montibus, Stephanie Maupetit-Mehouas, Astrid Espinadel, Marylou Dupre, Charles C Hong, Kenichiro Hata, Kazuhiko Nakabayashi, Antonius Plagge, Trsitan Bouschet, philippe arnaud, Isabelle Vaillant, Franck Court\\nbioRxiv 2023.09.26.559498; doi: https://doi.org/10.1101/2023.09.26.559498 \\n Add to Selected Citations\\n\\n\\n\\nTargeting EIF4A triggers an interferon response to synergize with chemotherapy and suppress triple-negative breast cancer \\nNa Zhao, Elena Kabotyanski, Alexander Saltzman, Anna Malovannaya, Xueying Yuan, Lucas Reineke, Nadia Lieu, Yang Gao, Diego Pedroza, Sebastian Calderon, Alex Smith, Clark Hamor, Kazem Safari, Sara Savage, Bing Zhang, Jianling Zhou, Luisa Soto, Susan Hilsenbeck, Cheng Fan, Charles Perou, Jeffrey Rosen\\nbioRxiv 2023.09.28.559973; doi: https://doi.org/10.1101/2023.09.28.559973 \\n Add to Selected Citations\\n\\n\\n\\nDirect extrusion of multifascicle prevascularized human skeletal muscle for volumetric muscle loss surgery \\nVan Thuy Duong, Thao Thi Dang, Van Phu Le, Thi Huong Le, Chanh Trung Nguyen, Huu Lam Phan, Jongmo Seo, Sung Hoon Back, Kyo-in Koo\\nbioRxiv 2023.09.26.559212; doi: https://doi.org/10.1101/2023.09.26.559212 \\n Add to Selected Citations\\n\\n\\n\\nBeauty beacon: correlated strategies for the Fisher runaway process \\nDaniil Ryabko, Angustias Vaca, Prudencio Pazoca\\nbioRxiv 2023.09.26.559589; doi: https://doi.org/10.1101/2023.09.26.559589 \\n Add to Selected Citations\\n\\n\\n\\nPeds1 deficiency in zebrafish results in myeloid cell apoptosis and exacerbated inflammation \\nAna B Arroyo, Sylwia D Tyrkalska, Eva Bastida-Martinez, Antonio J Monera-Girona, Joaquin Canton-Sandoval, Martin Bernal-Carrion, Diana Garcia-Moreno, Montserrat Elias-Arnanz, Victoriano Mulero\\nbioRxiv 2023.09.26.559500; doi: https://doi.org/10.1101/2023.09.26.559500 \\n Add to Selected Citations\\n\\n\\n\\nSelf-sustained rhythmic behavior of Synechocystis PCC 6803 under continuous light conditions in the absence of light-dark entrainment \\nLutz C. Berwanger, Nikolaus Thumm, Rahil Gholamipoor, Anika Wiegard, Jeannine Schlebusch, Markus Kollmann, Ilka M. Axmann\\nbioRxiv 2023.09.26.559469; doi: https://doi.org/10.1101/2023.09.26.559469 \\n Add to Selected Citations\\n\\n\\n\\nMitochondrial dsRNA from B-ALL cells stimulates mesenchymal stromal cells to become cancer associated fibroblasts \\nRichard J Burt, Aditi Dey, Ayse Akarca, Hermione Allen, Rodothea Amerikanou, Samantha Atkinson, David Auty, Jenny Chatzigerou, Emily Cutler, Jose Afonso Guerra-Assuncao, Kristina Kirschner, Ruchi Kumari, JIten Manji, Teresa Marafioti, Adele K. Fielding\\nbioRxiv 2023.09.26.559490; doi: https://doi.org/10.1101/2023.09.26.559490 \\n Add to Selected Citations\\n\\n\\n\\nSCancerRNA: Expression at the Single Cell Level and Interaction Resource of Non-coding RNA Biomarkers for Cancers \\nHongzhe Guo, Liyuan Zhang, Xinran Cui, Liang Cheng, Tianyi Zhao, Yadong Wang\\nbioRxiv 2023.09.26.559661; doi: https://doi.org/10.1101/2023.09.26.559661 \\n Add to Selected Citations\\n\\n\\n\\nHuman vaginal microbiota colonization is regulated by female sex 1 hormones in a mouse model \\nNuzhat Rahman, Firoz Mian, Charu Kaushic\\nbioRxiv 2023.09.27.559827; doi: https://doi.org/10.1101/2023.09.27.559827 \\n Add to Selected Citations\\n\\n\\n\\ninDrops-2: a flexible, versatile and cost-efficient droplet microfluidics approach for high-throughput scRNA-seq of fresh and preserved clinical samples \\nLinas Mazutis, Simonas Juzenas, Vaidotas Kiseliovas, Karolis Goda, Justina Zvirblyte, Juozas Nainys, Alvaro Quintinal-Villalonga\\nbioRxiv 2023.09.26.559493; doi: https://doi.org/10.1101/2023.09.26.559493 \\n Add to Selected Citations\\n\\n\\n\\nTransGeneSelector: A Transformer-based Approach Tailored for Key Gene Mining with Small Plant Transcriptomic Datasets \\nKerui Huang, Jianhong Tian, Lei Sun, Peng Xie, Shiqi Zhou, Aihua Deng, Ping Mo, Zhibo Zhou, Ming Jiang, Guiwu Li, Yun Wang, Xiaocheng Jiang\\nbioRxiv 2023.09.26.559592; doi: https://doi.org/10.1101/2023.09.26.559592 \\n Add to Selected Citations\\n\\n\\n\\nPaGeSearch: A Tool for Identifying Genes within Pathways in Unannotated Genomes \\nSohyoung Won, Jaewoong Yu, Heebal Kim\\nbioRxiv 2023.09.26.559665; doi: https://doi.org/10.1101/2023.09.26.559665 \\n Add to Selected Citations\\n\\n\\n\\nCell and tissue-specific glycosylation pathways and transcriptional regulation informed by single-cell transcriptomics \\nPanagiotis Chrysinas, Shriramprasad Venkatesan, Isaac Ang, Vishnu Ghosh, Changyou Chen, Sriram Neelamegham, Rudiyanto Gunawan\\nbioRxiv 2023.09.26.559616; doi: https://doi.org/10.1101/2023.09.26.559616 \\n Add to Selected Citations\\n\\n\\n\\nAddressing preferred orientation in single-particle cryo-EM through AI-generated auxiliary particles. \\nHui Zhang, Dihan Zheng, Qiurong Wu, Nieng Yan, Zuoqiang Shi, Mingxu Hu, Chenglong Bao\\nbioRxiv 2023.09.26.559492; doi: https://doi.org/10.1101/2023.09.26.559492 \\n Add to Selected Citations\\n\\n\\n\\nWhole-brain analysis of CO2 chemosensitive regions and identification of the retrotrapezoid and medullary raphe nuclei in the common marmoset (Callithrix jacchus) \\nAriana Z. Turk, Marissa Millwater, Shahriar SheikhBahaei\\nbioRxiv 2023.09.26.558361; doi: https://doi.org/10.1101/2023.09.26.558361 \\n Add to Selected Citations\\n\\n\\n\\nThe structural landscape and diversity of Pyricularia oryzae MAX effectors revisited \\nMounia Lahfa, Philippe BARTHE, Karine DEGUILLEN, Stella Cesari, Mouna RAJI, Thomas Kroj, Marie Le Naour--Vernet, Francois Hoh, Pierre Gladieux, Christian ROUMESTAND, Jerome Gracy, Nathalie Declerck, Andre Padilla\\nbioRxiv 2023.09.26.559520; doi: https://doi.org/10.1101/2023.09.26.559520 \\n Add to Selected Citations\\n\\n\\n\\nEvaluating goodness-of-fit of animal movement models using lineups \\nJohn Fieberg, Johannes Signer, Smith Freeman\\nbioRxiv 2023.09.26.559591; doi: https://doi.org/10.1101/2023.09.26.559591 \\n Add to Selected Citations\\n\\n\\n\\nIL-2-induced Stat3 Signaling is Critical for Effector Treg Cell Programming \\nEmma C Dean, Daniel F DiToro, Duy Pham, Min Gao, Blake F Frey, Stacey N Harbour, David A Figge, Aidan T Miller, Caleb R Glassman, K Christopher Garcia, Robin D Hatton, Casey Weaver\\nbioRxiv 2023.09.26.559434; doi: https://doi.org/10.1101/2023.09.26.559434 \\n Add to Selected Citations\\n\\n\\n\\nReducing the vicissitudes of heterologous prochiral substrate catalysis by alcohol dehydrogenases through machine learning algorithms \\nArindam Ghatak, Anirudh P Shanbhag, Santanu Datta\\nbioRxiv 2023.09.26.559656; doi: https://doi.org/10.1101/2023.09.26.559656 \\n Add to Selected Citations\\n\\n\\n\\nFungal Pathogen Gene Selection for Predicting the Onset of Infection Using a Multi-Stage Machine Learning Approach \\nGraham Thomas, Oliver Stoner, Fabrizio Costa, Ryan M Ames\\nbioRxiv 2023.09.26.559518; doi: https://doi.org/10.1101/2023.09.26.559518 \\n Add to Selected Citations\\n\\n\\n\\nA simple method to genetically differentiate invasive F1 Typha hybrids (T. x glauca) and advanced-generation/backcrossed hybrids from parent species (T. latifolia and T. angustifolia) in eastern Canada and northeastern USA \\nAvery Chambers, Braidy Chambers, Deven Bhargava, Alberto Aleman, Marcel Dorken, Joanna Freeland\\nbioRxiv 2023.09.26.559561; doi: https://doi.org/10.1101/2023.09.26.559561 \\n Add to Selected Citations\\n\\n\\n\\nTranscriptomic gene profiles in an ex vivo model of erythropoiesis to unravel molecular pathomechanisms in sickle cell disease \\nMatthis Tinguely, Lennart Opitz, Dominik Daniela Schaer, Florence Vallelian, Markus Schmugge, Francesca Daniela Franzoso\\nbioRxiv 2023.09.26.559266; doi: https://doi.org/10.1101/2023.09.26.559266 \\n Add to Selected Citations\\n\\n\\n\\nIntegrated single-cell multiome analysis reveals muscle fiber-type gene regulatory circuitry modulated by endurance exercise \\nAliza B Rubenstein, Gregory R Smith, Zidong Zhang, Xi Chen, Toby L. Chambers, Frederique Ruf-Zamojski, Natalia Mendelev, Wan Sze Cheng, Michel Zamojski, Mary Anne S. Amper, Venugopalan D. Nair, Andrew R. Marderstein, Stephen B. Montgomery, Olga G. Troyanskaya, Elena Zaslavsky, Todd Trappe, Scott Trappe, Stuart C. Sealfon\\nbioRxiv 2023.09.26.558914; doi: https://doi.org/10.1101/2023.09.26.558914 \\n Add to Selected Citations\\n\\n\\n\\nDetermining the effects of temperature on the evolution of bacterial tRNA pools \\nVatsal Jain, Alexander L Cope\\nbioRxiv 2023.09.26.559538; doi: https://doi.org/10.1101/2023.09.26.559538 \\n Add to Selected Citations\\n\\n\\n\\nStableMate: a new statistical method to select stable predictors in omics data \\nYidi Deng, Jiadong Mao, Jarny Choi, Kim-Anh Le Cao\\nbioRxiv 2023.09.26.559658; doi: https://doi.org/10.1101/2023.09.26.559658 \\n Add to Selected Citations\\n\\n\\n\\nDemocratizing access to microfluidics: Rapid prototyping of capillary microfluidics with a low-cost masked stereolithography 3D printer \\nKelsey M Leong, Mindy L Quach, Carrie H Lin, Cosette A Craig, Aileen Y Sun, Felix Guo, Timothy R Robinson, Ayokunle O Olanrewaju\\nbioRxiv 2023.09.26.558021; doi: https://doi.org/10.1101/2023.09.26.558021 \\n Add to Selected Citations\\n\\n\\n\\nCYP79B2 and CYP79B3 contribute to root branching through production of the auxin precursor indole-3-acetonitrile \\nEva van Zelm, Iko Tamar Koevoets, A Jessica Meyer, Kyra van der Velde, Thijs De Zeeuw, francel verstappen, Rens Holmer, Wouter Kohlen, Viola Willemsen, Charlotte M.M. Gommers, Christa Testerink\\nbioRxiv 2023.09.26.559630; doi: https://doi.org/10.1101/2023.09.26.559630 \\n Add to Selected Citations\\n\\n\\n\\nSpecies-specific epigenetic responses to drought stress of two sympatric oak species reflect their ecological preferences. \\nBernadette Rubio, Gregoire Le Provost, Benjamin Brachi, Theo Gerardin, Oliver Brendel, Jorg Tost, Christian Daviaud, Philippe Gallusci\\nbioRxiv 2023.09.26.559529; doi: https://doi.org/10.1101/2023.09.26.559529 \\n Add to Selected Citations\\n\\n\\n\\nIntegrative Transcriptomic Analysis of Anterior and Posterior IOP-Controlling Tissues in Glaucoma Reveals Enrichment of MHC-II Pathway and T-Cell Infiltration Signatures \\nShuxin Liang, Shichen Zhang, Hongqiu Wang, Shaojun Tang\\nbioRxiv 2023.09.26.558362; doi: https://doi.org/10.1101/2023.09.26.558362 \\n Add to Selected Citations\\n\\n\\n\\nGene transfer among viruses substantially contributes to gene gain of giant viruses \\nJunyi Wu, Lingjie Meng, Morgan Gaia, Hiroyuki Hikida, Yusuke Okazaki, Hisashi Endo, Hiroyuki Ogata\\nbioRxiv 2023.09.26.559659; doi: https://doi.org/10.1101/2023.09.26.559659 \\n Add to Selected Citations\\n\\n\\n\\nDeepGO-SE: Protein function prediction as Approximate Semantic Entailment \\nMaxat Kulmanov, Francisco J Guzman-Vega, Paula Duek Roggli, Lydie Lane, Stefan T Arold, Robert Hoehndorf\\nbioRxiv 2023.09.26.559473; doi: https://doi.org/10.1101/2023.09.26.559473 \\n Add to Selected Citations\\n\\n\\n\\nBacteria colonies modify their shear and compressive mechanical properties in response to different growth substrates \\nJakub A Kochanowski, Bobby Carroll, Merrill Asp, Emma Kaputa, Alison Elise Patteson\\nbioRxiv 2023.09.26.559510; doi: https://doi.org/10.1101/2023.09.26.559510 \\n Add to Selected Citations\\n\\n\\n\\nA chromosome-level reference genome of a Fabaceae species yam bean (Pachyrhizus erosus) \\nWei Fan, Xingyao Xiong, Fan Jiang, Lihua Yuan, Caishun Zhang, Fengjiao Bu\\nbioRxiv 2023.09.26.559645; doi: https://doi.org/10.1101/2023.09.26.559645 \\n Add to Selected Citations\\n\\n\\n\\nDetection of short identity by descent segments using low-frequency variants \\nChonghao Wang, Werner Pieter Veldsman, Lu Zhang\\nbioRxiv 2023.09.26.559464; doi: https://doi.org/10.1101/2023.09.26.559464 \\n Add to Selected Citations\\n\\n\\n\\nCancer associated fibroblasts mitigate the efficacy of the combination treatment of chemotherapy and BCL-xL targeting in triple negative breast cancer cells \\nLisa Nocquet, Julie Roul, Laurine Duarte, Mario Campone, Philippe P Juin, Frédérique Souazé\\nbioRxiv 2023.09.26.559287; doi: https://doi.org/10.1101/2023.09.26.559287 \\n Add to Selected Citations\\n\\n\\n\\nEndometrial zinc transporter Slc39a10/Zip10 is indispensable for progesterone responsiveness and successful pregnancy in mice \\nYui KAWATA, Jumpei TERAKAWA, Ayuu TAKESHITA, Takafumi NAMIKI, Atsuko KAGEYAMA, Michiko NOGUCHI, Hironobu MURAKAMI, Toshiyuki FUKADA, Junya ITO, Naomi KASHIWAZAKI\\nbioRxiv 2023.09.26.559476; doi: https://doi.org/10.1101/2023.09.26.559476 \\n Add to Selected Citations\\n\\n\\n\\nValidation of cross progeny variance genomic prediction using simulations and experimental data in winter elite bread wheat \\nClaire Oget-Ebrad, Emmanuel Heumez, Laure Duchalais, Ellen Goudemand-Dugue, Francois-Xavier Oury, Jean-Michel Elsen, Sophie Bouchet\\nbioRxiv 2023.09.26.558758; doi: https://doi.org/10.1101/2023.09.26.558758 \\n Add to Selected Citations\\n\\n\\n\\nBringing circuit theory into spatial occupancy models to assess landscape connectivity \\nMaelis Kervellec, Thibaut Couturier, Sarah Bauduin, Delphine Chenesseau, Pierre Defos du Rau, Nolwenn Drouet-Hoguet, Christophe Duchamp, Julien Steinmetz, Jean-Michel Vandel, Olivier Gimenez\\nbioRxiv 2023.09.26.559320; doi: https://doi.org/10.1101/2023.09.26.559320 \\n Add to Selected Citations\\n\\n\\n\\nFull-Length Single-Molecule Protein Fingerprinting \\nMike Filius, Raman van Wee, Carlos de Lannoy, Ilja Westerlaken, Zhesi Li, Sung Hyun Kim, Cecilia de Agrela Pinto, Yunfei Wu, Geert-Jan Boons, Martin Pabst, Dick de Ridder, Chirlmin Joo\\nbioRxiv 2023.09.26.559471; doi: https://doi.org/10.1101/2023.09.26.559471 \\n Add to Selected Citations\\n\\n\\n\\nThe global expansion from southern Africa by modern humans? Its signal may be shown more through the cranial shape of males than females \\nZarus Cenac\\nbioRxiv 2023.09.25.559024; doi: https://doi.org/10.1101/2023.09.25.559024 \\n Add to Selected Citations\\n\\n\\n\\nMultimodal identification of rare potent effector CD8 T cells in solid tumors \\nArja Ray, Molly Bassette, Kenneth H Hu, Lomax F. Pass, Bushra Samad, Alexis Combes, Vrinda Johri, Brittany Davidson, Grace Hernandez, Itzia Zaleta-Linares, Matthew F Krummel\\nbioRxiv 2023.09.26.559470; doi: https://doi.org/10.1101/2023.09.26.559470 \\n Add to Selected Citations\\n\\n\\n\\nAcceptability and sustainability of a point-of-care HPV ‘self-collect, screen-and-treat’ for cervical cancer prevention in Papua New Guinea: A qualitative exploration of key informants’ perspectives \\nHawa Camara, Somu Nosi, Gloria Munnull, Steven  G Badman, John Bolgna, Joseph Kuk, Glen Mola, Rebecca Guy, Andrew  J Vallely, Angela  M. Kelly-Hanku\\nmedRxiv 2023.09.27.23296249; doi: https://doi.org/10.1101/2023.09.27.23296249 \\n Add to Selected Citations\\n\\n\\n\\nDeep learning models of RNA base-pairing structures generalize to unseen folds and make accurate zero-shot predictions of base-base interactions of RNA complexes \\nmei lang, Thomas Litfin, Ke Chen, Jian Zhan, Yaoqi Zhou\\nbioRxiv 2023.09.26.559463; doi: https://doi.org/10.1101/2023.09.26.559463 \\n Add to Selected Citations\\n\\n\\n\\nCell proliferation and Notch signaling coordinate the formation of epithelial folds in the Drosophila leg \\nAlonso Rodriguez, Sergio Cordoba, Daniel Delipe-Cordero, Antonio Baonza, David Miguez, Carlos Estella\\nbioRxiv 2023.09.28.559944; doi: https://doi.org/10.1101/2023.09.28.559944 \\n Add to Selected Citations\\n\\n\\n\\nThe nuclear basket regulates distribution and mobility of nuclear pore complexes in budding yeast \\nJanka Zsok, Francois Simon, Goeksu Bayrak, Luljeta Isaki, Nina Kerff, Amy Wolstenholme, Lucien E Weiss, Elisa Dultz\\nbioRxiv 2023.09.28.558499; doi: https://doi.org/10.1101/2023.09.28.558499 \\n Add to Selected Citations\\n\\n\\n\\nThe polarity protein Yurt associates with the lateral domain via basic and hydrophobic motifs embedded in its FERM domain \\nCharles Alende, Clemence L. Gamblin, Francois Corriveau, Alexandra Jette, Frederique Parent-Prevost, Cornelia Biehler, Nathalie Majeau, Melanie Laurin, Patrick Laprise\\nbioRxiv 2023.09.28.559937; doi: https://doi.org/10.1101/2023.09.28.559937 \\n Add to Selected Citations\\n\\n\\n\\nBarcoding of small extracellular vesicles with CRISPR-gRNA enables comprehensive, subpopulation-specific analysis of their biogenesis/release regulators \\nKoki Kunitake, Tadahaya Mizuno, Kazuki Hattori, Chitose Oneyama, Mako Kamiya, Sadao Ota, Yasuteru Urano, Ryosuke Kojima\\nbioRxiv 2023.09.28.559700; doi: https://doi.org/10.1101/2023.09.28.559700 \\n Add to Selected Citations\\n\\n\\n\\nExonuclease Xrn1 regulates TORC1 signaling in response to SAM availability \\nMadeline M. McGinnis, Benjamin M Sutter, Samira Jahangiri, Benjamin P Tu\\nbioRxiv 2023.09.28.559955; doi: https://doi.org/10.1101/2023.09.28.559955 \\n Add to Selected Citations\\n\\n\\n\\nTranscriptome analysis reveals temporally regulated genetic networks during Drosophila border cell collective migration \\nEmily Burghardt, Jessica Rakijas, Antariksh Tyagi, Pralay Majumder, Bradley J.S.C. Olson, Jocelyn A. McDonald\\nbioRxiv 2023.09.27.559830; doi: https://doi.org/10.1101/2023.09.27.559830 \\n Add to Selected Citations\\n\\n\\n\\nVaccine Effectiveness Against Long COVID in Children: A Report from the RECOVER EHR Cohort \\nHanieh Razzaghi, Christopher Forrest, Kathryn Hirabayshi, Qiong Wu, Andrea Allen, Suchitra Rao, Yong Chen, Timothy Bunnell, Elizabeth A Chrischilles, Lindsey G Cowell, Mollie R Cummins, David A Hanauer, Miranda J Higginbotham, Benjamine Horne, Carol R Horowitz, Rhavi Jhaveri, Susan Kim, Aaron Mishkin, Jennifer Muszynski, Susanna Naggie, Nathan M Pajor, Anuradha Paranjape, Hayden T Schwenk, Marion R Sills, Yacob G Tedla, David A Williams, Charles Bailey\\nmedRxiv 2023.09.27.23296100; doi: https://doi.org/10.1101/2023.09.27.23296100 \\n Add to Selected Citations\\n\\n\\n\\nCombining stacked polygenic scores with clinical risk factors improves cardiovascular risk prediction in people with type 2 diabetes \\nKatarzyna Dziopa, Nishi Chaturvedi, Jasmine Gratton, Rory Maclean, Aroon Hingorani, Folkert Asselbergs, Chris Finan, Amand Floriaan Schmidt\\nmedRxiv 2022.09.01.22279477; doi: https://doi.org/10.1101/2022.09.01.22279477 \\n Add to Selected Citations\\n\\n\\n\\nPrebiotic Supplements Correct Oral Probiotic Deficiency for Lasting Allergy Relief \\nCliff Han\\nmedRxiv 2023.07.20.23291433; doi: https://doi.org/10.1101/2023.07.20.23291433 \\n Add to Selected Citations\\n\\n\\n\\nDevelopment and assessment of a machine learning tool for predicting emergency admission in Scotland \\nJames Liley, Gergo Bohner, Samuel R Emerson, Bilal A Mateen, Katie Borland, David Carr, Scott Heald, Samuel D Oduro, Jill Ireland, Keith Moffat, Rachel Porteous, Stephen Riddell, Simon Rogers, Ioanna Thoma, Nathan Cunningham, Chris Holmes, Katrina Payne, Sebastian J Vollmer, Catalina A Vallejos, Louis J M Aslett\\nmedRxiv 2021.08.06.21261593; doi: https://doi.org/10.1101/2021.08.06.21261593 \\n Add to Selected Citations\\n\\n\\n\\nEvaluating Clinical Outcomes in Patients Being Treated Exclusively via Telepsychiatry \\nCheryl Person, Nicola O\\'Connor, Lucy Koehler, Kartik Venkatachalam, Georgia Gaveras\\nmedRxiv 2023.09.27.23296225; doi: https://doi.org/10.1101/2023.09.27.23296225 \\n Add to Selected Citations\\n\\n\\n\\nHeterozygous loss-of-function SMC3 variants are associated with variable and incompletely penetrant growth and developmental features \\nMorad Ansari, Kamli N. W. Faour, Akiko Shimamura, Graeme Grimes, Emeline M. Kao, Erica R. Denhoff, Ana Blatnik, Daniel Ben-Isvy, Lily Wang, Benjamin M. Helm, Helen Firth, Amy M. Breman, Emilia K. Bijlsma, Aiko Iwata-Otsubo, Thomy J.L. de Ravel, Vincent Fusaro, Alan Fryer, Keith Nykamp, Lara G. Stuhn, Tobias B. Haack, G. Christoph Korenke, Panayiotis Constantinou, Kinga M. Bujakowska, Karen J. Low, Emily Place, Jennifer Humberson, Melanie P. Napier, Jessica Hoffman, Jane Juusola, Matthew A. Deardorff, Wanqing Shao, Shira Rockowitz, Ian Krantz, Maninder Kaur, Sarah Raible, Sabine Kliesch, Moriel Singer-Berk, Emily Groopman, Stephanie DiTroia, Sonia Ballal, Siddharth Srivastava, Kathrin Rothfelder, Saskia Biskup, Jessica Rzasa, Jennifer Kerkhof, Haley McConkey, Anne O\\'Donnell-Luria, Bekim Sadikovic, Sarah Hilton, Siddharth Banka, Frank Tuttelmann, Donald F. Conrad, Michael E. Talkowski, David R. FitzPatrick, Philip M. Boone\\nmedRxiv 2023.09.27.23294269; doi: https://doi.org/10.1101/2023.09.27.23294269 \\n Add to Selected Citations\\n\\n\\n\\nAn extended catalytic model to assess changes in risk for multiple reinfections with SARS-CoV-2 \\nBelinda Lombard, Cheryl Cohen, Anne von Gottberg, Jonathan Dushoff, Cari van Schalkwyk, Juliet Pulliam\\nmedRxiv 2023.09.27.23296231; doi: https://doi.org/10.1101/2023.09.27.23296231 \\n Add to Selected Citations\\n\\n\\n\\nSmartphone Apps for Cannabis Cessation: Quality Assessment and Content Analysis \\nSiddharth Seth, Sumedha Kushwaha, Reshma Prashad, Michael Chaiton\\nmedRxiv 2023.09.27.23296238; doi: https://doi.org/10.1101/2023.09.27.23296238 \\n Add to Selected Citations\\n\\n\\n\\nSex-specific socioeconomic inequalities in trajectories of anthropometry, blood pressure and blood-based biomarkers from birth to 18 years: a prospective cohort study \\nKate N O\\'Neill, Minhal Ahmed, Linda M O\\'Keeffe\\nmedRxiv 2023.09.27.23296220; doi: https://doi.org/10.1101/2023.09.27.23296220 \\n Add to Selected Citations\\n\\n\\n\\nTask performance to discriminate mild cognitive impairment from healthy aging \\nMelissa D Stockbridge, Argye E Hillis\\nmedRxiv 2023.09.27.23296235; doi: https://doi.org/10.1101/2023.09.27.23296235 \\n Add to Selected Citations\\n\\n\\n\\nTime trends and modifiable factors of contact tracing coverage in Geneva, Switzerland, June 2020 to February 2022 \\nDenis Mongin, Nils Burgisser, Covid-SMC Study Group, Delphine Sophie Courvoisier\\nmedRxiv 2023.03.22.23287577; doi: https://doi.org/10.1101/2023.03.22.23287577 \\n Add to Selected Citations\\n\\n\\n\\nFair Reinforcement Learning for Maternal Sepsis Treatment \\nSiân Carey, Ciarán McInerney, Tom Lawton, Ibrahim Habli, Owen Johnson, Leila Fahel, Alwyn Kotzé, Marc de Kamps\\nmedRxiv 2022.08.09.22278582; doi: https://doi.org/10.1101/2022.08.09.22278582 \\n Add to Selected Citations\\n\\n\\n\\nStriatal dopamine tone is positively associated with body mass index in humans as determined by PET using dual dopamine type-2 receptor antagonist tracers \\nValerie L Darcey, Juen Guo, Meible Chi, Stephanie T Chung, Amber B Courville, Isabelle Gallagher, Peter Herscovitch, Rebecca Howard, Melissa La Noire, Lauren Milley, Alex Schick, Michael Stagliano, Sara Turner, Nicholas Urbanski, Shana Yang, Eunha Yim, Nan Zhai, Megan S Zhou, Kevin D Hall\\nmedRxiv 2023.09.27.23296169; doi: https://doi.org/10.1101/2023.09.27.23296169 \\n Add to Selected Citations\\n\\n\\n\\nZero-shot prediction of drug repurposing with geometric deep learning and clinician centered design \\nKexin Huang, Payal Chandak, Qianwen Wang, Shreyas Havaldar, Akhil Vaid, Jure Leskovec, Girish Nadkarni, Benjamin S Glicksberg, Nils Gehlenborg, Marinka Zitnik\\nmedRxiv 2023.03.19.23287458; doi: https://doi.org/10.1101/2023.03.19.23287458 \\n Add to Selected Citations\\n\\n\\n\\nUrine tenofovir and dried blood spot tenofovir diphosphate concentrations and viraemia in people taking efavirenz and dolutegravir based antiretroviral therapy \\nJienchi Dorward, Katya Govender, Pravikrishnen Moodley, Richard John Lessells, Natasha Samsunder, Yukteshwar Sookrajh, Thomas Fanshawe, Philip Turner, Christopher Butler, Paul K Drain, Gail Hayward, Nigel Garrett\\nmedRxiv 2023.09.27.23296217; doi: https://doi.org/10.1101/2023.09.27.23296217 \\n Add to Selected Citations\\n\\n\\n\\nModel-based impact evaluation of new tuberculosis vaccines in aging populations under different modeling scenarios: the case of China. \\nMario Tovar, Joaquin Sanz, Yamir Moreno\\nmedRxiv 2023.09.27.23296224; doi: https://doi.org/10.1101/2023.09.27.23296224 \\n Add to Selected Citations\\n\\n\\n\\nThe Impact of Body Mass Index on Robotic Surgery Outcomes in Endometrial Cancer \\nEva Kadoch, Yoav Brezinov, Gabriel Levin, Florentin Racovitan, Susie Lau, Shannon Salvador, Walter Gotlieb\\nmedRxiv 2023.09.27.23296160; doi: https://doi.org/10.1101/2023.09.27.23296160 \\n Add to Selected Citations\\n\\n\\n\\nTemporal Trends in Antibiotic Resistance in Europe, 1998-2019 \\nMartin Emons, Francois Blanquart, Sonja Lehtinen\\nmedRxiv 2023.09.27.23296241; doi: https://doi.org/10.1101/2023.09.27.23296241 \\n Add to Selected Citations\\n\\n\\n\\nBlood-based multivariate methylation risk score for cognitive impairment and dementia \\nJarno Koetsier, Rachel Cavill, Rick A Reijnders, Joshua Harvey, Kay Deckers, Sebastian Köhler, Lars Eijssen, Rebecca G Smith, Adam R Smith, Joe Burrage, Emma M Walker, Gemma Shireby, Eilis Hannon, Emma Dempster, Timothy Frayling, Jonathan Mill, Valerija Dobricic, Yasmine Sommerer, Peter Johannsen, Michael Wittig, Andre Franke, Rick Vandenberghe, Jolien Schaeverbeke, Yvonne Freund-Levi, Lutz Frölich, Philip Scheltens, Charlotte Teunissen, Giovanni Frisoni, Olivier Blin, Jill Richardson, Régis Bordet, Sebastiaan Engelborghs, Ellen de Roeck, Pablo Martinez-Lage, Mikel Tainta, Alberto Lleó, Isabel Sala, Julius Popp, Gwedoline Peyratout, Frans Verhey, Magda Tsolaki, Ulf Andreasson, Kaj Blennow, Henrik Zetterberg, Johannes Streffer, Stephanie JB Vos, Simon Lovestone, Pieter-Jelle Visser, Lars Bertram, Katie Lunnon, Ehsan Pishva\\nmedRxiv 2023.09.27.23296143; doi: https://doi.org/10.1101/2023.09.27.23296143 \\n Add to Selected Citations\\n\\n\\n\\nMulti-task analysis of gene expression data on cancer public datasets \\nYasmmin Cortes Martins\\nmedRxiv 2023.09.27.23296213; doi: https://doi.org/10.1101/2023.09.27.23296213 \\n Add to Selected Citations\\n\\n\\n\\nUnintended events in long-distance neonatal interhospital transport in Western Australia: A comparison of neonatal specialist and non-neonatal specialist transport teams. \\nJacqueline Gardiner, Sam Athikarisamy, Joanne Blacker, Kylie McDonald, Mary Sharp, Jonathan W Davis\\nmedRxiv 2023.09.26.23296136; doi: https://doi.org/10.1101/2023.09.26.23296136 \\n Add to Selected Citations\\n\\n\\n\\nProteolytic activation of fatty acid synthase signals pan-stress resolution \\nHai Wei, Yi M Weaver, Chendong Yang, Yuan Zhang, Guoli Hu, Courtney M. Karner, Ralph J DeBerardinis, Benjamin P Weaver\\nbioRxiv 2023.09.27.559743; doi: https://doi.org/10.1101/2023.09.27.559743 \\n Add to Selected Citations\\n\\n\\n\\nGenomic analyses correspond with deep persistence of peoples of Blackfoot Confederacy from glacial times \\nDorothy First Rider, Annabel Crop Eared Wolf, John Murray, Alida de Flamingh, Andre Luiz Campelo dos Santos, Francois Lanoe, Maria N Zedeno, Michael DeGiorgio, John Lindo, Ripan Singh Malhi\\nbioRxiv 2023.09.25.559300; doi: https://doi.org/10.1101/2023.09.25.559300 \\n Add to Selected Citations\\n\\n\\n\\nLarval brooding damselfishes and shifting body proportions: does pelagic larval swimming constrain reef fish morphology? \\nJohn P. Lyons, Kathryn D. Kavanagh\\nbioRxiv 2023.09.25.559426; doi: https://doi.org/10.1101/2023.09.25.559426 \\n Add to Selected Citations\\n\\n\\n\\nHuman lower leg muscle pump acts like a stream diversion pump during locomotion \\nRoman A. Tauraginskii, Fedor Lurie, Sergei Simakov, Rishal Agalarov, Pavel Khramtsov, Maxim Babushkin, Denis Borsuk, Maxim Galchenko\\nbioRxiv 2023.09.25.559436; doi: https://doi.org/10.1101/2023.09.25.559436 \\n Add to Selected Citations\\n\\n\\n\\nFinding related sequences by a simple sum over alignments \\nMartin C Frith\\nbioRxiv 2023.09.26.559458; doi: https://doi.org/10.1101/2023.09.26.559458 \\n Add to Selected Citations\\n\\n\\n\\nHematologic setpoints are a stable and patient-specific deep phenotype \\nBrody H Foy, Rachel Petherbridge, Maxwell Roth, Christopher Mow, Hasmukh R Patel, Chhaya H Patel, Samantha N Ho, Evie Lam, Konrad J Karczewski, Veronica Tozzo, John H Higgins\\nmedRxiv 2023.09.26.23296146; doi: https://doi.org/10.1101/2023.09.26.23296146 \\n Add to Selected Citations\\n\\n\\n\\nTransient States During Oligonucleotides Hybridization: Insights from Annealing Kinetics of Mismatches and Bulges \\nMarco Todisco, Dian Ding, Jack W. Szostak\\nbioRxiv 2023.09.27.559754; doi: https://doi.org/10.1101/2023.09.27.559754 \\n Add to Selected Citations\\n\\n\\n\\nDescription of Protosticta sexcoloratus sp. nov. (Odonata, Platystictidae) from the Western Ghats, India \\nVivek A Chandran, Muneer P K, Madhavan M, Subin K Jose\\nbioRxiv 2023.09.25.559450; doi: https://doi.org/10.1101/2023.09.25.559450 \\n Add to Selected Citations\\n\\n\\n\\nEZH2 deletion does not impact acinar cell regeneration but restricts progression to pancreatic cancer in mice \\nEmilie Jaune-Pons, Xiaoyi Wang, Fatemeh Mousavi, Abdessamad El kaoutari, Kurt Berger, Charis Johnson, Mickenzie Martin, Saloni Aggarwal, Sukhman Brar, Khalid Muhammad, Parisa Shooshtari, Angela Mathison, Nelson Dusetti, Raul Urrutia, Gwen Lomberk, Christopher L Pin\\nbioRxiv 2023.09.25.559339; doi: https://doi.org/10.1101/2023.09.25.559339 \\n Add to Selected Citations\\n\\n\\n\\nActivation of the aryl hydrocarbon receptor inhibits neuropilin-1 upregulation on IL-2 responding CD4+ T cells \\nSimone Sandoval, Keegan Malany, Krista Thongphanh, Clarisa A Martinez, Michael L Goodson, Felipe da Costa Souza, Lo-Wei Lin, Jamie Pennington, Pamela J Lein, Nancy I Kerkvliet, Allison K Ehrlich\\nbioRxiv 2023.09.25.559429; doi: https://doi.org/10.1101/2023.09.25.559429 \\n Add to Selected Citations\\n\\n\\n\\nCITE-seq reveals inhibition of NF-kB pathway in B cells from vitamin D-treated multiple sclerosis patients \\nManon Galoppin, Manon Rival, Anais Louis, Saniya Kari, Sasha Soldati, Britta Engelhardt, Anne Astier, Philippe Marin, Eric Thouvenot\\nbioRxiv 2023.09.25.559400; doi: https://doi.org/10.1101/2023.09.25.559400 \\n Add to Selected Citations\\n\\n\\n\\nThe origin, divergence and inter-subspecies hybridization of house mice (Mus musculus) \\nMeidong Jing Sr., Yingjie Chen Sr., Wenbo Yuan IV, Ying Song II, Xiaoxin Bi II, Ling Huang II\\nbioRxiv 2023.09.26.559459; doi: https://doi.org/10.1101/2023.09.26.559459 \\n Add to Selected Citations\\n\\n\\n\\nOSCAA: A Two-Dimensional Gaussian Mixture Model for Copy Number Variation Association Analysis \\nXUANXUAN YU, Xizhi Luo, Guoshuai Cai, Feifei Xiao\\nbioRxiv 2023.09.25.559392; doi: https://doi.org/10.1101/2023.09.25.559392 \\n Add to Selected Citations\\n\\n\\n\\nDevelopment of LIBRA-seq for the Guinea Pig Model System as a Tool for the Evaluation of Antibody Responses to Multivalent HIV-1 Vaccines \\nMatthew J Vukovich, Nagarajan Raju, Prudence Kgagudi, Nelia P Manamela, Alexandra A Abu-Shmais, Kathryn R Gripenstraw, Perry T Wasdin, Shaunna Shen, Bridget Dwyer, Jumana Akoad, Rebecca M Lynch, David C Montefiori, Simone I Richardson, Penny L Moore, Ivelin S Georgiev\\nbioRxiv 2023.09.25.559382; doi: https://doi.org/10.1101/2023.09.25.559382 \\n Add to Selected Citations\\n\\n\\n\\nHematopoietic Stem Cells Undergo Differentiation State Reprogramming to Overcome Venetoclax Sensitivity in Patients with Myelodysplastic Syndromes \\nJuan Jose Rodriguez Sevilla, Irene Ganan-Gomez, Feyiang Ma, Kelly Chien, Guillermo Montalban-Bravo, Rashmi Kanagal-Shamanna, Sanam Loghavi, Vera Adema, Alexandre Bazinet, Guillermo Garcia-Manero, Simona Colla\\nbioRxiv 2023.09.25.559439; doi: https://doi.org/10.1101/2023.09.25.559439 \\n Add to Selected Citations\\n\\n\\n\\nScience abhors a surveillance vacuum: detection of ticks and tick-borne bacteria in southern New Mexico through passive surveillance \\nPaige  R Harman, Nicole  L Mendell, Maysee  M Harman, Puck  A Draney, Anna  T Boyle, Matthew  E Gompper, Teri  J Orr, Donald  H Bouyer, Pete  D Teel, Kathryn Hanley\\nbioRxiv 2023.09.25.559416; doi: https://doi.org/10.1101/2023.09.25.559416 \\n Add to Selected Citations\\n\\n\\n\\nCalling site preference reduces masking interference of acoustic signals among sympatric bush frogs and facilitates coexistence \\nAbhijith A V, Shomen Mukherjee\\nbioRxiv 2023.09.25.559099; doi: https://doi.org/10.1101/2023.09.25.559099 \\n Add to Selected Citations\\n\\n\\n\\nMicroevolution of clade II isolates of Candida auris highlights multifaceted intra-clade heterogeneity in acquiring resistance towards amphotericin B \\nAnshu Chauhan, Praveen Kumar, Mohit Kumar, Aswathy Narayanan, Kusum Yadav, Basharat Ali, Amandeep Saini, Ashutosh Singh, Atanu Banerjee, Shivaprakash M. Rudramurthy, Arunaloke Chakrabarti, Alok K. Mondal, Naseem A. Gaur, Kaustuv Sanyal, Rajendra Prasad\\nbioRxiv 2023.09.20.558730; doi: https://doi.org/10.1101/2023.09.20.558730 \\n Add to Selected Citations\\n\\n\\n\\nReduction in myocardial fibrosis in MDX mice on oral consumption of Aureobasidium Pullulans produced Neu REFIX Beta-glucans; holds potential as an adjuvant in managing post-transplantation organ fibrosis \\nSenthilkumar Preethy, Naoki Yamamoto, Kottoorathu Mammen Cherian, Rajasekaran Premsekar, Gary A Levy, Rajappa Senthilkumar, Samuel JK Abraham\\nbioRxiv 2023.09.25.559276; doi: https://doi.org/10.1101/2023.09.25.559276 \\n Add to Selected Citations\\n\\n\\n\\nThe European Reference Genome Atlas: piloting a decentralised approach to equitable biodiversity genomics \\nAnn M Mc Cartney, Giulio Formenti, Alice Mouton, Claudio Ciofi, Robert M Waterhouse, Camila J Mazzoni, Diego De Panis, Luisa S Schlude Marins, Henrique G Leitao, Genevieve Diedericks, Joseph Kirangwa, Marco Morselli, Judit Salces, Nuria Escudero, Alessio Iannucci, Chiara Natali, Hannes Svardal, Rosa Fernandez, Tim De Pooter, Geert Joris, Mojca Strazisar, Jo Wood, Katie E Herron, Ole Seehausen, Phillip C Watts, Felix Shaw, Robert P Davey, Alice Minotto, Jose Maria Fernandez Gonzalez, Astrid Bohne, Carla Alegria, Tyler Alioto, Paulo C Alves, Isabel R Amorim, Jean-Marc Aury, Niclas Backstrom, Petr Baldrian, Loriano Ballarin, Laima Baltrunaite, Endre Barta, Bertrand BedHom, Caroline Belser, Johannes Bergsten, Laurie Bertrand, Helena Bilandija, Mahesh Binzer-Panchal, Iliana Bista, Mark Blaxter, Paulo AV Borges, Guilherme Borges Dias, Mirte Bosse, Tom Brown, Remy Bruggmann, Elena Buena-Atienza, Josephine Burgin, Elena Buzan, Nicolas Casadei, Matteo Chiara, Sergio Chozas, Fedor F Ciampor Jr., Angelica Crottini, Corinne Cruaud, Fernando Cruz, Love Dalen, Alessio De Biase, Javier del Campo, Teo Delic, Alice B Dennis, Martijn FL Derks, Maria Angela Diroma, Mihajla Djan, Simone Duprat, Klara Eleftheriadi, Philine GD Feulner, Jean-Francois Flot, Giobbe Forni, Bruno Fosso, Pascal Fournier, Christine Fournier-Chambrillon, Toni Gabaldon, Shilpa Garg, Carmela Gissi, Luca Giupponi, Jessica Gomez-Garrido, Josefa Gonzalez, Miguel L Grilo, Bjoern Gruening, Thomas Guerin, Nadege Guiglielmoni, Marta Gut, Marcel P Haesler, Christoph Hahn, Balint Halpern, Peter Harrison, Julia Heintz, Maris Hindrikson, Jacob Hoglund, Kerstin Howe, Graham Hughes, Benjamin Istace, Mark J. Cock, Franc Jancekovic, Zophonias O Jonsson, Sagane Joye-Dind, Janne J. Koskimaki, Boris Krystufek, Justyna Kubacka, Heiner Kuhl, Szilvia Kusza, Karine Labadie, Meri Lahteenaro, Henrik Lantz, Anton Lavrinienko, Lucas Leclere, Ricardo Jorge Lopes, Ole Madsen, Ghislaine Magdelenat, Giulia Magoga, Tereza Manousaki, Tapio Mappes, Joao Pedro Marques, Gemma I Martinez Redondo, Florian Maumus, Hendrik-Jan Megens, Jose Melo-Ferreira, Sofia L Mendes, Matteo Montagna, Joao Moreno, Mai-Britt Mosbech, Monica Moura, Zuzana Musilova, Eugene Myers, Will J. Nash, Alexander Nater, Pamela Nicholson, Manuel Niell, Reindert Nijland, Benjamin Noel, Karin Noren, Pedro H Oliveira, Remi-Andre Olsen, Lino Ometto, Stephan Ossowski, Vaidas Palinauskas, Snaebjorn Palsson, Jerome P Panibe, Joana Pauperio, Martina Pavlek, Emilie Payen, Julia Pawlowska, Jaume Pellicer, Graziano Pesole, Joao Pimenta, Martin Pippel, Anna Maria Pirttila, Nikos Poulakakis, Jeena Rajan, Ruben MC Rego, Roberto Resendes, Philipp Resl, Ana Riesgo, Patrik Rodin-Morch, Andre ER Soares, Carlos Rodriguez Fernandes, Maria M. Romeiras, Guilherme Roxo, Lukas Ruber, Maria Jose Ruiz-Lopez, Urmas Saarma, Luis P Silva, Manuela Sim-Sim, Lucile Soler, Vitor C Sousa, Carla Sousa Santos, Alberto Spada, Milomir Stefanovic, Viktor Steger, Josefin Stiller, Matthias Stock, Torsten Hugo H Struck, Hiranya Sudasinghe, Riikka Tapanainen, Christian Tellgren-Roth, Helena Trindade, Yevhen Tukalenko, Ilenia Urso, Benoit Vacherie, Steven M Van Belleghem, Kees van Oers, Carlos Vargas-Chavez, Nevena Velickovic, Noel Vella, Adriana Vella, Cristiano Vernesi, Sara Vicente, Sara Villa, Olga Vinnere Pettersson, Filip AM Volckaert, Judit Voros, Patrick Wincker, Sylke Winkler\\nbioRxiv 2023.09.25.559365; doi: https://doi.org/10.1101/2023.09.25.559365 \\n Add to Selected Citations\\n\\n\\n\\nDistinct transcriptional profiles of maternal and fetal placental macrophages at term are associated with gravidity \\nNida Ozarslan, Joshua F. Robinson, Sirirak Buarpung, M. Yvonne Kim, Megan R. Ansbro, Jason Akram, Dennis J. Montoya, Moses R. Kamya, Abel Kakuru, Grant Dorsey, Philip J. Rosenthal, Genhong Cheng, Margaret E. Feeney, Susan J. Fisher, Stephanie L. Gaw\\nbioRxiv 2023.09.25.559419; doi: https://doi.org/10.1101/2023.09.25.559419 \\n Add to Selected Citations\\n\\n\\n\\nAssociation between Body Mass Index and Risk of Aortic Stenosis in Women \\nSilvana Kontogeorgos, Annika Rosengren, Tatiana Zverkova Sandström, Michael Fu, Martin Lindgren, Carmen Basic, Maria Teresia Svanvik, Demir Djekic, Erik Petter Thunstrom\\nmedRxiv 2023.09.26.23296191; doi: https://doi.org/10.1101/2023.09.26.23296191 \\n Add to Selected Citations\\n\\n\\n\\nClinical prediction models combining routine clinical measures identify participants with youth-onset diabetes who maintain insulin secretion in the range associated with type 2 diabetes: The SEARCH for Diabetes in Youth Study \\nAngus G Jones, Beverley Shields, Richard A Oram, Dana M Dabelea, William A Hagopian, Eva Lustigova, Amy S Shah, Julieanne Knupp, Amy K Mottl, Ralph B D`Agostino Jr., Adrienne Williams, Santica M Marcovina, Catherine Pihoker, Jasmin Divers, Maria J Redondo\\nmedRxiv 2023.09.27.23296128; doi: https://doi.org/10.1101/2023.09.27.23296128 \\n Add to Selected Citations\\n\\n\\n\\nUnlocking Esophageal Carcinoma’s Secrets: An integrated Omics Approach Unveils DNA Methylation as a pivotal Early Detection Biomarker with Clinical Implications. \\nAli Akbar, Li Zhang, Hong-Sheng Liu\\nmedRxiv 2023.09.26.23296198; doi: https://doi.org/10.1101/2023.09.26.23296198 \\n Add to Selected Citations\\n\\n\\n\\nReconstitution of Early Paclitaxel Biosynthetic Network \\nJack Chun-Ting Liu, Ricardo De La Pena, Christian Tocol, Elizabeth S. Sattely\\nbioRxiv 2023.09.27.559859; doi: https://doi.org/10.1101/2023.09.27.559859 \\n Add to Selected Citations\\n\\n\\n\\nEvolution of wheat blast resistance gene Rmg8 accompanied by differentiation of variants recognizing the powdery mildew fungus \\nSoichiro Asuke, Kohei Morita, Motoki Shimizu, Fumitaka Abe, Ryohei Terauchi, Chika Nago, Yoshino Takahashi, Mai Shibata, Motohiro Yoshioka, Mizuki Iwakawa, Mitsuko Kishi-Kaboshi, Zhuo Su, Shuhei Nasuda, Hirokazu Handa, Masaya Fujita, Makoto Tougou, Koichi Hatta, Naoki Mori, Yoshihiro Matsuoka, Kenji Kato, Yukio Tosa\\nbioRxiv 2023.09.26.559445; doi: https://doi.org/10.1101/2023.09.26.559445 \\n Add to Selected Citations\\n\\n\\n\\nEnhancing plant photosynthesis through activating cells deficient in photosynthetic apparatus \\nFenfen Miao, Yongyao Zhao, Ming-ju Amy Lyu, Bihai Shi, Fusang Liu, Xinguang Zhu\\nbioRxiv 2023.09.27.559740; doi: https://doi.org/10.1101/2023.09.27.559740 \\n Add to Selected Citations\\n\\n\\n\\nThe wheat powdery mildew resistance gene Pm4 also confers resistance to wheat blast \\nTom O\\'Hara, Andrew Steed, Rachel Goddard, Kumar Gaurav, Sanu Arora, Jesus Quiroz-Chavez, Ricardo Ramirez-Gonzalez, Roshani Badgami, David Gilbert, Javier Sanchez-Martin, Luzie Wingen, Cong Feng, Mei Jiang, Shifeng Cheng, Susanne Dreisigacker, Beat Keller, Brande Wulff, Cristobal Uauy, Paul Nicholson\\nbioRxiv 2023.09.26.559489; doi: https://doi.org/10.1101/2023.09.26.559489 \\n Add to Selected Citations\\n\\n\\n\\nLeaf δ13C reveals post-photosynthetic fractionation during ontogeny in a C4 grass \\nYong Zhi Yu, Hai Tao Liu, Fang Yang, Lei Li, Rudi Schaeufele, Guillaume Tcherkez, Hans Schnyder, Xiao Ying Gong\\nbioRxiv 2023.09.25.559424; doi: https://doi.org/10.1101/2023.09.25.559424 \\n Add to Selected Citations\\n\\n\\n\\nLipid constituents from Cissus trifoliata stems \\nLuis Fernando Mendez Lopez, Jose Luis Gonzalez Llerena, Bryan Alejandro Espinosa Rodriguez, Isaias Balderas Renteria, Maria del Rayo Camacho Corona\\nbioRxiv 2023.09.27.559825; doi: https://doi.org/10.1101/2023.09.27.559825 \\n Add to Selected Citations\\n\\n\\n\\nThe shape and volume of air, kernels, and cracks, in a nutshell \\nErik J Amézquita, Michelle Y Quigley, Patrick J Brown, Elizabeth Munch, Daniel H Chitwood\\nbioRxiv 2023.09.26.559651; doi: https://doi.org/10.1101/2023.09.26.559651 \\n Add to Selected Citations\\n\\n\\n\\nKinetochore and ionomic adaptation to whole genome duplication \\nSian M Bray, Tuomas Hamala, Min Zhou, Silvia Busoms, Sina Fischer, Stuart D Desjardins, Terezie Mandakova, Chris Moore, Thomas C Mathers, Laura Cowan, Patrick Monnahan, Jordan Koch, Eva M Wolf, Martin A Lysak, Filip Kolar, James D Higgins, Marcus A Koch, Levi Yant\\nbioRxiv 2023.09.27.559727; doi: https://doi.org/10.1101/2023.09.27.559727 \\n Add to Selected Citations\\n\\n\\n\\nNon-destructive, whole-plant phenotyping reveals dynamic changes in water use efficiency, photosynthetic efficiency, and rhizosphere acidification of sorghum cultivars under osmotic stress \\nDaniel Ginzburg, Jack A Cox, Seung Y Rhee\\nbioRxiv 2023.09.26.559576; doi: https://doi.org/10.1101/2023.09.26.559576 \\n Add to Selected Citations\\n\\n\\n\\nTranscriptional reactivation of the tRNASer/tRNATyr gene cluster in Arabidopsis thaliana root tip \\nGuillaume Hummel, Priyanka Kumari, Long Wang, Yan-Xia Mai, Nan Wang, Negjmedin Shala, Emir Can Kaya, Jia-Wei Wang, Chang Liu\\nbioRxiv 2023.09.27.559738; doi: https://doi.org/10.1101/2023.09.27.559738 \\n Add to Selected Citations\\n\\n\\n\\nSeqVerify: A quality-assurance pipeline for whole-genome sequencing data \\nMerrick D Pierson Smela, Valerio Pepe, George M Church\\nbioRxiv 2023.09.27.559766; doi: https://doi.org/10.1101/2023.09.27.559766 \\n Add to Selected Citations\\n\\n\\n\\nTranscriptional acclimation to warming temperatures of the Australian alpine herb Wahlenbergia ceracea \\nRocco Fabrizio Notarnicola, Pieter Andrew Arnold, Zhi-Ping Feng, Jemimah Hamilton, Ashley Jones, Stella Loke, Adrienn B Nicotra, Benjamin Schwessinger, Diep R Ganguly\\nbioRxiv 2023.09.27.559694; doi: https://doi.org/10.1101/2023.09.27.559694 \\n Add to Selected Citations\\n\\n\\n\\nLight management by algal aggregates in living photosynthetic hydrogels. \\nSing Teng Chua, Alyssa Smith, Swathi Murthy, Maria Murace, Han Yang, Michael Kuhl, Pietro Cicuta, Alison G. Smith, Daniel Wangpraseurt, Silvia Vignolini\\nbioRxiv 2023.09.28.559714; doi: https://doi.org/10.1101/2023.09.28.559714 \\n Add to Selected Citations\\n\\n\\n\\nDiscovering paracrine regulators of cellular composition from spatial transcriptomics data using SPER \\nAdam L Haber, Tianxiao Zhao\\nbioRxiv 2023.09.27.559863; doi: https://doi.org/10.1101/2023.09.27.559863 \\n Add to Selected Citations\\n\\n\\n\\nObesity-elicited macrophages shape CD9hi progenitor fate to promote adipose tissue fibrosis and dysfunction \\nHouassa Askia, Sebastien Dussaud, Marina Chaves de Oliveira, Amanda Oliveira, Lucie Poupel, Clementine Rebiere, Melissa Ouhachi, Raoul Manuel, Fatiha Merabtene, Laurent Genser, Judith Aron-Wisnewsky, Martine Moreau, Laurent Yvan-Charvet, Adaliene Ferreira, Thierry Huby, Karine Clement, Genevieve Marcelin, Emmanuel Gautier\\nbioRxiv 2023.09.26.559540; doi: https://doi.org/10.1101/2023.09.26.559540 \\n Add to Selected Citations\\n\\n\\n\\nPost-mortem AT-8 reactive tau species correlate with non-plaque Aβ levels in the frontal cortex of non-AD and AD brains \\nNauman Malik, Mohi Miah, Alessandro Galgani, Kirsty McAleese, Lauren Walker, Fiona LeBeau, Johannes Attems, Tiago Outeiro, Alan Thomas, David Koss\\nbioRxiv 2023.09.27.559720; doi: https://doi.org/10.1101/2023.09.27.559720 \\n Add to Selected Citations\\n\\n\\n\\nBaRDIC: robust peak calling for RNA-DNA interaction data \\nDmitry E. Mylarshchikov, Arina I. Nikolskaya, Olesja D. Bogomaz, Anastasia A. Zharikova, Andrey A. Mironov\\nbioRxiv 2023.09.21.558815; doi: https://doi.org/10.1101/2023.09.21.558815 \\n Add to Selected Citations\\n\\n\\n\\nLesions in a songbird vocal circuit increase variability in song syntax \\nAvani Koparkar, Timothy L. Warren, Jonathan D. Charlesworth, Sooyoon Shin, Michael S. Brainard, Lena Veit\\nbioRxiv 2023.07.25.550508; doi: https://doi.org/10.1101/2023.07.25.550508 \\n Add to Selected Citations\\n\\n\\n\\nAn altered cell type-specific subcellular distribution of translesion synthesis DNA polymerase kappa (PolK) in aging neurons \\nMofida Abdelmageed, Shilpi Paul, Anirban Paul\\nbioRxiv 2023.09.27.559771; doi: https://doi.org/10.1101/2023.09.27.559771 \\n Add to Selected Citations\\n\\n\\n\\nReduced discrimination between signals of danger and safety but not overgeneralization is linked to exposure to childhood adversity in healthy adults \\nMaren Klingelhoefer-Jens, Katharina Hutterer, Miriam A. Schiele, Elisabeth Leehr, Dirk Schuemann, Karoline Rosenkranz, Joscha Boehnlein, Jonathan Repple, Juergen Deckert, Katharina Domschke, Udo Dannlowski, Ulrike Lueken, Andreas Reif, Marcel Romanos, Peter Zwanzger, Paul Pauli, Matthias Gamer, Tina B. Lonsdorf\\nbioRxiv 2023.09.26.559474; doi: https://doi.org/10.1101/2023.09.26.559474 \\n Add to Selected Citations\\n\\n\\n\\nDecryption of sequence, structure, and functional features of SINE repeat elements in SINEUP non–coding RNA–mediated post–transcriptional gene regulation \\nHarshita Sharma, Matthew N Z Valentine, Naoko Toki, Hiromi Sueki Nishiyori, Stefano Gustincich, Hazuki Takahashi, Piero Carninci\\nbioRxiv 2023.05.22.541671; doi: https://doi.org/10.1101/2023.05.22.541671 \\n Add to Selected Citations\\n\\n\\n\\nFrequency- and circuit- specific effects of septohippocampal deep brain stimulation in mice as measured by functional ultrasound imaging. \\nLindsey M. Crown, Kofi Agyeman, Wooseong Choi, Nancy Zepeda, Steven Siegel, Charles Liu, Vasileios Christopoulos, Darrin Lee\\nbioRxiv 2023.05.21.541598; doi: https://doi.org/10.1101/2023.05.21.541598 \\n Add to Selected Citations\\n\\n\\n\\nContact-FP: a dimerization-dependent fluorescent protein toolkit for visualizing membrane contact site dynamics \\nGregory E. Miner, Sidney Y. Smith, Wendy K. Showalter, Christina M. So, Joey V. Ragusa, Alex E. Powers, Maria Clara Zanellati, Chih-Hsuan Hsu, Michelle F. Marchan, Sarah Cohen\\nbioRxiv 2023.09.27.559820; doi: https://doi.org/10.1101/2023.09.27.559820 \\n Add to Selected Citations\\n\\n\\n\\nCorrelation of calcium multiphoton with ultrastructural STED imaging of the slit diaphragm in the same glomerulus. \\nEva Wiesner, Julia Binz-Lotter, Agnes Hackl, David Unnersjoe-Jess, Nelli Rutkowski, Thomas Benzing, Matthias J Hackl\\nbioRxiv 2023.09.27.559730; doi: https://doi.org/10.1101/2023.09.27.559730 \\n Add to Selected Citations\\n\\n\\n\\nMacrophage-expressed Neuropilin 2 promotes plaque formation in ApoE knockout mice and is essential for pro-inflammatory macrophage polarisation and gene expression. \\nJacob Fernando-Sayers, Jennifer L Harman, Matthew C Gage, Ian C Zachary, Caroline AP Pellet-Many\\nbioRxiv 2023.09.27.558806; doi: https://doi.org/10.1101/2023.09.27.558806 \\n Add to Selected Citations\\n\\n\\n\\nMaturation and detoxification of synphilin-1 inclusion bodies regulated by sphingolipids \\nXiuling Cao, Xiang Wu, Lei Zhao, Ju Zheng, Xuejiao Jin, Xinxin Hao, Joris Winderickx, Shenkui Liu, Lihua Chen, Beidong Liu\\nbioRxiv 2023.09.27.559647; doi: https://doi.org/10.1101/2023.09.27.559647 \\n Add to Selected Citations\\n\\n\\n\\nBenchmarking of deep neural networks for predicting personal gene expression from DNA sequence highlights shortcomings \\nAlexander Sasse, Bernard Ng, Anna Spiro, Shinya Tasaki, David Bennett, Chrisopher Gaiteri, Philip L. De Jager, Maria Chikina, Sara Mostafavi\\nbioRxiv 2023.03.16.532969; doi: https://doi.org/10.1101/2023.03.16.532969 \\n Add to Selected Citations\\n\\n\\n\\nA Window into Mammalian Basement Membrane Development: Insights from the mTurq2-Col4a1 Mouse Model \\nRebecca A. Jones, Brandon Trejo, Parijat Sil, Katherine A. Little, H. Amalia Pasolli, Bradley Joyce, Eszter Posfai, Danelle Devenport\\nbioRxiv 2023.09.27.559396; doi: https://doi.org/10.1101/2023.09.27.559396 \\n Add to Selected Citations\\n\\n\\n\\nCytoplasmic fluidization triggers breaking spore dormancy in fission yeast. \\nKeiichiro Sakai, Yohei Kondo, Yuhei Goto, Kazuhiro Aoki\\nbioRxiv 2023.09.27.559686; doi: https://doi.org/10.1101/2023.09.27.559686 \\n Add to Selected Citations\\n\\n\\n\\nA data assimilation method to track time-varying changes in the excitation-inhibition balance using scalp EEG \\nHiroshi Yokoyama, Keiichi Kitajo\\nbioRxiv 2022.12.16.520838; doi: https://doi.org/10.1101/2022.12.16.520838 \\n Add to Selected Citations\\n\\n\\n\\nConflicts are parametrically encoded: initial evidence for a cognitive space view to reconcile the debate of domain-general and domain-specific cognitive control \\nGuochun Yang, Haiyan Wu, Qi Li, Xun Liu, Zhongzheng Fu, Jiefeng Jiang\\nbioRxiv 2023.02.13.528292; doi: https://doi.org/10.1101/2023.02.13.528292 \\n Add to Selected Citations\\n\\n\\n\\nLRRK2 kinase dependent and independent function on endolysosomal repair promotes macrophage cell death \\nRebecca Morrison, Simeon R G Mihaylov, Chak Hon Luk, Angela Rodgers, Natalia Athanasiadi, Huw Morris, Enrica Pellegrino, Sila K Ultanir, Maximiliano G Gutierrez\\nbioRxiv 2023.09.27.559807; doi: https://doi.org/10.1101/2023.09.27.559807 \\n Add to Selected Citations\\n\\n\\n\\nGut microbial community in proboscis monkeys: implications for effects of geographical and social factors \\nLilian Jose, Wanyi Lee, Goro Hanya, Augustine Tuuga, Benoit Goossens, Joseph Tangah, Ikki Matsuda, Vijay Kumar Subbiah\\nbioRxiv 2023.03.14.532648; doi: https://doi.org/10.1101/2023.03.14.532648 \\n Add to Selected Citations\\n\\n\\n\\nBET inhibition rescues ciliogenesis and ameliorates pancreatitis-driven phenotypic changes in mice with Par3 loss. \\nMario A Shields, Anastasia E Metropulos, Christina Spaulding, Tomonori Hirose, Shigeo Ohno, Thao ND Pham, Hidayatullah G Munshi\\nbioRxiv 2023.09.14.557654; doi: https://doi.org/10.1101/2023.09.14.557654 \\n Add to Selected Citations\\n\\n\\n\\nMembrane Bound O-Acyltransferase 7 (MBOAT7) Shapes Lysosomal Lipid Homeostasis and Function to Control Alcohol-Associated Liver Injury \\nVenkateshwari Varadharajan, Iyappan Ramachandiran, William J Massey, Raghav Jain, Rakhee Banerjee, Anthony J Horak III, Megan R McMullen, Emily Huang, Annette Bellar, Shuhui W Lorkowski, Kailash Gulshan, Robert N Helsley, Isabella James, Vai Pathak, Jaividhya Dasarathy, Nicole Welch, Srinivasan Dasarathy, David Streem, Ofer Reizes, Daniela S Allende, Jonathan D Smith, Judith Simcox, Laura E Nagy, J. Mark Brown\\nbioRxiv 2023.09.26.559533; doi: https://doi.org/10.1101/2023.09.26.559533 \\n Add to Selected Citations\\n\\n\\n\\nFusion-Negative Rhabdomyosarcoma xenografts in murine tongue provide a model for the study of local invasion, intravasation and metastasis at cellular and subcellular levels in live animals \\nSarah M Hammoudeh, Yeap Ng, Bih-Rong Wei, Thomas Madsen, R. Mark Simpson, Roberto Weigert, Paul A Randazzo\\nbioRxiv 2023.09.21.558858; doi: https://doi.org/10.1101/2023.09.21.558858 \\n Add to Selected Citations\\n\\n\\n\\nThe IRE1α/XBP1/Myomaker axis drives myoblast fusion in adult skeletal muscle \\nAniket S Joshi, Meiricris Tomaz Da Silva, Anirban Roy, Tatiana E. Koike, Mingfu Wu, Micah B. Castillo, Preethi H Gunaratne, Yu Liu, Takao Iwawaki, Ashok Kumar\\nbioRxiv 2023.09.26.559625; doi: https://doi.org/10.1101/2023.09.26.559625 \\n Add to Selected Citations\\n\\n\\n\\nSample preparation and data collection for Serial Block Face Scanning Electron Microscopy of Mammalian Cell Monolayers \\nNoelle Antao, Joseph Sall, Christopher Petzold, Damian Charles Ekiert, Gira Bhabha, Feng-Xia Liang\\nbioRxiv 2023.09.26.559595; doi: https://doi.org/10.1101/2023.09.26.559595 \\n Add to Selected Citations\\n\\n\\n\\nSeptins Enable T Cell Contact Guidance via Amoeboid-Mesenchymal Switch \\nAlexander S Zhovmer, Alexis Manning, Chynna Smith, Jian Wang, Xuefei Ma, Denis Tsygankov, Nikolay V Dokholyan, Alexander X. Cartagena-Rivera, Rakesh Singh, Erdem D Tabdanov\\nbioRxiv 2023.09.26.559597; doi: https://doi.org/10.1101/2023.09.26.559597 \\n Add to Selected Citations\\n\\n\\n\\nThe herpesvirus UL49.5 protein hijacks a cellular C-degron pathway to drive TAP transporter degradation \\nMagda Wachalska, Celeste Riepe, Magdalena J. Slusarz, Malgorzata Graul, Lukasz S. Borowski, Wenjie Qiao, MIchalina Foltynska, Jan  E. Carette, Krystyna Bienkowska-Szewczyk, Roman J. Szczesny, Ron R. Kopito, Andrea D. Lipinska\\nbioRxiv 2023.09.27.559663; doi: https://doi.org/10.1101/2023.09.27.559663 \\n Add to Selected Citations\\n\\n\\n\\nTemporal Dynamics and Stoichiometry in Notch Signaling - from Notch Synaptic Complex Formation to NICD Nuclear Entry \\nLena Tveriakhina, Gustavo Scanavachi, Emily D. Egan, Ricardo Bango Da Cunha Correia, Alexandre P. Martin, Julia M. Rogers, Jeremy S. Yodh, Jon C Aster, Tom Kirchhausen, Stephen Blacklow\\nbioRxiv 2023.09.27.559780; doi: https://doi.org/10.1101/2023.09.27.559780 \\n Add to Selected Citations\\n\\n\\n\\nBioinformatic investigation of discordant sequence data for SARS-CoV-2: insights for robust genomic analysis during pandemic surveillance \\nSara E. Zufan, Katherine A. Lau, Angela Donald, Tuyet Hoang, Charles S.P. Foster, Chisha Sikazwe, Torsten Theis, William D. Rawlinson, Susan A. Ballard, Timothy P. Stinear, Benjamin P. Howden, Amy V. Jennison, Torsten Seemann\\nbioRxiv 2023.02.01.526694; doi: https://doi.org/10.1101/2023.02.01.526694 \\n Add to Selected Citations\\n\\n\\n\\nDynamic ventral disc contraction is necessary for Giardia attachment and host pathology \\nChristopher Nosala, Kari D Hagen, Sarah L Guest, Nicholas A Hilton, Antonia Mueller, Michael Laue, Christian Klotz, Anton Aebischer, Scott C Dawson\\nbioRxiv 2023.07.04.547600; doi: https://doi.org/10.1101/2023.07.04.547600 \\n Add to Selected Citations\\n\\n\\n\\nDevelopment of novel genetically-encoded fluorescent probes to track ceramides during phagocytosis \\nVladimir Girik, Larissa Van Ek, Isabelle Dentand Quadri, Maral Azam, Maria Cruz Cobo, Marion Mandavit, Isabelle Riezman, Howard Riezman, Anne-Claude Gavin, Paula Nunes-Hasler\\nbioRxiv 2023.09.27.559623; doi: https://doi.org/10.1101/2023.09.27.559623 \\n Add to Selected Citations\\n\\n\\n\\nA quantitative approach to measure and predict microbiome response to antibiotics \\nVincent Tu, Yue Ren, Ceylan Tanes, Sagori Mukhopadhyay, Scott G. Daniel, Hongzhe Li, Kyle Bittinger\\nbioRxiv 2023.01.27.525904; doi: https://doi.org/10.1101/2023.01.27.525904 \\n Add to Selected Citations\\n\\n\\n\\nA fluidization transition of the cell envelope sets up a universal volume expansion pacemaker \\nAvik Mukherjee, Yanqing Huang, Seungeun Oh, Carlos Sanchez, Yu-Fang Chang, Xili Liu, Gary Andrew Bradshaw, Nina Catherine Benites, Johan Paulsson, Marc W. Kirschner, Yongjin Sung, Jens Elgeti, Markus Basan\\nbioRxiv 2023.08.31.555748; doi: https://doi.org/10.1101/2023.08.31.555748 \\n Add to Selected Citations\\n\\n\\n\\nEndogenous tagging using split mNeonGreen in human iPSCs for live imaging studies \\nMathieu C Husser, Nhat P Pham, Chris Law, Flavia R B Araujo, Vincent J J Martin, Alisa Piekny\\nbioRxiv 2023.06.28.546942; doi: https://doi.org/10.1101/2023.06.28.546942 \\n Add to Selected Citations\\n\\n\\n\\nThe domed architecture of Giardias ventral disc is necessary for attachment and host pathogenesis \\nKari D Hagen, Christopher Nosala, Nicholas Hilton, Antonia Mueller, David Holthaus, Michael Laue, Christian Klotz, Anton Aebischer, Scott C Dawson\\nbioRxiv 2023.07.02.547441; doi: https://doi.org/10.1101/2023.07.02.547441 \\n Add to Selected Citations\\n\\n\\n\\nActin-dependent recruitment of AGO2 to the zonula adherens \\nMary Catherine Bridges, Joyce Nair-Menon, Alyssa Risner, Douglas W. Jimenez, Amanda C. Daulagala, Christina Kingsley, Madison E. Davis, Antonis Kourtidis\\nbioRxiv 2022.03.10.483874; doi: https://doi.org/10.1101/2022.03.10.483874 \\n Add to Selected Citations\\n\\n\\n\\nDesign of an effective sgRNA for CRISPR/Cas9 knock-ins in polyploid Synechocystis sp. PCC 6803 \\nMaria Isabel Nares Rodriguez, Esther Karunakaran\\nbioRxiv 2023.09.25.559380; doi: https://doi.org/10.1101/2023.09.25.559380 \\n Add to Selected Citations\\n\\n\\n\\nMinimal determinants for durable B cell antibody response towards virus-like antigens \\nWei-Yun Wholey, Alexander R. Meyer, Sekou-Tidiane Yoda, Bryce Chackerian, Julie Zikherman, Wei Cheng\\nbioRxiv 2023.02.20.529089; doi: https://doi.org/10.1101/2023.02.20.529089 \\n Add to Selected Citations\\n\\n\\n\\nUnsupervised pattern discovery in spatial gene expression atlas reveals mouse brain regions beyond established ontology \\nRobert Cahill, Yu Wang, R. Patrick Xian, Alex Lee, Hongkui Zeng, Bin Yu, Bosiljka Tasic, Reza Abbasi-Asl\\nbioRxiv 2023.03.10.531984; doi: https://doi.org/10.1101/2023.03.10.531984 \\n Add to Selected Citations\\n\\n\\n\\nComparison of insect and human cytochrome b561 proteins: Insights into candidate ferric reductases in insects \\nJessica  D. Holst, Laura  G. Murphy, Maureen  J. Gorman, Emily Ragan\\nbioRxiv 2023.09.01.555970; doi: https://doi.org/10.1101/2023.09.01.555970 \\n Add to Selected Citations\\n\\n\\n\\nFlight power muscles have a coordinated, causal role in controlling hawkmoth pitch turns \\nLeo J Wood, Joy Putney, Simon Sponberg\\nbioRxiv 2023.09.27.559785; doi: https://doi.org/10.1101/2023.09.27.559785 \\n Add to Selected Citations\\n\\n\\n\\nMolecular and network disruptions in neurodevelopment uncovered by single cell transcriptomics analysis of CHD8 heterozygous cerebral organoids \\nMaider Astorkia, Yang Liu, Erika Pedrosa, Herbert Lachman, Deyou Zheng\\nbioRxiv 2023.09.27.559752; doi: https://doi.org/10.1101/2023.09.27.559752 \\n Add to Selected Citations\\n\\n\\n\\nSystematic perturbations of SETD2, NSD1, NSD2, NSD3 and ASH1L reveals their distinct contributions to H3K36 methylation \\nGerry Shipman, Reinnier Padilla, Cynthia Horth, Bo Hu, Eric Bareke, Francisca Vitorino, Joanna Gongora, Benjamin Garcia, Chao Lu, Jacek Majewski\\nbioRxiv 2023.09.27.559313; doi: https://doi.org/10.1101/2023.09.27.559313 \\n Add to Selected Citations\\n\\n\\n\\nAn amphioxus neurula stage cell atlas supports a complex scenario for the emergence of vertebrate head mesoderm \\nXavier Grau-Bové, Lucie Subirana, Lydvina Meister, Anaël Soubigou, Ana Neto, Anamaria Elek, Oscar Fornas, Jose Luis Gomez-Skarmeta, Juan J Tena, Manuel Irimia, Stéphanie Bertrand, Arnau Sebé-Pedrós, Hector Escriva\\nbioRxiv 2023.09.26.559513; doi: https://doi.org/10.1101/2023.09.26.559513 \\n Add to Selected Citations\\n\\n\\n\\nQuantitative trait loci mapping of circulating metabolites in cerebrospinal fluid to uncover biological mechanisms involved in brain-related phenotypes \\nLianne M. Reus, Toni Boltz, Marcelo Francia, Merel Bot, Naren Ramesh, Maria Koromina, Yolande A.L. Pijnenburg, Anouk den Braber, Wiesje  M. van der Flier, Pieter Jelle Visser, Sven J van der Lee, Betty Tijms, Charlotte E. Teunissen, Loes M Olde Loohuis, Roel A. Ophoff\\nbioRxiv 2023.09.26.559021; doi: https://doi.org/10.1101/2023.09.26.559021 \\n Add to Selected Citations\\n\\n\\n\\nReach, adoption, and implementation strategies of a telehealth falls prevention program: perspectives from francophone communities across Canada \\nJennifer O\\'Neil, Nathalie Dionne, Sylvie Marchand, Dominique Cardinal, Grant Handrigan, Jacinthe Savard\\nmedRxiv 2023.09.27.23295569; doi: https://doi.org/10.1101/2023.09.27.23295569 \\n Add to Selected Citations\\n\\n\\n\\nTrans-biobank Mendelian randomization analyses identify opposing pathways in plasma low-density lipoprotein-cholesterol lowering and gallstone disease \\nGuoyi Yang, Amy Mason, Dipender Gill, C M Schooling, Stephen Burgess\\nmedRxiv 2023.09.27.23296205; doi: https://doi.org/10.1101/2023.09.27.23296205 \\n Add to Selected Citations\\n\\n\\n\\nPatient Perception of Endoscopic and Medical Therapies for Weight Loss \\nMonica Saumoy, Yinglin Gao, Kelly Allison, Peter Cronholm, Octavia Pickett-Blakely, Michael Kochman, Nikhil Thiruvengadam\\nmedRxiv 2023.09.26.23296199; doi: https://doi.org/10.1101/2023.09.26.23296199 \\n Add to Selected Citations\\n\\n\\n\\nAssociations of Combined Phenotypic Aging and Genetic Risk with Incident Cancer: A Prospective Cohort Study \\nLijun Bian, Zhimin Ma, Xiangjin Fu, Chen Ji, Tianpei Wang, Caiwang Yan, Juncheng Dai, Hongxia Ma, Zhibin Hu, Hongbing Shen, Lu Wang, Meng Zhu, Guangfu Jin\\nmedRxiv 2023.09.27.23296204; doi: https://doi.org/10.1101/2023.09.27.23296204 \\n Add to Selected Citations\\n\\n\\n\\nThe potential health and economic impacts of new tuberculosis vaccines under varying delivery strategies in Delhi and Gujarat, India: a modelling study \\nRebecca A Clark, Allison Portnoy, Chathika Krishan Weerasuriya, Tom Sumner, Roel Bakker, Rebecca C Harris, Kirankumar Rade, Sanjay Kumar Mattoo, Dheeraj Tumu, Nicolas A Menzies, Richard G White\\nmedRxiv 2023.09.27.23296211; doi: https://doi.org/10.1101/2023.09.27.23296211 \\n Add to Selected Citations\\n\\n\\n\\nAssociation of Depression Symptoms and Biomarkers of Risk on Clinical Outcomes in HFrEF \\nAndrew Sherwood, James A. Blumenthal, Robert J Mentz, Gary G. Koch, Joseph G Rogers, Patricia P Chang, Christopher V Chien, Kirkwood F. Adams, Mark Paul Donahue, Lisa Rose-Jones, Brian C. Jensen, Kristy Johnson, Alan Hinderliter\\nmedRxiv 2023.09.26.23296194; doi: https://doi.org/10.1101/2023.09.26.23296194 \\n Add to Selected Citations\\n\\n\\n\\nDeep mutational scanning of Pneumocystis jirovecii dihydrofolate reductase reveals allosteric mechanism of resistance to an antifolate \\nFrancois D. Rouleau, Alexandre K Dubé, Isabelle Gagnon-Arsenault, Soham Dibyachintan, Alicia Pageau, Philippe C Després, Patrick Lagüe, Christian R Landry\\nbioRxiv 2023.09.27.559786; doi: https://doi.org/10.1101/2023.09.27.559786 \\n Add to Selected Citations\\n\\n\\n\\nIntimate partner violence and its correlates in middle-aged and older adults during the COVID-19 pandemic: A multi-country secondary analysis \\nGwendolyn Chang, Joseph Tucker, Kate Walker, Claire Chu, Naomi Miall, Rayner Tan, Dan Wu\\nmedRxiv 2023.09.26.23296197; doi: https://doi.org/10.1101/2023.09.26.23296197 \\n Add to Selected Citations\\n\\n\\n\\nInterpretable Metric Learning in Comparative Metagenomics: The Adaptive Haar-like Distance \\nEvan  D. Gorman, Manuel  E. Lladser\\nbioRxiv 2023.09.27.559681; doi: https://doi.org/10.1101/2023.09.27.559681 \\n Add to Selected Citations\\n\\n\\n\\nAdiposity and metabolic health in Asian populations: An epidemiological study using Dual X-Ray Absorptiometry \\nTheresia Mina, Wubin Xie, Dorrain Low Yan Wen, Xiaoyan Wang, Benjamin Lam Chih Chiang, Nilanjana Sadhu, Hong Kiat Ng, Nor Azizah Abdul Aziz, Terry Tong Yoke Yin, Swat Kim kerk, Wee Lin Choo, Guo Liang Low, Halimah Ibrahim, Li Ming Lim, Gervais Wansaicheong, Rinkoo Dalan, Yik Weng Yew, Paul Elliott, Elio Riboli, Marie Loh Chiew Shia, Joanne Ngeow Yuen Yie, Eng Sing Lee, Jimmy Lee Chee Keong, James Best, John Chambers\\nmedRxiv 2023.09.26.23296180; doi: https://doi.org/10.1101/2023.09.26.23296180 \\n Add to Selected Citations\\n\\n\\n\\nLessons from a multilaboratorial task force for diagnosis of a fatal toxoplasmosis outbreak in captive primates in Brazil \\nFrancine  Bittencourt Schiffler, Silvia  Bahadian Moreira, Asheley Henrique  Barbosa Pereira, Igor  Falco Arruda, Filipe Romero  Rebello Moreira, Mirela D\\'arc, Ingra  Morales Claro, Thalita  de Abreu Pissinatti, Liliane  Tavares de Faria Cavalcante, Thamiris  dos Santos Miranda, Matheus Augusto  Calvano Cosentino, Renata  Carvalho de Oliveira, Jorlan  Fernandes de Jesus, Matheus  Ribeiro da Silva Assis, Jonathan  Gonçalves de Oliveira, Thayssa  Alves Coelho da Silva, Rafael  Mello Galliez, Debora  Souza Faffe, Jaqueline  Goes de Jesus, Marise  Sobreira Bezerra da Silva, Matheus  Filgueira Bezerra, Orlando  da Costa Ferreira Junior, Amilcar Tanuri, Terezinha Marta  Pereira Pinto Castiñeiras, Renato  Santana Aguiar, Nuno  Rodrigues Faria, Alzira  Paiva Almeida, Alcides Pissinatti, Ester  Cerdeira Sabino, Maria Regina  Reis Amendoeira, Elba Regina  Sampaio de Lemos, Daniel  Guimarães Ubiali, Andre  FA Santos\\nbioRxiv 2023.09.27.559677; doi: https://doi.org/10.1101/2023.09.27.559677 \\n Add to Selected Citations\\n\\n\\n\\nMapping the biochemical landscape of rubisco \\nNoam Prywes, Naiya R Phillips, Luke M Oltrogge, Benoit de Pins, Aidan E Cowan, Leah J Taylor-Kearney, Hana A Chang, Laina N Hall, Abhishek Bhatt, Patrick Shih, Ron Milo, David F Savage\\nbioRxiv 2023.09.27.559826; doi: https://doi.org/10.1101/2023.09.27.559826 \\n Add to Selected Citations\\n\\n\\n\\nClinically actionable pharmacogenomic landscape of antidepressants and antipsychotics in Qatar: A population-based cohort study \\nDinesh Velayutham, Kholoud MHN Abdulla Bastaki, the Qatar Genome Program Research Consortium, Suhaila Ghuloum, Muhammad Waqar Azeem, Munir Pirmohamed, Puthen Veettil Jithesh\\nmedRxiv 2023.09.27.23296201; doi: https://doi.org/10.1101/2023.09.27.23296201 \\n Add to Selected Citations\\n\\n\\n\\n5-hydroxymethylcytosines regulate gene expression as a passive DNA demethylation resisting epigenetic mark in proliferative somatic cells \\nAlex Wei, Hongjie Zhang, Qi Qiu, Emily B. Fabyanic, Peng Hu, Hao Wu\\nbioRxiv 2023.09.26.559662; doi: https://doi.org/10.1101/2023.09.26.559662 \\n Add to Selected Citations\\n\\n\\n\\nAssessing dose-response relations of lipid traits with coronary artery disease, all-cause mortality, and cause-specific mortality: a linear and non-linear Mendelian randomization study \\nGuoyi Yang, Amy Mason, Angela Wood, C M Schooling, Stephen Burgess\\nmedRxiv 2023.09.27.23296203; doi: https://doi.org/10.1101/2023.09.27.23296203 \\n Add to Selected Citations\\n\\n\\n\\nThe Use of Electronic Health Record Embedded MRC-ICU as a Metric for Critical Care Pharmacist Workload \\nAndrew Webb, Bayleigh Carver, Sandra Rowe, Andrea Sikora\\nmedRxiv 2023.09.27.23296158; doi: https://doi.org/10.1101/2023.09.27.23296158 \\n Add to Selected Citations\\n\\n\\n\\nHost metabolic pathways essential for malaria and related hemoparasites in the infection of nucleated cells \\nMarina Maurizio, Maria Masad, Kerry Woods, Reto Caldelari, John G Doench, Arunasalam Naguleswaran, Denis Joly, Martin Gonzalez Fernandez, Jonas Zemp, Melanie Borteele, Vassily Hatzimanikatis, Volker Heussler, Sven Rottenberg, Philipp Olias\\nbioRxiv 2023.09.27.559824; doi: https://doi.org/10.1101/2023.09.27.559824 \\n Add to Selected Citations\\n\\n\\n\\nSoil surveillance for monitoring soil-transmitted helminth infections: method development and field testing in three countries \\nMalathi Manuel, Heather Amato, Nils Pilotte, Benard Chieng, Joel Siko, Michael Harris, Maya Nadimpalli, Venkateshprabhu Janagaraj, Parfait Houngbegnon, Rajeshkumar Rajendiran, Joel Thamburaj, Saravanakumar  Puthupalayam Kaliappan, Allison  R Sirois, Gretchen Walch, Will Oswald, Kristjana  Hrönn Ásbjörnsdóttir, Sean Galagan, Judd L Walson, Steven  A Williams, Adrian  J.F. Luty, Sammy Njenga, Moudachirou Ibikounle, Sitara  S. R. Ajjampur, Amy Pickering\\nmedRxiv 2023.09.26.23296174; doi: https://doi.org/10.1101/2023.09.26.23296174 \\n Add to Selected Citations\\n\\n\\n\\nSex-Related Differences in Long-term Outcomes across the Spectrum of Ejection Fraction in Heart Failure Patients \\nAkane Kawai, WET-HF investivators, Yuji Nagatomo, Midori Yukino Iwashita, Yukinori Ikegami, Makoto Takei, Takashi Kohno, Ayumi Goda, Atsushi Mizuno, Mitsunobu Kitamura, Munehisa Sakamoto, Shintaro Nakano, Yasuyuki Shiraishi, Shun Kohsaka, Takeshi Adachi, Tsutomu Yoshikawa\\nmedRxiv 2023.09.26.23296192; doi: https://doi.org/10.1101/2023.09.26.23296192 \\n Add to Selected Citations\\n\\n\\n\\nStarch can expedite the screening for bacterial aflatoxin degraders \\nNatalie Sandlin, Babak Momeni\\nbioRxiv 2023.09.27.559811; doi: https://doi.org/10.1101/2023.09.27.559811 \\n Add to Selected Citations\\n\\n\\n\\nPotential of MALDI-TOF MS biotyping to detect deltamethrin resistance in the dengue vector Aedes aegypti. \\nLionel ALMERAS, Monique  Melo Costa, Remy Amalvict, Joseph Guillet, Isabelle Dusfour, Jean-Philippe David, Vincent Corbel\\nbioRxiv 2023.09.27.559680; doi: https://doi.org/10.1101/2023.09.27.559680 \\n Add to Selected Citations\\n\\n\\n\\nClusterin can mediate apoptosis-induced molecular mechanisms in immune thrombocytopenia \\nTea Stein, Christina Bitsina, Markus Schmugge, Francesca Daniela Franzoso\\nbioRxiv 2023.09.26.559483; doi: https://doi.org/10.1101/2023.09.26.559483 \\n Add to Selected Citations\\n\\n\\n\\nIdentification of Host Restriction Factors Critical for Recombinant AAV Transduction of Polarized Human Airway Epithelium \\nSiyuan Hao, Xiujuan Zhang, Kang Ning, Zehua Feng, Soo Yeun Park, Cagla Aksu Kuz, Shane McFarlin, Donovan Richart, Fang Cheng, Elizabeth Yan Zhang, Aaron Zhang-Chen, Ziying Yan, Jianming Qiu\\nbioRxiv 2023.09.27.559795; doi: https://doi.org/10.1101/2023.09.27.559795 \\n Add to Selected Citations\\n\\n\\n\\nPCR-based survey of methane-cycling archaea in methane-soaked subsurface sediments of Guaymas Basin, Gulf of California \\nJohn Hinkle, Paraskevi Mara, David Beaudoin, Virginia P Edgcomb, Andreas P Teske\\nbioRxiv 2023.09.27.559568; doi: https://doi.org/10.1101/2023.09.27.559568 \\n Add to Selected Citations\\n\\n\\n\\nEngineered Lactiplantibacillus plantarum as a Biosensor Probe for the Lungs \\nMichael Brasino, Eli Wagnell, Elise C Manalo, Samuel Drennan, Jared M Fischer, Justin Merritt\\nbioRxiv 2023.09.27.559779; doi: https://doi.org/10.1101/2023.09.27.559779 \\n Add to Selected Citations\\n\\n\\n\\nTemporal profiling of Salmonella transcriptional dynamics during macrophage infection using a comprehensive reporter library \\nTaylor H Nguyen, Oscar R. Diaz, Manohary Rajendram, Daniel S.C. Butler, Benjamin X. Wang, Jay C. D. Hinton, Denise M. Monack, Kerwyn Casey Huang\\nbioRxiv 2023.09.27.559620; doi: https://doi.org/10.1101/2023.09.27.559620 \\n Add to Selected Citations\\n\\n\\n\\nNovel insights into phage biology of the pathogen Clostridioides difficile based on the active virome \\nMiriam Antonia Schueler, Rolf Daniel, Anja Poehlein\\nbioRxiv 2023.09.27.559748; doi: https://doi.org/10.1101/2023.09.27.559748 \\n Add to Selected Citations\\n\\n\\n\\nColonic spermidine promotes the proliferation and migration of intestinal epithelial cells. \\nMadison Flory, Angela Gao, Morgan Morrow, Ashfaqul Alam\\nbioRxiv 2023.09.26.559404; doi: https://doi.org/10.1101/2023.09.26.559404 \\n Add to Selected Citations\\n\\n\\n\\nPre-piRNA trimming safeguards piRNAs against erroneous targeting by RNA-dependent RNA Polymerase \\nWEN TANG, Benjamin Pastore, Hannah L Hertz\\nbioRxiv 2023.09.26.559619; doi: https://doi.org/10.1101/2023.09.26.559619 \\n Add to Selected Citations\\n\\n\\n\\nMassively parallel identification of sequence motifs triggering ribosome-associated mRNA quality control \\nKatharine Y Chen, Heungwon Park, Arvind Rasi Subramaniam\\nbioRxiv 2023.09.27.559793; doi: https://doi.org/10.1101/2023.09.27.559793 \\n Add to Selected Citations\\n\\n\\n\\nA SUMO E3 ligase promotes long non-coding RNA transcription to regulate small RNA-directed DNA elimination \\nSalman Shehzada, Tomoko Noto, Julie Saksouk, Kazufumi Mochizuki\\nbioRxiv 2023.09.27.558415; doi: https://doi.org/10.1101/2023.09.27.558415 \\n Add to Selected Citations\\n\\n\\n\\n53BP1 interacts with the RNA primer from Okazaki fragments to support their processing during unperturbed DNA replication \\nMelissa Leriche, Clara Bonnet, Jagannath Jana, Gita Chhetri, Sabrina Mennour, Sylvain Martineau, Vincent Pennaneach, Didier BUSSO, Xavier Veaute, Pascale Bertrand, sarah AE lambert, Kumar Somyajit, Patricia Uguen, Stephan Vagner\\nbioRxiv 2023.09.27.559698; doi: https://doi.org/10.1101/2023.09.27.559698 \\n Add to Selected Citations\\n\\n\\n\\nRlmQ: A Newly Discovered rRNA Modification Enzyme Bridging RNA Modification and Virulence Traits in Staphylococcus aureus \\nRoberto Bahena-Ceron, Chloe Teixeira, Jose R Jaramillo Ponce, Philippe Wolff, Florence Couzon, Pauline François, Bruno Klaholz, François Vandenesch, Pascale Romby, Karen Moreau, Stefano Marzi\\nbioRxiv 2023.09.27.559750; doi: https://doi.org/10.1101/2023.09.27.559750 \\n Add to Selected Citations\\n\\n\\n\\nCROTONdb: genome-wide variant effect prediction database for CRISPR/Cas9 editing outcomes \\nVictoria R Li, Alicja Tadych, Aaron K Wong, Zijun Zhang\\nbioRxiv 2023.09.26.559657; doi: https://doi.org/10.1101/2023.09.26.559657 \\n Add to Selected Citations\\n\\n\\n\\nQuantifying nutrient colimitation of microbial growth \\nNoelle A. Held, Aswin Krishna, Donat Crippa, Anastasia Dragan, Michael Manhart\\nbioRxiv 2023.09.27.559472; doi: https://doi.org/10.1101/2023.09.27.559472 \\n Add to Selected Citations\\n\\n\\n\\nDairy practices select for thermostable endolysins in phages \\nFrank Oechslin, Xiaojun Zhu, Carlee Morency, Vincent Somerville, Rong Shi, Sylvain Moineau\\nbioRxiv 2023.09.27.559745; doi: https://doi.org/10.1101/2023.09.27.559745 \\n Add to Selected Citations\\n\\n\\n\\nUninheritable but widespread bacterial symbiont mediates insecticide detoxification of an agricultural invasive pest Spodoptera frugiperda \\nYunhua Zhang, Feng Ju\\nbioRxiv 2023.09.26.559648; doi: https://doi.org/10.1101/2023.09.26.559648 \\n Add to Selected Citations\\n\\n\\n\\nBroad intraspecies killing activity in Pseudomonas syringae due to the combinatorial action of LPS-interacting bacteriocins \\nSavannah Weaver, Emma Casamassima, Anh Evy Nguyen, David A Baltrus\\nbioRxiv 2023.09.27.559845; doi: https://doi.org/10.1101/2023.09.27.559845 \\n Add to Selected Citations\\n\\n\\n\\nDiurnal cycles drive rhythmic physiology and promote survival in facultative phototrophic bacteria \\nCamille Tinguely, Mélanie Paulméry, Céline Terrettaz, Diego Gonzalez\\nbioRxiv 2023.09.27.559767; doi: https://doi.org/10.1101/2023.09.27.559767 \\n Add to Selected Citations\\n\\n\\n\\nBacteria Contribute Exopolysaccharides to an Algal-Bacterial Joint Extracellular Matrix \\nValeria Lipsman, Olesia Shlakhter, Jorge Rocha, Einat Segev\\nbioRxiv 2023.09.27.559704; doi: https://doi.org/10.1101/2023.09.27.559704 \\n Add to Selected Citations\\n\\n\\n\\nGenomic and Phenotypic Comparison of Polyhydroxyalkanoates Producing Strains of genus Caldimonas/Schlegelella \\nJana Musilova, Xenie Kourilova, Kristyna Hermankova, Matej Bezdicek, Anastasiia Ieremenko, Pavel Dvorak, Stanislav Obruca, Karel Sedlar\\nbioRxiv 2023.09.27.559687; doi: https://doi.org/10.1101/2023.09.27.559687 \\n Add to Selected Citations\\n\\n\\n\\nInfluence of preprocessing, distortion correction and cardiac triggering on the quality of diffusion MR images of spinal cord \\nKurt Schilling, Anna Combes, Karthick Ramadass, Francois Rheault, Grace Sweeney, Logan Prock, Subramaniam Sriram, Julien Cohen-Adad, John C Gore, Bennett A Landman, Seth Smith, Kristin P. O\\'Grady\\nbioRxiv 2023.09.26.559530; doi: https://doi.org/10.1101/2023.09.26.559530 \\n Add to Selected Citations\\n\\n\\n\\nConditional Chemoconnectomics: A Set of Libraries Targeting All Chemical Transmission Corresponding Genes Efficiently \\nRenbo Mao, Jianjun Yu, Bowen Deng, Xihuimin Dai, Yuyao Du, Sujie Du, Wenxia Zhang, Yi Rao\\nbioRxiv 2023.09.26.559642; doi: https://doi.org/10.1101/2023.09.26.559642 \\n Add to Selected Citations\\n\\n\\n\\nAdaptive Group Behavior of Fragile X Mice in Unfamiliar Environments \\nGabriele Giua, Benjamin Strauss, Olivier Lassalle, Pascale Chavis, Olivier J Manzoni\\nbioRxiv 2023.09.26.559495; doi: https://doi.org/10.1101/2023.09.26.559495 \\n Add to Selected Citations\\n\\n\\n\\nFAM210B is dispensable for erythroid differentiation in adult mice \\nMark Perfetto, Catherine M Rondelli, Samantha Gillis, Amber N Stratman, Yvette Y Yien\\nbioRxiv 2023.09.26.559581; doi: https://doi.org/10.1101/2023.09.26.559581 \\n Add to Selected Citations\\n\\n\\n\\nA Phase 2, randomized, double-blind, placebo-controlled multi-center trial sub-study for the clinical effects of paridiprubart treatment in hospitalized critically ill patients with COVID-19 ARDS \\nBlair Gordon, Fiona Allum, Michael Brooks, Nishani Rajakulendran, Emmanouil Rampakakis, John Sampalis, EB05 Study Investigators\\nmedRxiv 2023.09.21.23295853; doi: https://doi.org/10.1101/2023.09.21.23295853 \\n Add to Selected Citations\\n\\n\\n\\nShort- and long-term experience-dependent neuroplasticity interact during the perceptual learning of concurrent speech \\nJessica MacLean, Jack Stirn, Alexandria Sisson, Gavin Bidelman\\nbioRxiv 2023.09.26.559640; doi: https://doi.org/10.1101/2023.09.26.559640 \\n Add to Selected Citations\\n\\n\\n\\nDifferential Patterns of Associations within Audiovisual Integration Networks in Children with ADHD \\nMohammad Zamanzadeh, Abbas Pourhedayat, Fatemeh Bakouie, Fatemeh Hadaeghi\\nbioRxiv 2023.09.26.559610; doi: https://doi.org/10.1101/2023.09.26.559610 \\n Add to Selected Citations\\n\\n\\n\\nDopamine neuron dysfunction and loss in the PrknR275W mouse model of Juvenile Parkinsonism \\nMaria Regoni, Letizia Zanetti, Martina Sevegnani, Chiara Domenicale, Stefano Magnabosco, Jyoti C. Patel, Megan K Fernandes, Elena Monzani, Stefano Comai, Laura Cherchi, Andrea Ciammola, Flavia Valtorta, Michele Morari, Giovanni Piccoli, Margaret E Rice, Jenny Sassone\\nbioRxiv 2023.09.26.559326; doi: https://doi.org/10.1101/2023.09.26.559326 \\n Add to Selected Citations\\n\\n\\n\\nNeural circuit mechanisms underlying context-specific halting in Drosophila. \\nNeha Sapkal, Nino Mancini, Divya Sthanu Kumar, Nico Spiller, Kazuma Murakami, Gianna Vitelli, Benjamin Bargeron, Kate Maier, Katharina Eichler, Gregory S. X. E. Jefferis, Philip Shiu, Gabriella R. Sterne, Salil S Bidaye\\nbioRxiv 2023.09.25.559438; doi: https://doi.org/10.1101/2023.09.25.559438 \\n Add to Selected Citations\\n\\n\\n\\nAxon morphology and intrinsic cellular properties determine repetitive transcranial magnetic stimulation threshold for plasticity \\nChristos Galanis, Lena Neuhaus, Nicholas Hananeia, Zsolt Turi, Peter Jedlicka, Andreas Vlachos\\nbioRxiv 2023.09.25.559399; doi: https://doi.org/10.1101/2023.09.25.559399 \\n Add to Selected Citations\\n\\n\\n\\nThe impact of overall light-level on the reverse Pulfrich effect \\nVictor Rodriguez-Lopez, Benjamin M Chin, Johannes Burge\\nbioRxiv 2023.09.27.559782; doi: https://doi.org/10.1101/2023.09.27.559782 \\n Add to Selected Citations\\n\\n\\n\\nSensorimotor Integration Supporting Perception Requires Syngap1 Expression in Cortex \\nThomas Vaissiere, Sheldon Michaelson, Thomas Creson, Jessie Goins, Daniel Fürth, Diana Balazsfi, Camilo Rojas, Randall Golovin, Konstantinos Meletis, Courtney Miller, Daniel H. O\\'Connor, Gavin Rumbaugh\\nbioRxiv 2023.09.27.559787; doi: https://doi.org/10.1101/2023.09.27.559787 \\n Add to Selected Citations\\n\\n\\n\\nSpatial transcriptomics and single-nucleus RNA sequencing reveal a transcriptomic atlas of adult human spinal cord \\nDonghang Zhang, Yali Chen, Yiyong Wei, Hongjun Chen, Yujie Wu, Lin Wu, Jin Li, Qiyang Ren, Changhong Miao, Tao Zhu, Jin Liu, Bowen Ke, Cheng Zhou\\nbioRxiv 2023.09.26.559582; doi: https://doi.org/10.1101/2023.09.26.559582 \\n Add to Selected Citations\\n\\n\\n\\nDevelopmental Ramifications of the Screen Age: Associations between Screen Time and Neural Maturation in a Massive Adolescent Cohort \\nAlexander S Atalay, Benjamin T Newman, T Jason Druzgal\\nbioRxiv 2023.09.27.559772; doi: https://doi.org/10.1101/2023.09.27.559772 \\n Add to Selected Citations\\n\\n\\n\\nGenomic evolution of SARS-CoV-2 variants of concern under in vitro neutralising selection pressure following two doses of the Pfizer-BioNTech BNT162b2 COVID-19 vaccine \\nKerri Basile, Jessica E. Agius, Winkie Fong, Kenneth McPhie, Michael Fennel, Danny Ko, Linda Hueston, Connie Lam, Alicia Arnott, Sharon C-A Chen, Susan Maddocks, Matthew V. N. O Sullivan, Dominic Dwyer, Vitali Sintchenko, Jen Kok, Rebecca J Rockett, CIDMLS COVID-19 Study Group\\nbioRxiv 2023.09.24.558921; doi: https://doi.org/10.1101/2023.09.24.558921 \\n Add to Selected Citations\\n\\n\\n\\nIndividual Behavioral Variability Across Time and Contexts in Dendrobates tinctorius Poison Frogs \\nKatharina M Soto, Eva K Fischer, Faith O Hardin, Harmen P Alleyne\\nbioRxiv 2023.09.25.559405; doi: https://doi.org/10.1101/2023.09.25.559405 \\n Add to Selected Citations\\n\\n\\n\\nListening to classical music influences brain connectivity in post-stroke aphasia: a pilot study \\nMaryane Chea, Amina Ben Salah, Monica N. Toba, Ryan Zeineldin, Brigitte C. Kaufmann, Agnes Weill-Chounlamountry, Lionel Naccache, Eleonore Bayen, Paolo Bartolomeo\\nbioRxiv 2023.09.27.559699; doi: https://doi.org/10.1101/2023.09.27.559699 \\n Add to Selected Citations\\n\\n\\n\\nAvoidance, confusion or solitude? Modelling how noise pollution affects whale migration \\nStuart T Johnston, Kevin J Painter\\nbioRxiv 2023.09.27.559669; doi: https://doi.org/10.1101/2023.09.27.559669 \\n Add to Selected Citations\\n\\n\\n\\nGut microbiota promotes enteroendocrine cell maturation and mitochondrial function \\nAlfahadh Alsudayri, Shane Perelman, Annika Chura, Melissa Brewer, Maddie McDevitt, Catherine Drerup, Lihua Ye\\nbioRxiv 2023.09.27.558332; doi: https://doi.org/10.1101/2023.09.27.558332 \\n Add to Selected Citations\\n\\n\\n\\nPrimate Saccade Rhythmicity \\nTim Näher, Yufeng Zhang, Martina Pandinelli, Pascal Fries\\nbioRxiv 2023.09.27.559710; doi: https://doi.org/10.1101/2023.09.27.559710 \\n Add to Selected Citations\\n\\n\\n\\nVision plays a calibrating role in discriminating threat-related vocal emotions \\nFederica Falagiarda, Valeria Occelli, Olivier Collignon\\nbioRxiv 2023.09.27.559716; doi: https://doi.org/10.1101/2023.09.27.559716 \\n Add to Selected Citations\\n\\n\\n\\nSleep and circadian rhythm disruption by NPTX2 loss of function \\nSeung-Eon Roh, Meifang Xiao, Ana Delgado, Chuljung Kwak, Alena Savonenko, Arnold Bakker, Hyung-Bae Kwon, Paul Worley\\nbioRxiv 2023.09.26.559408; doi: https://doi.org/10.1101/2023.09.26.559408 \\n Add to Selected Citations\\n\\n\\n\\nTrait level somatic arousal modulates fMRI neural synchrony to naturalistic stimuli \\nKeva Klamer, Joshua Craig, KiAnna Sullivan, Christina Haines, Chelsea Ekstrand\\nbioRxiv 2023.09.27.559823; doi: https://doi.org/10.1101/2023.09.27.559823 \\n Add to Selected Citations\\n\\n\\n\\nNeural bases of reading fluency: A systematic review and meta-analysis \\nMarissa Marko Lee, Catherine Stoodley\\nbioRxiv 2023.09.25.559403; doi: https://doi.org/10.1101/2023.09.25.559403 \\n Add to Selected Citations\\n\\n\\n\\nMitochondrial Dysfunction and Neuronal Anomalies in POLG Mutant Midbrain Organoids \\nAnbin Chen, Tsering Yangzom, Gareth J Sullivan, Kristina Xiao Liang\\nbioRxiv 2023.09.27.559684; doi: https://doi.org/10.1101/2023.09.27.559684 \\n Add to Selected Citations\\n\\n\\n\\nComparative Analysis of SARS-CoV-2 Antigenicity across Assays and in Human and Animal Model Sera \\nBarbara Mühlemann, Samuel Hedley Wilks, Lauren Baracco, Meriem Bekliz, Juan Manuel Carreno, Victor M Corman, Meredith E Davis-Gardner, Wanwisa Dejnirattisai, Michael Diamond, Danny Doueck, Christian Drosten, Isabella Eckerle, Venkata Vishwanadh Edara, Madison Ellis, Ron AM Fouchier, Matthew Frieman, Sucheta Godbole, Bart Haagmans, Peter Halfmann, Amy R Henry, Terry C Jones, Leah Katzelnick, Yoshihiro Kawaoka, Janine Kimpel, Florian Krammer, Lilin Lai, Chang Liu, Sabrina Lusvarghi, Benjamin Meyer, Juthathip Mongkolsapaya, David Montefiori, Anna Z Mykytyn, Antonia Netzl, Simon Pollett, Annika Rössler, Gavin Screaton, Xiaoying Shen, Alex Sigal, Viviana Simon, Rahul Subramanian, Piyada Supasa, Mehul Suthar, Sina Tureli, Wei Wang, Carol D. Weiss, Derek J Smith\\nbioRxiv 2023.09.27.559689; doi: https://doi.org/10.1101/2023.09.27.559689 \\n Add to Selected Citations\\n\\n\\n\\nNotch signalling plays a critical role in patterning the ventral mesoderm during early embryogenesis in Drosophila melanogaster \\nMarvel Megaly, Gregory Foran, Arsala Ali, Anel Turgambayeva, Ryan D. Hallam, Aleksandar Necakov\\nbioRxiv 2023.09.27.558900; doi: https://doi.org/10.1101/2023.09.27.558900 \\n Add to Selected Citations\\n\\n\\n\\nConvergent generation of atypical prions in knock-in mouse models of genetic prion disease \\nSurabhi Mehra, Matthew E.C. Bourkas, Lech Kaczmarczyk, Erica Stuart, Hamza Arshad, Jennifer K. Griffin, Kathy L. Frost, Daniel J. Walsh, Surachai Supattapone, Stephanie A. Booth, Walker Scot Jackson, Joel C Watts\\nbioRxiv 2023.09.26.559572; doi: https://doi.org/10.1101/2023.09.26.559572 \\n Add to Selected Citations\\n\\n\\n\\nElegansBot: Development of equation of motion deciphering locomotion including omega turns of Caenorhabditis elegans \\nTaegon Chung, Iksoo Chang, Sangyeol Kim\\nbioRxiv 2023.09.26.559644; doi: https://doi.org/10.1101/2023.09.26.559644 \\n Add to Selected Citations\\n\\n\\n\\nEfficient Search for Circuit Structure by \\'Smooth-Index\\' Matrix-Reordering \\nAlexander Borst, Winfried Denk\\nbioRxiv 2023.09.26.559501; doi: https://doi.org/10.1101/2023.09.26.559501 \\n Add to Selected Citations\\n\\n\\n\\nDefective mesenchymal Bmpr1a-mediated BMP signaling causes congenital pulmonary cysts \\nYongfeng Luo, Ke Cao, Joanne Chiu, Hui Chen, Hong-Jun Wang, Matthew E. Thornton, Brendan H. Grubbs, Martin Kolb, Michael S. Parmacek, Yuji Mishina, Wei Shi\\nbioRxiv 2023.09.26.559527; doi: https://doi.org/10.1101/2023.09.26.559527 \\n Add to Selected Citations\\n\\n\\n\\nA Reinforcement Learning Model with Choice Traces for a Progressive Ratio Schedule \\nKeiko Ihara, Yu Shikano, Sae Kato, Sho Yagishita, Kenji F Tanaka, Norio Takata\\nbioRxiv 2023.09.26.559283; doi: https://doi.org/10.1101/2023.09.26.559283 \\n Add to Selected Citations\\n\\n\\n\\nMapping the task-general and task-specific neural correlates of speech production: meta-analysis and fMRI direct comparisons of category fluency and picture naming \\nGina F Humphreys, Matthew Lambon Ralph\\nbioRxiv 2023.09.27.559692; doi: https://doi.org/10.1101/2023.09.27.559692 \\n Add to Selected Citations\\n\\n\\n\\nProteome profiling of brain vessels in a mouse model of cerebrovascular pathology \\nArsalan S Haqqani, Zainab Mianoor, Alexandra T Star, Flavie Detcheverry, Danica B Stanimirovic, Edith Hamel, AmanPreet Badhwar\\nbioRxiv 2023.09.27.559361; doi: https://doi.org/10.1101/2023.09.27.559361 \\n Add to Selected Citations\\n\\n\\n\\nGas5 regulates early life stress-induced anxiety and spatial memory \\nSourav Banerjee, Dipanjana Banerjee, Sania Sultana\\nbioRxiv 2023.09.27.559778; doi: https://doi.org/10.1101/2023.09.27.559778 \\n Add to Selected Citations\\n\\n\\n\\np38-MAPK is prerequisite for the synthesis of SARS-CoV-2 protein \\nPriyasi Mittal, Nitin Khandelwal, Yogesh Chander, Assim Verma, Ram Kumar, Chayanika Putatunda, Sanjay Barua, Baldev Raj Gulati, Naveen Kumar\\nbioRxiv 2023.09.27.559660; doi: https://doi.org/10.1101/2023.09.27.559660 \\n Add to Selected Citations\\n\\n\\n\\nChronic Intermittent Hypoxia reveals role of the Postinspiratory Complex in swallow pattern generation \\nAlyssa D Huff, Marlusa Karlen-Amarante, Luiz M Oliveira, Jan-Marino Ramirez\\nbioRxiv 2023.09.26.559560; doi: https://doi.org/10.1101/2023.09.26.559560 \\n Add to Selected Citations\\n\\n\\n\\nDISTINCT METABOLIC STATES DIRECT RETINAL PIGMENT EPITHELIUM CELL FATE DECISIONS \\nJ. Raul Perez-Estrada, Jared A Tangeman, Maeve Proto-Newton, Harshavardhan Sanaka, Byran Smucker, Katia Del Rio-Tsonis\\nbioRxiv 2023.09.26.559631; doi: https://doi.org/10.1101/2023.09.26.559631 \\n Add to Selected Citations\\n\\n\\n\\nNeocortico-hippocampal ripple-based coordination during naturalistic encoding \\nMarta Silva, Xiongbo Wu, Marc Sabio, Estefania Conde-Blanco, Pedro Roldan, Antonio Donaire, Mar Carreno, Nikolai Axmacher, Christopher Baldassano, Lluis Fuentemilla\\nbioRxiv 2023.09.27.559739; doi: https://doi.org/10.1101/2023.09.27.559739 \\n Add to Selected Citations\\n\\n\\n\\nOculomotor routines for perceptual judgments \\nAvi Aizenman, Karl Gegenfurtner, Alexander Goettker\\nbioRxiv 2023.09.27.559695; doi: https://doi.org/10.1101/2023.09.27.559695 \\n Add to Selected Citations\\n\\n\\n\\nAfferents to Action: Cortical proprioceptive processing assessed with corticokinematic coherence specifically predicts gross motor skills \\nScott J Mongold, Christian Georgiev, Thomas Legrand, Mathieu Bourguignon\\nbioRxiv 2023.09.26.559273; doi: https://doi.org/10.1101/2023.09.26.559273 \\n Add to Selected Citations\\n\\n\\n\\nA tale of two receptors: simultaneous targeting of NMDARs and 5-HT4Rs exerts additive effects against stress \\nBriana K Chen, Victor M Luna, Michelle Jin, Abhishek Shah, Margaret E Shannon, Michaela M Pauers, Brenna L Williams, Vananh Pham, Holly C Hunsberger, Alain M Gardier, Indira Mendez-David, Denis J David, Christine Ann Denny\\nbioRxiv 2023.09.27.559065; doi: https://doi.org/10.1101/2023.09.27.559065 \\n Add to Selected Citations\\n\\n\\n\\nCircuit-based intervention corrects excessive dentate gyrus output in the Fragile X mouse model \\nPanyue Deng, Ajeet Kumar, Valeria Cavalli, Vitaly Klyachko\\nbioRxiv 2023.09.27.559792; doi: https://doi.org/10.1101/2023.09.27.559792 \\n Add to Selected Citations\\n\\n\\n\\nIdentifying the cortical face network with dynamic face stimuli: A large group fMRI study \\nDavid Pitcher, Geena Ianni, Kelsey Holiday, Leslie Ungerleider\\nbioRxiv 2023.09.26.559583; doi: https://doi.org/10.1101/2023.09.26.559583 \\n Add to Selected Citations\\n\\n\\n\\nAltered hierarchical gradients of intrinsic neural timescales in mild cognitive impairment and Alzheimer\\'s disease \\nAIYING ZHANG, Kenneth Wengler, Xi Zhu, Guillermo Horga, Terry Goldberg, seonjoo lee, For Alzheimer\\'s Disease Neuroimaging Initiative\\nbioRxiv 2023.09.26.559549; doi: https://doi.org/10.1101/2023.09.26.559549 \\n Add to Selected Citations\\n\\n\\n\\nCapturing temporal heterogeneity of communities: a temporal β-diversity based on Hill numbers and time series analysis \\nDaniel  Joaquin Sánchez-Ochoa, Edgar  J González, María  del Coro Arizmendi, Patricia Koleff, Raúl Martell-Dubois, Jorge  Arturo Meave, Hibraim  Adan Perez-Mendoza\\nbioRxiv 2023.09.25.559413; doi: https://doi.org/10.1101/2023.09.25.559413 \\n Add to Selected Citations\\n\\n\\n\\nThe most prominent electrophysiological effect of ketamine is not sufficient to cause anesthesia \\nBenjamin T Acland, Ben Julian A Palanca, Janine D Bijsterbosch, Lawrence H Snyder\\nbioRxiv 2023.09.26.559594; doi: https://doi.org/10.1101/2023.09.26.559594 \\n Add to Selected Citations\\n\\n\\n\\nExpansion of interferon signaling-associated gene (ISAG)-hi T cells in early-onset Alzheimer\\'s disease \\nDaniel W. Sirkis, Caroline Warly Solsberg, Taylor P. Johnson, Ethan G. Geier, Luke W. Bonham, Bruce L. Miller, Gil D. Rabinovici, Jennifer S. Yokoyama\\nbioRxiv 2023.09.26.559634; doi: https://doi.org/10.1101/2023.09.26.559634 \\n Add to Selected Citations\\n\\n\\n\\nPiezo1 agonist restores meningeal lymphatic vessels, drainage, and brain-CSF perfusion in craniosynostosis and aged mice \\nMatt J Matrongolo, Phillip S Ang, Junbing Wu, Aditya Jain, Joshua K Thackray, Akash Reddy, Chi Chang Sung, Gaetan Barbet, Young-Kwon Hong, Max Tischfield\\nbioRxiv 2023.09.27.559761; doi: https://doi.org/10.1101/2023.09.27.559761 \\n Add to Selected Citations\\n\\n\\n\\nThe Temporal Dynamics of Metacognitive Experiences Track Rational Adaptations in Task Performance \\nLuc Vermeylen, Senne Braem, Ivan I Ivanchei, Kobe Desender, Carlos González-García, J.M. García-Román, María Ruz, Wim Notebaert\\nbioRxiv 2023.09.26.559523; doi: https://doi.org/10.1101/2023.09.26.559523 \\n Add to Selected Citations\\n\\n\\n\\nRandomness impacts specific priors building, visual exploration, and perception in object recognition \\nCecile Gal, Ioana Tincas, Vasile Vlad Moca, Andrei Ciuparu, Emanuela Loredana Dan, Marie L. Smith, Teodora Gliga, Raul Cristian Muresan\\nbioRxiv 2023.09.26.559544; doi: https://doi.org/10.1101/2023.09.26.559544 \\n Add to Selected Citations\\n\\n\\n\\nA Scalable Staining Strategy for Whole-Brain Connectomics \\nXiaotang Lu, Yuelong Wu, Richard L. Schalek, Yaron Meirovitch, Daniel R. Berger, Jeff W. Lichtman\\nbioRxiv 2023.09.26.558265; doi: https://doi.org/10.1101/2023.09.26.558265 \\n Add to Selected Citations\\n\\n\\n\\nMMORF - FSL\\'s MultiMOdal Registration Framework \\nFrederik J Lange, Christoph Arthofer, Andreas Bartsch, Gwenaelle Douaud, Paul McCarthy, Stephen M Smith, Jesper L R Andersson\\nbioRxiv 2023.09.26.559484; doi: https://doi.org/10.1101/2023.09.26.559484 \\n Add to Selected Citations\\n\\n\\n\\nBed Nucleus of the Stria Terminalis (BNST) neurons containing the serotonin 5HT2c receptor modulate operant alcohol self-administration behavior in mice \\nMeghan E Flanigan, Carol Gianessi, Megan E Castle, Winifred Dorlean, Tori Sides, Thomas L Kash\\nbioRxiv 2023.09.26.559653; doi: https://doi.org/10.1101/2023.09.26.559653 \\n Add to Selected Citations\\n\\n\\n\\nA multiplexed, single-cell sequencing screen identifies compounds that increase neurogenic reprogramming of murine Muller glia. \\nAmy Tresenrider, Marcus Hooper, Levi Todd, Faith Kierney, Cole Trapnell, Nicolai Blasdel, Thomas A Reh\\nbioRxiv 2023.09.26.559569; doi: https://doi.org/10.1101/2023.09.26.559569 \\n Add to Selected Citations\\n\\n\\n\\nContribution of image statistics and semantics in local vs. distributed EEG decoding of rapid serial visual presentation \\nEric Lutzow Holm, Diego Fernandez Slezak, Enzo Tagliazucchi\\nbioRxiv 2023.09.26.559617; doi: https://doi.org/10.1101/2023.09.26.559617 \\n Add to Selected Citations\\n\\n\\n\\nEmergence of a somatosensory tonotopic map for substrate vibration in the brainstem \\nKuo-Sheng Lee, Dominica de Thomas Wagner, Mark Sanders, Daniel Huber\\nbioRxiv 2023.09.26.559502; doi: https://doi.org/10.1101/2023.09.26.559502 \\n Add to Selected Citations\\n\\n\\n\\nCell Typing and Sub-typing Based on Detecting Characteristic Subspaces of Morphological Features Derived from Neuron Images \\nSujun Zhao, Penghao Qian, Lijuan Liu\\nbioRxiv 2023.09.26.559442; doi: https://doi.org/10.1101/2023.09.26.559442 \\n Add to Selected Citations\\n\\n\\n\\nSex divergent behavioral responses in platform-mediated avoidance and glucocorticoid receptor blockade \\nCarly J Halcomb, Trey R Philipp, Parker S Dhillon, J. Hunter Cox, Ricardo Aguilar-Alvarez, Samantha O Vanderhoof, Aaron M Jasnow\\nbioRxiv 2023.09.26.559122; doi: https://doi.org/10.1101/2023.09.26.559122 \\n Add to Selected Citations\\n\\n\\n\\nMediodorsal thalamic nucleus mediates resistance to ethanol through Cav3.1 T-type Ca2+ regulation of neural activity \\ncharles-francois V Latchoumane, Joon-Hyuk Lee, Seong-Wook Kim, Jinhyun Kim, Hee-Sup Shin\\nbioRxiv 2023.09.25.558585; doi: https://doi.org/10.1101/2023.09.25.558585 \\n Add to Selected Citations\\n\\n\\n\\nIndividuals learning to drive solo before age 18 have superior spatial navigation ability compared with those who learn later \\nEmre Yavuz, Ed Manley, Christoffer J. Gahnstrom, Sarah Goodroe, Antoine Coutrot, Michael Hornberger, Hugo J. Spiers\\nbioRxiv 2023.09.26.559011; doi: https://doi.org/10.1101/2023.09.26.559011 \\n Add to Selected Citations\\n\\n\\n\\nWild sulphur-crested cockatoos match human activity rhythms to access food in the urban environment \\nGaelle Fehlmann, John M Martin, Kamran Safi, Lucy M Aplin\\nbioRxiv 2023.09.26.555651; doi: https://doi.org/10.1101/2023.09.26.555651 \\n Add to Selected Citations\\n\\n\\n\\nMixed Multi-Level Visual, Reward, and Motor Signals in Dorsomedial Frontal Cortex Area F7 during Active Naturalistic Video Exploration \\nFarid Aboharb, Stephen Serene, Julia Sliwa, Winrich A Freiwald\\nbioRxiv 2023.09.25.559420; doi: https://doi.org/10.1101/2023.09.25.559420 \\n Add to Selected Citations\\n\\n\\n\\nA 4D transcriptomic map for the evolution of multiple sclerosis-like lesions in the marmoset brain \\nJing-Ping Lin, Alexis Brake, Maxime Donadieu, Amanda Lee, Riki Kawaguchi, Pascal Sati, Daniel H. Geschwind, Steven Jacobson, Dorothy P. Schafer, Daniel S. Reich\\nbioRxiv 2023.09.25.559371; doi: https://doi.org/10.1101/2023.09.25.559371 \\n Add to Selected Citations\\n\\n\\n\\nPoly ADP-ribosylation regulates Arc expression and promotes adaptive stress-coping \\nEliyahu Dahan, Tzeela Taub, Arthur Vovk, Jade Manier, Raphael Avneri, Elad Lax\\nbioRxiv 2023.09.26.558845; doi: https://doi.org/10.1101/2023.09.26.558845 \\n Add to Selected Citations\\n\\n\\n\\nQuantifying brain connectivity signatures by means of polyconnectomic scoring \\nIlan Libedinsky, Koen Helwegen, Laura Guerrero Simon, Marius Gruber, Jonathan Repple, Tilo Kircher, Udo Dannlowski, Martijn van den Heuvel\\nbioRxiv 2023.09.26.559327; doi: https://doi.org/10.1101/2023.09.26.559327 \\n Add to Selected Citations\\n\\n\\n\\nsurvInTime - Exploring surveillance methods and data analysis on Brazilian respiratory syndrome dataset and community mobility changes \\nYasmmin Cortes Martins, Ronaldo da Silva Francisco Jr.\\nbioRxiv 2023.09.26.559599; doi: https://doi.org/10.1101/2023.09.26.559599 \\n Add to Selected Citations\\n\\n\\n\\nWorking together to control mutation: how collective peroxide detoxification determines microbial mutation rate plasticity \\nRowan Green, Hejie Wang, Carol Botchey, Nancy Zhang, Charles Wadsworth, Andrew McBain, Pawel Paszek, Rok Krasovec, Christopher G. Knight\\nbioRxiv 2023.09.27.557722; doi: https://doi.org/10.1101/2023.09.27.557722 \\n Add to Selected Citations\\n\\n\\n\\nSARS-CoV-2 Omicron BA.2.86: less neutralization evasion compared to XBB sub-variants \\nYaling An, Xuemei Zhou, Lifeng Tao, Haitang Xie, Dedong Li, Ruyue Wang, Hua Hu, Zepeng Xu, Lianpan Dai, Kun Xu, George F. Gao\\nbioRxiv 2023.09.26.559580; doi: https://doi.org/10.1101/2023.09.26.559580 \\n Add to Selected Citations\\n\\n\\n\\nDevelopment of a mutant aerosolized ACE2 that neutralizes SARS-CoV-2 in vivo \\nDaniel L. Kober, Marley C. Caballero Van Dyke, Jennifer L. Eitson L. Eitson, Daniel Rosenbaum, John W Schoggins\\nbioRxiv 2023.09.26.559550; doi: https://doi.org/10.1101/2023.09.26.559550 \\n Add to Selected Citations\\n\\n\\n\\nSurvey of white-footed mice in Connecticut, USA reveals low SARS-CoV-2 seroprevalence and infection with divergent betacoronaviruses \\nRebecca Earnest, Anne M. Hahn, Nicole M. Feriancek, Matthew Brandt, Renata B. Filler, Zhe Zhao, Mallery I. Breban, Chantal B.F. Vogels, Nicholas F.G. Chen, Robert T. Koch, Abbey J. Porzucek, Afeez Sodeinde, Alexa Garbiel, Claire Keanna, Hannah Litwak, Heidi R. Stuber, Jamie L. Cantoni, Virginia E. Pitzer, Ximena A. Olarte Castillo, Laura B. Goodman, Craig B. Wilen, Megan A. Linske, Scott C. Williams, Nathan D. Grubaugh\\nbioRxiv 2023.09.22.559030; doi: https://doi.org/10.1101/2023.09.22.559030 \\n Add to Selected Citations\\n\\n\\n\\nTransposition of HOPPLA in siRNA-deficient plants suggests a limited effect of the environment on retrotransposon mobility in Brachypodium distachyon \\nMichael Thieme, Nikolaos Minadakis, Christophe Himber, Bettina Keller, Wenbo Xu, Kinga rutowicz, Calvin Matteoli, Marcel Boehrer, Bart Rymen, Debbie Laudencia-Chingcuanco, John P. Vogel, Richard Sibout, Christoph Stritt, Todd Blevins, Anne C Roulin\\nbioRxiv 2023.09.25.559196; doi: https://doi.org/10.1101/2023.09.25.559196 \\n Add to Selected Citations\\n\\n\\n\\nAnnotated Compendium of 102 Breast Cancer Gene-Expression Datasets \\nIfeanyichukwu O Nwosu, Daniel D Tabler, Greg Chipman, Stephen R Piccolo\\nbioRxiv 2023.09.22.559045; doi: https://doi.org/10.1101/2023.09.22.559045 \\n Add to Selected Citations\\n\\n\\n\\nUnveiling the Domain-Specific and RAS Isoform-Specific Details of BRAF Regulation \\nTarah Elizabeth Trebino, Borna Markusic, Haihan Nan, Shrhea Banerjee, Zhihong Wang\\nbioRxiv 2023.04.24.538112; doi: https://doi.org/10.1101/2023.04.24.538112 \\n Add to Selected Citations\\n\\n\\n\\nSynapses and Ca2+ activity in oligodendrocyte precursor cells can predict where myelin sheaths form \\nJiaxing Li, Tania Miramontes, Tim Czopka, Kelly R Monk\\nbioRxiv 2022.03.18.484955; doi: https://doi.org/10.1101/2022.03.18.484955 \\n Add to Selected Citations\\n\\n\\n\\nSUMOylation of Bonus, the Drosophila homolog of Transcription Intermediary Factor 1, safeguards germline identity by recruiting repressive chromatin complexes to silence tissue-specific genes \\nBaira Godneeva, Maria Ninova, Katalin Fejes Toth, Alexei Aravin\\nbioRxiv 2023.04.14.536936; doi: https://doi.org/10.1101/2023.04.14.536936 \\n Add to Selected Citations\\n\\n\\n\\nA supervised Bayesian factor model for the identification of multi-omics signatures \\nJeremy P. Gygi, Anna Konstorum, Shrikant Pawar, Edel Aron, Steven H. Kleinstein, Leying Guan\\nbioRxiv 2023.01.25.525545; doi: https://doi.org/10.1101/2023.01.25.525545 \\n Add to Selected Citations\\n\\n\\n\\nParkinsons VPS35[D620N] mutation induces LRRK2 mediated lysosomal association of RILPL1 and TMEM55B \\nProsenjit Pal, Matthew Taylor, Pui Yiu Lam, Francesca Tonelli, Chloe A Hecht, Pawel Lis, Raja S Nirujogi, Toan K Phung, Wondwossen M Yeshaw, Ebsy Jaimon, Rotimi Fasimoye, Emily A Dickie, Melanie Wightman, Thomas Macartney, Suzanne R. Pfeffer, Dario R Alessi\\nbioRxiv 2023.06.07.544051; doi: https://doi.org/10.1101/2023.06.07.544051 \\n Add to Selected Citations\\n\\n\\n\\nEffects of combined exposure to ethanol and delta-9-tetrahydrocannabinol during adolescence on synaptic plasticity in the prefrontal cortex of Long Evans rats \\nLinyuan Shi, Shuo Kang, Chan Young Choi, Brynn L Noonan, Lauren K Carrica, Nu-Chu Liang, Joshua M Gulley\\nbioRxiv 2023.08.14.553087; doi: https://doi.org/10.1101/2023.08.14.553087 \\n Add to Selected Citations\\n\\n\\n\\nProteasome-dependent degradation of histone H1 subtypes is mediated by its C-terminal domain. \\nDaniel R Garcia-Gomis, Jordi Lopez-Gomez, Anna Calderon, Marta R Andres, Inma Ponte, Alicia Roque\\nbioRxiv 2023.06.17.545431; doi: https://doi.org/10.1101/2023.06.17.545431 \\n Add to Selected Citations\\n\\n\\n\\nBacterial histone HBb from Bdellovibrio bacteriovorus compacts DNA by bending \\nYimin Hu, Samuel Schwab, Silvia Deiss, Pedro Escudeiro, Joe D. Joiner, Marcus D. Hartmann, Andrei N. Lupas, Birte Hernandez Alvarez, Vikram Alva, Remus T. Dame\\nbioRxiv 2023.02.26.530074; doi: https://doi.org/10.1101/2023.02.26.530074 \\n Add to Selected Citations\\n\\n\\n\\nErasable Hippocampal Neural Signatures Predict Memory Discrimination \\nNathaniel Reid Kinsky, Daniel J Orlin, Evan A Ruesch, Kamran Diba, Steve Ramirez\\nbioRxiv 2023.02.02.526824; doi: https://doi.org/10.1101/2023.02.02.526824 \\n Add to Selected Citations\\n\\n\\n\\nIntracellular trafficking of furin enhances cellular intoxication by recombinant immunotoxins based on Pseudomonas exotoxin A \\nBrian D. Grossman, Jack D. Sanford, Yuyi Zhu, Cynthia B. Zeller, John E. Weldon\\nbioRxiv 2023.05.31.542721; doi: https://doi.org/10.1101/2023.05.31.542721 \\n Add to Selected Citations\\n\\n\\n\\nCholinergic control of GnRH neuron physiology and luteinizing hormone secretion: involvement of ACh/GABA co-transmission \\nCsaba Vastagh, Imre Farkas, Veronika Csillag, Masahiko Watanabe, Imre Kallo, Zsolt Liposits\\nbioRxiv 2023.08.22.554236; doi: https://doi.org/10.1101/2023.08.22.554236 \\n Add to Selected Citations\\n\\n\\n\\nEnsembled best subset selection using summary statistics for polygenic risk prediction \\nTony Chen, Haoyu Zhang, Rahul Mazumder, Xihong Lin\\nbioRxiv 2023.09.25.559307; doi: https://doi.org/10.1101/2023.09.25.559307 \\n Add to Selected Citations\\n\\n\\n\\nA population modification gene drive targeting both Saglin and Lipophorin impairs Plasmodium transmission in Anopheles mosquitoes \\nEmily I Green, Etienne Jaouen, Dennis Klug, Roenick Proveti Olmo, Amandine Gautier, Stephanie A Blandin, Eric Marois\\nbioRxiv 2022.07.08.499187; doi: https://doi.org/10.1101/2022.07.08.499187 \\n Add to Selected Citations\\n\\n\\n\\nTargeting mosquito X-chromosomes reveals complex transmission dynamics of sex ratio distorting gene drives \\nDaniella An Haber, Yael Arien, Lee Benjamin Lamdan, Yehonathan Alcalay, Chen Zecharia, Flavia Krsticevic, Elad Shmuel Yonah, Rotem Daniel Avraham, Nikolai Windbichler, Philippos Aris Papathanos\\nbioRxiv 2023.08.20.554004; doi: https://doi.org/10.1101/2023.08.20.554004 \\n Add to Selected Citations\\n\\n\\n\\nInterface-acting nucleotide controls polymerization dynamics at microtubule plus- and minus-ends \\nLauren A McCormick, Joseph M Cleary, William O Hancock, Luke M Rice\\nbioRxiv 2023.05.03.539131; doi: https://doi.org/10.1101/2023.05.03.539131 \\n Add to Selected Citations\\n\\n\\n\\nSingle nuclei transcriptomics reveal the differentiation trajectories of periosteal skeletal/stem progenitor cells in bone regeneration \\nSimon Perrin, Cecile-Aurore Wotawa, Vincent Bretegnier, Marine Luka, Fanny Coulpier, Cecile Masson, Mickael Menager, Celine Colnot\\nbioRxiv 2023.06.23.546220; doi: https://doi.org/10.1101/2023.06.23.546220 \\n Add to Selected Citations\\n\\n\\n\\nOscillatory brain activity as unified control mechanism for working memory and social cognition \\nElisabeth V. C. Friedrich, Yannik Hilla, Elisabeth F Sterner, Simon S Ostermeier, Larissa Behnke, Paul Sauseng\\nbioRxiv 2023.02.13.528423; doi: https://doi.org/10.1101/2023.02.13.528423 \\n Add to Selected Citations\\n\\n\\n\\nUncertainty-aware single-cell annotation with a hierarchical reject option \\nLauren Theunissen, Thomas Mortier, Yvan Saeys, Willem Waegeman\\nbioRxiv 2023.09.25.559294; doi: https://doi.org/10.1101/2023.09.25.559294 \\n Add to Selected Citations\\n\\n\\n\\nAssessing Folic Acid Supplementation Adherence Among Pregnant Women in LMICs and its Impact on Neural Tube Defect Incidence: A Systematic Review Protocol \\nJeremiah Oluwatomi Itodo Daniel, Abdulmuizz Oluwatomi Sulaiman, Chiwendu Isiakpona, Joanne Igoli, Chukwunonso Emmanuel Chukwumaeze, Eghosa Morgan, Olatomiwa Olukoya\\nmedRxiv 2023.09.26.23296200; doi: https://doi.org/10.1101/2023.09.26.23296200 \\n Add to Selected Citations\\n\\n\\n\\nGenerative AI Mitigates Representation Bias Using Synthetic Health Data \\nNicolo Micheletti, Raffaele Marchesi, Nicholas I-Hsien Kuo, Sebastiano Barbieri, Giuseppe Jurman, Venet Osmani\\nmedRxiv 2023.09.26.23296163; doi: https://doi.org/10.1101/2023.09.26.23296163 \\n Add to Selected Citations\\n\\n\\n\\nMode of inheritance needs to be accounted for in interpreting genotype-phenotype links in monogenic disorders \\nAntoni Riera-Escamilla, Corrine Kolka Welt, Maris Laan\\nmedRxiv 2023.09.26.23296051; doi: https://doi.org/10.1101/2023.09.26.23296051 \\n Add to Selected Citations\\n\\n\\n\\nProtocol for Systematic Review of Interventions to Improve Sleep Health in Adolescents Living in Low-and-Middle-Income Countries \\nOluwatosin Eunice Olorunmoteni, Jeremiah Oluwatomi Itodo Daniel, Temitayo Ayantayo, Virginia Guvoeke, Lordstrong Akano\\nmedRxiv 2023.09.26.23296177; doi: https://doi.org/10.1101/2023.09.26.23296177 \\n Add to Selected Citations\\n\\n\\n\\nImproved assessment of Schistosoma community infection through data resampling methodology \\nDavid Gurarie, Anirban Mondal, Martial Ndeffo-Mbah\\nmedRxiv 2023.09.26.23296164; doi: https://doi.org/10.1101/2023.09.26.23296164 \\n Add to Selected Citations\\n\\n\\n\\nAssessing generalizability of an AI-based visual test for cervical cancer screening \\nSyed Rakin Ahmed, Didem Egemen, Brian Befano, Ana Cecilia Rodriguez, Jose Jeronimo, Kanan Desai, Carolina Teran, Karla Alfaro, Joel Fokom-Domgue, Kittipat Charoenkwan, Chemtai Mungo, Rebecca Luckett, Rakiya Saidu, Taina Raiol, Ana Ribeiro, Julia C. Gage, Silvia de Sanjose, Jayashree Kalpathy-Cramer, Mark Schiffman\\nmedRxiv 2023.09.26.23295263; doi: https://doi.org/10.1101/2023.09.26.23295263 \\n Add to Selected Citations\\n\\n\\n\\nA Retrospective Case Study of Successful Translational Research: Cardiovascular Disease Risk Assessment, Experiences in Community Engagement \\nMichael Eliot Bales, Jifeng Zhu, Christine Anne Ganzer, Farid Aboharb, Allegra Keeler, Krista Ryon, Brett Jared Ehrmann, Julianne Imperato-McGinley, The Heart to Heart Consortium\\nmedRxiv 2023.09.26.23296161; doi: https://doi.org/10.1101/2023.09.26.23296161 \\n Add to Selected Citations\\n\\n\\n\\nEvaluation of HOTAIR, HOXD8, HOXD9, HOXD11 Gene Expression Levels in Turkish Patients with Acute Myeloid Leukemia and Chronic Myeloid Leukemia: A Single Center Experience \\nEsma Saraymen, Yakut Erdem, Hilal Akalin, Nazife Tascioglu, Berkay Saraymen, Serhat Celik, Yesim Ozdemir, Leylagul Kaynar, Mustafa Cetin, Yusuf Ozkul\\nmedRxiv 2023.09.26.23296073; doi: https://doi.org/10.1101/2023.09.26.23296073 \\n Add to Selected Citations\\n\\n\\n\\nDigital thErapy For Improved tiNnitus carE Study (DEFINE): Protocol for a Randomised Controlled Trial \\nMatthew  E Smith, Dhiraj Sharma, Oliver Rivero-Arias, Kim Rand, Lucy Barrack, Emma Ogburn, Michael Young, Polly Field, Jan Multmeier, Jameel Muzaffar\\nmedRxiv 2023.09.25.23296108; doi: https://doi.org/10.1101/2023.09.25.23296108 \\n Add to Selected Citations\\n\\n\\n\\nISARIC COVID-19 Clinical Data Report: 10 January 2023 \\nISARIC Clinical Characterisation Group, J. Kenneth Baillie, Joaquin Baruch, Abigail Beane, Lucille Blumberg, Fernando Augusto Bozza, Tessa Broadley, Aidan Burrell, Gail Carson, Barbara Wanjiru Citarella, Jake Dunning, Loubna Elotmani, Noelia Garcia Barrio, Jean-Christophe Goffard, Bronner Goncalves, Matthew Hall, Madiha Hashmi, Peter Horby, Waasila Jassat, Christiana Kartsonaki, Bharath Kumar Tirupakuzhi Vijayaraghavan, Pavan Kumar Vecham, Cedric Laouenan, Samantha Lissauer, Ignacio Martin-Loeches, France Mentre, Ben Morton, Daniel Munblit, Nikita A. Nekliudov, Alistair Nichol, David S.Y. Ong, Prasan Kumar Panda, Miguel Pedrera Jimenez, Michelle Petrovic, Nagarajan Ramakrishnan, Grazielle Viana Ramos, Claire Roger, Amanda Rojek, Oana Sandulescu, Malcolm G. Semple, Pratima Sharma, Sally Shrapnel, Louise Sigfrid, Benedict Sim Lim Heng, Budha Charan Singh, Emily Somers, Anca Streinu-Cercel, Fabio S. Taccone, Jia Wei, Evert-Jan Wils, Xin Ci Wong, Kyle Young, Piero L. Olliaro, Laura Merson\\nmedRxiv 2020.07.17.20155218; doi: https://doi.org/10.1101/2020.07.17.20155218 \\n Add to Selected Citations\\n\\n\\n\\nHigh-flow nasal cannula vs non-invasive ventilation in acute hypoxia: Propensity score matched study \\nElizabeth S Munroe, Ina Prevalska, Madison Hyer, William J Meurer, Jarrod M Mosier, Mark A Tidswell, Hallie C Prescott, Lai Wei, Henry Wang, Christopher M Fung\\nmedRxiv 2023.09.26.23296167; doi: https://doi.org/10.1101/2023.09.26.23296167 \\n Add to Selected Citations\\n\\n\\n\\nLongitudinal analysis within one hospital in sub-Saharan Africa over 20 years reveals repeated replacements of dominant clones of Klebsiella pneumoniae and stresses the importance to include temporal patterns for vaccine design considerations \\nEva Heinz, Oliver Pearse, Allan Zuza, Sithembile Bilima, Chisomo Msefula, Patrick Musicha, Patriciah Siyabu, Edith Tewesa, Fabrice E Graf, Rebecca Lester, Samantha Lissauer, Jennifer Cornick, Joseph M Lewis, Kondwani Kawaza, Nicholas R Thomson, Nicholas A Feasey\\nmedRxiv 2023.09.26.23296137; doi: https://doi.org/10.1101/2023.09.26.23296137 \\n Add to Selected Citations\\n\\n\\n\\nDNA mismatch repair gene variant classification: evaluating the utility of somatic mutations and mismatch repair deficient colonic crypts and endometrial glands \\nRomy Walker, Khalid Mahmood, Julia Como, Mark Clendenning, Jihoon E Joo, Peter Georgeson, Sharelle Joseland, Susan G Preston, Bernard J Pope, Rachel Austin, Jasmina Bojadzieva, Ainsley Campbell, James M Chan, Emma Edwards, Margaret Gleeson, Annabel Goodwin, Marion T Harris, Emilia Ip, Judy Kirk, Julia Mansour, Helen Marfan, Cassandra Nichols, Nicholas Pachter, Abiramy Ragunathan, Allan Spigelman, Rachel Susman, Michael Christie, Mark A Jenkins, Rish K Pai, Christophe Rosty, Finlay A Macrae, Ingrid M Winship, Daniel D Buchanan\\nmedRxiv 2023.09.26.23295173; doi: https://doi.org/10.1101/2023.09.26.23295173 \\n Add to Selected Citations\\n\\n\\n\\nExploration of ChatGPT application in diabetes education: a multi-dataset, multi-reviewer study \\nZhen Ying, Yujuan Fan, Jiaping Lu, Ping Wang, Lin Zou, Qi Tang, Yizhou Chen, Xiaoying Li, Ying Chen\\nmedRxiv 2023.09.27.23296144; doi: https://doi.org/10.1101/2023.09.27.23296144 \\n Add to Selected Citations\\n\\n\\n\\nEffectiveness of Hyperbaric Oxygen Therapy Compared to Surgical Free Tissue Flap for Osteoradionecrosis of the Jaw: A Meta-Analysis \\nMegha K Sheth, Casey Collet, De-Chen Lin, Uttam K Sinha\\nmedRxiv 2023.09.25.23296038; doi: https://doi.org/10.1101/2023.09.25.23296038 \\n Add to Selected Citations\\n\\n\\n\\nComparing the Diagnostic Performance of qPCR, ddPCR, and NGS Liquid Biopsies for HPV-Associated Cancers \\nSaskia Naegele, Daniel A. A. Ruiz-Torres, Yan Zhao, Deborah Goss, Daniel L. Faden\\nmedRxiv 2023.09.26.23296002; doi: https://doi.org/10.1101/2023.09.26.23296002 \\n Add to Selected Citations\\n\\n\\n\\nIndicators of Co-occurrence of Mood Disorder with Chronic Medical Conditions: Evidence from an Administrative Claims Data Analysis \\nKarishma Chhabria, Randa Hamden, Trudy Millard Krause, Mbemba Jabbi\\nmedRxiv 2023.09.26.23296173; doi: https://doi.org/10.1101/2023.09.26.23296173 \\n Add to Selected Citations\\n\\n\\n\\nReal World Experience and Clinical Utility of EsoGuard® - Interim Data from the Lucid Registry \\nRichard Englehardt, Jason B Samarasena, Nikolai A Bildzukewicz, Rachelle Hamblin, Victoria T Lee, Suman Verma, Brian J deGuzman, Lishan Aklog\\nmedRxiv 2023.09.26.23296162; doi: https://doi.org/10.1101/2023.09.26.23296162 \\n Add to Selected Citations\\n\\n\\n\\nAutostent: A Semi-Automated Approach to Designing Customized 3D-Printed Oral Radiation Stents for Patients with Head and Neck Cancer \\nAnshuman Agrawal, Mohamed M Zaid, Millicent A Roach, Lianchun Xiao, Mark S Chambers, Eugene J Koay\\nmedRxiv 2023.09.26.23296170; doi: https://doi.org/10.1101/2023.09.26.23296170 \\n Add to Selected Citations\\n\\n\\n\\niQC: machine-learning-driven prediction of surgical procedure uncovers systematic confounds of cancer whole slide images in specific medical centers \\nAndrew J Schaumberg, Michael S Lewis, Ramin Nazarian, Ananta Wadhwa, Nathanael Kane, Graham Turner, Purushotham Karnam, Poornima Devineni, Nicholas Wolfe, Randall Kintner, Matthew B Rettig, Beatrice S Knudsen, Isla P Garraway, Saiju Pyarajan\\nmedRxiv 2023.09.19.23295798; doi: https://doi.org/10.1101/2023.09.19.23295798 \\n Add to Selected Citations\\n\\n\\n\\nThe impact of spectral basis set composition on estimated levels of cingulate glutamate and its associations with different personality traits \\nVerena Franziska Demler, Elisabeth F Sterner, Martin Wilson, Claus Zimmer, Franziska Knolle\\nmedRxiv 2023.07.12.23292540; doi: https://doi.org/10.1101/2023.07.12.23292540 \\n Add to Selected Citations\\n\\n\\n\\nResource allocation strategies for insecticide-treated bed nets to achieve malaria eradication \\nNora Schmit, Hillary M Topazian, Matteo Pianella, Giovanni D Charles, Peter Winskill, Michael White, Katharina Hauck, Azra C Ghani\\nmedRxiv 2023.04.16.23288647; doi: https://doi.org/10.1101/2023.04.16.23288647 \\n Add to Selected Citations\\n\\n\\n\\nReduced effectiveness of repeat influenza vaccination: distinguishing among within-season waning, recent clinical infection, and subclinical infection \\nQifang Bi, Barbra A Dickerman, Huong Q McLean, Emily T Martin, Manjusha Gaglani, Karen J Wernli, G.K. Balasubramani, Brendan Flannery, Marc Lipsitch, Sarah Cobey\\nmedRxiv 2023.03.12.23287173; doi: https://doi.org/10.1101/2023.03.12.23287173 \\n Add to Selected Citations\\n\\n\\n\\nPrediction of individual survival and trial outcome for anti-PDL1 treatment in non-small cell lung cancer using blood markers-based kinetics-machine learning \\nSébastien Benzekry, Mélanie Karslsen, Célestin Bigarré, Abdessamad El Kaoutari, Bruno Gomes, Martin Stern, Ales Neubert, René Bruno, François Mercier, Suresh Vatakuti, Peter Curle, Candice Jamois\\nmedRxiv 2023.09.26.23296135; doi: https://doi.org/10.1101/2023.09.26.23296135 \\n Add to Selected Citations\\n\\n\\n\\nAnalysis of Potential Risk Factors of COVID-19 Based on Variants: Omicron, Delta, and Alpha \\nDharshini Kannan, Sreenidhi Muppiri\\nmedRxiv 2023.09.26.23295911; doi: https://doi.org/10.1101/2023.09.26.23295911 \\n Add to Selected Citations\\n\\n\\n\\nRecruitment, Consent and DNA Sample Acquisition in a U.S. Precision Health Cohort During the COVID-19 Pandemic \\nAllyson M. Derry, Yvette Strong, Davia Schioppo, Joni Cotter, Geisa M. Wilkins, Laura I. Siquieros, Andrea Ouyang, Kathleen Hulseman, Joseph Petrosino, Lorrin Liang, Megan Stevenson, Tiffany Elianne Aguilera, Alexandria L. Soto, Katherin Meurer, Alison L. Herman, Inessa Cohen, Guido J. Falcone, Erin E. Longbrake, Cassius I. Ochoa Chaar, Kelly M. Anastasio, Michael F. Murray\\nmedRxiv 2023.09.25.23289158; doi: https://doi.org/10.1101/2023.09.25.23289158 \\n Add to Selected Citations\\n\\n\\n\\nER-export and ARFRP1/AP-1-dependent delivery of SARS-CoV-2 Envelope to lysosomes controls late stages of viral replication \\nGuy J Pearson, Harriet Mears, Malgorzata Broncel, Ambrosius P Snijders, David LV Bauer, Jeremy G Carlton\\nbioRxiv 2021.06.30.450614; doi: https://doi.org/10.1101/2021.06.30.450614 \\n Add to Selected Citations\\n\\n\\n\\nAntigenicity and receptor affinity of SARS-CoV-2 BA.2.86 spike \\nQian Wang, Yicheng Guo, Liyuan Liu, Logan T Schwanz, Zhiteng Li, Jerren Ho, Richard M Zhang, Sho Iketani, Jian Yu, Yiming Huang, Yiming Qu, Riccardo Valdez, Adam S Lauring, Aubree Gordon, Harris H Wang, Lihong Liu, David D Ho\\nbioRxiv 2023.09.24.559214; doi: https://doi.org/10.1101/2023.09.24.559214 \\n Add to Selected Citations\\n\\n\\n\\nNucleolar dynamics are determined by the ordered assembly of the ribosome \\nJessica Sheu-Gruttadauria, Xiaowei Yan, Nico Stuurman, Stephen N. Floor, Ronald D. Vale\\nbioRxiv 2023.09.26.559432; doi: https://doi.org/10.1101/2023.09.26.559432 \\n Add to Selected Citations\\n\\n\\n\\nRNA-based sensitive fungal pathogen detection \\nJulia Micheel, Franziska Aron, Abdulrahman A. Kelani, Christian Girbardt, Matthew G. Blango, Grit Walther, Damian Wollny\\nbioRxiv 2023.09.26.559494; doi: https://doi.org/10.1101/2023.09.26.559494 \\n Add to Selected Citations\\n\\n\\n\\nOrganizing the coactivity structure of the hippocampus from robust to flexible memory \\nGiuseppe Pietro Gava, Laura Lefèvre, Tabitha Broadbelt, Stephen B McHugh, Vitor Lopes-dos-Santos, Demi Brizee, Katja Hartwich, Hanna Sjoberg, Pavel V. Perestenko, Robert Toth, Andrew Sharott, David Dupret\\nbioRxiv 2023.09.26.558975; doi: https://doi.org/10.1101/2023.09.26.558975 \\n Add to Selected Citations\\n\\n\\n\\nHaloTag display enables quantitative single-particle characterization and functionalization of engineered extracellular vesicles \\nRoxana E Mitrut, Devin M Stranford, Jonathan M Chan, Matthew D Bailey, Minrui Luo, Thomas J Meade, Muzhou Wang, Joshua Nathaniel Leonard\\nbioRxiv 2023.09.25.559433; doi: https://doi.org/10.1101/2023.09.25.559433 \\n Add to Selected Citations\\n\\n\\n\\nA new cell culture resource for investigations of reptilian gene function \\nSukhada P Samudra, Sungdae Park, Elizabeth A Esser, Tryggvi P McDonald, Arianna M Borges, Jonathan Eggenschwiler, Douglas B. Menke\\nbioRxiv 2023.09.25.559349; doi: https://doi.org/10.1101/2023.09.25.559349 \\n Add to Selected Citations\\n\\n\\n\\nEvidence for kilometer-scale biophysical features at the Gulf Stream front \\nPatrick Clifton Gray, Ivan Savelyev, Nicolas Cassar, Marina Levy, Emmanuel Boss, Yoav Lehahn, Guillaume Bourdin, Kate A Thompson, Anna Windle, Jessica Gronniger, Sheri Floge, Dana Hunt, Greg Silsbe, Zackary I Johnson, David W Johnston\\nbioRxiv 2023.09.25.559383; doi: https://doi.org/10.1101/2023.09.25.559383 \\n Add to Selected Citations\\n\\n\\n\\nDendrites support formation and reactivation of sequential memories through Hebbian plasticity \\nAlessio Quaresima, Hartmut Fitz, Renato Duarte, Karl Magnus Petersson, Peter Hagoort\\nbioRxiv 2023.09.26.559322; doi: https://doi.org/10.1101/2023.09.26.559322 \\n Add to Selected Citations\\n\\n\\n\\nElevated genetic risk for multiple sclerosis originated in Steppe Pastoralist populations \\nWilliam Barrie, Yaoling Yang, Evan Irving-Pease, Kathrine E. Attfield, Gabriele Scorrano, Lise Torp Jensen, Angelos P. Armen, Evangelos Antonios Dimopoulos, Aaron Stern, Alba Refoyo-Martinez, Alice Pearson, Abigail Ramsøe, Charleen Gaunitz, Fabrice Demeter, Marie Louise S. Jørkov, Stig Bermann Møller, Bente Springborg, Lutz Klassen, Inger Marie Hyldgård, Niels Wickmann, Lasse Vinner, Thorfinn Sand Korneliussen, Morten E. Allentoft, Martin Sikora, Kristian Kristiansen, Santiago Rodriguez, Rasmus Nielsen, Astrid K. N. Iversen, Daniel J. Lawson, Lars Fugger, Eske Willerslev\\nbioRxiv 2022.09.23.509097; doi: https://doi.org/10.1101/2022.09.23.509097 \\n Add to Selected Citations\\n\\n\\n\\nDeep learning insights into the architecture of the mammalian egg-sperm fusion synapse \\nArne Elofsson, Ling Han, Enrica Bianchi, Gavin J. Wright, Luca Jovine\\nbioRxiv 2023.07.27.550195; doi: https://doi.org/10.1101/2023.07.27.550195 \\n Add to Selected Citations\\n\\n\\n\\nDeep learning to extract the meteorological by-catch of wildlife cameras \\nJamie Alison, Stephanie Payne, Jake M Alexander, Anne D Bjorkman, Vincent Ralph Clarke, Onalenna Gwate, Maria Huntsaar, Evelin Iseli, Jonathan Lenoir, Hjalte Mads Rosenstand Mann, Sandy-Lynn Steenhuisen, Toke Thomas Høye\\nbioRxiv 2023.09.25.558780; doi: https://doi.org/10.1101/2023.09.25.558780 \\n Add to Selected Citations\\n\\n\\n\\nOn the pH-dependence of α-synuclein amyloid polymorphism and the role of secondary nucleation in seeding experiments \\nLukas Frey, Dhiman Ghosh, Bilal M. Qureshi, David Rhyner, Ricardo Guerrero-Ferreira, Aditya Pokharna, Witek Kwiatkowski, Tetjana Serdiuk, Paola Picotti, Roland Riek, Jason Greenwald\\nbioRxiv 2023.06.25.546428; doi: https://doi.org/10.1101/2023.06.25.546428 \\n Add to Selected Citations\\n\\n\\n\\nA dynamic Hedgehog gradient orients tracheal cartilage rings \\nEvan P Kingsley, Darcy Mishkind, Tom W Hiscock, Clifford J Tabin\\nbioRxiv 2023.09.25.559425; doi: https://doi.org/10.1101/2023.09.25.559425 \\n Add to Selected Citations\\n\\n\\n\\nSmurfness-based two-phase model of ageing helps deconvolve the ageing transcriptional signature \\nFlaminia Zane, Hayet Bouzid, Sofia Sosa Marmol, Mira Brazane, Savandara Besse, Julia Lisa Molina, Celine Cansell, Fanny Aprahamian, Sylvere Durand, Jessica Ayache, Christophe Antoniewski, CLEMENT CARRE, Nicolas Todd, Michael Rera\\nbioRxiv 2022.11.22.517330; doi: https://doi.org/10.1101/2022.11.22.517330 \\n Add to Selected Citations\\n\\n\\n\\nDNA Tetrahedral Nanocages as a Promising Nanocarrier for Dopamine Delivery in Neurological Disorders \\nRamesh Singh, Krupa Kansara, Sandip Mandal, Ritu Varshney, Sharad Gupta, Ashutosh Kumar, Prabal Kumar Maiti, Dhiraj D Bhatia\\nbioRxiv 2023.09.19.558434; doi: https://doi.org/10.1101/2023.09.19.558434 \\n Add to Selected Citations\\n\\n\\n\\nThe left posterior angular gyrus is engaged by autobiographical recall not object-semantics, or event-semantics: Evidence from contrastive propositional speech production \\nGina F Humphreys, Ajay D Halai, Francesca Branzi, Matthew Lambon Ralph\\nbioRxiv 2022.04.04.487000; doi: https://doi.org/10.1101/2022.04.04.487000 \\n Add to Selected Citations\\n\\n\\n\\nCell autonomous polarization by the planar cell polarity signaling pathway \\nAlexis T Weiner, Bomsoo Cho, Kaye L. Suyama, Jeffrey D. Axelrod\\nbioRxiv 2023.09.26.559449; doi: https://doi.org/10.1101/2023.09.26.559449 \\n Add to Selected Citations\\n\\n\\n\\nPoles apart - the structure and composition of the bird community in bamboo in the Eastern Himalaya \\nSidharth Srinivasan, Dumbar Kumar Pradhan, Micah Rai, Aman Biswakarma, Umesh Srinivasan\\nbioRxiv 2023.01.27.525938; doi: https://doi.org/10.1101/2023.01.27.525938 \\n Add to Selected Citations\\n\\n\\n\\nHigh performance enrichment-based genome sequencing to support the investigation of hepatitis A virus outbreaks \\nSara E. Zufan, Karolina Mercoulia, Jason Kwong, Louise M. Judd, Benjamin P. Howden, Torsten Seemann, Timothy P. Stinear\\nbioRxiv 2023.07.21.548488; doi: https://doi.org/10.1101/2023.07.21.548488 \\n Add to Selected Citations\\n\\n\\n\\nSpatial cell fate manipulation of human pluripotent stem cells by controlling the microenvironment using photocurable hydrogel. \\nZHE WANG, Akira Numada, Yusuke Oda, Fumi Wagai, Masatoshi Ohgushi, Mototsugu Eiraku\\nbioRxiv 2022.10.02.510557; doi: https://doi.org/10.1101/2022.10.02.510557 \\n Add to Selected Citations\\n\\n\\n\\nN-6-methyladenosine (m6A) Promotes the Nuclear Retention of mRNAs with Intact 5\\' Splice Site Motifs \\nEliza S Lee, Harrison W Smith, Sean SJ Ihn, Leticia Scalize de Olivera, Yifan E Wang, Robert Y Jomphe, Syed Nabeel-Shah, Shuye Pu, Jack F Greenblatt, Alexander F Palazzo\\nbioRxiv 2023.06.20.545713; doi: https://doi.org/10.1101/2023.06.20.545713 \\n Add to Selected Citations\\n\\n\\n\\nAbsolute number of three populations of interneurons and GABAergic synapses in the human hippocampus \\nVirag Takacs, Peter Papp, Aron Orosz, Zsuzsanna Bardoczi, Tamas Zsoldos, Masahiko Watanabe, Zsofia Magloczky, Peter Gombas, Tamás F. Freund, Gabor Nyiri\\nbioRxiv 2023.09.26.559559; doi: https://doi.org/10.1101/2023.09.26.559559 \\n Add to Selected Citations\\n\\n\\n\\nNanoconnectivity: a quantitative approach to examine the structural complexity of the brain \\nMarta Turegano-Lopez, Felix de las Pozas, Andrea Santuy, Jose-Rodrigo Rodriguez, Javier Defelipe, Angel Merchan-Perez\\nbioRxiv 2023.09.26.559505; doi: https://doi.org/10.1101/2023.09.26.559505 \\n Add to Selected Citations\\n\\n\\n\\nDysregulation of astrocyte-secreted pleiotrophin contributes to neuronal structural and functional phenotypes in Down Syndrome \\nAshley N Brandebura, Quinn N Asbell, Mariel Kristine B Micael, Nicola J Allen\\nbioRxiv 2023.09.26.559633; doi: https://doi.org/10.1101/2023.09.26.559633 \\n Add to Selected Citations\\n\\n\\n\\nThe oxytocin-modulated brain circuit that synchronizes heart rate with breathing \\nJulie GM Buron, Ambre A Linossier, Christian Gestreau, Fabienne Schaller, Roman Tyzio, Marie-Solenne Felix, Valery Matarazzo, Muriel Thoby-Brisson, Francoise Muscatelli, Clement Menuet\\nbioRxiv 2023.09.26.559512; doi: https://doi.org/10.1101/2023.09.26.559512 \\n Add to Selected Citations\\n\\n\\n\\nOligodendrocyte maturation alters the cell death mechanisms that cause demyelination \\nTimothy W. Chapman, Enrique T. Piedra, Robert A. Hill\\nbioRxiv 2023.09.26.557781; doi: https://doi.org/10.1101/2023.09.26.557781 \\n Add to Selected Citations\\n\\n\\n\\nSynchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition \\nMitchell R. Vollger, Jonas Korlach, Kiara C. Eldred, Elliott Swanson, Jason G. Underwood, Katherine M. Munson, Yong-Han H. Cheng, Jane Ranchalis, Yizi Mao, Elizabeth E. Blue, Ulrike Schwarze, Christopher T. Saunders, Aaron M. Wenger, Aimee Allworth, Sirisak Chanprasert, Brittney L. Duerden, Ian Glass, Martha Horike-Pyne, Michelle Kim, Kathleen A. Leppig, Ian J. McLaughlin, Jessica Ogawa, Elisabeth A. Rosenthal, Sam Sheppeard, Stephanie M. Sherman, Samuel Strohbehn, Amy L. Yuen, Thomas A. Reh, Peter H. Byers, Michael J. Bamshad, Fuki M. Hisama, Gail P. Jarvik, Yasemin Sancak, Katrina M. Dipple, Andrew B. Stergachis\\nbioRxiv 2023.09.26.559521; doi: https://doi.org/10.1101/2023.09.26.559521 \\n Add to Selected Citations\\n\\n\\n\\nCdc42 activation is necessary for heterosynaptic cooperation and competition \\nMariana Nunes, Natalia Madeira, Rosalina Fonseca\\nbioRxiv 2023.09.26.559525; doi: https://doi.org/10.1101/2023.09.26.559525 \\n Add to Selected Citations\\n\\n\\n\\nRegulation of Chromatin Architecture by Transcription Factor Binding \\nStephanie Portillo, Suckwoo Chung, Jill Hoffman, Tamar Schlick\\nbioRxiv 2023.09.26.559535; doi: https://doi.org/10.1101/2023.09.26.559535 \\n Add to Selected Citations\\n\\n\\n\\nA first morphological and electrophysiological characterization of FCs of the mouse cerebellum. \\nAntonin Singer, Lea Vinel, Federico Trigo, Isabel Llano, Martin Oheim\\nbioRxiv 2023.09.26.559486; doi: https://doi.org/10.1101/2023.09.26.559486 \\n Add to Selected Citations\\n\\n\\n\\nSingle-cell multi-omics map of human foetal blood in Down\\'s Syndrome \\nAndrew R. Marderstein, Marco De Zuani, Haoliang Xue, Jon Bezney, Shuo Wong, Tim H. H. Coorens, Stephen B. Montgomery, Ana Cvejic\\nbioRxiv 2023.09.25.559431; doi: https://doi.org/10.1101/2023.09.25.559431 \\n Add to Selected Citations\\n\\n\\n\\nCRISPR-Cas inhibits plasmid transfer and immunizes bacteria against antibiotic resistance acquisition in manure \\nChahat Upreti, Pranav Kumar, Lisa Durso, Kelli Palmer\\nbioRxiv 2023.09.26.559507; doi: https://doi.org/10.1101/2023.09.26.559507 \\n Add to Selected Citations\\n\\n\\n\\nSpecialized Bacteroidetes dominate the Arctic Ocean during marine spring blooms \\nÁlvaro Redondo-Río, Christopher J. Mundy, Javier Tamames, Carlos Pedrós-Alió\\nbioRxiv 2023.09.26.559482; doi: https://doi.org/10.1101/2023.09.26.559482 \\n Add to Selected Citations\\n\\n\\n\\nConnectome Operations For FSL ExEcution (COFFEE): a turnkey pipeline for preprocessing of fMRI data \\nMark M McAvoy, Lei Liu, Ruiwen Zhou, Benjamin A Philip\\nbioRxiv 2023.09.19.558290; doi: https://doi.org/10.1101/2023.09.19.558290 \\n Add to Selected Citations\\n\\n\\n\\nAntagonism of kappa opioid receptors accelerates the development of L-DOPA-induced dyskinesia in a preclinical model of moderate dopamine depletion \\nAndrew J Flores, Mitchell J Bartlett, Blake T Seaton, Grace Samtani, Morgan R Sexauer, Nathan C Weintraub, James R Siegenthaler, Dong Lu, Michael L Heien, Frank Porreca, Scott J Sherman, Torsten Falk\\nbioRxiv 2023.07.31.551112; doi: https://doi.org/10.1101/2023.07.31.551112 \\n Add to Selected Citations\\n\\n\\n\\nMicrodroplet-enabled high-throughput cultivation of vaginal bacteria using cervicovaginal fluids \\nCorine M Jackman, James Y Tan, Xiaoxia Nina Lin\\nbioRxiv 2023.09.26.559375; doi: https://doi.org/10.1101/2023.09.26.559375 \\n Add to Selected Citations\\n\\n\\n\\nSTK25 is an IRF5 kinase that promotes TLR7/8-mediated inflammation \\nMatthew R. Rice, Bharati Matta, Loretta Wang, Surya Indukuri, Betsy J Barnes\\nbioRxiv 2023.09.26.559637; doi: https://doi.org/10.1101/2023.09.26.559637 \\n Add to Selected Citations\\n\\n\\n\\nProgressive community, biogeochemical and evolutionary remodeling of the soil microbiome underpins long-term desert ecosystem restoration \\nQiong Chen, Mengyi Yuan, Liuyiqi Jiang, Xin Wei, Zhen Liu, Chen Peng, Zinuo Huang, Dongmei Tang, Xiangrong Wu, Jing Sun, Qing Liu, Xiaowei Zhu, Peng Gao, Laibin Huang, Meng Wang, Mingkai Jiang, Chao Jiang\\nbioRxiv 2023.09.26.559499; doi: https://doi.org/10.1101/2023.09.26.559499 \\n Add to Selected Citations\\n\\n\\n\\nActivation-induced shift in nutrient preference and function-specific nutrient dependence in human neutrophils \\nEmily C Britt, Xin Qing, James A Votava, Jorgo Lika, Andrew Wagner, Simone Shen, Nicholas L Arp, Hamidullah Khan, Stefan M Schieke, Christopher D Fletcher, Anna Huttenlocher, Jing Fan\\nbioRxiv 2023.09.25.559385; doi: https://doi.org/10.1101/2023.09.25.559385 \\n Add to Selected Citations\\n\\n\\n\\nHigh-throughput Automated Muropeptide Analysis (HAMA) Reveals Peptidoglycan Composition of Gut Microbial Cell Walls \\nYa-Chen Hsu, Pin-Rui Su, Lin-Jie Huang, Kum-Yi Cheng, Chun-hsien Chen, Cheng-Chih Hsu\\nbioRxiv 2023.04.17.537164; doi: https://doi.org/10.1101/2023.04.17.537164 \\n Add to Selected Citations\\n\\n\\n\\nIdentification of a new gregarine parasite in mass mortality events of freshwater pearl mussels (Margaritifera margaritifera) in Sweden \\nAnders Alfjorden, Ioana Onut Brännström, Niklas Wengström, Arni Kristmundsson, Mahwash Jamy, David Persson, Fabien Burki\\nbioRxiv 2023.09.26.559503; doi: https://doi.org/10.1101/2023.09.26.559503 \\n Add to Selected Citations\\n\\n\\n\\nGradient boosted decision trees reveal nuances of auditory discrimination behaviour \\nCarla Seoyun Griffiths Ms, Jules M Lebert Mr, Joseph Sollini Dr, Jennifer K Bizley Dr\\nbioRxiv 2023.06.16.545302; doi: https://doi.org/10.1101/2023.06.16.545302 \\n Add to Selected Citations\\n\\n\\n\\nSediment source and dose influence the larval performance of the threatened coral Orbicella faveolata \\nXaymara Serrano, Stephanie  M. Rosales, Margaret  W. Miller, Ana  M. Palacio-Castro, Olivia  M. Williamson, Andrea Gomez, Andrew  C. Baker\\nbioRxiv 2023.09.22.559035; doi: https://doi.org/10.1101/2023.09.22.559035 \\n Add to Selected Citations\\n\\n\\n\\nA Mixture Model for Finite But Unbounded Human Lifetimes \\nFei Huang, Ross Maller, Brandon Milholland, Xu Ning\\nbioRxiv 2022.01.07.474862; doi: https://doi.org/10.1101/2022.01.07.474862 \\n Add to Selected Citations\\n\\n\\n\\nDeciphering the Genomes of Motility-Deficient Mutants of Vibrio alginolyticus 138-2 \\nKunio Ihara, Kazuma Uesaka, Keita Inaba, Noriko Nishioka, Seiji Kojima, Michio Homma\\nbioRxiv 2023.09.26.559574; doi: https://doi.org/10.1101/2023.09.26.559574 \\n Add to Selected Citations\\n\\n\\n\\nAAO2 impairment improves aldehyde detoxification by AAO3 in Arabidopsis leaves exposed to UVC or Rose Bengal. \\nZhadyrassyn Nurbekova, Sudhakar Srivastava, Nja Zai Du, Veronika Turečková, Miroslav Strnad, Rustem Omarov, Moshe Sagi\\nbioRxiv 2023.09.22.559040; doi: https://doi.org/10.1101/2023.09.22.559040 \\n Add to Selected Citations\\n\\n\\n\\nUnveiling multi-drug resistant (MDR) gram negative pathogenic bacteria from poultry chickens in the Noakhali region of Bangladesh \\nAdnan Munim, Shuvo  Chandra Das, Md. Murad Hossain, Ithmam Hami, Mridul  Gope Topu, Shipan Das Gupta\\nbioRxiv 2023.09.26.559636; doi: https://doi.org/10.1101/2023.09.26.559636 \\n Add to Selected Citations\\n\\n\\n\\nPervasive GxE interactions shape adaptive trajectories and the exploration of the phenotypic space in artificial selection experiments \\nArnaud Desbiez-Piat, Adrienne Ressayre, Elodie MARCHADIER, Alicia Noly, Carine Remoue, Clementine Vitte, Harry Belcram, Aurelie Bourgais, Nathalie Galic, Martine Le Guilloux, Maud Irene Tenaillon, Christine Dillmann\\nbioRxiv 2023.01.13.523786; doi: https://doi.org/10.1101/2023.01.13.523786 \\n Add to Selected Citations\\n\\n\\n\\nComparative analysis of the molecular starvation response of Southern Ocean copepods \\nCory A. Berger, Deborah K. Steinberg, Louise A. Copeman, Ann M. Tarrant\\nbioRxiv 2023.08.17.553703; doi: https://doi.org/10.1101/2023.08.17.553703 \\n Add to Selected Citations\\n\\n\\n\\nSemantic Representation of Neural Circuit Knowledge in Caenorhabditis elegans \\nSharan J Prakash, Kimberly Van Auken, David P Hill, Paul W Sternberg\\nbioRxiv 2023.04.28.538760; doi: https://doi.org/10.1101/2023.04.28.538760 \\n Add to Selected Citations\\n\\n\\n\\nThe Time-Course of Food Representation in the Human Brain \\nDenise Moerel, James Psihoyos, Thomas A Carlson\\nbioRxiv 2023.06.06.543985; doi: https://doi.org/10.1101/2023.06.06.543985 \\n Add to Selected Citations\\n\\n\\n\\nThe genomic landscape of metallic color variation in ground beetles \\nYi-Ming Weng, David H Kavahaugh, Bryan Rubio-Perez, Jad Salman, Mikhail A Kats, Sean D Schoville\\nbioRxiv 2023.09.25.559374; doi: https://doi.org/10.1101/2023.09.25.559374 \\n Add to Selected Citations\\n\\n\\n\\nInhibition of Neisseria gonorrhoeae complement-mediated killing during acute gonorrhoea is dependent upon the IgG2:IgG3 antibody ratio \\nSamantha A McKeand, Sian E Faustini, Alex Cook, Nikki Kennett, Mark T Drayson, Adam F. Cunningham, Ian R. Henderson, Jonathan D C Ross, Jeffrey A Cole, Amanda E Rossiter-Pearson\\nbioRxiv 2023.09.26.558794; doi: https://doi.org/10.1101/2023.09.26.558794 \\n Add to Selected Citations\\n\\n\\n\\nNeural signatures of natural behavior in socializing macaques \\nCamille Testard, Sebastien Tremblay, Felipe Parodi, Ronald W DiTullio, Arianna Acevedo-Ithier, Kristin Gardiner, Konrad Paul Kording, Michael Platt\\nbioRxiv 2023.07.05.547833; doi: https://doi.org/10.1101/2023.07.05.547833 \\n Add to Selected Citations\\n\\n\\n\\nThe temporal dynamics of lncRNA Firre-mediated epigenetic and transcriptional regulation \\nChristian Much, Erika Lasda, Isabela T Pereira, Tenaya K Vallery, Daniel Ramirez, Jordan Lewandowski, Robin D Dowell, Michael J Smallegan, John L Rinn\\nbioRxiv 2022.05.15.492001; doi: https://doi.org/10.1101/2022.05.15.492001 \\n Add to Selected Citations\\n\\n\\n\\nOct4 redox sensitivity potentiates reprogramming and differentiation \\nZuolian Shen, Yifan Wu, Asit Mana, Kimberley J Evason, Mahesh B Chandrasekharan, Dean Tantin\\nbioRxiv 2023.02.21.529404; doi: https://doi.org/10.1101/2023.02.21.529404 \\n Add to Selected Citations\\n\\n\\n\\nCryo-EM structures of human SID-1 transmembrane family proteins and implications for their low-pH-dependent RNA transport activity \\nLe Zheng, Tingting Yang, Hangtian Guo, Chen Qi, Yuchi Lu, Haonan Xiao, Yan Gao, Yue Liu, Yixuan Yang, Mengru Zhou, Henry Nguyen, Yun Zhu, Fei Sun, Chen-Yu Zhang, Xiaoyun Ji\\nbioRxiv 2023.09.25.559159; doi: https://doi.org/10.1101/2023.09.25.559159 \\n Add to Selected Citations\\n\\n\\n\\nReplication-dead gammaherpesvirus vaccine protects against acute replication, reactivation from latency, and lethal challenge in mice \\nWesley A Bland, Shana M Owens, Kyle McEvoy, Chad H Hogan, Luciarita Boccuzzi, Varvara Kirillov, Camille Khairallah, Brian S Sheridan, James Craig Forrest, Laurie T Krug\\nbioRxiv 2023.09.26.559621; doi: https://doi.org/10.1101/2023.09.26.559621 \\n Add to Selected Citations\\n\\n\\n\\nMolecular mechanisms controlling the biogenesis of the TGF-β signal Vg1 \\nP. C. Dave P. Dingal, Adam N. Carte, Tessa G. Montague, Medel B Lim Suan Jr., Alexander F. Schier\\nbioRxiv 2021.04.25.441333; doi: https://doi.org/10.1101/2021.04.25.441333 \\n Add to Selected Citations\\n\\n\\n\\nAlpha-synuclein expression in neuron modulates the Japanese encephalitis virus infection \\nAnjali Gupta, Anshuman Mohapatra, Harpreet Kaur, Ajanta Sharma, Nitin Chaudhary, Sachin Kumar\\nbioRxiv 2023.09.25.559402; doi: https://doi.org/10.1101/2023.09.25.559402 \\n Add to Selected Citations\\n\\n\\n\\nBacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems \\nLydia R Chambers, Qiaozhen Ye, Jiaxi Cai, Minheng Gong, Hannah E Ledvina, Huilin Zhou, Aaron T Whiteley, Raymond T Suhandynata, Kevin D Corbett\\nbioRxiv 2023.09.26.559546; doi: https://doi.org/10.1101/2023.09.26.559546 \\n Add to Selected Citations\\n\\n\\n\\nGut microbial genomes with paired isolates from China signify probiotic and cardiometabolic effects \\nPan Huang, Quanbin Dong, Yifeng Wang, Yunfan Tian, Shunhe Wang, Chengcheng Zhang, Leilei Yu, Fengwei Tian, Xiaoxiang Gao, Hang Guo, Shanrong Yi, Mingyang Li, Yang Liu, Qingsong Zhang, Wenwei Lu, Gang Wang, Bo Yang, Shumao Cui, Dongxu Hua, Xiuchao Wang, Yuwen Jiao, Lu Liu, Qiufeng Deng, Beining Ma, Tingting Wu, Huayiyang Zou, CGMR Consortium, Jing Shi, Haifeng Zhang, Daming Fan, Yanhui Sheng, jianxin Zhao, Liming Tang, Hao Zhang, Wei Sun, Wei Chen, Xiangqing Kong, Lianmin Chen, Qixiao Zhai\\nbioRxiv 2023.09.25.559444; doi: https://doi.org/10.1101/2023.09.25.559444 \\n Add to Selected Citations\\n\\n\\n\\nA green-fluorescent siderophore protects bacterial communities from UV damage \\nÖzhan Özkaya, Jos Kramer, Tobias E Wechsler, Rolf Kümmerli\\nbioRxiv 2023.09.26.559457; doi: https://doi.org/10.1101/2023.09.26.559457 \\n Add to Selected Citations\\n\\n\\n\\nMolecular principles of CRISPR-Cas13 mismatch intolerance enable selective silencing of point-mutated oncogenic RNA with single-base precision \\nCarolyn Shembrey, Ray Yang, Joshua Casan, Wenxin Hu, Teresa Sadras, Krishneel Prasad, Jake Shortt, Ricky W Johnstone, Joseph A Trapani, Paul G Ekert, Mohamed Fareh\\nbioRxiv 2023.09.26.557083; doi: https://doi.org/10.1101/2023.09.26.557083 \\n Add to Selected Citations\\n\\n\\n\\nIdentification of a Hypoxia-induced Inflammatory Cell Death Pathway \\nAbhishek Bhardwaj, Maria Antonelli, Beatrix Ueberheide, Benjamin G. Neel\\nbioRxiv 2023.08.05.552118; doi: https://doi.org/10.1101/2023.08.05.552118 \\n Add to Selected Citations\\n\\n\\n\\nLuteinizing hormone stimulates ingression of granulosa cells within the mouse preovulatory follicle. \\nCorie M Owen, Laurinda A. Jaffe\\nbioRxiv 2023.04.21.537855; doi: https://doi.org/10.1101/2023.04.21.537855 \\n Add to Selected Citations\\n\\n\\n\\nThe interplay of uncertainty, relevance and learning influences auditory categorization \\nJanaki Sheth, Jared Collina, Eugenio Piasini, Konrad Paul Kording, Yale Cohen, Maria Neimark Geffen\\nbioRxiv 2022.12.01.518777; doi: https://doi.org/10.1101/2022.12.01.518777 \\n Add to Selected Citations\\n\\n\\n\\nAntidepressants stimulate lipoprotein(a) macropinocytosis via serotonin-enhanced cell surface binding \\nNikita Deo, Halima Siddiqui, Golnoush Madani, Malcolm Rutledge, Michael Williams, Sally PA McCormick, Gregory M.I. Redpath\\nbioRxiv 2021.03.26.437114; doi: https://doi.org/10.1101/2021.03.26.437114 \\n Add to Selected Citations\\n\\n\\n\\nIntra and interspecific diversity in a tropical plant clade alter herbivory and ecosystem resilience \\nAri Grele, Tara J Massad, Kathryn A Uckele, Lee Dyer, Yasmine Antonini, Laura Braga, Matthew L Forister, Lidia Sulca-Garro, Massuo Kato, Humberto G Lopez, André R Nascimento, Tom Parchman, Wilmer R Simbaña, Angela M Smilanich, John O Stireman, Eric J Tepe, Thomas Walla, Lora A Richards\\nbioRxiv 2023.03.05.531210; doi: https://doi.org/10.1101/2023.03.05.531210 \\n Add to Selected Citations\\n\\n\\n\\nThe time course of injury-risk after return-to-play in professional football \\nGuangze Zhang, Michel Brink, Karen aus der Fuenten, Tobi Tross, Peter Willeit, Tim Meyer, Koen A.P.M. Lemmink, Anne Hecksteden\\nmedRxiv 2023.09.25.23295972; doi: https://doi.org/10.1101/2023.09.25.23295972 \\n Add to Selected Citations\\n\\n\\n\\nGenomic signatures of adaptation in native lizards exposed to human-introduced fire ants \\nBraulio A. Assis, Alexis P. Sullivan, Stephanie Marciniak, Christina M. Bergey, Vanessa Garcia, Zachary A. Szpiech, Tracy Langkilde, George H Perry\\nbioRxiv 2023.09.24.559217; doi: https://doi.org/10.1101/2023.09.24.559217 \\n Add to Selected Citations\\n\\n\\n\\nMacroevolutionary dynamics in micro-organisms: generalists give rise to specialists across biomes in the ubiquitous bacterial phylum Myxococcota. \\nDaniel Padfield, Suzanne Kay, Rutger Vos, Chris Quince, Michiel Vos\\nbioRxiv 2023.09.26.559479; doi: https://doi.org/10.1101/2023.09.26.559479 \\n Add to Selected Citations\\n\\n\\n\\nDiet-induced obesity impacts influenza disease severity and transmission dynamics in ferrets \\nVictoria A. Meliopoulos, Rebekah Honce, Brandi Livingston, Virginia Hargest, Pamela Freiden, Lauren Lazure, Pamela H Brigleb, Erik A Karlsson, Heather Tillman, E Kaity Allen, David Boyd, Paul Thomas, Stacey Schultz-Cherry\\nbioRxiv 2023.09.26.558609; doi: https://doi.org/10.1101/2023.09.26.558609 \\n Add to Selected Citations\\n\\n\\n\\nMating system is associated with seed phenotypes upon loss of RNA-directed DNA Methylation in Brassicaceae \\nKelly J Dew-Budd, Hiu Tung Chow, Timmy Kendall, Brandon C David, James A Rozelle, Rebecca A Mosher, Mark A Beilstein\\nbioRxiv 2023.07.03.547469; doi: https://doi.org/10.1101/2023.07.03.547469 \\n Add to Selected Citations\\n\\n\\n\\nA novel bivalent interaction mode underlies a non-catalytic mechanism for Pin1-mediated Protein Kinase C regulation \\nXiao-Ru Chen, Karuna Dixit, Yuan Yang, Mark I McDermott, Hasan T Imam, Vytas A Bankaitis, Tatyana I Igumenova\\nbioRxiv 2023.09.18.558341; doi: https://doi.org/10.1101/2023.09.18.558341 \\n Add to Selected Citations\\n\\n\\n\\nDisulfide bridge-dependent dimerization triggers FGF2 membrane translocation into the extracellular space \\nFabio Lolicato, Julia Steringer, Roberto Saleppico, Daniel Beyer, Jaime Fernandez-Sobaberas, Sebastian Unger, Steffen Klein, Petra Riegerova, Xiao Jakob Schmitt, Shreyas Kaptan, Sabine Wegehingel, Hans-Michael Mueller, Christian Freund, Martin Hof, Radek Sachl, Petr Chlanda, Ilpo Vattulainen, Walter Nickel\\nbioRxiv 2023.04.12.536539; doi: https://doi.org/10.1101/2023.04.12.536539 \\n Add to Selected Citations\\n\\n\\n\\nA synthetic Notch (synNotch) system linking intratumoral immune-cancer cell communication to a synthetic blood biomarker assay \\nYangHao Fu, TianDuo Wang, John A. Ronald\\nbioRxiv 2023.09.26.559329; doi: https://doi.org/10.1101/2023.09.26.559329 \\n Add to Selected Citations\\n\\n\\n\\nOrthogonality of sensory and contextual categorical dynamics embedded in a continuum of responses from the second somatosensory cortex \\nLucas Bayones, Antonio Zainos, Manuel Alvarez, Ranulfo Romo, Alessio Franci, Roman Rossi-Pool\\nbioRxiv 2023.09.22.559070; doi: https://doi.org/10.1101/2023.09.22.559070 \\n Add to Selected Citations\\n\\n\\n\\nMechanism for Vipp1 spiral formation, ring biogenesis and membrane repair \\nSouvik Naskar, Andrea Merino, Javier Espadas, Jayanti Singh, Aurelien Roux, Adai Colom, Harry H Low\\nbioRxiv 2023.09.26.559607; doi: https://doi.org/10.1101/2023.09.26.559607 \\n Add to Selected Citations\\n\\n\\n\\nVirulence and transmission biology of the widespread, ecologically important pathogen of zooplankton, Spirobacillus cienkowskii \\nNina Wale, Claire B Freimark, Justin Ramirez, Ahmad Kafri, Marcin K Dziuba, Rebecca Bilich, Meghan A Duffy\\nbioRxiv 2023.09.26.557596; doi: https://doi.org/10.1101/2023.09.26.557596 \\n Add to Selected Citations\\n\\n\\n\\nUsing synchronized brain rhythms to bias memory-guided decisions \\nJohn J Stout Jr., Allison E George, Suhyeong Kim, Henry L Hallock, Amy L Griffin\\nbioRxiv 2023.04.02.535279; doi: https://doi.org/10.1101/2023.04.02.535279 \\n Add to Selected Citations\\n\\n\\n\\nCharacterising the Protein-Protein Interaction Between MDM2 and 14-3-3σ; Proof of Concept for Small Molecule Stabilisation \\nJake A Ward, Beatriz Romartinez-Alonso, Danielle F Kay, Jeddidiah Bellamy-Carter, Bethany Thurairajah, Jaswir Basran, Hanna Kwon, Aneika C Leney, Salvador Macip, Pietro Roversi, Frederick W Muskett, Richard G Doveston\\nbioRxiv 2023.09.26.559467; doi: https://doi.org/10.1101/2023.09.26.559467 \\n Add to Selected Citations\\n\\n\\n\\nLanTERN: a fluorescent sensor that specifically binds lanthanides \\nEthan M. Jones, Yang Su, Chris Sander, Quincey A. Justman, Michael Springer, Pamela A. Silver\\nbioRxiv 2023.09.26.559553; doi: https://doi.org/10.1101/2023.09.26.559553 \\n Add to Selected Citations\\n\\n\\n\\nAn enhancer RNA recruits MLL1 to regulate transcription of Myb \\nJuhyun Kim, Luis Francisco Diaz, Matthew James Miller, Benjamin Leadem, Ivan Krivega, Ann Dean\\nbioRxiv 2023.09.26.559528; doi: https://doi.org/10.1101/2023.09.26.559528 \\n Add to Selected Citations\\n\\n\\n\\nAn isoform-resolution transcriptomic atlas of colorectal cancer from long-read single-cell sequencing \\nZhongxiao Li, Bin Zhang, Jia Jia Chan, Hossein Tabatabaeian, Qing Yun Tong, Xiao Hong Chew, Xiaonan Fan, Patrick Driguez, Charlene Chan, Faith Cheong, Shi Wang, Bei En Siew, Ian Jse-Wei Tan, Kai Yin Lee, Bettina Lieske, Wai-Kit Cheong, Dennis Kappei, Ker-Kan Tan, Xin Gao, Yvonne Tay\\nbioRxiv 2023.04.21.536771; doi: https://doi.org/10.1101/2023.04.21.536771 \\n Add to Selected Citations\\n\\n\\n\\nUsing a function-first \"scout fragment\"-based approach to develop allosteric covalent inhibitors of conformationally dynamic helicase mechanoenzymes \\nJared R Ramsey, Patrick M M Shelton, Tyler K. Heiss, Paul Dominic B Olinares, Lauren E Vostal, Heather Soileau, Michael Grasso, Sara Warrington, Stephanie Adaniya, Michael Miller, Shan Sun, David J Huggins, Robert W Myers, Brian T Chait, Ekaterina V Vinogradova, Tarun M Kapoor\\nbioRxiv 2023.09.25.559391; doi: https://doi.org/10.1101/2023.09.25.559391 \\n Add to Selected Citations\\n\\n\\n\\nCharacterization of the bioaccumulation and toxicity of copper pyrithione, an antifouling compound, on juveniles of rainbow trout \\nCharlotte Bourdon, Jérôme Cachot, Patrice Gonzalez, Patrice Couture\\nbioRxiv 2023.01.31.526498; doi: https://doi.org/10.1101/2023.01.31.526498 \\n Add to Selected Citations\\n\\n\\n\\nHigh plasticity of ribosomal DNA organization in budding yeast \\nShuangying Jiang, Zelin Cai, Yun Wang, Cheng Zeng, Jiaying Zhang, Wenfei Yu, Chenghao Su, Shijun Zhao, Ying Chen, Yue Shen, Yingxin Ma, Yizhi Cai, Junbiao Dai\\nbioRxiv 2023.09.26.559284; doi: https://doi.org/10.1101/2023.09.26.559284 \\n Add to Selected Citations\\n\\n\\n\\nThe impact of phage and phage resistance on microbial community dynamics \\nEllinor  Opsal Alseth, Rafael Custodio, Sarah  A Sundius, Rachel  A Kuske, Sam  P. Brown, Edze  R Westra\\nbioRxiv 2023.09.26.559468; doi: https://doi.org/10.1101/2023.09.26.559468 \\n Add to Selected Citations\\n\\n\\n\\nA ready-to-use logistic Verhulst model implemented in R shiny to estimate growth parameters of microorganisms \\nMarc Garel, Lloyd Izard, Marthe Vienne, David Nerini, Badr Al Ali, Christian Tamburini, Severine Martini\\nbioRxiv 2022.07.29.501982; doi: https://doi.org/10.1101/2022.07.29.501982 \\n Add to Selected Citations\\n\\n\\n\\nStructural and functional insights into the enzymatic plasticity of the SARS-CoV-2 NiRAN Domain \\nGabriel I Small, Olga Federova, Paul Dominic B Olinares, Joshua Chandanani, Anoosha Banerjee, Young Joo Choi, Henrik Molina, Brian Chait, Seth A Darst, Elizabeth A Campbell\\nbioRxiv 2023.09.25.558837; doi: https://doi.org/10.1101/2023.09.25.558837 \\n Add to Selected Citations\\n\\n\\n\\nMolecular tuning of sea anemone stinging \\nLily S He, Yujia Qi, Corey AH Allard, Wendy A Valencia-Montoya, Stephanie P Krueger, Keiko Weir, Agnese Seminara, Nicholas W Bellono\\nbioRxiv 2023.06.15.545144; doi: https://doi.org/10.1101/2023.06.15.545144 \\n Add to Selected Citations\\n\\n\\n\\nQuinoclamine inhibits Shiga toxin production in enterohemorrhagic Escherichia coli \\nOiti Kar, Hsiao-Cheng Feng, Hiroyuki Hirano, Ching-Hao Teng, Hiroyuki Osada, Masayuki Hashimoto\\nbioRxiv 2023.09.26.559460; doi: https://doi.org/10.1101/2023.09.26.559460 \\n Add to Selected Citations\\n\\n\\n\\nExperimental procedures for flow cytometry of wild type mouse brain: A systematic review \\nRobert C Sharp, Dylan T Guenther, Matthew James Farrer\\nbioRxiv 2023.07.06.547972; doi: https://doi.org/10.1101/2023.07.06.547972 \\n Add to Selected Citations\\n\\n\\n\\nRolling the evolutionary dice: Neisseria commensals as proxies for elucidating the underpinnings of antibiotic resistance mechanisms and evolution in human pathogens \\nKelly M Frost, Sierra L Charron-Smith, Terence C Cotsonas, Daniel C Dimartino, Rachel C Eisenhart, Eric T Everingham, Elle C Holland, Kainat Imtiaz, Cory J Kornowicz, Lydia E Lenhard, Liz H Lynch, Nadia P Moore, Kavya Phadke, Makayla L Reed, Samantha R Smith, Liza L Ward, Crista B Wadsworth\\nbioRxiv 2023.09.26.559611; doi: https://doi.org/10.1101/2023.09.26.559611 \\n Add to Selected Citations\\n\\n\\n\\nProlonged delays in human microbiota transmission after a controlled antibiotic perturbation \\nKatherine Shaohua Xue, Sophie Jean Walton, Doran A Goldman, Maike L Morrison, Adrian J Verster, Autumn B Parrott, Feiqiao Brian Yu, Norma Neff, Noah A Rosenberg, Benjamin D Ross, Dmitri Petrov, Kerwyn Casey Huang, Benjamin H Good, David Relman\\nbioRxiv 2023.09.26.559480; doi: https://doi.org/10.1101/2023.09.26.559480 \\n Add to Selected Citations\\n\\n\\n\\nCross-regulation and cross-talk of conserved and accessory two-component regulatory systems orchestrate Pseudomonas copper resistance \\nSylvie Elsen, Victor SIMON, Ina Attrée\\nbioRxiv 2023.09.26.559466; doi: https://doi.org/10.1101/2023.09.26.559466 \\n Add to Selected Citations\\n\\n\\n\\nMetabolic Phenotypes of Overweight/Obesity and Risk of Cardiovascular Diseases in Postmenopausal Women: Findings from the China PEACE Million Persons Project \\nDanying Deng, Zhiqiang Nie, Jiabin Wang, Chaolei Chen, Wenbin Wang, Yanchen Zhu, Qingyu Guan, Yanqiu Ou, Yingqing Feng\\nmedRxiv 2023.09.25.23296122; doi: https://doi.org/10.1101/2023.09.25.23296122 \\n Add to Selected Citations\\n\\n\\n\\nAntifungal efficacy of natural antiseptic products against Candida auris \\nWing-Gi Wu, Kristine Shik Luk, Mei-Fan Hung, Wing-Yi Tsang, Kin-Ping Lee, Bosco Hoi-Shiu Lam, Ka-Lam Cheng, Wing-Sze Cheung, Hau-Ling Tang, Wing-Kin To\\nbioRxiv 2023.09.22.559057; doi: https://doi.org/10.1101/2023.09.22.559057 \\n Add to Selected Citations\\n\\n\\n\\nDesign of Cytotoxic T Cell Epitopes by Machine Learning of Human Degrons \\nNicholas L Truex, Somesh Mohapatra, Mariane Melo, Jacob Rodriguez, Na Li, Wuhbet Abraham, Deborah Sementa, Faycal Touti, Derin B Keskin, Catherine J Wu, Darrell J Irvine, Rafael Gomez-Bombarelli, Bradley L. Pentelute\\nbioRxiv 2023.08.22.554289; doi: https://doi.org/10.1101/2023.08.22.554289 \\n Add to Selected Citations\\n\\n\\n\\nRCoV19: A One-stop Hub for SARS-CoV-2 Genome Data Integration, Variants Monitoring, and Risk Pre-warning \\nCuiping Li, Lina Ma, Dong Zou, Rongqin Zhang, Xue Bai, Lun Li, Gangao Wu, Tianhao Huang, Wei Zhao, Enhui Jin, Yiming Bao, Shuhui Song\\nbioRxiv 2023.09.24.558358; doi: https://doi.org/10.1101/2023.09.24.558358 \\n Add to Selected Citations\\n\\n\\n\\nPeptidoglycan-tethered and free forms of the Braun lipoprotein are in dynamic equilibrium in Escherichia coli \\nYucheng LIANG, Jean-Emmanuel HUGONNET, Filippo Rusconi, Michel Arthur\\nbioRxiv 2023.09.26.559517; doi: https://doi.org/10.1101/2023.09.26.559517 \\n Add to Selected Citations\\n\\n\\n\\nKetogenic diet reduces a neurobiological craving signature in alcohol use disorder \\nCorinde Wiers, Peter Manza, Gene-Jack Wang, Nora D Volkow\\nmedRxiv 2023.09.25.23296094; doi: https://doi.org/10.1101/2023.09.25.23296094 \\n Add to Selected Citations\\n\\n\\n\\nIdentification of potent and orally efficacious phosphodiesterase inhibitors in Cryptosporidium parvum-infected immunocompromised mice \\nJose E Teixeira, Makafui Gasonoo, Peter Miller, Jubilee Ajiboye, Alexandra C Cameron, Erin Stebbins, Scott D Campbell, David W Griggs, Thomas Spangenberg, Marvin J Meyers, Christopher D Huston\\nbioRxiv 2023.09.26.559556; doi: https://doi.org/10.1101/2023.09.26.559556 \\n Add to Selected Citations\\n\\n\\n\\nMultiscale effects of perturbed translation dynamics inform antimalarial design \\nLeonie Anton, Wenjing Cheng, Meseret Haile, David Wayne Cobb, Xiyan Zhu, Leyan Han, Emerson Li, Anjali Nair, Carolyn Lin Lee, Hangjun Ke, Guoan Zhang, Emma H. Doud, Chi-Min Ho\\nbioRxiv 2023.09.03.556115; doi: https://doi.org/10.1101/2023.09.03.556115 \\n Add to Selected Citations\\n\\n\\n\\nIn silico analyses identify sequence contamination thresholds for Nanopore-generated SARS-CoV-2 sequences \\nAyooluwa Bolaji, Ana  T. Duggan\\nbioRxiv 2023.09.26.559465; doi: https://doi.org/10.1101/2023.09.26.559465 \\n Add to Selected Citations\\n\\n\\n\\nWhen paleontology meets genomics: complete mitochondrial genomes of two saber-toothed cats\\' species (Felidae: Machairodontinae) \\nIgor Henrique Rodrigues-Oliveira, Iuri Batista da Silva, Renan Rodrigues Rocha, Rafael Augusto Silva Soares, Fabiano Bezerra Menegidio, Caroline Garcia, Rubens Pasa, Karine Frehner Kavalco\\nbioRxiv 2023.09.14.557696; doi: https://doi.org/10.1101/2023.09.14.557696 \\n Add to Selected Citations\\n\\n\\n\\nThe Healthy Brain Initiative (HBI): A prospective cohort study protocol \\nLilah M. Besser, Stephanie Chrisphonte, Michael J. Kleiman, Deirdre O\\'Shea, Amie Rosenfeld, Magda Tolea, James E. Galvin\\nmedRxiv 2023.09.21.23295908; doi: https://doi.org/10.1101/2023.09.21.23295908 \\n Add to Selected Citations\\n\\n\\n\\nPatient and Hospital Characteristics Associated with Disparities in Acute Stroke Treatment in Community and Academic Hospitals \\nJerry Shen, Laura Corlin, Kathryn E Coté, Megan E Pudlo, Lester Y Leung\\nmedRxiv 2023.09.25.23296119; doi: https://doi.org/10.1101/2023.09.25.23296119 \\n Add to Selected Citations\\n\\n\\n\\nOncogene EVI1 Drives Acute Myeloid Leukemia Via a Targetable Interaction with CTBP2 \\nDorien Pastoors, Marije Havermans, Roger Mulet-Lazaro, Duncan Brian, Willy Noort, Julius Grasel, Remco Hoogenboezem, Leonie Smeenk, Jeroen AA Demmers, Tariq Enver, Richard WJ Groen, Eric Bindels, Ruud Delwel\\nbioRxiv 2023.07.11.548358; doi: https://doi.org/10.1101/2023.07.11.548358 \\n Add to Selected Citations\\n\\n\\n\\nElectrostatic Interactions Dictate Bile Salt Hydrolase Substrate Preference \\nKien P. Malarney, Pamela V. Chang\\nbioRxiv 2023.09.25.559308; doi: https://doi.org/10.1101/2023.09.25.559308 \\n Add to Selected Citations\\n\\n\\n\\nFrom recency to central tendency biases in working memory: a unifying network model \\nVezha Boboeva, Alberto Pezzotta, Claudia Clopath, Athena Akrami\\nbioRxiv 2022.05.16.491352; doi: https://doi.org/10.1101/2022.05.16.491352 \\n Add to Selected Citations\\n\\n\\n\\nCONORM: Context-Aware Entity Normalization for Adverse Drug Event Detection \\nAnthony Yazdani, Hossein Rouhizadeh, Alban Bornet, Douglas Teodoro\\nmedRxiv 2023.09.26.23296150; doi: https://doi.org/10.1101/2023.09.26.23296150 \\n Add to Selected Citations\\n\\n\\n\\nDiagnostic Accuracy of Carotid Plaque Instability by Noninvasive Imaging: A Systematic Review and Meta-analysis \\nDavid Pakizer, Jiří Kozel, Patrick Taffé, Jolanda Elmers, Janusz Feber, Patrik Michel, David Školoudík, Gaia Sirimarco\\nmedRxiv 2023.09.25.23296124; doi: https://doi.org/10.1101/2023.09.25.23296124 \\n Add to Selected Citations\\n\\n\\n\\nCost-effectiveness and budget impact of decentralising childhood tuberculosis diagnosis: a mathematical modelling study in six high tuberculosis incidence countries \\nMarc d\\'Elbée, Martin Harker, Nyashadzaishe Mafirakureva, Mastula Nanfuka, Minh Huyen Ton Nu Nguyet, Jean-Voisin Taguebue, Raoul Moh, Celso Khosa, Ayeshatu Mustapha, Juliet Mwanga, Laurence Borand, Sylvie Kwedi Nolna, Eric Komena, Saniata Cumbe, Jacob Mugisha, Naome Natukunda, Tan Eang Mao, Jėrôme Wittwer, Antoine Bénard, Tanguy Bernard, Hojoon Sohn, Maryline Bonnet, Eric Wobudeya, Olivier Marcy, Pete James Dodd, TB-Speed Health Economics Study Group\\nmedRxiv 2023.09.26.23295962; doi: https://doi.org/10.1101/2023.09.26.23295962 \\n Add to Selected Citations\\n\\n\\n\\nSearching the mechanisms of mRNA buffering: the releasing-shuttling model \\nQirun Wang, Jie Lin\\nbioRxiv 2023.08.23.554439; doi: https://doi.org/10.1101/2023.08.23.554439 \\n Add to Selected Citations\\n\\n\\n\\nMetabolic and mitochondria alterations induced by SARS-CoV-2 accessory proteins ORF3a, ORF9b, ORF9c and ORF10 \\nBlanca D. Lopez-Ayllon, Silvia Marin, Marcos Farinas Fernandez, Transito Garcia-Garcia, Raul Fernandez-Rodriguez, Ana de Lucas-Rius, Natalia Redondo, Laura Mendoza-Garcia, Carles Foguet, Jouzas Grigas, Alba Calvet, Jose Manuel Villalba, Maria Josefa Rodriguez Gomez, Diego Megias, Biagio Mandracchia, Daniel Luque, Juan Jose Lozano, Cristina Calvo, Timothy M. Thomson, Juan Jose Garrido, Marta Cascante, Maria Montoya\\nbioRxiv 2023.09.26.559506; doi: https://doi.org/10.1101/2023.09.26.559506 \\n Add to Selected Citations\\n\\n\\n\\nGenomic insights into pediatric intestinal inflammatory and eosinophilic disorders using single-cell RNA-sequencing \\nMarissa R Keever-Keigher, Rachel Chevalier, Lisa Harvey, Veronica Williams, Carrie A Vyhlidal, Atif A Ahmed, Jeffery J Johnston, Daniel A Louiselle, Elin Grundberg, Tomi Pastinen, Craig A Friesen, Craig Smail, Valentina Shakhnovich\\nmedRxiv 2023.09.26.23295909; doi: https://doi.org/10.1101/2023.09.26.23295909 \\n Add to Selected Citations\\n\\n\\n\\nOutcomes Following Surgical Repair of Sinus Venosus Atrial Septal Defects: A Systematic Review \\nRyaan EL-Andari, Muhammad Moolla, Kevin John, Ashley Slingerland, Sandra Campbell, Jeevan Nagendran, Anoop Mathew\\nmedRxiv 2023.09.21.23295934; doi: https://doi.org/10.1101/2023.09.21.23295934 \\n Add to Selected Citations\\n\\n\\n\\nAssessment of Accuracy and Safety of LabTest Checker (LTC-AI) \\nDawid Szumilas, Anna Ochmann, Katarzyna Zięba, Jerzy Chudek, Bartłomiej Bartoszewicz, Anna Kubrak, Sebastian Makuch, Siddarth Agrawal, Grzegorz Mazur\\nmedRxiv 2023.09.26.23296157; doi: https://doi.org/10.1101/2023.09.26.23296157 \\n Add to Selected Citations\\n\\n\\n\\nSpecies and habitat specific changes in bird activity in an urban environment during Covid 19 lockdown \\nCongnan Sun, Yoel Hassin, Arjan Boonman, Assaf Shwartz, Yossi Yovel\\nbioRxiv 2023.06.03.543542; doi: https://doi.org/10.1101/2023.06.03.543542 \\n Add to Selected Citations\\n\\n\\n\\nBMP signaling pathway member expression is enriched in enteric neural progenitors and required for zebrafish enteric nervous system development \\nJoshua A Moore, Arielle S Noah, Eileen W Singleton, Rosa A Uribe\\nbioRxiv 2023.09.24.559191; doi: https://doi.org/10.1101/2023.09.24.559191 \\n Add to Selected Citations\\n\\n\\n\\nWavelet transform analysis reveals differences between patients with impaired left ventricular systolic function and healthy individuals \\nMarcin Gruszecki, Damian Kaufmann, Michal Swiatczak, Krzysztof Mlodzinski, J. Patrick Neary, Jyotpal Singh, Jacek Ruminski, Ludmila Danilowicz-Szymanowicz\\nmedRxiv 2023.09.25.23296125; doi: https://doi.org/10.1101/2023.09.25.23296125 \\n Add to Selected Citations\\n\\n\\n\\nQuantitative Assessment of Morphological Changes in Lipid Droplets and Lipid-Mito Interactions with Aging in Brown Adipose \\nAmber Crabtree, Kit Neikirk, Aaron Whiteside, Jessica McKenzie, Zer Vue, Larry Vang, Trinity C Owens, Ashton Oliver, Faben Zeleke, Heather K. Beasley, Edgar Garza-Lopez, Estevao Scudese, Alice Koh, Suzanne Navarro, Jacob Lam, Benjamin Kirk, Margaret Mungai, Mariya T Sweetwyne, Ho-Jin Koh, Elma Zaganjor, Steven M. Damo, Jennifer A. Gaddy, Clintoria Williams, Melanie McReynolds, Andrea G. Marshall, Antentor Hinton Jr.\\nbioRxiv 2023.09.23.559135; doi: https://doi.org/10.1101/2023.09.23.559135 \\n Add to Selected Citations\\n\\n\\n\\nMuscle cell atrophy induced by miR-155-5p reveals molecular targets in skeletal muscle disorders \\nLeticia Oliveira Lopes, Sarah Santiloni Cury, Diogo de Moraes, Jakeline Santos Oliveira, Grasieli de Oliveira, Otavio Cabral-Marques, Geysson Javier Fernandez, Mario Hiroyuki Hirata, Da-Zhi WANG, Maeli Dal-Pai-Silva, Robson Carvalho, Paula P Freire\\nbioRxiv 2023.09.21.558746; doi: https://doi.org/10.1101/2023.09.21.558746 \\n Add to Selected Citations\\n\\n\\n\\nChromosome-scale Genome Assembly of the West Indian Fruit Fly Anastrepha obliqua (Diptera: Tephritidae) \\nSheina B. Sim, Carlos Congrains, Sandra Marcela Velasco-Cuervo, Renée Lynn Corpuz, Angela Nani Kauwe, Brian Scheffler, Scott Michael Geib\\nbioRxiv 2023.09.22.559041; doi: https://doi.org/10.1101/2023.09.22.559041 \\n Add to Selected Citations\\n\\n\\n\\nDevelopment of a novel patient-reported mental wellbeing scale for patients with chronic gastroduodenal symptoms: The Alimetry® Gut-Brain Wellbeing Survey \\nMikaela Law, Isabella Pickering, Gayl Humphrey, Gabrielle Sebaratnam, Gabriel Schamberg, Katie Simpson, Chris Varghese, Peng Du, Charlotte Daker, I-Hsuan Huang, Armen Gharibans, Gregory O\\'Grady, Christopher N Andrews, Stefan Calder\\nmedRxiv 2023.09.25.23296105; doi: https://doi.org/10.1101/2023.09.25.23296105 \\n Add to Selected Citations\\n\\n\\n\\nCardiopulmonary Exercise Testing in a Prospective Multicenter Cohort of Older Adults: The Study of Muscle, Mobility and Aging (SOMMA) \\nCody Wolf, Terri L. Blackwell, Eileen Johnson, Nancy W. Glynn, Barbara J Nicklas, Stephen B Kritchevsky, Elvis A. Carnero, Peggy Cawthon, Steven Cummings Cummings, Frederico G. S. Toledo, Anne B Newman, Daniel E. Forman, Bret H. Goodpaster\\nmedRxiv 2023.09.25.23296120; doi: https://doi.org/10.1101/2023.09.25.23296120 \\n Add to Selected Citations\\n\\n\\n\\nDetection of M. tuberculosis DNA in TB contacts PBMC does not associate with blood RNA signatures for incipient tuberculosis \\nJoshua Rosenheim, Markos Abebe, Stephen Reece, Mahdad Noursadeghi, Adrian R Martineau\\nmedRxiv 2023.09.26.23296131; doi: https://doi.org/10.1101/2023.09.26.23296131 \\n Add to Selected Citations\\n\\n\\n\\nAn RNA foundation model enables discovery of disease mechanisms and candidate therapeutics \\nAlbi Celaj, Alice Jiexin Gao, Tammy T.Y. Lau, Erle M. Holgersen, Alston Lo, Varun Lodaya, Christopher B. Cole, Robert E. Denroche, Carl Spickett, Omar Wagih, Pedro O. Pinheiro, Parth Vora, Pedrum Mohammadi-Shemirani, Steve Chan, Zach Nussbaum, Xi Zhang, Helen Zhu, Easwaran Ramamurthy, Bhargav Kanuparthi, Michael Iacocca, Diane Ly, Ken Kron, Marta Verby, Kahlin Cheung-Ong, Zvi Shalev, Brandon Vaz, Sakshi Bhargava, Farhan Yusuf, Sharon Samuel, Sabriyeh Alibai, Zahra Baghestani, Xinwen He, Kirsten Krastel, Oladipo Oladapo, Amrudha Mohan, Arathi Shanavas, Magdalena Bugno, Jovanka Bogojeski, Frank Schmitges, Carolyn Kim, Solomon Grant, Rachana Jayaraman, Tehmina Masud, Amit Deshwar, Shreshth Gandhi, Brendan J. Frey\\nbioRxiv 2023.09.20.558508; doi: https://doi.org/10.1101/2023.09.20.558508 \\n Add to Selected Citations\\n\\n\\n\\nTetraspanin CD81 promotes leukemia stem cell function and represents a new therapeutic vulnerability in acute myeloid leukemia \\nFanny Gonzales, Pauline Peyrouze, Thomas Boyer, Soizic Guihard, Francois Sevrin, Djohana LAURENT, Adriana Plesa, Adeline Barthelemy, Antonino Bongiovanni, Nicolas Pottier, Claude PREUDHOMME, Nicolas DUPLOYEZ, Celine Berthon, Christophe ROUMIER, Meyling CHEOK\\nbioRxiv 2023.09.20.558656; doi: https://doi.org/10.1101/2023.09.20.558656 \\n Add to Selected Citations\\n\\n\\n\\nExpanding the DNA editing toolbox: novel lambda integrase variants targeting microalgal and human genome sequences. \\nSiau  Jia Wei, Asim  Azhar Siddiqui, Lau  Sze Yi, Srinivasaraghavan Kannan, Sabrina Peter, Zeng Yingying, Chandra Verma, Peter Droge, John  F. Ghadessy\\nbioRxiv 2023.09.22.559039; doi: https://doi.org/10.1101/2023.09.22.559039 \\n Add to Selected Citations\\n\\n\\n\\nCladosporium species detoxify multiple water micropollutants of emerging concern using diverse strategies \\nMaria Louise Leth, Kai Tang, Trine Sørensen, Aaron John Andersen, Claus Hélix-Nielsen, Birgitte Andersen, Jens Frisvad, Teis Esben Sondergaard, Henrik Rasmus Andersen, Maher Abou Hachem\\nbioRxiv 2023.09.25.559271; doi: https://doi.org/10.1101/2023.09.25.559271 \\n Add to Selected Citations\\n\\n\\n\\nComparison of NAFLD, MAFLD, MASLD and pure-MASLD characteristics and mortality outcomes in United States adults \\nRui Song, Zhao Li, Yingzhi Zhang, Jiahe Tan, Zhiwei Chen\\nmedRxiv 2023.09.26.23296130; doi: https://doi.org/10.1101/2023.09.26.23296130 \\n Add to Selected Citations\\n\\n\\n\\nDeterminants of health as predictors for differential antibody responses following SARS-CoV-2 primary and booster vaccination in an at-risk, longitudinal cohort \\nFelipe Echeverri Tribin, Erin Williams, Valeska Testamarck, Juan  Manuel Carreño, Dominika Bielak, Temima Yellin, Florian Krammer, Michael Hoffer, Suresh Pallikkuth, Savita Pahwa\\nmedRxiv 2023.09.25.23296114; doi: https://doi.org/10.1101/2023.09.25.23296114 \\n Add to Selected Citations\\n\\n\\n\\nExploring the effects of Dasatinib, Quercetin, and Fisetin on DNA methylation clocks: a longitudinal study on senolytic interventions \\nEdwin Lee, Natalia Carreras-Gallo, Leilani Lopez, Logan Turner, Aaron Lin, Tavis L Mendez, Hannah Went, Alan Tomusiak, Eric Verdin, Michael Corley, Lishomwa Ndhlovu, Ryan Smith, Varun B Dwaraka\\nmedRxiv 2023.09.22.23295961; doi: https://doi.org/10.1101/2023.09.22.23295961 \\n Add to Selected Citations\\n\\n\\n\\nGenomic surveillance of Clostridioides difficile transmission and virulence in a healthcare setting \\nErin P. Newcomer, Skye Fishbein, Kailun Zhang, Tiffany Hink, Kimberly A. Reske, Candice Cass, Zainab H. Iqbal, Emily L. Struttman, Erik R. Dubberke, Gautam Dantas\\nmedRxiv 2023.09.26.23295023; doi: https://doi.org/10.1101/2023.09.26.23295023 \\n Add to Selected Citations\\n\\n\\n\\nImmunologic predictors of vaccine responsiveness in patients with lymphoma and CLL \\nElise A Chong, Kingsley Gideon Kumashie, Emeline R Chong, Joseph Fabrizio, Aditi Gupta, Jakub Svoboda, Stefan K Barta, Kristy M Walsh, Ellen Napier, Rachel Lundburg, Sunita D Nasta, James N Gerson, Daniel J Landsburg, Joyce Gonzalez, Andrew Gaano, Madison E Weirick, Christopher M McAllister, Moses Awofolaju, Gavin N John, Shane C Kammerman, Josef Novaceck, Raymone Pajarillo, Kendall A Lundgreen, Nicole Tanenbaum, Sigrid Gouma, Elizabeth M Drapeau, Sharon Adamski, Kurt D Andrea, Ajinkya Pattekar, Amanda Hicks, Scott Korte, Harsh Sharma, Sarah Herring, Justine C Williams, Jacob T Hamilton, Paul Bates, Scott E Hensley, Eline T Luning Prak, Allison R Greenplate, E John Wherry, Stephen J Schuster, Marco Ruella, Laura A Vella\\nmedRxiv 2023.09.26.23295903; doi: https://doi.org/10.1101/2023.09.26.23295903 \\n Add to Selected Citations\\n\\n\\n\\nIncorporating Nanopore Sequencing into a Diverse Diagnostic Toolkit for Incontinentia Pigmenti \\nSimone Ahting, Denny Popp, Henry Oppermann, Vincent Strehlow, Maria Fasshauer, Bernt Popp, Maike Karnstedt, Isabell Schumann\\nmedRxiv 2023.09.26.23295778; doi: https://doi.org/10.1101/2023.09.26.23295778 \\n Add to Selected Citations\\n\\n\\n\\nEvaluation of computer vision syndrome in the sample of students from the College of Medicine/University of Kerbala result of online learning with the isolation of COVID-19 \\nmahdy hameed AbuRagheif Sr.\\nmedRxiv 2023.09.25.23295828; doi: https://doi.org/10.1101/2023.09.25.23295828 \\n Add to Selected Citations\\n\\n\\n\\nLandscape of blood group antigens and alleles in the Indian population from whole genome sequences \\nMercy Rophina, Rahul C Bhoyar, Mohamed Imran, Vigneshwar Senthivel, Mohit Kumar Divakar, Anushree Mishra, Bani Jolly, Sridhar Sivasubbu, Vinod Scaria\\nmedRxiv 2023.09.26.23296145; doi: https://doi.org/10.1101/2023.09.26.23296145 \\n Add to Selected Citations\\n\\n\\n\\nVoice-Enabled Response Analysis Agent (VERAA): Leveraging Large Language Models to Map Voice Responses in SDoH Survey \\nRishivardhan Krishnamoorthy, Vishal Nagarajan, Hayden Pour, Supreeth Shashikumar, Aaron Boussina, Emilia Farcas, Shamim Nemati, Christopher S. Josef\\nmedRxiv 2023.09.25.23295917; doi: https://doi.org/10.1101/2023.09.25.23295917 \\n Add to Selected Citations\\n\\n\\n\\nAn assessment of census-tract level socioeconomic position as a modifier of the relationship between PM 2.5 concentrations and cardiovascular emergency department visits in Missouri \\nZachary H. McCann, Howard H. Chang, Rohan D\\'Souza, Noah Scovronick, Stefanie Ebelt\\nmedRxiv 2023.09.25.23296034; doi: https://doi.org/10.1101/2023.09.25.23296034 \\n Add to Selected Citations\\n\\n\\n\\nA curriculum-based approach to make healthcare inclusive for the transgender population: challenges and lessons from Pakistan \\nAkash Kumar Ahuja, Manzar Abbas, Mian Arsam Haroon, Muhammad Abdullah Javed, Areeba Memon, Rida Irfan, Zohair Karim, Areesh Bhatti, Mehsa Hashim, Janeeta Hamid, Asaad Nafees\\nmedRxiv 2023.09.26.23295951; doi: https://doi.org/10.1101/2023.09.26.23295951 \\n Add to Selected Citations\\n\\n\\n\\nLongitudinal Trajectories of Symptom Change During Antidepressant Treatment Among Managed Care Patients with Co-Occurring Depression and Anxiety \\nJudith Cukor, Zhenxing Xu, Veer Vekaria, Fei Wang, Mark Olfson, Samprit Banerjee, Gregory E Simon, George S Alexopoulos, Jyotishman Pathak\\nmedRxiv 2023.09.25.23295985; doi: https://doi.org/10.1101/2023.09.25.23295985 \\n Add to Selected Citations\\n\\n\\n\\nHealthcare Costs and Healthcare Utilization Outcomes of Vitamin D3 Supplementation at 5000 IU Daily During a 10.9 month Observation Period within a Pragmatic Randomized Clinical Trial \\nPatrick J LaRiccia, Teresa Cafaro, Dibato John, Noud van Helmond, Ludmil V Mitrev, Brigid Bandomer, Tracy L Brobyn, Krystal Hunter, Satyajeet Roy, Kevin Q Ng, Helen Goldstein, Alan Tsai, Denise Thwing, Mary Ann Maag, Myung K Chung\\nmedRxiv 2023.09.25.23296104; doi: https://doi.org/10.1101/2023.09.25.23296104 \\n Add to Selected Citations\\n\\n\\n\\nFunctional genomics implicates natural killer cells as potential key drivers in the pathogenesis of ankylosing spondylitis \\nMarcos Chinas, Daniela Fernandez-Salinas, Vitor RC Aguiar, Victor E Nieto-Caballero, Micah Lefton, Peter A Nigrovic, Joerg Ermann, Maria Gutierrez-Arcelus\\nmedRxiv 2023.09.21.23295912; doi: https://doi.org/10.1101/2023.09.21.23295912 \\n Add to Selected Citations\\n\\n\\n\\nAccessPD - a next generation registry to accelerate Parkinson\\'s disease research \\nYun-Hsuan Chang, Maria Teresa Perinan, Matt Wilson, Alastair Noyce\\nmedRxiv 2023.09.26.23295976; doi: https://doi.org/10.1101/2023.09.26.23295976 \\n Add to Selected Citations\\n\\n\\n\\nNear-term forecasting of Covid-19 cases and hospitalisations in Aotearoa New Zealand \\nMichael J Plank, Leighton M Watson, Oliver J Maclaren\\nmedRxiv 2023.09.25.23296118; doi: https://doi.org/10.1101/2023.09.25.23296118 \\n Add to Selected Citations\\n\\n\\n\\nDevelopment and validation of the Japanese version of the Lesbian, Gay, Bisexual, and Transgender Development of Clinical Skills Scale \\nYusuke Kanakubo, Yoshifumi Sugiyama, Eriko Yoshida, Takuya Aoki, Rieko Mutai, Masato Matsushima, Tadao Okada\\nmedRxiv 2023.09.26.23296058; doi: https://doi.org/10.1101/2023.09.26.23296058 \\n Add to Selected Citations\\n\\n\\n\\nNoninvasive Quantification of Radiation-Induced Lung Injury using a Targeted Molecular Imaging Probe \\nEric Abston, Iris Zhou, Jonathan Saenger, Sergey Shuvaev, Eman A Akam, Shadi Esfahani, Lida Hariri, Nicholas Rotile, Elizabeth Crowley, Sydney Montesi, Valerie Humblet, Grae Arabasz, Melin Khandekar, Ciprian Catana, Florian Fintelmann, Peter Caravan, Michael Lanuti\\nmedRxiv 2023.09.25.23295897; doi: https://doi.org/10.1101/2023.09.25.23295897 \\n Add to Selected Citations\\n\\n\\n\\nA novel ultrasensitive assay for plasma p-tau217: performance in individuals with subjective cognitive decline and early Alzheimers disease \\nFernando Gonzalez-Ortiz, Pamela C Ferreira, Armand Gonzalez, Laia Montoliu-Gaya, Paula Ortiz-Romero, Przemyslaw R Kac, Michael Turton, Hlin Kvartsberg, Nicholas J Ashton, Henrik Zetterberg, Peter Harrison, Bruna Bellaver, Guilherme Povala, Victor L Villemagne, Tharick Pascoal, Mary Ganguli, Anne D Cohen, Carolina Minguillon, Jose Contador, Marc Suarez-Calvet, Thomas K Karikari, Kaj Blennow\\nmedRxiv 2023.09.26.23296134; doi: https://doi.org/10.1101/2023.09.26.23296134 \\n Add to Selected Citations\\n\\n\\n\\nDeep Learning based retinal layer segmentation in optical coherence tomography scans of patients with inherited retinal diseases \\nFranziska Eckardt, Robin Mittas, Nastassya Horlava, Johannes Schiefelbein, Ben Asani, Stylianos Michalakis, Maximilian Gerhardt, Claudia Priglinger, Daniel Keeser, Nikolaos Koutsouleris, Siegfried G. Priglinger, Fabian J Theis, Tingying Peng, Benedikt Schworm\\nmedRxiv 2023.09.26.23295852; doi: https://doi.org/10.1101/2023.09.26.23295852 \\n Add to Selected Citations\\n\\n\\n\\nAttendance at remote versus face-to-face outpatient appointments in an NHS Trust \\nGabriele Keziah Kerr, Geva Greenfield, Benedict Hayhoe, Fiona Gaughran, Kristoffer Halvorsrud, Mariana Pinto da Costa, Nirandeep Rehill, Rosalind Raine, Azeem Majeed, Ceire Costelloe, Ana Luísa Neves, Thomas Beaney\\nmedRxiv 2023.09.22.23295958; doi: https://doi.org/10.1101/2023.09.22.23295958 \\n Add to Selected Citations\\n\\n\\n\\nDevelopment and validation of a machine learning model for prediction of type 2 diabetes in patients with mental illness \\nMartin Bernstorff, Lasse Hansen, Kenneth Enevoldsen, Jakob Damgaard, Frida Haestrup, Erik Perfalk, Andreas Aalkjaer Danielsen, Soeren Dinesen Oestergaard\\nmedRxiv 2023.09.26.23296141; doi: https://doi.org/10.1101/2023.09.26.23296141 \\n Add to Selected Citations\\n\\n\\n\\nAdverse childhood experiences across Multiracial and monoracial groups with and without Indigenous ancestry \\nTracy Lam-Hine, Corinne A. Riddell, Patrick T. Bradshaw, Michael Omi, Amani M. Allen\\nmedRxiv 2023.09.25.23296116; doi: https://doi.org/10.1101/2023.09.25.23296116 \\n Add to Selected Citations\\n\\n\\n\\nSelf-identification as a Crucial Aspect of Inclusion of Sexual and Gender Minorities in a Study of Adverse Childhood Experiences in Haiti \\nJoseph Heger, Phycien Paul, Caleb Jean-Baptiste, Elmondo Odans, Abraham A Salinas-Miranda, Erin Lynch, Emily Stevens, Guitele J Rahill, Manisha Joshi\\nmedRxiv 2023.09.24.23295796; doi: https://doi.org/10.1101/2023.09.24.23295796 \\n Add to Selected Citations\\n\\n\\n\\nDiscovery of Urinary Metabolite Biomarkers of Psychiatric Disorders Using Two-Sample Mendelian Randomization \\nJihan K Zaki, Jakub Tomasik, Jade A McCune, Oren A Scherman, Sabine Bahn\\nmedRxiv 2023.09.26.23296078; doi: https://doi.org/10.1101/2023.09.26.23296078 \\n Add to Selected Citations\\n\\n\\n\\nAn unsupervised machine learning approach to predict recovery from traumatic spinal cord injury \\nSarah C. Brueningk, Lucie Bourguignon, Louis Lukas, EMSCI Study Group, Doris Maier, Rainer Abel, Norbert Weidner, Ruediger Rupp, Fred Geisler, John L.K. Kramer, James Guest, Armin Curt, Catherine R. Jutzeler\\nmedRxiv 2023.09.26.23295361; doi: https://doi.org/10.1101/2023.09.26.23295361 \\n Add to Selected Citations\\n\\n\\n\\nProgrammatic considerations for chikungunya vaccine introduction in countries at risk of chikungunya outbreaks: stakeholder analysis \\nMegan Auzenbergs, Clara Maure, Hyolim Kang, Oliver J Brady, Sushant Sahastrabuddhe, Kaja Abbas\\nmedRxiv 2023.09.26.23296129; doi: https://doi.org/10.1101/2023.09.26.23296129 \\n Add to Selected Citations\\n\\n\\n\\nMODELING CYSTIC FIBROSIS PATIENT PROGNOSIS: NOMOGRAMS TO PREDICT LUNG TRANSPLANTATION AND SURVIVAL PRIOR TO HIGHLY EFFECTIVE MODULATOR THERAPY \\nAnnalisa Piccorelli, Jerry Nick\\nmedRxiv 2023.09.25.23296112; doi: https://doi.org/10.1101/2023.09.25.23296112 \\n Add to Selected Citations\\n\\n\\n\\nDeterminants of anti-retroviral therapy adherence among adolescents living with HIV in the Kingdom of Eswatini \\nLondiwe  D. Hlophe, Constance  S Shumba, Diribsa  T Bedada, Peter  S Nyasulu\\nmedRxiv 2023.09.25.23296111; doi: https://doi.org/10.1101/2023.09.25.23296111 \\n Add to Selected Citations\\n\\n\\n\\nDeveloping and validating the Nepalese Abuse Assessment Screen (N-AAS) for identifying domestic violence among pregnant women in Nepal \\nPratibha Manandhar, ADVANCE 2 study team\\nmedRxiv 2023.09.25.23296107; doi: https://doi.org/10.1101/2023.09.25.23296107 \\n Add to Selected Citations\\n\\n\\n\\nResearch ethics review during the COVID-19 pandemic An international study \\nFabio Salamanca-Buentello, Rachel Katz, Diego  S. Silva, Ross  E.G. Upshur, Maxwell  J. Smith\\nmedRxiv 2023.09.24.23296056; doi: https://doi.org/10.1101/2023.09.24.23296056 \\n Add to Selected Citations\\n\\n\\n\\nHow demographic factors matter for antimicrobial resistance – quantification of the patterns and impact of variation in prevalence of resistance by age and sex. \\nNaomi  R Waterlow, Ben  S Cooper, Julie  V Robotham, Gwenan  Mary Knight\\nmedRxiv 2023.09.24.23296060; doi: https://doi.org/10.1101/2023.09.24.23296060 \\n Add to Selected Citations\\n\\n\\n\\nHighly efficient and sensitive membrane-based concentration process allows quantification, surveillance, and sequencing of viruses in large volumes of wastewater. \\nGhina El Soufi, Leo Di Jorio, Zuzana Gerber, Nicolas Cluzel, Jeanne Van Assche, Damien Delafoy, Robert Olaso, Christian Daviaud, Thomas Loustau, Christian Schwartz, Dominique Trebouet, Olivier Hernalsteens, Vincent Marechal, Stephanie Raffestin, Dominique Rousset, Carine Van Lint, Jean-Francois Deleuze, Mickael Boni, OBEPINE Consortium, Olivier ROHR, Maud Villain-Gambier, Clementine Wallet\\nmedRxiv 2023.09.25.23296071; doi: https://doi.org/10.1101/2023.09.25.23296071 \\n Add to Selected Citations\\n\\n\\n\\nPerivascular space enlargement accelerates with hypertension, white matter hyperintensities, chronic inflammation, and Alzheimer\\'s disease pathology: evidence from a three-year longitudinal multicentre study \\nInga Menze, Jose Bernal, Pinar Kaya, Çağla Aki, Malte Pfister, Jonas Geisendörfer, Renat Yakupov, Michael T. Heneka, Frederic Brosseron, Matthias C. Schmid, Wenzel Glanz, Enise I. Incesoy, Michaela Butryn, Ayda Rostamzadeh, Dix Meiberth, Oliver Peters, Lukas Preis, Dominik Lammerding, Daria Gref, Josef Priller, Eike J. Spruth, Slawek Altenstein, Andrea Lohse, Stefan Hetzer, Anja Schneider, Klaus Fliessbach, Okka Kimmich, Ina R. Vogt, Jens Wiltfang, Claudia Bartels, Björn H. Schott, Niels Hansen, Peter Dechent, Katharina Buerger, Daniel Janowitz, Robert Perneczky, Boris-Stephan Rauchmann, Stefan Teipel, Ingo Kilimann, Doreen Goerss, Christoph Laske, Matthias H. Munk, Carolin Sanzenbacher, Petra Hinderer, Klaus Scheffler, Annika Spottke, Nina Roy-Kluth, Falk Lüsebrink, Katja Neumann, Frank Jessen, Stefanie Schreiber, Emrah Düzel, Gabriel Ziegler\\nmedRxiv 2023.09.25.23296088; doi: https://doi.org/10.1101/2023.09.25.23296088 \\n Add to Selected Citations\\n\\n\\n\\nComparative performance of COVID-19 test methods in healthcare workers during the omicron wave \\nEmma Catherine Tornberg, Alexander Tomlinson, Nicholas T.T. Oshiro, Esraa Derfalie, Marcel Curlin\\nmedRxiv 2023.09.25.23296099; doi: https://doi.org/10.1101/2023.09.25.23296099 \\n Add to Selected Citations\\n\\n\\n\\nEnhancing HIV testing yield in southern Mozambique: the effect of a Ministry of Health training module in targeted provider-initiated testing and counselling \\nAnna Saura Lazaro, Sheila Fernandez Luis, Tacilta Nhampossa, Laura Fuente Soro, Elisa Lopez Varela, Edson Bernardo, Orvalho Augusto, Teresa Sanchez, Paula Vaz, Stanley C. Wei, Peter Kerndt, Nelly Honwana, Peter Young, Guita Amane, Fernando Boene, Denise Naniche\\nmedRxiv 2023.09.25.23296036; doi: https://doi.org/10.1101/2023.09.25.23296036 \\n Add to Selected Citations\\n\\n\\n\\nMetabolic Insights into Iron Deposition in Relapsing-Remitting Multiple Sclerosis via 7T Magnetic Resonance Spectroscopic Imaging \\nAlexandra Lipka, Wolfgang Bogner, Assunta Dal-Bianco, Gilbert J. Hangel, Paulus S. Rommer, Bernhard Strasser, Stanislav Motyka, Lukas Hingerl, Thomas Berger, Fritz Leutmezer, Stephan Gruber, Siegfried Trattnig, Eva Niess\\nmedRxiv 2023.03.28.23287856; doi: https://doi.org/10.1101/2023.03.28.23287856 \\n Add to Selected Citations\\n\\n\\n\\nThe Impact of Intermittent Hypoxemia on Type 1 Retinopathy of Prematurity in Preterm Infants \\nSamaneh Rabienia Haratbar, Li Chen, Qiang Cheng, Dara Singh, Faraneh Fathi, Mehrana Mohtasebi, Xuhui Liu, Abhijit Patwardhan, Prasad Bhandary, Henrietta Bada, Guoqiang Yu, Elie Abujawdeh\\nmedRxiv 2023.09.25.23295922; doi: https://doi.org/10.1101/2023.09.25.23295922 \\n Add to Selected Citations\\n\\n\\n\\nAnalysis of a Pilot Study Delivering Cancer Survivorship Education to Community Healthcare Professionals Utilizing the Project ECHO model \\nAshley Pariser, Kevin N Johns, Dena Champion, Andrea Roberts, Susan Fugett, Erin Holley, Candice Schreiber, Carolyn J Presley, Jalyn Todd, Andrew Honeychuck, Katherine Hunt, Yurong Lu, Bhuvaneswari Ramaswamy, Seuli Bose Brill\\nmedRxiv 2023.09.25.23296103; doi: https://doi.org/10.1101/2023.09.25.23296103 \\n Add to Selected Citations\\n\\n\\n\\nAntimicrobial susceptibility testing reveals reduced susceptibility to azithromycin and other antibiotics in Legionella pneumophila serogroup 1 isolates from Portugal \\nCorrado Minetti, Rachael Barton, Caitlin Farley, Brad Owen Spiller, Raquel Esaguy Rodrigues, Paulo Goncalves\\nmedRxiv 2023.09.25.23296014; doi: https://doi.org/10.1101/2023.09.25.23296014 \\n Add to Selected Citations\\n\\n\\n\\nPharmacokinetic effects of a single-dose nutritional ketone ester supplement on brain ketone and glucose metabolism in alcohol use disorder \\nXinyi Li, Anthony J Young, Lais S Pereira-Rufino, Zhenhao Shi, Juliana I Byanyima, Sianneh Vesslee, Rishika Reddy, Timothy Pond, Mark Elliott, Ravinder Reddy, Robert K Doot, Jan-Willem van der Veen, Henry Kranzler, Jacob G Dubroff, Corinde E Wiers\\nmedRxiv 2023.09.25.23296090; doi: https://doi.org/10.1101/2023.09.25.23296090 \\n Add to Selected Citations\\n\\n\\n\\nSomatic mutations reveal hyperactive Notch signaling and racial disparities in prurigo nodularis \\nAhmad Rajeh, Hannah L Cornman, Anuj Gupta, Mindy D Szeto, Anusha Kambala, Olusola Oladipo, Varsha Parthasarathy, Junwen Deng, Sarah Wheelan, Thomas Pritchard, Madan M Kwatra, Yevgeniy R Semenov, Alexander Gusev, Srinivasan Yegnasubramanian, Shawn G Kwatra\\nmedRxiv 2023.09.25.23295810; doi: https://doi.org/10.1101/2023.09.25.23295810 \\n Add to Selected Citations\\n\\n\\n\\nNumber of children and mid- to later-life cognitive function and cognitive impairment in rural South Africa: Evidence from \"Health and Aging in Africa: A Longitudinal Study of an INDEPTH Community in South Africa\" (HAALSI) \\nMeredith L Phillips, Lindsay C Kobayashi, Rishika Chakraborty, Ryan Wagner, Nomsa Mahlalela, Joshua W Brown, Jaroslaw Harezlak, Christina Ludema, Molly Rosenberg\\nmedRxiv 2023.09.25.23296101; doi: https://doi.org/10.1101/2023.09.25.23296101 \\n Add to Selected Citations\\n\\n\\n\\nPrognostic Insights from Longitudinal Multicompartment Study of Host-Microbiota Interactions in Critically Ill Patients. \\nGeorgios D Kitsios, Khaled Sayed, Adam Fitch, Haopu Yang, Noel Britton, Faraaz Shah, William Bain, John Evankovich, Shulin Qin, Xiaohong Wang, Kelvin Li, Asha Patel, Yingze Zhang, Josiah Radder, Charles Dela Cruz, Daniel Okin, Ching Ying Huang, Daria van Tyne, Panayiotis V Benos, Barbara Methe, Peggy Lai, Alison Morris, Bryan McVerry\\nmedRxiv 2023.09.25.23296086; doi: https://doi.org/10.1101/2023.09.25.23296086 \\n Add to Selected Citations\\n\\n\\n\\nStrong early impact of letrozole on ovulation induction outperforms clomiphene citrate in polycystic ovary syndrome: a systematic review with meta-analysis \\nRita Zsuzsanna Vajna, András Mihály Géczi, Fanni Adél Meznerics, Nándor Ács, Péter Hegyi, Emma Zoé Feig, Péter Fehérvári, Szilvia Kiss-Dala, Szabolcs Várbíró, Judit Réka Hetthessy, Levente Sára\\nmedRxiv 2023.09.25.23296113; doi: https://doi.org/10.1101/2023.09.25.23296113 \\n Add to Selected Citations\\n\\n\\n\\nForecasting drug overdose mortality by age in the United States at the national and county levels \\nLucas Böttcher, Tom Chou, Maria  R. D\\'Orsogna\\nmedRxiv 2023.09.25.23296097; doi: https://doi.org/10.1101/2023.09.25.23296097 \\n Add to Selected Citations\\n\\n\\n\\nBarriers and facilitators to women general practitioners career progression, a systematic review \\nLaura A Jefferson, Elin Webster, Su Golder, Katie Barnett, Nicola Greenwood, Veronica Dale, Karen Bloor\\nmedRxiv 2023.09.26.23296133; doi: https://doi.org/10.1101/2023.09.26.23296133 \\n Add to Selected Citations\\n\\n\\n\\nHousehold income does not affect the pleiotropy of schizophrenia genetic liability with mental and physical health outcomes \\nManuela Kouakou, Brenda Cabrera Mendoza, Gita A Pathak, Tyrone Cannon, Renato Polimanti\\nmedRxiv 2023.09.25.23296085; doi: https://doi.org/10.1101/2023.09.25.23296085 \\n Add to Selected Citations\\n\\n\\n\\nDietary Assessment and Prevention of Hypertension in Nigeria: Protocol for a Retrospective Cross-sectional Study for the Development and Validation of a Food Frequency Questionnaire for Clinical Use. \\nNimisoere  Peace Batubo, J  Bernadette Moore, Michael  A Zulyniak\\nmedRxiv 2023.09.25.23296109; doi: https://doi.org/10.1101/2023.09.25.23296109 \\n Add to Selected Citations\\n\\n\\n\\nMindWatch: A Smart Cloud-based AI solution for Suicide Ideation Detection leveraging Large Language Models \\nRuna Bhaumik, Vineet Srivastava, Arash Jalali, Shanta Ghosh, Ranganathan Chandrasekaran\\nmedRxiv 2023.09.25.23296062; doi: https://doi.org/10.1101/2023.09.25.23296062 \\n Add to Selected Citations\\n\\n\\n\\nHigh-level anxiety is associated with worse clinical symptoms and aberrant brain networks in Parkinson\\'s disease \\nZhichun Chen, Guanglu Li, Liche Zhou, Lina Zhang, Jun Liu\\nmedRxiv 2023.09.25.23296076; doi: https://doi.org/10.1101/2023.09.25.23296076 \\n Add to Selected Citations\\n\\n\\n\\n\"Tranq-Dope\" Overdose and Mortality: Lethality Induced by Fentanyl and Xylazine \\nMark Smith, Samantha Biancorosso, Jacob Camp, Salome Hailu, Alexandra Johansen, Mackenzie Morris, Hannah Carlson\\nbioRxiv 2023.09.25.559379; doi: https://doi.org/10.1101/2023.09.25.559379 \\n Add to Selected Citations\\n\\n\\n\\nTenofovir Activation is Diminished in the Brain and Liver of Creatine Kinase Brain-Type Knockout Mice \\nColten D. Eberhard, Eric P. Mosher, Namandje N. Bumpus, Benjamin C. Orsburn\\nbioRxiv 2023.09.25.559370; doi: https://doi.org/10.1101/2023.09.25.559370 \\n Add to Selected Citations\\n\\n\\n\\nAugmenting complex and dynamic performance through mindfulness-based cognitive training: an evaluation of training adherence, trait mindfulness, personality and resting-state EEG. \\nChloe Dziego, Ina Bornkessel-Schlesewsky, Matthias Schlesewsky, Ruchi Sinha, Maarten  A Immink, Zachariah  R Cross\\nbioRxiv 2023.09.24.559207; doi: https://doi.org/10.1101/2023.09.24.559207 \\n Add to Selected Citations\\n\\n\\n\\nResuscitating Sleeping Beauties: reviving a six-hundred-year-old amoeba and endosymbiont. \\nIssam Hasni, Mamadou Lamine Tall, Matthieu Le Bailly, Philippe Colson, Anthony Levasseur, Didier Raoult, Bernard La Scola, Michel Drancourt\\nbioRxiv 2023.09.22.558946; doi: https://doi.org/10.1101/2023.09.22.558946 \\n Add to Selected Citations\\n\\n\\n\\nFARMS: Framework for Animal and Robot Modeling and Simulation \\nJonathan Arreguit, Shravan Tata Ramalingasetty, Auke Jan Ijspeert\\nbioRxiv 2023.09.25.559130; doi: https://doi.org/10.1101/2023.09.25.559130 \\n Add to Selected Citations\\n\\n\\n\\nAlteration of mechanical stresses in the murine brain by age and hemorrhagic stroke \\nSiyi Zheng, Rohin Banerji, Rob LeBourdais, Sue Zhang, Eric Dubois, Timothy M O\\'Shea, Hadi Nia\\nbioRxiv 2023.09.25.559368; doi: https://doi.org/10.1101/2023.09.25.559368 \\n Add to Selected Citations\\n\\n\\n\\nTemporal interference stimulation disrupts spike timing in the primate brain \\nPedro G. Vieira, Matthew R. Krause, Christopher C. Pack\\nbioRxiv 2023.09.25.559340; doi: https://doi.org/10.1101/2023.09.25.559340 \\n Add to Selected Citations\\n\\n\\n\\nA suite of engineered mice for interrogating psychedelic drug actions \\nYi-ting Chiu, Ariel Deutch, Wei Wang, Gavin P Schmitz, Karen Huang, Dewran Kocak, Pierre Lorach, Kasey Bowye, Bei Liu, Noah Sciaky, Kunjie Hua, Chongguang Chen, Sarah Mott, Jesse Niehaus, Jeff DiBerto, Justin English, Jessica Walsh, Gregory Scherrer, Melissa Herman, Zhu Wu, William C Wetsel, Bryan L Roth\\nbioRxiv 2023.09.25.559347; doi: https://doi.org/10.1101/2023.09.25.559347 \\n Add to Selected Citations\\n\\n\\n\\nImpact of COVID-19 Pandemic on Sleep Including HRV and Physical Activity as Mediators: A Causal ML Approach \\nElahe Khatibi, Mahyar Abbasian, Iman Azimi, Sina Labbaf, Mohammad Feli, Jessica Borelli, Nikil Dutt, Amir Rahmani\\nmedRxiv 2023.06.08.23291008; doi: https://doi.org/10.1101/2023.06.08.23291008 \\n Add to Selected Citations\\n\\n\\n\\nMedical history predicts phenome-wide disease onset and enables the rapid response to emerging health threats \\nJakob Steinfeldt, Benjamin Wild, Thore Manuel Buergel, Maik Pietzner, Julius Valten Upmeier zu Belzen, Andre Vauvelle, Stefan Hegselmann, Spiros Denaxas, Harry Hemingway, Claudia Langenberg, Ulf Landmesser, John Deanfield, Roland Eils\\nmedRxiv 2023.03.10.23286918; doi: https://doi.org/10.1101/2023.03.10.23286918 \\n Add to Selected Citations\\n\\n\\n\\nHeartbeat-evoked neural response abnormalities in generalized anxiety disorder during peripheral adrenergic stimulation \\nCharles Verdonk, Adam R. Teed, Evan J. White, Xi Ren, Jennifer L. Stewart, Martin P. Paulus, Sahib S. Khalsa\\nmedRxiv 2023.06.09.23291166; doi: https://doi.org/10.1101/2023.06.09.23291166 \\n Add to Selected Citations\\n\\n\\n\\nPrevalence and factors associated with abdominal and total obesity among low-income urban population of Kiambu County, Kenya. \\nGrace  Wambura Mbuthia, James Mwangi, James  Odhiambo Oguta, Catherine Akoth, Karani Magutah, Rosemary Kawira, Caroline Nyariki, Nickson Kimutai, Agnes Kinyua, Stephen  T. McGarvey\\nmedRxiv 2023.08.08.23293819; doi: https://doi.org/10.1101/2023.08.08.23293819 \\n Add to Selected Citations\\n\\n\\n\\nThe shared genetic risk architecture of neurological and psychiatric disorders: a genome-wide analysis \\nOlav Bjerkehagen Smeland, Gleda Kutrolli, Shahram Bahrami, Vera Fominykh, Nadine Parker, Guy Hindley, Linn Rodevand, Piotr Jaholkowski, Markos Tesfaye, Pravesh Parekh, Torbjorn Elvsaashagen, Andrew David Grotzinger, The International Multiple Sclerosis Genetics Consortium (IMSGC), The International Headache Genetics Consortium (IHGC), Nils Eiel Steen, Dennis van der Meer, Kevin S O\\'Connell, Srdjan David Djurovic, Anders Dale, Alexey Shadrin, Oleksandr Frei, Ole Andreassen\\nmedRxiv 2023.07.21.23292993; doi: https://doi.org/10.1101/2023.07.21.23292993 \\n Add to Selected Citations\\n\\n\\n\\nToward building a master health facility list database in Senegal: approaches and outputs from triangulating secondary health facility data \\nDaouda M Gueye, Alioune Badara Ly, Babacar Gueye, Papa Ibrahima Ndour, Nancy Fullman, Patrick Y Liu, Khadim Mbaye, Aliou Diallo, Ibrahima Diatta, Saly Amos Diatta, Mouhamadou Moustapha Mane, Gloria Ikilezi, Moussa Sarr\\nmedRxiv 2023.05.22.23290283; doi: https://doi.org/10.1101/2023.05.22.23290283 \\n Add to Selected Citations\\n\\n\\n\\nUndergraduates\\' reactions to errors mediates the association between growth mindset and study strategies \\nAnastasia Chouvalova, Anisha S Navlekar, Devin J Mills, Mikayla Adams, Sami Daye, Fatima De Anda, Lisa B Limeri\\nbioRxiv 2023.09.25.559345; doi: https://doi.org/10.1101/2023.09.25.559345 \\n Add to Selected Citations\\n\\n\\n\\nHarnessing the endogenous Type I-C CRISPR-Cas system for genome editing in Bifidobacterium breve \\nXiao Han, Lulu Chang, Haiqin Chen, jianxin Zhao, Fengwei Tian, R. Paul Ross, Catherine Stanton, Douwe van Sinderen, Wei Chen, Bo Yang\\nbioRxiv 2023.09.25.559376; doi: https://doi.org/10.1101/2023.09.25.559376 \\n Add to Selected Citations\\n\\n\\n\\nHidden HCN channels permit pathway-specific synaptic amplification in L2/3 pyramidal neurons \\nViktor Janos Olah, Jing Wu, Leonard K Kaczmarek, Matthew JM Rowan\\nbioRxiv 2023.09.25.559198; doi: https://doi.org/10.1101/2023.09.25.559198 \\n Add to Selected Citations\\n\\n\\n\\nPro-resolution lipid mediator maresin-1 ameliorates inflammation, promotes neuroprotection, and prevents disease progression in experimental models of multiple sclerosis \\nInsha Zahoor, Mohammad Nematullah, Mohammad Ejaz Ahmed, Mena Fatma, Sajad Mir, Kamesh Ayasolla, Mirela Cerghet, Suresh Palaniyandi, Yang Mao-Draayer, Ramandeep Rattan, Shailendra Giri\\nbioRxiv 2023.09.25.559216; doi: https://doi.org/10.1101/2023.09.25.559216 \\n Add to Selected Citations\\n\\n\\n\\nA rotational velocity estimate constructed through visuomotor competition updates the fly\\'s neural compass \\nBrad K Hulse, Angel Stanoev, Daniel B Turner-Evans, Johannes Seelig, Vivek Jayaraman\\nbioRxiv 2023.09.25.559373; doi: https://doi.org/10.1101/2023.09.25.559373 \\n Add to Selected Citations\\n\\n\\n\\nThe ghost of selection past: evolution and conservation relevance of the kākāpō color polymorphism \\nLara Urban, Anna W Santure, Lydia Uddstrom, Andrew Digby, Deidre Vercoe, Daryl Eason, Jodie Crane, Kakapo Recovery Team, Matthew J Wylie, Tane Davis, Marissa F LeLec, Joseph Guhlin, Simon Poulton, Jon Slate, Alana Alexander, Patricia Fuentes-Cross, Peter K Dearden, Neil J Gemmell, Farhan Azeem, Marvin Weyland, Harald G L Schwefel, Cock van Oosterhout, Hernan E Morales\\nbioRxiv 2023.09.25.559367; doi: https://doi.org/10.1101/2023.09.25.559367 \\n Add to Selected Citations\\n\\n\\n\\nUntargeted proteomics identifies plant substrates of the bacterial-derived ADP-ribosyltransferase AvrRpm1 \\nSimranjit Kaur, Thomas Colby, Domenika Thieme, Carsten Proksch, Susanne Matschi, Ivan Matić, Lennart Wirthmueller\\nbioRxiv 2023.09.25.558804; doi: https://doi.org/10.1101/2023.09.25.558804 \\n Add to Selected Citations\\n\\n\\n\\nVentral CA1 Population Codes for Anxiety \\nSean Chih-Hsiung Lim, Stefano Fusi, Rene Hen\\nbioRxiv 2023.09.25.559358; doi: https://doi.org/10.1101/2023.09.25.559358 \\n Add to Selected Citations\\n\\n\\n\\nBioXTAS RAW 2: new developments for a free open-source program for small angle scattering data reduction and analysis \\nJesse B Hopkins\\nbioRxiv 2023.09.25.559353; doi: https://doi.org/10.1101/2023.09.25.559353 \\n Add to Selected Citations\\n\\n\\n\\nThe Application of Label-free Detection Using a Tapered Optical Fiber System for Head and Neck Cancer and Infectious Biomolecules \\nCasey Collet, Cong Deng, Chaminda Ranathunga, Partha P Banerjee, Uttam Sinha\\nbioRxiv 2023.09.25.559133; doi: https://doi.org/10.1101/2023.09.25.559133 \\n Add to Selected Citations\\n\\n\\n\\nMetabolic health tracking using Ultrahuman M1 continuous glucose monitoring platform in non- and pre-diabetic Indians: a multi-armed observational study \\nMonik Chaudhry, Mohit Kumar, Vatsal Singhal, Bhuvan Srinivasan\\nmedRxiv 2023.09.20.23295642; doi: https://doi.org/10.1101/2023.09.20.23295642 \\n Add to Selected Citations\\n\\n\\n\\nContext-aware single-cell multiome approach identified cell-type specific lung cancer susceptibility genes \\nErping Long, Jinhu Yin, Ju Hye Shin, Yuyan Li, Alexander Kane, Harsh Patel, Thong Luong, Jun Xia, Younghun Han, Jinyoung Byun, Tongwu Zhang, Wei Zhao, Maria Teresa Landi, Nathaniel Rothman, Qing Lan, Yoon Soo Chang, Fulong Yu, Christopher Amos, Jianxin Shi, Jin Gu Lee, Eun Young Kim, Jiyeon Choi\\nbioRxiv 2023.09.25.559336; doi: https://doi.org/10.1101/2023.09.25.559336 \\n Add to Selected Citations\\n\\n\\n\\nProteinFlow: a Python Library to Pre-Process Protein Structure Data for Deep Learning Applications \\nElizaveta Kozlova, Arthur Valentin, Aous Khadhraoui, Daniel Nakhaee-Zadeh Gutierrez\\nbioRxiv 2023.09.25.559346; doi: https://doi.org/10.1101/2023.09.25.559346 \\n Add to Selected Citations\\n\\n\\n\\nNON-ENDOSCOPIC ESOPHAGEAL SAMPLING DEVICE AND BIOMARKER PANEL FOR DETECTION OF BARRETT′S ESOPHAGUS (BE) AND ESOPHAGEAL ADENOCARCINOMA (EAC) \\nHelen R Moinova, Suman Verma, John Dumot, Ashley Faulx, Prasad G Iyer, Marcia Irene Canto, Jean S Wang, Nicholas J Shaheen, Prashanthi N Thota, Lishan Aklog, Joseph E Willis, Sanford D Markowitz, Amitabh Chak\\nmedRxiv 2023.06.06.23291048; doi: https://doi.org/10.1101/2023.06.06.23291048 \\n Add to Selected Citations\\n\\n\\n\\nThe impact of size on middle-ear sound transmission in elephants, the largest terrestrial mammal \\nCaitlin E O\\'Connell-Rodwell, Jodie L Berezin, Anbuselvan Dharmarajan, Michael E Ravicz, Yihan Hu, Xiying Guan, Kevin N O\\'Connor, Sunil Puria\\nbioRxiv 2023.09.25.559337; doi: https://doi.org/10.1101/2023.09.25.559337 \\n Add to Selected Citations\\n\\n\\n\\nCellPolaris: Decoding Cell Fate through Generalization Transfer Learning of Gene Regulatory Networks \\nGuihai Feng, Xin Qin, Jiahao Zhang, Wuliang Huang, Yiyang Zhang, Wentao Cui, Shirui Li, Yao Chen, Wenhao Liu, Yao Tian, Yana Liu, Jingxi Dong, Ping Xu, Zhenpeng Man, Guole Liu, Zhongming Liang, Xinlong Jiang, Xiaodong Yang, Pengfei Wang, Ge Yang, Hongmei Wang, Xuezhi Wang, Ming-Han Tong, Yuanchun Zhou, Shihua Zhang, Yiqiang Chen, Yong Wang, Xin Li\\nbioRxiv 2023.09.25.559244; doi: https://doi.org/10.1101/2023.09.25.559244 \\n Add to Selected Citations\\n\\n\\n\\nLocation of joint involvement differentiates Rheumatoid arthritis into different clinical subsets \\nTjardo Daniel Maarseveen, Marc P Maurits, Nils Steinz, Sytske Anne Bergstra, Bianca B.M. Boxma-de Klerk, Annette H. M. Van der Helm-van Mil, Cornelia F. Allaart, Marcel Reinders, Thomas W.J. Huizinga, Erik B van den Akker, Rachel Knevel\\nmedRxiv 2023.09.19.23295482; doi: https://doi.org/10.1101/2023.09.19.23295482 \\n Add to Selected Citations\\n\\n\\n\\nCalorimetric Analysis Using DNA Thermal Stability to Determine Protein Concentration \\nMatthew W Eskew, Patrick W Reardon, Albert S Benight\\nbioRxiv 2023.09.25.559360; doi: https://doi.org/10.1101/2023.09.25.559360 \\n Add to Selected Citations\\n\\n\\n\\nThe grass that built the Central Highland of Madagascar: environmental niches and morphological diversity of Loudetia simplex \\nTchana Ornella Mariany Almary, Joseph White, Fitiavana Rasaminirina, Jacqueline Razanatsoa, Caroline Lehmann, Mijoro Rakotoarinivo, Helene Ralimanana, Maria S. Vorontsova, George P. Tiley\\nbioRxiv 2023.09.25.559324; doi: https://doi.org/10.1101/2023.09.25.559324 \\n Add to Selected Citations\\n\\n\\n\\nSex-Specific Association Between Genetic Risk of Psychiatric Disorders and Cardiovascular Diseases \\nJiayue-Clara Jiang, Kritika Singh, Lea K. Davis, Naomi R. Wray, Sonia Shah\\nmedRxiv 2022.10.08.22280805; doi: https://doi.org/10.1101/2022.10.08.22280805 \\n Add to Selected Citations\\n\\n\\n\\nlista-GEM: the genome-scale metabolic reconstruction of Lipomyces starkeyi \\nEduardo Almeida, Mauricio Ferreira, Wendel Silveira\\nbioRxiv 2023.09.25.559328; doi: https://doi.org/10.1101/2023.09.25.559328 \\n Add to Selected Citations\\n\\n\\n\\nWITHDRAWN: Complement C3-dependent glutamatergic synapse elimination in the developing hippocampus is region- and synapse-specific \\nEric W Salter, Gang Lei, Sun-Lim Choi, Liam T Ralph, Lijia Zhang, Fuzi Jin, Ashish Kadia, Junhui Wang, John Georgiou, Graham L Collingridge\\nbioRxiv 2020.05.20.106930; doi: https://doi.org/10.1101/2020.05.20.106930 \\n Add to Selected Citations\\n\\n\\n\\nNeuroanatomical, electrophysiological, and morphological characterization of melanin-concentrating hormone cells coexpressing cocaine- and amphetamine-regulated transcript \\nPersephone A Miller, Jesukhogie Williams-Ikhenoba, Aditi S Sankhe, Brendan Hoffe, Melissa J Chee\\nbioRxiv 2023.09.25.559204; doi: https://doi.org/10.1101/2023.09.25.559204 \\n Add to Selected Citations\\n\\n\\n\\nHaploinsufficiency underlies the neurodevelopmental consequences of SLC6A1/GAT-1 variants \\nDina Buitrago Silva, Marena Trinidad, Alicia Ljungdahl, Jezrael L. Revalde, Geoffrey Y. Berguig, William Wallace, Cory S. Patrick, Lorenzo Bomba, Michelle Arkin, Shan Dong, Karol Estrada, Keino Hutchinson, Jonathan H. LeBowitz, Avner Schlessinger, Katrine M. Johannesen, Rikke S. Moller, Kathleen M. Giacomini, Steven Froelich, Stephan J. Sanders, Arthur Wuster\\nmedRxiv 2022.03.09.22271804; doi: https://doi.org/10.1101/2022.03.09.22271804 \\n Add to Selected Citations\\n\\n\\n\\nGenerative Diffusion Models for Antibody Design, Docking, and Optimization \\nZhangzhi Peng, Chenchen Han, Xiaohan Wang, Dapeng Li, Fajiie Yuan\\nbioRxiv 2023.09.25.559190; doi: https://doi.org/10.1101/2023.09.25.559190 \\n Add to Selected Citations\\n\\n\\n\\nGenetic variability in pathways associates with pesticide-induced nervous system disease in the United States \\nMarissa Kosnik, Philipp Antczak, Peter Fantke\\nbioRxiv 2023.09.25.559342; doi: https://doi.org/10.1101/2023.09.25.559342 \\n Add to Selected Citations\\n\\n\\n\\nPanEffect: A pan-genome visualization tool for variant effects in maize \\nCarson M Andorf, Olivia C Haley, Rita K Hayford, John L Portwood II, Shatabdi Sen, Ethalinda KS Cannon, Jack M Gardiner, Margaret R Woodhouse\\nbioRxiv 2023.09.25.559155; doi: https://doi.org/10.1101/2023.09.25.559155 \\n Add to Selected Citations\\n\\n\\n\\nMassive lateral gene transfer under strain coexistence in the gut \\nNelson Frazao, Elsa Seixas, Hugo Barreto, Manolo Mischler, Daniela Guleresi, Isabel Gordo\\nbioRxiv 2023.09.25.559333; doi: https://doi.org/10.1101/2023.09.25.559333 \\n Add to Selected Citations\\n\\n\\n\\nSuppression of HIV and cocaine-induced neurotoxicity and inflammation by cell penetrable itaconate esters \\nB Celia Cui, Marina Aksenova, Aliaksandra Sikirzhytskaya, Diana Odhiambo, Elizaveta Korunova, Vitali Sikirzhytski, Hao Ji, Diego Altomare, Eugenia Broude, Norma Frizzell, Rosemarie Booze, Michael D Wyatt, Michael Shtutman\\nbioRxiv 2023.09.25.559154; doi: https://doi.org/10.1101/2023.09.25.559154 \\n Add to Selected Citations\\n\\n\\n\\nGlacier extinction homogenizes functional diversity \\nNora Khelidj, Marco Caccianiga, Bruno EL Cerabolini, Duccio Tampucci, Gianalberto Losapio\\nbioRxiv 2023.09.25.559332; doi: https://doi.org/10.1101/2023.09.25.559332 \\n Add to Selected Citations\\n\\n\\n\\nOxytocin, but not vasopressin, decreases willingness to harm others by promoting moral emotions of guilt and shame \\nXiaoxiao Zheng, Jiayuan Wang, Xi Yang Sr., Lei Xu, Benjamin Becker, Barbara J Sahakian, Trevor Robbins, Keith M M Kendrick\\nbioRxiv 2023.09.25.559242; doi: https://doi.org/10.1101/2023.09.25.559242 \\n Add to Selected Citations\\n\\n\\n\\nGenome-wide family prediction unveils molecular mechanisms underlying the regulation of agronomic traits in Urochloa ruziziensis \\nFelipe Bitencourt Martins, Alexandre Hild Aono, Aline da Costa Lima Moraes, Rebecca Caroline Ulbricht Ferreira, Mariane de Mendonça Vilela, Marco Pessoa-Filho, Mariana Rodrigues Motta, Rosângela Maria Simeão, Anete Pereira de Souza\\nbioRxiv 2023.09.25.559305; doi: https://doi.org/10.1101/2023.09.25.559305 \\n Add to Selected Citations\\n\\n\\n\\nTopography influences diurnal and seasonal microclimate fluctuations in hilly terrain environments of coastal California \\nAji John, Julian D Olden, Meagan F Oldfather, Matthew M Kling, David D Ackerly\\nbioRxiv 2023.09.22.559081; doi: https://doi.org/10.1101/2023.09.22.559081 \\n Add to Selected Citations\\n\\n\\n\\nNeuronal aging is associated with declined autophagy caused by reduced functionality of cGAS-STING signaling \\nSergio Passarella, Karina Brandes, Evelyn Dankert, Paola Cavalli, Andrea Kroeger, Daniela C. Dieterich, Peter Landgraf\\nbioRxiv 2023.09.25.559304; doi: https://doi.org/10.1101/2023.09.25.559304 \\n Add to Selected Citations\\n\\n\\n\\nNutritional condition drives spatial variation in physiology of Antarctic lipid-storing copepods \\nCory A Berger, Deborah K Steinberg, Ann M. Tarrant\\nbioRxiv 2023.09.25.559317; doi: https://doi.org/10.1101/2023.09.25.559317 \\n Add to Selected Citations\\n\\n\\n\\nSHIP1 modulation and proteome characterization of microglia \\nErpan Ahat, Zanyu Shi, Shaoyou Chu, Hai Hoang Bui, Emily R. Mason, Disha M Soni, Kenneth D. Roth, Michael James Chalmers, Adrian L. Oblak, Jie Zhang, Jesus A. Gutierrez, Timothy Richardson\\nbioRxiv 2023.09.25.559310; doi: https://doi.org/10.1101/2023.09.25.559310 \\n Add to Selected Citations\\n\\n\\n\\nA Novel Subtype of Myeloproliferative Neoplasms Driven by a MYC-Alarmin Axis \\nNicole D Vincelette, Xiaoqing Yu, Andrew T Kuykendall, Jungwon Moon, Siyuan Su, Chia-Ho Cheng, Rinzine Sammut, Tiffany N Razabdouski, Hai Vu Nguyen, Erika A Eksioglu, Onyee Chan, Najla Al Ali, Parth C Patel, Dae Hyun Lee, Shima Nakanishi, Renan B Ferreira, Qianxing Mo, Suzanne Cory, Harshani R Lawrence, Ling Zhang, Daniel J Murphy, Rami S Komrokji, Daesung Lee, Scott H Kaufmann, John L Cleveland, Seongseok Yun\\nbioRxiv 2023.09.25.559138; doi: https://doi.org/10.1101/2023.09.25.559138 \\n Add to Selected Citations\\n\\n\\n\\nDifferential mutation rates in plant meristematic layers \\nKirk R Amundson, Mohan Prem Anand Marimuthu, Oanh Nguyen, Konsam Sarika, Isabelle J DeMarco, Angelina Phan, Isabelle M Henry, Luca Comai\\nbioRxiv 2023.09.25.559363; doi: https://doi.org/10.1101/2023.09.25.559363 \\n Add to Selected Citations\\n\\n\\n\\nFrom buds to shoots: Insights into grapevine development from the Witch\\'s Broom bud sport \\nEleanore J Ritter, Peter Cousins, Michelle Quigley, Aidan Kile, Sunil Kumar Kenchanmane Raju, Daniel H Chitwood, Chad Niederhuth\\nbioRxiv 2023.09.25.559343; doi: https://doi.org/10.1101/2023.09.25.559343 \\n Add to Selected Citations\\n\\n\\n\\nNuclear lamin A-associated proteins are required for centromere assembly \\nAdriana Landeros, Destiny Wallace, Amit Rahi, Christine Magdongon, Praveen Suraneni, Mohammed Amin, Manas Chakraborty, Stephen Adam, Daniel Foltz, Dileep Varma\\nbioRxiv 2023.09.25.559341; doi: https://doi.org/10.1101/2023.09.25.559341 \\n Add to Selected Citations\\n\\n\\n\\nIRAK1 is a critical mediator of low molecular weight hyaluronic acid-induced stemness in high-grade serous ovarian cancer \\nDavid Standing, Prasad Dandawate, Sumedha Gunewardena, Obdulia Covarrubias-Zambrano, Katherine F Roby, Dineo Khabele, Andrea Jewell, Ossama Tawfik, Stefan H Bossmann, Andrew K Godwin, Scott J Weir, Roy A Jensen, Shrikant Anant\\nbioRxiv 2023.09.25.559366; doi: https://doi.org/10.1101/2023.09.25.559366 \\n Add to Selected Citations\\n\\n\\n\\nModeling Cardiomyocyte Signaling and Metabolism Predicts Genotype to Phenotype Mechanisms in Hypertrophic Cardiomyopathy \\nAli Khalilimeybodi, Padmini Rangamani\\nbioRxiv 2023.09.25.559356; doi: https://doi.org/10.1101/2023.09.25.559356 \\n Add to Selected Citations\\n\\n\\n\\n1.7 GHz long-term evolution radiofrequency electromagnetic field with efficient thermal control has no effect on the proliferation of different human cell types \\nJaeseong Goh, Dongwha Suh, Sangbong Jeon, Youngseung Lee, Nam Kim, Kiwon Song\\nbioRxiv 2023.09.25.559414; doi: https://doi.org/10.1101/2023.09.25.559414 \\n Add to Selected Citations\\n\\n\\n\\nOrganelle landscape analysis using a multi-parametric particle-based method \\nYoshitaka Kurikawa, Ikuko Koyama-Honda, Keiko Igarashi, Norito Tamura, Seiichi Koike, Noboru Mizushima\\nbioRxiv 2023.09.25.559448; doi: https://doi.org/10.1101/2023.09.25.559448 \\n Add to Selected Citations\\n\\n\\n\\nThe Topological Properties of the Protein Universe \\nChristian  Degnbol Madsen, Agnese Barbensi, Stephen Y Zhang, Lucy Ham, Alessia David, Douglas E.V. Pires, Michael P.H. Stumpf\\nbioRxiv 2023.09.25.559443; doi: https://doi.org/10.1101/2023.09.25.559443 \\n Add to Selected Citations\\n\\n\\n\\nQuantitative Profiling of Lysosomal pH Heterogeneity using Fluorescence Lifetime Imaging Microscopy \\nDinghuan Deng, Youchen Guan, Baiping Wang, Hui Zheng, Ayse Sena Mutlu, Meng Carla Wang\\nbioRxiv 2023.09.25.559395; doi: https://doi.org/10.1101/2023.09.25.559395 \\n Add to Selected Citations\\n\\n\\n\\nCo-evolution of spheroid fluidization and mechanical fracture in a morphogenic cascade \\nSelwin Kaixiang Wu, Celestine Z. Ho, Fuqiang Sun, Yuting Lou, Christina Bao-Xian Huang, Jingwei Xiao, Murat Shagirov, Ivan Yow, Jasmine Fei Li Chin, Suzie Verma, Alpha Yap, Yuan Lin, Tetsuya Hiraiwa, Boon Chuan Low\\nbioRxiv 2023.09.25.559247; doi: https://doi.org/10.1101/2023.09.25.559247 \\n Add to Selected Citations\\n\\n\\n\\nTRIP13 localizes to synapsed chromosomes and functions as a dosage-sensitive regulator of meiosis \\nJessica Y Chotiner, N. Adrian Leu, Fang Yang, Isabella G Cossu, Yongjuan Guan, Huijuan Lin, P. Jeremy Wang\\nbioRxiv 2023.09.25.559355; doi: https://doi.org/10.1101/2023.09.25.559355 \\n Add to Selected Citations\\n\\n\\n\\nNeurOne: High-performance Motor Unit-Computer Interface for the Paralyzed \\nDominik I. Braun, Daniela Souza de Oliveira, Patricia Bayer, Matthias Ponfick, Thomas Mehari Kinfe, Alessandro Del Vecchio\\nmedRxiv 2023.09.25.23295902; doi: https://doi.org/10.1101/2023.09.25.23295902 \\n Add to Selected Citations\\n\\n\\n\\nThe effect of single mutations in Zika virus envelope on escape from broadly neutralizing antibodies \\nLeslie Goo, Caroline Kikawa, Catiana H Cartwright-Acar, Jackson B Stuart, Maya Contreras, Lisa M Levoir, Matthew J. Evans, Jesse D. Bloom\\nbioRxiv 2023.09.13.557606; doi: https://doi.org/10.1101/2023.09.13.557606 \\n Add to Selected Citations\\n\\n\\n\\nAcupuncture at Sanyinjiao SP6 for patients with primary dysmenorrhea: Protocol for a systematic review and meta-analysis \\nXuemei Wang, Jiao Yang, Jing Liu, Yepeng Yang, Zhiyuan Zhang, Ping Xie\\nmedRxiv 2023.09.24.23296053; doi: https://doi.org/10.1101/2023.09.24.23296053 \\n Add to Selected Citations\\n\\n\\n\\nPseudoirreversible inhibition elicits persistent efficacy of a sphigosine-1-phosphate receptor-1 antagonist \\nYuya Maruyama, Yusuke Ohsawa, Takayuki Suzuki, Yuko Yamauchi, Kohsuke Ohno, Hitoshi Inoue, Akitoshi Yamamoto, Morimichi Hayashi, Yuji Okuhara, Wataru Muramatsu, Kano Namiki, Maki Miyauchi, Takahisa Miyao, Naho Hagiwara, Tatsuya Ishikawa, Kenta Horie, Mio Hayama, Nobuko Akiyama, Takatsugu Hirokawa, Taishin Akiyama\\nbioRxiv 2023.05.08.539826; doi: https://doi.org/10.1101/2023.05.08.539826 \\n Add to Selected Citations\\n\\n\\n\\nMicrobially-catalyzed conjugation of GABA and tyramine to bile acids \\nMichael Mullowney, Aretha Fiebig, Matthew Schnizlein, Mary McMillin, Jason Koval, David Rubin, Sushila Dalal, Mitch Sogin, Eugene Chang, Ashley Sidebottom, Sean Crosson\\nbioRxiv 2023.09.25.559407; doi: https://doi.org/10.1101/2023.09.25.559407 \\n Add to Selected Citations\\n\\n\\n\\nJMJD3 Deficiency in Midbrain Dopamine Neurons Disturbs Dopamine Synthesis and Aggravates Chronic Inflammatory Pain. \\nXi-Biao He, Fang Guo, Wei Zhang, Weidong Le, Yong Zheng, Yongjun Ma, Sang-Hun Lee, Huijing Wang, Yi Wu, Qinming Zhou\\nbioRxiv 2022.12.01.518657; doi: https://doi.org/10.1101/2022.12.01.518657 \\n Add to Selected Citations\\n\\n\\n\\nSpatial correspondence in relative space regulates serial dependence \\nJaeseob Lim, Sang-Hun Lee\\nbioRxiv 2023.06.27.546681; doi: https://doi.org/10.1101/2023.06.27.546681 \\n Add to Selected Citations\\n\\n\\n\\nEPIGENETIC MODULATION VIA THE C-TERMINAL TAIL OF H2A.Z \\nLaszlo Imre, Peter Nanasi Jr., Ibtissem Benhamza, Kata Nora Enyedi, Gabor Mocsar, Rosevalentine Bosire, Eva Hegedus, Erfaneh Firouzi Niaki, Agota Csoti, Zsuzsanna Darula, Eva Csosz, Szilard Poliska, Beata Scholtz, Gabor Mezo, Zsolt Bacso, Marc H.T. Timmers, Masayuki Kusakabe, Margit Balazs, Gyorgy Vamosi, Juan Ausio, Peter Cheung, Katalin Toth, David Tremethick, Masahiko Harata, Gabor Szabo\\nbioRxiv 2021.02.22.432230; doi: https://doi.org/10.1101/2021.02.22.432230 \\n Add to Selected Citations\\n\\n\\n\\nNew insights into functional divergence and adaptive evolution of uncultured bacteria in anammox community by complete genome-centric analysis \\nYi-Cheng Wang, Yanping Mao, Hui-Min Fu, Jin Wang, Xun Weng, Zi-Hao Liu, Xiao-Wei Xu, Peng Yan, Fang Fang, Jin-Song Guo, Yu Shen, You-Peng Chen\\nbioRxiv 2023.08.15.553441; doi: https://doi.org/10.1101/2023.08.15.553441 \\n Add to Selected Citations\\n\\n\\n\\nDiffSegR: An RNA-Seq data driven method for differential expression analysis using changepoint detection \\nArnaud Liehrmann, Etienne Delannoy, Alexandra Launay-Avon, Elodie Gilbault, Olivier Loudet, Benoit Castandet, Guillem Rigaill\\nbioRxiv 2023.06.05.543691; doi: https://doi.org/10.1101/2023.06.05.543691 \\n Add to Selected Citations\\n\\n\\n\\nMetabolic plasticity of serine metabolism is crucial for cGAS/STING-signalling and innate immune response to viral infections in the gut \\nBjoern Becker, Felix Wottawa, Mohamed Bakr, Eric Koncina, Lisa Mayr, Julia Kugler, Guang Yang, Samuel J Windross, Laura Neises, Neha Mishra, Danielle Harris, Florian Tran, Lina Welz, Julian Schwaerzler, Zoltan Banki, Stephanie T Stengel, Go Ito, Christina Kroetz, Olivia Coleman, Christian Jaeger, Dirk Haller, Soren R. Paludan, Richard Blumberg, Arthur Kaser, Luka Cicin-Sain, Stefan Schreiber, Elisabeth Letellier, Timon E. Adolph, Philip Rosenstiel, Johannes Meiser, Konrad Aden\\nbioRxiv 2022.05.17.492340; doi: https://doi.org/10.1101/2022.05.17.492340 \\n Add to Selected Citations\\n\\n\\n\\nThe Social Role of Self Control \\nRobin I.M. Dunbar, Susanne Shultz\\nbioRxiv 2020.10.26.354852; doi: https://doi.org/10.1101/2020.10.26.354852 \\n Add to Selected Citations\\n\\n\\n\\nSingle-cell characterization of human GBM reveals regional differences in tumor-infiltrating leukocyte activation \\nPhilip Schmassmann, Julien Roux, Steffen Dettling, Sabrina Hogan, Tala Shekarian, Tomas A. Martins, Marie-Francoise Ritz, Sylvia Herter, Marina Bacac, Gregor Hutter\\nbioRxiv 2022.06.17.496574; doi: https://doi.org/10.1101/2022.06.17.496574 \\n Add to Selected Citations\\n\\n\\n\\nMicrofluidic interfaces for chronic bidirectional access to the brain. \\nSimone Marcigaglia, Robin De Plus, Charysse Vandendriessche, Marie-Lynn Cuypers, Jordi Cools, Luis Hoffman, Roosmarijn Vandenbroucke, Maarten Dewilde, Sebastian Haesler\\nbioRxiv 2023.09.22.558790; doi: https://doi.org/10.1101/2023.09.22.558790 \\n Add to Selected Citations\\n\\n\\n\\nLUZP1 regulates the assembly of stress fibers by promoting maturation of contractile actomyosin bundles \\nLiang Wang, Hoi Ying Tsang, Ziyi Yan, Sari Tojkander, Katarzyna Ciuba, Konstantin Kogan, Xiaonan Liu, Hongxia Zhao\\nbioRxiv 2023.09.08.556811; doi: https://doi.org/10.1101/2023.09.08.556811 \\n Add to Selected Citations\\n\\n\\n\\nA diversified, widespread microbial gene cluster encodes homologs of methyltransferases involved in methanogenesis. \\nDuncan James Kountz, Emily P. Balskus\\nbioRxiv 2023.07.31.551370; doi: https://doi.org/10.1101/2023.07.31.551370 \\n Add to Selected Citations\\n\\n\\n\\nBackWards - Unveiling the Brain\\'s Topographic Organization of Paraspinal Sensory Input \\nAlexandros Guekos, David M Cole, Monika A Doerig, Philipp Staempfli, Louis Schibli, Philipp A Schuetz, Petra Schweinhardt, Michael Lukas Meier\\nbioRxiv 2022.08.05.502912; doi: https://doi.org/10.1101/2022.08.05.502912 \\n Add to Selected Citations\\n\\n\\n\\nRandom genetic drift sets an upper limit on mRNA splicing accuracy in metazoans \\nFlorian Benitiere, Anamaria Necsulea, Laurent Duret\\nbioRxiv 2022.12.09.519597; doi: https://doi.org/10.1101/2022.12.09.519597 \\n Add to Selected Citations\\n\\n\\n\\nComparative Interactome Analysis of α-arrestin Families in Human and Drosophila \\nKyung-Tae Lee, Inez K. Pranoto, Soon-Young Kim, Hee-Joo Choi, Ngoc Bao To, Hansong Chae, Jeong-Yeon Lee, Jung-Eun Kim, Young V Kwon, Jin-Wu Nam\\nbioRxiv 2023.06.11.544486; doi: https://doi.org/10.1101/2023.06.11.544486 \\n Add to Selected Citations\\n\\n\\n\\nIdentification of FAM53C as a cytosolic-anchoring inhibitory binding protein of the kinase DYRK1A \\nYoshihiko Miyata, Eisuke Nishida\\nbioRxiv 2023.04.21.537794; doi: https://doi.org/10.1101/2023.04.21.537794 \\n Add to Selected Citations\\n\\n\\n\\nIntegrating biodiversity assessments into local conservation planning: the importance of assessing suitable data sources \\nThibaut Ferraille, Christian Kerbiriou, Charlotte Bigard, Fabien Claireau, John D. Thompson\\nbioRxiv 2023.05.09.539999; doi: https://doi.org/10.1101/2023.05.09.539999 \\n Add to Selected Citations\\n\\n\\n\\nAdaptive Functions of Structural Variants in Human Brain Development \\nWanqiu Ding, Xiangshang Li, Jie Zhang, Mingjun Ji, Mengling Zhang, Xiaoming Zhong, Yong Cao, Xiaoge Liu, Chunqion Li, Chunfu Xiao, Jiaxin Wang, Ting Li, Qing Yu, Fan Mo, Boya Zhang, Jianhuan Qi, Jie-Chun Yang, Juntian Qi, Lu Tian, Xinwei Xu, QI Peng, Wei-Zhen Zhou, Zhijin Liu, Aisi Fu, Xiuqing Zhang, Jian-jun Zhang, Yujie Sun, Baoyang Hu, Ni A. An, Li Zhang, Chuan-Yun Li\\nbioRxiv 2023.09.25.558917; doi: https://doi.org/10.1101/2023.09.25.558917 \\n Add to Selected Citations\\n\\n\\n\\nWhite matter tract microstructure, macrostructure, and associated cortical gray matter morphology across the lifespan \\nKurt Schilling, Jordan A Chad, Maxime Chamberland, Victor Nozais, Francois Rheault, Derek B Archer, Muwei Li, Yurui Gao, Leon Y Cai, Flavio Del\\'Acqua, Allen Newton, Daniel Moyer, John C Gore, Catherine Lebel, Bennett  A Landman\\nbioRxiv 2023.09.25.559330; doi: https://doi.org/10.1101/2023.09.25.559330 \\n Add to Selected Citations\\n\\n\\n\\nRegulation of the mouse ventral tegmental area by melanin-concentrating hormone \\nC Duncan Spencer, Persephone A Miller, Jesukhogie G Williams-Ikhenoba, Ralitsa G Nikolova, Melissa J Chee\\nbioRxiv 2023.09.25.559435; doi: https://doi.org/10.1101/2023.09.25.559435 \\n Add to Selected Citations\\n\\n \\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n View Selected Citations (0) \\n\\n\\n\\n Add All Citations Remove All Citations \\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nSubject Areas \\n\\n\\n\\n\\n\\n\\n\\nAll Articles \\n\\n\\n\\n\\n\\nAnimal Behavior and Cognition (4672)\\nBiochemistry (10338)\\nBioengineering (7658)\\nBioinformatics (26300)\\nBiophysics (13501)\\nCancer Biology (10672)\\nCell Biology (15412)\\nClinical Trials (138)\\nDevelopmental Biology (8487)\\nEcology (12806)\\nEpidemiology (2067)\\nEvolutionary Biology (16831)\\nGenetics (11382)\\nGenomics (15469)\\nImmunology (10601)\\nMicrobiology (25167)\\nMolecular Biology (10206)\\nNeuroscience (54383)\\nPaleontology (399)\\nPathology (1667)\\nPharmacology and Toxicology (2889)\\nPhysiology (4334)\\nPlant Biology (9235)\\nScientific Communication and Education (1586)\\nSynthetic Biology (2555)\\nSystems Biology (6773)\\nZoology (1461)\\n \\n\\n\\n\\n\\n\\n\\n\\n \\n  \\tRefine Search  \\n\\n\\n\\nArticle Type\\n Confirmatory Results 7\\n Contradictory Results 1\\n New Results 475\\n Regular Article 484\\n Withdrawn 1\\n\\n\\n\\nSubject\\n Addiction Medicine 2\\n Allergy and Immunology 1\\n Animal Behavior and Cognition 9\\n Biochemistry 15\\n Bioengineering 9\\n Bioinformatics 34\\n Biophysics 14\\n Cancer Biology 17\\n Cardiovascular Medicine 9\\n Cell Biology 37\\n Dermatology 1\\n Developmental Biology 13\\n Ecology 16\\n Endocrinology (including Diabetes Mellitus and Metabolic Disease) 3\\n Epidemiology 11\\n Evolutionary Biology 22\\n Gastroenterology 6\\n Genetic and Genomic Medicine 13\\n Genetics 6\\n Genomics 28\\n HIV/AIDS 3\\n Health Economics 2\\n Health Informatics 9\\n Health Policy 1\\n Health Systems and Quality Improvement 1\\n Hematology 1\\n Immunology 12\\n Infectious Diseases (except HIV/AIDS) 8\\n Intensive Care and Critical Care Medicine 6\\n Microbiology 72\\n Molecular Biology 28\\n Neurology 11\\n Neuroscience 104\\n Nutrition 1\\n Obstetrics and Gynecology 2\\n Oncology 8\\n Ophthalmology 1\\n Otolaryngology 2\\n Pathology 5\\n Pediatrics 2\\n Pharmacology and Therapeutics 3\\n Pharmacology and Toxicology 5\\n Physiology 5\\n Plant Biology 17\\n Primary Care Research 1\\n Psychiatry and Clinical Psychology 5\\n Public and Global Health 24\\n Radiology and Imaging 4\\n Rehabilitation Medicine and Physical Therapy 1\\n Rheumatology 1\\n Scientific Communication and Education 2\\n Sports Medicine 1\\n Synthetic Biology 7\\n Systems Biology 8\\n Transplantation 1\\n Zoology 2\\n Show More Show Less\\n\\n\\nSource\\n bioRxiv 484\\n medRxiv 145\\n\\n\\n\\n\\n\\n\\n\\n \\n\\n\\n\\n \\n \\n\\n \\n\\n\\n\\n\\n\\n\\n\\n\\n\\n'"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup = BeautifulSoup(response.text, 'html.parser')\n",
    "soup.text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "num_read = 10\n",
    "response = requests.get(f'https://api.biorxiv.org/details/biorxiv/2023-08-21/2023-08-28/{num_read}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'messages': [{'status': 'ok',\n",
       "   'interval': '2023-08-21:2023-08-28',\n",
       "   'cursor': '10',\n",
       "   'count': 100,\n",
       "   'count_new_papers': 947,\n",
       "   'total': 1299}],\n",
       " 'collection': [{'doi': '10.1101/2020.05.26.104687',\n",
       "   'title': 'Pictorial balance is a bottom-up aesthetic property mediated by eye movements. A model of a primitive visual operating system explains balance and visual properties of pictures',\n",
       "   'authors': 'Corwin, D. M.',\n",
       "   'author_corresponding': 'David M Corwin',\n",
       "   'author_corresponding_institution': 'David Corwin MD',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '18',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2020.05.26.104687.source.xml',\n",
       "   'abstract': 'The concept of pictorial balance as a visual center-of-mass effect originated at the beginning of the 20th century to explain why some paintings evoke feelings of unity and harmony. It had never been mentioned in art commentary of the previous 500 years and is an unsubstantiated but widely assumed illusion. However, the visual basis for this illusion exists, and it is the object of this paper to explain what balance is and what it says about the visual system. A computer model was created of an unnamed pictorial effect known to painters when a picture feels unusually unified and harmonious. It was first described by a 17th century critic Roger de Piles, and there are descriptions of it by renowned artists. It was thought by the author that this effect was due to the painting being in a state of perfect balance. Computer modeling found that it had bilateral luminance symmetry with a lighter lower half by [~]1.07 and that balance is based solely on luminosity. A study was done in which the identical picture in two different frames could seem different to subjects who viewed them in sequence; the sole difference being the total balance of the image. The study did not show that the subjects could see the effect but did show that the method of calculating balance using an algorithm was correct. It was found that low averaged balanced pairs could be distinguished from high averaged balanced pairs. This method views an organism as a collection of lights all moving together against a ground. This is divided by a vertical line into two equally luminous parts and further divided in half by a horizontal line. Four virtual lines are drawn parallel to the above two to create a picture. Light from each quadrant is spread to fill its respective quadrant, and the upper half is arbitrarily filtered to make it darker so that organisms illuminated from above would appear more balance. Balance is calculated by a center of quadrant luminance equation analogous to a center-of-mass calculation. It is suggested that object balance is a residual primitive visual processing system identifying organisms. An ecological explanation of the study is presented. Balance and the visual concepts involved can help painters create more visually attractive paintings if desired and for everyone to understand the neuro-aesthetic effects of paintings.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2020.07.13.195107',\n",
       "   'title': \"Active synthesis of type I collagen homotrimer in Dupuytren's fibrosis is unaffected by anti-TNF-α treatment\",\n",
       "   'authors': 'Williamson, K.; Lee, K. J.; Beamish, E. L.; Carter, A.; Gumbs, J. A.; Cooper, G.; Cheung, G.; Brown, D.; Pettitt, R.; Comerford, E. J.; Clegg, P. D.; Canty-Laird, E. G.',\n",
       "   'author_corresponding': 'Elizabeth G Canty-Laird',\n",
       "   'author_corresponding_institution': 'University of Liverpool',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2020.07.13.195107.source.xml',\n",
       "   'abstract': 'Dupuytrens disease is a common fibroproliferative disease of the palmar fascia of the hand with advanced cases treated surgically. Anti-tumour necrosis factor (TNF) injection has undergone phase 2 trials and may be effective in slowing early-stage disease progression. Here we sought to determine how new synthesis of type I collagen in Dupuytrens differs from normal palmar fascia samples and to analyse the role of TNF in aberrant collagen synthesis. Model non-fibrotic, but fibrous connective tissues, were used to analyse active type I collagen protein synthesis in development, ageing and degenerative disease, where it was restricted to early development and ruptured tissue. Dupuytrens tissue was shown to actively synthesise type I collagen, including abnormal type I collagen homotrimer. TNF- reduced COL1A2 gene expression only in the presence of serum in 2D cell culture and had opposing effects on collagen protein production in the presence or absence of serum. TNF- had only limited effects in 3D tendon-like constructs. Anti-TNF did not reduce type I collagen synthesis in 3D tendon-like constructs or prevent type I collagen homotrimer synthesis in Dupuytrens tissue. Hence, modulation of the TNF- pathway in Dupuytrens disease is unlikely to prevent the pathological collagen accumulation that is characteristic of fibrosis.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2020.08.07.240150',\n",
       "   'title': 'Egg size variation in the context of polyandry: a case study using long-term field data from snowy plovers',\n",
       "   'authors': 'Eberhart-Hertel, L. J.; Falcao Rodrigues, L.; Krietsch, J.; Eberhart-Hertel, A. G.; Cruz-Lopez, M.; Vazquez-Rojas, K. A.; Gonzalez-Medina, E.; Schroeder, J.; Küpper, C.',\n",
       "   'author_corresponding': 'Lourenço  Falcão Rodrigues',\n",
       "   'author_corresponding_institution': 'Universidad Autónoma de Madrid',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'evolutionary biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2020.08.07.240150.source.xml',\n",
       "   'abstract': 'Gamete size variation between the sexes is central to the concept of sex roles, however, to what extent gamete size variation within the sexes relates to sex role variation remains unclear. Comparative and theoretical studies suggest that, when clutch size is invariable, polyandry is linked to a reduction of egg size, while increased female-female competition for mates favors early breeding when females cannot monopolize multiple males. To understand whether and how breeding phenology, egg size, and mating behavior are related at the individual level, we studied the reproductive histories of 424 snowy plover females observed in the wild over a 15-year period. Egg size, but not polyandry, were highly repeatable for individual females. Consistent with theoretical predictions, we found that polyandrous females were the earliest breeders and that early clutches contained smaller eggs than clutches initiated later. Neither egg size nor mating behavior showed clear signs of an age-related deterioration, on the contrary, prior experience acquired either through age or local recruitment enabled females to nest early. Taken together, these results suggest that gamete size variation is not linked to mating behavior at the individual level, and, consequently, the adaptive potential of such variation appears to be limited.\\n\\nTeaser TextComparative avian analyses have linked female polyandry and sex-role reversal to the production of smaller eggs: the notion being that smaller eggs allow polyandrous females to lay multiple sequential clutches quickly. Our research, however, has found that the notion is more complex at the individual level. We found that females who started breeding early in the season were more likely to be polyandrous and produce smaller eggs. Experience was also found to give older and locally-raised females a competitive advantage over younger and inexperienced females vying for early breeding opportunities. Despite this, we found limited evidence that egg size reduction gives a competitive advantage in the scramble for mates. Instead, seasonal variations in egg size can likely be attributed to other factors, such as the survival of early-hatching chicks, resource limitations during egg production, and individual differences. We found that senescence had little impact on egg size or mating behavior. Future research should also examine how seasonality, age, and mating behavior impact the reproductive fitness of males, who provide more parental care than females in many sex role reversed systems. By studying these effects in concert, we can gain a deeper understanding of sex roles in natural populations.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2020.10.23.349860',\n",
       "   'title': \"Imaging α-synuclein pathologies in animal models and patients with Parkinson's and related diseases\",\n",
       "   'authors': 'Endo, H.; Ono, M.; Takado, Y.; Matsuoka, K.; Takahashi, M.; Tagai, K.; Kataoka, Y.; Hirata, K.; Takahata, K.; Seki, C.; Kokubo, N.; Fujinaga, M.; Mori, W.; Nagai, Y.; Mimura, K.; Kumata, K.; Kikuchi, T.; Shimozawa, A.; Mishra, S. K.; Yamaguchi, Y.; Shimizu, H.; Kakita, A.; Takuwa, H.; Shinotoh, H.; Shimada, H.; Kimura, Y.; Ichise, M.; Suhara, T.; Minamimoto, T.; Sahara, N.; Kawamura, K.; Zhang, M.-R.; Hasegawa, M.; Higuchi, M.',\n",
       "   'author_corresponding': 'Maiko  Ono',\n",
       "   'author_corresponding_institution': 'National Institutes for Quantum and Radiological Science and Technology',\n",
       "   'date': '2023-08-23',\n",
       "   'version': '5',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/23/2020.10.23.349860.source.xml',\n",
       "   'abstract': 'Deposition of -synuclein fibrils is implicated in Parkinsons disease (PD) and dementia with Lewy bodies (DLB), while in vivo detection of -synuclein pathologies in these illnesses has been challenging. Here, we have developed a small-molecule ligand, C05-05, for visualizing -synuclein deposits in the brains of living subjects. In vivo optical and positron emission tomography (PET) imaging of mouse and marmoset models demonstrated that C05-05 captured a dynamic propagation of fibrillogenesis along neural pathways followed by disruptions of these structures. High-affinity binding of 18F-C05-05 to -synuclein aggregates in human brain tissues was also proven by in vitro assays. Notably, PET-detectable 18F-C05-05 signals were intensified in the midbrains of PD and DLB patients as compared to healthy controls, providing the first demonstration of visualizing -synuclein pathologies in these illnesses. Collectively, we propose a new imaging technology offering neuropathology-based translational assessments of PD and allied disorders towards diagnostic and therapeutic research and development.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2020.10.25.352823',\n",
       "   'title': 'Label-free composition determination for biomolecular condensates with an arbitrarily large number of components',\n",
       "   'authors': 'McCall, P. M.; Kim, K.; Ruer-Gruss, M.; Peychl, J.; Guck, J.; Hyman, A. A.; Brugues, J.',\n",
       "   'author_corresponding': 'Jan  Brugues',\n",
       "   'author_corresponding_institution': 'Max Planck Institute',\n",
       "   'date': '2023-08-25',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/25/2020.10.25.352823.source.xml',\n",
       "   'abstract': 'Biomolecular condensates are membrane-less organelles made of multiple components, often including several distinct proteins and nucleic acids. However, current tools to measure condensate composition are limited and cannot capture this complexity quantitatively, as they either require fluorescent labels, which we show can perturb composition, or can distinguish only 1-2 components. Here, we describe a label-free method based on quantitative phase microscopy to measure the composition of condensates with an arbitrarily large number of components. We first validate the method empirically in binary mixtures, revealing sequence-encoded density variation and complex aging dynamics for condensates composed of full-length proteins. In simplified multi-component protein/RNA condensates, we uncover a regime of constant condensate density and a large range of protein:RNA stoichiometry when varying average composition. The unexpected decoupling of density and composition highlights the need to determine molecular stoichiometry in multi-component condensates. We foresee this approach enabling the study of compositional regulation of condensate properties and function.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.03.15.435343',\n",
       "   'title': 'SPIRO - the automated Petri plate imaging platform designed by biologists, for biologists',\n",
       "   'authors': 'Ohlsson, J. A.; Leong, J. X.; Elander, P. H.; Ballhaus, F.; Holla, S.; Dauphinee, A. N.; Johansson, J.; Lommel, M.; Hofmann, G.; Betner, S.; Sandgren, M.; Schumacher, K.; Bozhkov, P. V.; Minina, E. A.',\n",
       "   'author_corresponding': 'Alyona E.A. Minina',\n",
       "   'author_corresponding_institution': 'Department of Molecular Sciences, Uppsala BioCenter, Swedish University of Agricultural Sciences and Linnean Center for Plant Biology, P.O. Box 7015, Uppsala, S',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'plant biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2021.03.15.435343.source.xml',\n",
       "   'abstract': 'Manual imaging of plants, fungi and bacterial colonies grown on Petri plates for phenotyping assays is common, despite the procedures being time-consuming and laborious. The main rea-son for this is the limited availability of automated phenotyping facilities, whereas constructing a custom automated solution can be a daunting task for biologists.\\n\\nHere, we describe SPIRO, the Smart Plate Imaging Robot, an automated platform that acquires time-lapse photos of up to four vertically oriented Petri plates in a single experiment. SPIRO is catered specifically to biologists needs, requiring no engineering or programming expertise for assembly and operation. SPIRO has a small footprint optimized for standard incubators, it al-lows remote access to data without interfering with sample growth and is equipped with an LED light source for imaging in the dark. Importantly, the robot can be easily customized for specific uses, as all information about SPIRO is released under open-source licenses. SPIROs excellent image quality is suitable for automated image processing, which we demonstrate on the example of seed germination and root growth assays. We illustrate the benefits of SPIRO in proof-of-concept experiments which yielded a novel insight on the role of plant autophagy in breaking seed dormancy.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.04.03.438339',\n",
       "   'title': 'From treadmill to trails: predicting performance of runners',\n",
       "   'authors': 'Crowell, B.',\n",
       "   'author_corresponding': 'Benjamin  Crowell',\n",
       "   'author_corresponding_institution': 'Fullerton College',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'physiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2021.04.03.438339.source.xml',\n",
       "   'abstract': 'Previous laboratory studies have measured the energetic costs to humans of running at uphill and downhill slopes on a treadmill. This work investigates the extension of those results to the prediction of relative performance of athletes running on flat, hilly, or very mountainous outdoor courses. Publicly available race results in the Los Angeles area provided a set of 109,000 times, with 2200 runners participating in more than one race, so that their times could be compared under different conditions. I compare with the results of a traditional model in which the only parameters considered are total distance and elevation gain. Both the treadmill-based model and the gain-based model have some shortcomings, leading to the creation of a hybrid model that combines the best features of each.\\n\\nAuthor summaryRunning a race on a road allows absolute measures of performance. Trail running, however, has traditionally been thought of as a sport in which the only valid comparison is between different runners competing on the same course on the same day. Even the exact measurement of distance is considered to be unimportant, since courses and conditions vary so much.\\n\\nAn extreme example is the relatively new genre of \"vertical\" races, in which runners race up a mountain. In a typical example, the competitors cover a horizontal distance of 5 km, while climbing about 1000 m. The winner in one such race had a time almost triple that expected for a state-champion high school runner in a 5k road race. Clearly no comparison can be made here without taking into account the amount of climbing.\\n\\nIn noncompetitive contexts, many runners venture onto mountain trails, lightly dressed and with little equipment, so that it becomes important to be able to anticipate whether they will have the endurance needed to be able to safely complete a planned route. Again, this is impossible without some model of the effect of hill climbing.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.04.20.440673',\n",
       "   'title': 'Modeling myosin with interacting linkages',\n",
       "   'authors': 'Omabegho, T.',\n",
       "   'author_corresponding': 'Tosan  Omabegho',\n",
       "   'author_corresponding_institution': 'None',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '7',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2021.04.20.440673.source.xml',\n",
       "   'abstract': 'In this study, I describe a model in which mechanical linkages dynamically interact in a stepwise and reversible manner, and use it to model the chemical cycle and lever arm action of the biomolecular motor myosin. Myosin is emulated using a series of multivalent chemical reactions between a linkage enzyme and four reactants: a cleaveable fuel, two cleavage products, and ligand. Geometric coupling between the fuel and ligand binding sites--an analog for negative allosteric coupling--allows reaction sequences similar to nucleotide exchange to take place that in turn drive the \"strokes\" of the machines lever arm. Cyclic chemical behavior is demonstrated by stochastic simulation, and mechanical activity by a series of logical arguments. I show how a reciprocal and nonreciprocal conformational cycle emerge from the allosteric rules designed to achieve chemical cycling, and how the non-reciprocal cycle can break directional symmetry along a track like structure. A dimeric construct is used to demonstrate how directed motion can be designed by inhibition of the reciprocal cycle and reinforcement of the non-reciprocal cycle, through allosteric feedback between the units of the dimer. By showing how the chemomechanical cycle of a biomolecular motor can be recreated with simple geometric and chemical principles, this work may help advance the rational design of allosteric mechanisms, and the development of synthetic molecular motors.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.06.09.447689',\n",
       "   'title': 'TRβ Agonism Induces Tumor Suppression and Enhances Drug Efficacy in Anaplastic Thyroid Cancer in Female Mice',\n",
       "   'authors': 'Gillis, N. E.; Cozzens, L. M.; Wilson, E. R.; Smith, N. M.; Tomczak, J. A.; Bolf, E. L.; Carr, F. E.',\n",
       "   'author_corresponding': 'Frances E Carr',\n",
       "   'author_corresponding_institution': 'University of Vermont, Larner College of Medicine',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'cancer biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2021.06.09.447689.source.xml',\n",
       "   'abstract': 'Thyroid hormone receptor beta (TR{beta}) is a recognized tumor suppressor in numerous solid cancers. The molecular signaling of TR{beta} has been elucidated in several cancer types through re-expression models. Remarkably, the potential impact of selective activation of endogenous TR{beta} on tumor progression remains largely unexplored. We used cell-based and in vivo assays to evaluate the effects of the TR{beta} agonist Sobetirome (GC-1) on a particularly aggressive and dedifferentiated cancer, anaplastic thyroid cancer (ATC). Here we report that GC-1 reduced the tumorigenic phenotype, decreased cancer stem-like cell populations, and induced re-differentiation of the ATC cell lines with different mutational backgrounds. Of note, this selective activation of TR{beta} amplified the effects of therapeutic agents in blunting the aggressive cell phenotype and stem-cell growth. In xenograft assays, GC-1 alone inhibited tumor growth and was as effective as the kinase inhibitor, Sorafenib. These results indicate that selective activation of TR{beta} not only induces a tumor suppression program de novo but enhances the effectiveness of anti-cancer agents revealing potential novel combination therapies for ATC and other aggressive solid tumors.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.07.13.452279',\n",
       "   'title': 'A Lethal Genetic Incompatibility between Naturally Hybridizing Species in Mitochondrial Complex I',\n",
       "   'authors': 'Moran, B. M.; Payne, C. Y.; Powell, D. L.; Iverson, E. N.; Donny, A. E.; Banerjee, S. M.; Langdon, Q. K.; Gunn, T. R.; Rodriguez-Soto, R. A.; Madero, A.; Baczenas, J. J.; Kleczko, K. M.; Liu, F.; Matney, R.; Singhal, K.; Leib, R. D.; Hernandez-Perez, O.; Corbett-Detig, R.; Frydman, J.; Gifford, C.; Schartl, M.; Havird, J. C.; Schumer, M.',\n",
       "   'author_corresponding': 'Benjamin Michael Moran',\n",
       "   'author_corresponding_institution': 'Stanford University',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'evolutionary biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2021.07.13.452279.source.xml',\n",
       "   'abstract': 'The evolution of reproductive barriers is the first step in the formation of new species and can help us understand the diversification of life on Earth. These reproductive barriers often take the form of \"hybrid incompatibilities,\" where alleles derived from two different species no longer interact properly in hybrids. Theory predicts that hybrid incompatibilities may be more likely to arise at rapidly evolving genes and that incompatibilities involving multiple genes should be common, but there has been sparse empirical data to evaluate these predictions. Here, we describe a mitonuclear incompatibility involving three genes in physical contact within respiratory Complex I in naturally hybridizing swordtail fish species. Individuals homozygous for specific mismatched protein combinations fail to complete embryonic development or die as juveniles, while those heterozygous for the incompatibility have reduced function of Complex I and unbalanced representation of parental alleles in the mitochondrial proteome. We find that the impacts of different genetic interactions on survival are non-additive, highlighting subtle complexity in the genetic architecture of hybrid incompatibilities. We document the evolutionary history of the genes involved, showing for the first time that an incompatibility has been transferred between species via hybridization. This work thus provides the first glimpse into the genetic architecture, physiological impacts, and evolutionary origin of a complex incompatibility impacting naturally hybridizing species.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.09.13.459985',\n",
       "   'title': 'Impacts of pleiotropy and migration on repeated genetic adaptation',\n",
       "   'authors': 'Battlay, P.; Yeaman, S.; Hodgins, K.',\n",
       "   'author_corresponding': 'Paul  Battlay',\n",
       "   'author_corresponding_institution': 'School of Biological Sciences, Monash University, Melbourne, Victoria, Australia',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '5',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'genetics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2021.09.13.459985.source.xml',\n",
       "   'abstract': 'Studies of trait-mapping and local adaptation often identify signatures of genetically repeated evolution, where different species independently evolve similar phenotypes using the same genes. Such patterns appear incongruent with current estimations of quantitative trait architecture: with hundreds or thousands or genes contributing to a trait, why would selection make repeated use of the same genes? This incongruence is particularly puzzling in light of the Fisher-Orr model, which predicts that repeated use of the same gene should decrease with increasing trait dimensionality, as alleles of large-effect are unlikely to be beneficial when there is high pleiotropy. When adaptation occurs for a single trait in spatially heterogeneous environments with migration, large-effect alleles are more strongly favoured, but it is unclear whether this result would generalize to multiple traits, especially if there are trade-offs between them. Here, we explore this question using individual-based simulations of a two-patch model with multiple quantitative traits governed by mutations with pleiotropic effects. We first confirm that repeatability is lower in spatially homogeneous environments, and then explore the effect of trait dimensionality and pleiotropy in a spatially heterogeneous environment. While repeatability decreases with increasing trait dimensionality, trade-offs in mutation effects on traits do not strongly limit the contribution of a locus of large effect to repeated adaptation, particularly under increased migration. These results suggest that evolutionary convergence will be more pronounced for local rather than global adaptation. Whereas pleiotropy limits repeatability in a single-population model, when there is local adaptation with gene flow, repeatability can be increased if some loci are able to produce alleles of large effect, even with pleiotropic trade-offs.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.10.18.464001',\n",
       "   'title': 'Discovery of embryonically derived bipotent endothelial-macrophage progenitor cells in postnatal aorta',\n",
       "   'authors': 'Williamson, A. E.; Liyanage, S.; Hassanshahi, M.; Dona, M. S. I.; Toledo-Flores, D.; Tran, D. X. A.; Dimasi, C.; Schwarz, N.; Fernando, S.; Salagaras, T.; Long, A.; Kazenwadel, J.; Harvey, N. L.; Drummond, G. R.; Vinh, A.; Chandrakanthan, V.; Misra, A.; Neufeld, Z.; Tan, J. T. M.; Martelotto, L. G.; Polo, J. M.; Bonder, C. S.; Pinto, A. R.; Sharma, S.; Nicholls, S. J.; Bursill, C. A.; Psaltis, P. J.',\n",
       "   'author_corresponding': 'Peter J Psaltis',\n",
       "   'author_corresponding_institution': 'South Australian Health and Medical Research Institute, University of Adelaide and Central Adelaide Local Health Network',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2021.10.18.464001.source.xml',\n",
       "   'abstract': 'BackgroundYolk sac (YS) progenitors are a source of macrophages and endothelial cells in some tissues that are thought to be maintained postnatally by self-renewal in their differentiated states. How this is achieved remains poorly understood.\\n\\nMethods and ResultsSingle-cell digests from mouse aortas selectively formed macrophage colony-forming units (CFU-M) in methylcellulose, which self-renewed in secondary cultures from single cells. CFU-M comprised a homogeneous population of Lin-CD45+/LoCD11b- F4/80-Sca-1+c-Kit+ progenitor cells that co-expressed fractalkine receptor (CX3CR1) and colony stimulating factor-1 receptor (CSF1R). These progenitors displayed high proliferative activity from adult aorta even at steady state. Flt3Cre lineage mapping revealed their independence from Flt3+ bone marrow hematopoietic progenitor cells. They were especially abundant in neonatal aorta, with subsequent age-related decline, suggesting prenatal seeding, which was confirmed by finding their emergence in YS after embryonic day (E) 7.5 and from aorta-gonad-mesonephros by E10.5. Inducible fate-mapping using Csf1rMer-iCre-Mer and Cx3cr1YFP-creER mice established that these progenitors originate from an E8.5 CSF1R+ and E8.5-9.5 CX3CR1+ source, together with macrophages and endothelial cells in the aortic wall. Complementary differentiation studies revealed aortic progenitors to be vasculogenic and bipotent for macrophages and endothelium, contributing to adventitial neovascularization in aortic ring assays and forming perfused blood vessels and macrophages after transfer into ischemic hindlimb. Single-cell RNA sequencing showed their relatively homogeneous myelopoietic and angiogenic gene expression profile without expression of mature myeloid or endothelial genes. Finally, we found that aortic progenitors also express angiotensin converting enzyme (ACE) and angiotensin II receptor, AGTR2, and established regulatory roles for angiotensin II, which augmented their proliferative, self-renewal and differentiation properties in vitro and expansion in aorta in vivo.\\n\\nConclusionOur discovery of aortic endothelial-macrophage progenitors adds to the recognized fate of YS progenitors in postnatal tissues. These bipotent cells may help explain the local renewal of YS-derived tissue-resident macrophages and endothelial cells after birth.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.10.27.466202',\n",
       "   'title': 'FEMA: Fast and efficient mixed-effects algorithm for large sample whole-brain imaging data',\n",
       "   'authors': 'Parekh, P.; Fan, C. C.; Frei, O.; Palmer, C. E.; Smith, D. M.; Makowski, C.; Iverson, J.; Pecheva, D.; Holland, D.; Loughnan, R.; Nedelec, P.; Thompson, W. K.; Hagler, D. J.; Andreassen, O. A.; Jernigan, T. L.; Nichols, T.; Dale, A. M.',\n",
       "   'author_corresponding': 'Chun Chieh  Fan',\n",
       "   'author_corresponding_institution': 'Population Neuroscience and Genetics Lab, University of California, San Diego',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2021.10.27.466202.source.xml',\n",
       "   'abstract': 'The linear mixed-effects model (LME) is a versatile modeling approach to deal with correlations among observations. Many large-scale neuroimaging datasets with complex designs have increased the need for LME, however LME has seldom been used in whole-brain imaging analyses due to its heavy computational requirements. In this paper, we introduce a fast and efficient mixed-effects algorithm (FEMA) that makes whole-brain vertex-wise, voxel-wise, and connectome-wide LME analyses in large samples possible. We validate FEMA with extensive simulations, showing that the estimates of the fixed effects are equivalent to standard maximum likelihood estimates but obtained with orders of magnitude improvement in computational speed. We demonstrate the applicability of FEMA by studying the cross-sectional and longitudinal effects of age on region-of-interest level and vertex-wise cortical thickness, as well as connectome-wide functional connectivity values derived from resting state functional MRI, using longitudinal imaging data from the Adolescent Brain Cognitive DevelopmentSM Study release 4.0. Our analyses reveal distinct spatial patterns for the annualized changes in vertex-wise cortical thickness and connectome-wide connectivity values in early adolescence, highlighting a critical time of brain maturation. The simulations and application to real data show that FEMA enables advanced investigation of the relationships between large numbers of neuroimaging metrics and variables of interest while considering complex study designs, including repeated measures and family structures, in a fast and efficient manner. The source code for FEMA is available via: https://github.com/cmig-research-group/cmig_tools/.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.10.27.466202',\n",
       "   'title': 'FEMA: Fast and efficient mixed-effects algorithm for large sample whole-brain imaging data',\n",
       "   'authors': 'Parekh, P.; Fan, C. C.; Frei, O.; Palmer, C. E.; Smith, D. M.; Makowski, C.; Iverson, J.; Pecheva, D.; Holland, D.; Loughnan, R.; Nedelec, P.; Thompson, W. K.; Hagler, D. J.; Andreassen, O. A.; Jernigan, T. L.; Nichols, T.; Dale, A. M.',\n",
       "   'author_corresponding': 'Chun Chieh  Fan',\n",
       "   'author_corresponding_institution': 'Population Neuroscience and Genetics Lab, University of California, San Diego',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '5',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2021.10.27.466202.source.xml',\n",
       "   'abstract': 'The linear mixed-effects model (LME) is a versatile modeling approach to deal with correlations among observations. Many large-scale neuroimaging datasets with complex designs have increased the need for LME, however LME has seldom been used in whole-brain imaging analyses due to its heavy computational requirements. In this paper, we introduce a fast and efficient mixed-effects algorithm (FEMA) that makes whole-brain vertex-wise, voxel-wise, and connectome-wide LME analyses in large samples possible. We validate FEMA with extensive simulations, showing that the estimates of the fixed effects are equivalent to standard maximum likelihood estimates but obtained with orders of magnitude improvement in computational speed. We demonstrate the applicability of FEMA by studying the cross-sectional and longitudinal effects of age on region-of-interest level and vertex-wise cortical thickness, as well as connectome-wide functional connectivity values derived from resting state functional MRI, using longitudinal imaging data from the Adolescent Brain Cognitive DevelopmentSM Study release 4.0. Our analyses reveal distinct spatial patterns for the annualized changes in vertex-wise cortical thickness and connectome-wide connectivity values in early adolescence, highlighting a critical time of brain maturation. The simulations and application to real data show that FEMA enables advanced investigation of the relationships between large numbers of neuroimaging metrics and variables of interest while considering complex study designs, including repeated measures and family structures, in a fast and efficient manner. The source code for FEMA is available via: https://github.com/cmig-research-group/cmig_tools/.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.10.28.466284',\n",
       "   'title': 'Emergence of robust global modules from local interactions and smooth gradients',\n",
       "   'authors': 'Khona, M.; Chandra, S.; Fiete, I.',\n",
       "   'author_corresponding': 'Ila  Fiete',\n",
       "   'author_corresponding_institution': 'MIT',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '5',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2021.10.28.466284.source.xml',\n",
       "   'abstract': 'Modular structure and function are ubiquitous in biology, from the scale of ecosystems to the organization of animal bodies and brains. However, the mechanisms of modularity emergence over development remain unclear. Here we introduce the principle of peak selection, a process in which two local interactions self-organize discontinuous module boundaries from a smooth global gradient, unifying the positional hypothesis and the Turing pattern formation hypothesis for morphogenesis. Applied to the brains grid cell networks, peak selection results in the spontaneous emergence of functionally distinct modules with discretely spaced spatial periods. Applied to ecological systems, a generalization of the process results in discrete systems-level niches. The dynamics exhibits emergent self-scaling to variations in system size and \"topological robustness\" [1] that renders module emergence and module properties insensitive to most parameters. Peak selection substantially ameliorates the fine-tuning requirement of continuous attractor dynamics even within single modules. It makes a detail-independent prediction that grid module period ratios should approximate adjacent integer ratios, furnishing the most accurate match to data to date, with additional predictions to connect physiology, connectomics, and transcriptomics data. In sum, our results indicate that local competitive interactions combined with low-information global gradients can lead to robust global module emergence.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.11.04.467365',\n",
       "   'title': 'WITHDRAWN: Whi5 and Stb1 define redundant pathways through which the G1 cyclin Cln3 promotes cell division',\n",
       "   'authors': 'Honey, S.; Futcher, B.',\n",
       "   'author_corresponding': 'Bruce  Futcher',\n",
       "   'author_corresponding_institution': 'Stony Brook University',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'withdrawn',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'genetics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2021.11.04.467365.source.xml',\n",
       "   'abstract': 'Withdrawal StatementThe authors have withdrawn their manuscript owing to a defect in the WHI5* STB1* strain used in Fig. 3, Fig. 4, and Table 2. In this strain, for reasons still unknown to us, the GAL-CLN3 gene is expressed moderately well even in the absence of galactose, and is only slightly induced upon the addition of galactose. Since CLN3 is not being substantially induced in the experiments of Fig. 3 and Fig. 4 in this WHI5* STB1* strain, no conclusion can be drawn as to whether there is or is not signaling from CLN3 to Start in a WHI5* STB1* mutant. Results in the manuscript not involving this strain appear to be correct. If you have any questions, please contact the corresponding author.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.12.07.471480',\n",
       "   'title': 'RoPod, a customizable toolkit for non-invasive root imaging, reveals cell type-specific dynamics of plant autophagy',\n",
       "   'authors': 'Guichard, M.; Holla, S.; Wernerova, D.; Grossmann, G. E. A.; Minina, A. E. A.',\n",
       "   'author_corresponding': 'Marjorie  Guichard',\n",
       "   'author_corresponding_institution': 'Heinrich-Heine-University Duesseldorf',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2021.12.07.471480.source.xml',\n",
       "   'abstract': 'Arabidopsis root is a classic model system in plant cell and molecular biology. The sensitivity of plant roots to local environmental perturbation challenges data reproducibility and incentivizes further optimization of imaging and phenotyping tools.\\n\\nHere we present RoPod, an easy-to-use toolkit for low-stress live time-lapse imaging of Arabidopsis roots. RoPod comprises a dedicated protocol for plant cultivation and a customizable 3D-printed vessel with integrated microscopy-grade glass that serves simultaneously as a growth and imaging chamber. RoPod reduces impact of sample handling, preserves live samples for prolonged imaging sessions, and facilitates application of treatments during image acquisition.\\n\\nWe describe a protocol for RoPods fabrication and provide illustrative application pipelines for monitoring root hair growth and autophagic activity. Furthermore, we showcase how the use of RoPods advanced our understanding of plant autophagy, a major catabolic pathway and a key player in plant fitness. Specifically, we obtained fine time resolution for autophagy response to commonly used chemical modulators of the pathway and revealed previously overlooked cell type-specific changes in the autophagy response. These results will aid a deeper understanding of the physiological role of autophagy and provide valuable guidelines for choosing sampling time during end-point assays currently employed in plant autophagy research.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.12.07.471672',\n",
       "   'title': 'Adaptive Ensemble Refinement of Protein Structures in High Resolution Electron Microscopy Density Maps with Radical Augmented Molecular Dynamics Flexible Fitting',\n",
       "   'authors': 'Sarkar, D.; Lee, H.; Vant, J. W.; Turilli, M.; Vermaas, J. V.; Jha, S.; Singharoy, A.',\n",
       "   'author_corresponding': 'Abhishek  Singharoy',\n",
       "   'author_corresponding_institution': 'School of Molecular Sciences, Arizona State University, Tempe, Arizona 85281, USA',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2021.12.07.471672.source.xml',\n",
       "   'abstract': 'Recent advances in cryo-electron microscopy (cryo-EM) have enabled modeling macromolecular complexes that are essential components of the cellular machinery. The density maps derived from cryo-EM experiments are often integrated with manual, knowledge or artificial intelligence driven, and physics-guided computational methods to build, fit, and refine molecular structures. Going beyond a single stationary-structure determination scheme, it is becoming more common to interpret the experimental data with an ensemble of models, which contributes to an average observation. Hence, there is a need to decide on the quality of an ensemble of protein structures on-the-fly, while refining them against the density maps. We introduce such an adaptive decision making scheme during the molecular dynamics flexible fitting (MDFF) of biomolecules. Using RADICAL-Cybertools, and the new RADICAL augmented MDFF implementation (R-MDFF) is examined in high-performance computing environments for refinement of two protein systems, Adenylate Kinase and Carbon Monoxide Dehy-drogenase. For the test cases, use of multiple replicas in flexible fitting with adaptive decision making in R-MDFF improves the overall correlation to the density by 40% relative to the refinements of the brute-force MDFF. The improvements are particularly significant at high, 2 - 3 [A] map resolutions. More importantly, the ensemble model captures key features of biologically relevant molecular dynamics that is inaccessible to a single-model interpretation. Finally, the pipeline is applicable to systems of growing sizes, which is demonstrated using ensemble refinement of capsid proteins from Chimpanzee adenovirus. The overhead for decision making remaining low and robust to computing environments. The software is publicly available on GitHub and includes a short user guide to install the R-MDFF on different computing environments, from local Linux based workstations to High Performance Computing (HPC) environments.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.12.20.473481',\n",
       "   'title': 'Inhibition of autophagy as a novel therapy for the treatment of neurofibromatosis type 1 tumors',\n",
       "   'authors': 'Stevens, M.; Wang, Y.; Bouley, S. J.; Mandigo, T. R.; Sharma, A.; Sengupta, S.; Housden, A. E.; Oltean, S.; Perrimon, N.; Walker, J. A.; Housden, B. E.',\n",
       "   'author_corresponding': 'Benjamin E Housden',\n",
       "   'author_corresponding_institution': 'University of Exeter',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'cancer biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2021.12.20.473481.source.xml',\n",
       "   'abstract': 'Neurofibromatosis type 1 (NF1) is a genetic disorder associated with various symptoms including the formation of benign tumors along nerves. Drug treatments are currently limited. The MEK inhibitor selumetinib is used for a subset of cases but is not always effective and can cause side effects. Therefore, there is a clear need to discover new drugs to target NF1-deficient tumor cells. Using a Drosophila cell model of NF1, we performed synthetic lethal screens to identify novel drug targets. We identified 54 candidates, which were validated with variable dose analysis as a secondary screen. Five candidates could be targeted using existing drugs, with autophagy inhibitors (chloroquine (CQ) and bafilomycin A1) showing the greatest potential for selectively killing NF1-deficient Drosophila cells. When further investigating autophagy-related genes, we found that 14 out of 30 genes tested had a synthetic lethal interaction with NF1. These 14 genes are involved in the regulation of all aspects of the autophagy pathway and can be targeted with additional autophagy drugs, although none were as effective as CQ. The lethal effect of autophagy inhibitors was conserved in a panel of human NF1-deficient Schwann cell lines, highlighting their translational potential. The effect of CQ was also conserved in a Drosophila NF1 in vivo model and in a xenografted NF1-deficient tumor cell line in mice, with CQ treatment resulting in a more significant reduction in tumor growth than selumetinib treatment. Furthermore, combined treatment with CQ and selumetinib resulted in a further reduction in NF1-deficient cell viability. In conclusion, NF1-deficient cells are vulnerable to disruption of the autophagy pathway. This pathway represents a promising therapeutic target for NF1-associated tumors, and CQ was identified as a promising candidate drug for the treatment of NF1 tumors.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.12.21.473383',\n",
       "   'title': 'Primary Cilia Directionality Reveals a Slow Reverse Migration of Principal Neurons for Postnatal Positioning and Lamination in the Cerebral Cortex',\n",
       "   'authors': 'Yang, J.; Mirhosseiniardakani, S.; Qiu, L.; Bicja, K.; Lin, K. J.; Chen, X.',\n",
       "   'author_corresponding': 'Xuanmao  Chen',\n",
       "   'author_corresponding_institution': 'University of New Hampshire',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2021.12.21.473383.source.xml',\n",
       "   'abstract': 'Currently little is known about the roles of neuronal primary cilia in postnatal neurodevelopment. Here we show that primary cilia of principal neurons in the cerebral cortex do not fully arise until the postnatal stage. Neuronal primary cilia undergo marked changes in morphology and orientation postnatally, concurrent with principal cell positioning. Primary cilia of early- and late-born hippocampal principal neurons display opposite orientations, while primary cilia of both early and late-born neocortical principal neurons point toward one direction. In contrast, astrocytes and interneurons in the hippocampus, and neurons in nucleated brain regions do not display specific cilia directionality. Guided by the clue of cilia directionality, we discovered that principal neurons in the hippocampal CA1, subiculum, postsubiculum, and neocortex undergo a previously unnoticed, slow but substantial \"reverse migration\" for postnatal positioning and proper lamination. Further, selective disruption of primary cilia function in the forebrain using Ift88 or Arl13b conditional knockout mice leads to cortical expansion, altered cortical lamination, or gyrification in the retrosplenial cortex that is formed by reverse migration. Collectively, this work has identified reverse migration as a novel fundamental process for principal cell positioning that helps shape lamination and gyrification in the cerebral cortex, and primary cilia orientation indicates the direction of reverse migration of principal neurons.\\n\\nHighlightsO_LIPrimary cilia of early- and late-born CA1 neurons display opposite orientations, while primary cilia of neocortical principal neurons generally point towards one direction (the pial surface).\\nC_LIO_LIPrincipal neurons in the CA1 superficial sublayer, subiculum, postsubiculum, and neocortex undergo a reverse migration for postnatal cell positioning that refines cerebral cortical lamination.\\nC_LIO_LIPrimary cilia orientation indicates the direction of reverse migration of principal neurons in the cerebral cortex.\\nC_LIO_LISelective ablation of primary cilia in the forebrain leads to cortical expansion, altered lamination, and gyrification that is formed by the reverse migration of principal neurons.\\nC_LIO_LIArl13b modulates the ciliation and postnatal positioning of hippocampal principal neurons.\\nC_LI',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.12.24.474127',\n",
       "   'title': 'Neural correlates of individual odor preference in Drosophila',\n",
       "   'authors': 'Churgin, M.; Lavrentovich, D.; Smith, M. A.-Y.; Gao, R.; Boyden, E. S.; de Bivort, B. L.',\n",
       "   'author_corresponding': 'Benjamin L de Bivort',\n",
       "   'author_corresponding_institution': 'Harvard University',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2021.12.24.474127.source.xml',\n",
       "   'abstract': 'Behavior varies even among genetically identical animals raised in the same environment. However, little is known about the circuit or anatomical origins of this individuality. We show individual Drosophila odor preferences (odor-vs-air and odor-vs-odor) are predicted by idiosyncratic calcium dynamics in olfactory receptor neurons (ORNs) and projection neurons (PNs), respectively. Variation in ORN presynaptic density also predicts odor-vs-odor preference. The ORN-PN synapse appears to be a locus of individuality where microscale variation gives rise to idiosyncratic behavior. Finally, simulating microscale stochasticity in ORN-PN synapses of a 3,062 neuron model of the antennal lobe recapitulates patterns of variation in PN calcium responses matching experiments. Our results demonstrate how physiological and microscale structural circuit variations can give rise to individual behavior, even when genetics and environment are held constant.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.01.02.474742',\n",
       "   'title': 'Thermoring Basis for the PIP2-Dependent Cold Switches in the TRPM8 Biothermometer',\n",
       "   'authors': 'Wang, G.',\n",
       "   'author_corresponding': 'Guangyu  Wang',\n",
       "   'author_corresponding_institution': 'University of California Davis',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2022.01.02.474742.source.xml',\n",
       "   'abstract': 'The menthol sensor TRPM8 can also be activated by cold and thus serves as a biothermometer in a primary afferent sensory neuron for noxious cold detection. However, the underlying design maps are unknown. Here, a series of cold-sensitive factors, together with a recently developed grid thermodynamic model, were employed to cross-examine if the available 3D structures of closed and open mouse TRPM8 with or without Ca2+ and PIP2 and cooling agents bound can be directly used for cold detection. The results demonstrated that noncovalent interactions as identified in those structures could form a systematic fluidic grid-like mesh network along the PIP2-dependent gating pathway. While the biggest Grid21 near the lower gate may serve as a cold switch with a matched threshold of 22 {degrees}C for PIP2 binding, the biggest Grid26 in the pore domain may act as another cold switch with a matched threshold 7 {degrees}C to stabilize the open state. Further, PIP2-dependent channel opening upon a cold incubation could create a matched temperature sensitivity of -28. Finally, both gating states passed the test of the available cold-sensitive factors including pH. Therefore, the captured structures of closed and open TRPM8 at low temperature could be used for noxious cold detection. (200 words)',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2021.12.31.474032',\n",
       "   'title': 'Analysis of SARS-CoV-2 Omicron Neutralization Data up to 2022-01-28',\n",
       "   'authors': 'Netzl, A.; Tureli, S.; LeGresley, E.; Mühlemann, B.; Wilks, S. H.; Smith, D. J.',\n",
       "   'author_corresponding': 'Derek  J.  Smith',\n",
       "   'author_corresponding_institution': 'University of Cambridge',\n",
       "   'date': '2023-08-25',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'immunology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/25/2021.12.31.474032.source.xml',\n",
       "   'abstract': 'The rapid spread of the Omicron BA.1 (B.1.1.529) SARS-CoV-2 (Severe Acute Respiratory Syndrome Coronavirus 2) variant in 2021 resulted in international efforts to quickly assess its escape from immunity generated by vaccines and previous infections. Numerous laboratories published Omicron neutralization data as preprints and reports. Here, we use forest plots and antigenic cartography to analyze aggregated Omicron neutralization data from 49 reporting laboratories up to 2022-01-28. We found that, in twice vaccinated individuals, titer fold drop of Omicron relative to wild type is more than 17x, likely substantially higher given the number of measurements below the assay detection limit. Moreover, after a third dose with an mRNA vaccine, the titer fold drop to Omicron is considerably less at 7x, and triple vaccination reduces fold drops across SARS-CoV-2 variants. We demonstrate that it is possible to build reliable antigenic cartography maps from this collated data.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.01.07.475310',\n",
       "   'title': 'Generalized Michaelis-Menten rate law with time-varying molecular concentrations',\n",
       "   'authors': 'Lim, R.; Martin, T. L. P.; Chae, J.; Kim, W.; Ghim, C.-M.; Kim, P.-J.',\n",
       "   'author_corresponding': 'Pan-Jun  Kim',\n",
       "   'author_corresponding_institution': 'Hong Kong Baptist University',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '6',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'biochemistry',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.01.07.475310.source.xml',\n",
       "   'abstract': 'The Michaelis-Menten (MM) rate law has been the dominant paradigm of modeling biochemical rate processes for over a century with applications in biochemistry, biophysics, cell biology, systems biology, and chemical engineering. The MM rate law and its remedied form stand on the assumption that the concentration of the complex of interacting molecules, at each moment, approaches an equilibrium (quasi-steady state) much faster than the molecular concentrations change. Yet, this assumption is not always justified. Here, we relax this quasi-steady state requirement and propose the generalized MM rate law for the interactions of molecules with active concentration changes over time. Our approach for time-varying molecular concentrations, termed the effective time-delay scheme (ETS), is based on rigorously estimated time-delay effects in molecular complex formation. With particularly marked improvements in protein-protein and protein-DNA interaction modeling, the ETS provides an analytical framework to interpret and predict rich transient or rhythmic dynamics (such as autogenously-regulated cellular adaptation and circadian protein turnover), which goes beyond the quasi-steady state assumption.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.01.20.477049',\n",
       "   'title': \"A quantitative, genome-wide analysis in Drosophila reveals transposable elements' influence on gene expression is species-specific\",\n",
       "   'authors': 'Fablet, M.; Salces-Ortiz, J.; Jacquet, A.; Menezes, B. F.; Dechaud, C.; Veber, P.; Nous, C.; Rebollo, R.; Vieira, C.',\n",
       "   'author_corresponding': 'Marie  Fablet',\n",
       "   'author_corresponding_institution': 'University Claude Bernard Lyon 1',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '5',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'genomics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.01.20.477049.source.xml',\n",
       "   'abstract': 'Transposable elements (TEs) are parasite DNA sequences that are able to move and multiply along the chromosomes of all genomes. They can be controlled by the host through the targeting of silencing epigenetic marks, which may affect the chromatin structure of neighboring sequences, including genes. In this study, we used transcriptomic and epigenomic high-throughput data produced from ovarian samples of several Drosophila melanogaster and Drosophila simulans wild-type strains, in order to finely quantify the influence of TE insertions on gene RNA levels and histone marks (H3K9me3 and H3K4me3). Our results reveal a stronger epigenetic effect of TEs on ortholog genes in D. simulans compared to D. melanogaster. At the same time, we uncover a larger contribution of TEs to gene H3K9me3 variance within genomes in D. melanogaster, which is evidenced by a stronger correlation of TE numbers around genes with the levels of this chromatin mark in D. melanogaster. Overall, this work contributes to the understanding of species-specific influence of TEs within genomes. It provides a new light on the considerable natural variability provided by TEs, which may be associated with contrasted adaptive and evolutionary potentials.\\n\\nSignificance StatementTransposable elements (TEs) are parasitic DNA sequences that are widespread components of all genomes. In this study, we combined genomic, transcriptomic and epigenomic high-throughput data produced from ovarian samples of Drosophila melanogaster and Drosophila simulans wild-type strains, in order to finely quantify the genome-wide influence of TE insertions on gene expression. Our results uncover contrasted patterns depending on the strain, which may have evolutionary impacts.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.02.15.480512',\n",
       "   'title': 'Awake responses suggest inefficient dense coding in the mouse retina',\n",
       "   'authors': 'Boissonnet, T.; Tripodi, M.; Asari, H.',\n",
       "   'author_corresponding': 'Hiroki  Asari',\n",
       "   'author_corresponding_institution': 'EMBL Rome',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.02.15.480512.source.xml',\n",
       "   'abstract': 'The structure and function of the vertebrate retina have been extensively studied across species with an isolated, ex vivo preparation. Retinal function in vivo, however, remains elusive, especially in awake animals. Here we performed single-unit extracellular recordings in the optic tract of head-fixed mice to compare the output of awake, anesthetized, and ex vivo retinas. While the visual response properties were overall similar across conditions, we found that awake retinal output had in general 1) faster kinetics with less variability in the response latencies; 2) a larger dynamic range; and 3) higher firing activity, by [~]20 Hz on average, for both baseline and visually evoked responses. Our modeling analyses further showed that such awake response patterns convey comparable total information but less efficiently, and allow for a linear population decoder to perform significantly better than the anesthetized or ex vivo responses. These results highlight distinct retinal behavior in awake states, in particular suggesting that the retina employs dense coding in vivo, rather than sparse efficient coding as has been often assumed from ex vivo studies.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.02.16.480703',\n",
       "   'title': 'Tensor decomposition reveals coordinated multicellular patterns of transcriptional variation that distinguish and stratify disease individuals',\n",
       "   'authors': 'Mitchel, J.; Gordon, M. G.; Perez, R. K.; Biederstedt, E.; Bueno, R.; Ye, C. J.; Kharchenko, P.',\n",
       "   'author_corresponding': 'Peter  Kharchenko',\n",
       "   'author_corresponding_institution': 'Harvard Medical School',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.02.16.480703.source.xml',\n",
       "   'abstract': 'Tissue- and organism-level biological processes often involve coordinated action of multiple distinct cell types. Current computational methods for the analysis of single-cell RNA-sequencing (scRNA-seq) data, however, are not designed to capture co-variation of cell states across samples, in part due to the low number of biological samples in most scRNA-seq datasets. Recent advances in sample multiplexing have enabled population-scale scRNA-seq measurements of tens to hundreds of samples. To take advantage of such datasets, here we introduce a computational approach called single-cell Interpretable Tensor Decomposition (scITD). This method extracts \"multicellular gene expression patterns\" that capture how sample-specific expression states of a cell type are correlated with the expression states of other cell types. Such multicellular patterns can reveal molecular mechanisms underlying coordinated changes of different cell types within the tissue, and can be used to stratify individuals in a clinically-relevant and reproducible manner. We first validated the performance of scITD using in vitro experimental data and simulations. We then applied scITD to scRNA-seq data on peripheral blood mononuclear cells (PBMCs) from 115 patients with systemic lupus erythematosus and 56 healthy controls. We recapitulated a well-established pan-cell-type signature of interferon-signaling that was associated with the presence of anti-dsDNA autoantibodies and a disease activity index. We further identified a novel multicellular pattern linked to nephritis, which was characterized by an expansion of activated memory B cells along with helper T cell activation. Our approach also sheds light on ligand-receptor interactions potentially mediating these multicellular patterns. As validation, we demonstrated that these expression patterns also stratified donors from a pediatric SLE dataset by the same phenotypic attributes. Lastly, we found the interferon multicellular pattern and others to be conserved in a COVID-19 dataset, pointing to the presence of both general and disease-specific patterns of inter-individual immune variation. Overall, scITD is a flexible method for exploring co-variation of cell states in multi-sample single-cell datasets, which can yield new insights into complex non-cell-autonomous dependencies that define and stratify disease.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.02.21.481254',\n",
       "   'title': 'Microglial TNFα controls GABAAR plasticity, sleep slow waves and memory consolidation',\n",
       "   'authors': 'Pinto, M. J.; Bizien, L.; Fabre, J. M.; Thukanovic, N.; Cottin, L.; Lepetz, V.; Henderson, F.; Pujol, M.; Sala, R. W.; Tarpin, T.; Popa, D.; Triller, A.; Lena, C.; Fabre, V.; Bessis, A.',\n",
       "   'author_corresponding': 'Alain  Bessis',\n",
       "   'author_corresponding_institution': \"Institut de Biologie de l'Ecole normale superieure\",\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.02.21.481254.source.xml',\n",
       "   'abstract': 'Microglia sense the changes in their environment. How microglia actively translate these changes into suitable cues to adapt brain physiology is unknown. We reveal an activity-dependent regulation of cortical inhibitory synapses by microglia, driven by purinergic signaling acting on P2RX7 and mediated by microglia-derived TNF. We demonstrate that sleep induces microglia-dependent synaptic enrichment of GABAARs in a manner dependent on microglial TNF and P2RX7. We further show that microglia-specific depletion of TNF alters slow waves during NREM sleep and blunt memory consolidation in sleep-dependent learning tasks. Together, our results reveal that microglia orchestrate sleep-intrinsic plasticity of synaptic GABAARs, sculpt sleep slow waves and support memory consolidation.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.02.25.481917',\n",
       "   'title': 'Functional Plasticity in Chromosome-Microtubule Coupling on the Evolutionary Time Scale',\n",
       "   'authors': 'Sankaranarayanan, S. R.; Polisetty, S. D.; Das, K.; Dumbrepatil, A.; Medina-Pritchard, B.; Singleton, M.; Jeyaprakash, A. A.; Sanyal, K.',\n",
       "   'author_corresponding': 'Kaustuv  Sanyal',\n",
       "   'author_corresponding_institution': 'Jawaharlal Nehru Centre for Advanced Scientific Research',\n",
       "   'date': '2023-08-25',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'genetics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/25/2022.02.25.481917.source.xml',\n",
       "   'abstract': 'The Dam1 complex is essential for mitotic progression across evolutionarily divergent fungi. Upon analyzing amino acid (aa) sequences of Dad2, a Dam1 complex subunit, we identified a conserved 10-aa-long Dad2 signature sequence (DSS). An arginine residue (R126) in the DSS is essential for viability in Saccharomyces cerevisiae that possesses point centromeres. The corresponding arginine residues are functionally important but not essential for viability in Candida albicans and Cryptococcus neoformans, both carry several kilobases long regional centromeres. The purified recombinant Dam1 complex containing either Dad2{Delta}DSS or Dad2R126A failed to bind microtubules (MTs) or form any visible rings like the wild-type complex. Intriguingly, functional analysis revealed that the requirement of the conserved arginine residue for chromosome biorientation and mitotic progression reduced with increasing centromere length. We propose that plasticity of the invariant arginine of Dad2 in organisms with regional centromeres is achieved by conditional elevation of the kinetochore protein(s) to enable multiple kinetochore MTs (kMTs) to bind to each chromosome. The capacity of a chromosome to bind multiple kMTs may mask deleterious effects of such lethal mutations.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.03.17.484757',\n",
       "   'title': 'Distinguishing fine structure and summary representation of sound textures from neural activity',\n",
       "   'authors': 'Berto, M.; Ricciardi, E.; Pietrini, P.; Weisz, N.; Bottari, D.',\n",
       "   'author_corresponding': 'Martina  Berto',\n",
       "   'author_corresponding_institution': 'IMT School for Advanced Studies Lucca',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.03.17.484757.source.xml',\n",
       "   'abstract': 'The auditory system relies on both local and summary representations; acoustic local features exceeding system constraints are compacted into a set of summary statistics. Such compression is pivotal for sound-object recognition. Here, we assessed whether computations subtending local and statistical representations of sounds could be distinguished at the neural level. A computational auditory model was employed to extract auditory statistics from natural sound textures (i.e., fire, rain) and to generate synthetic exemplars where local and statistical properties were controlled. Twenty-four human participants were passively exposed to auditory streams while the EEG was recorded. Each stream could consist of short, medium, or long sounds to vary the amount of acoustic information. Short and long sounds were expected to engage local or summary statistics representations, respectively. Data revealed a clear dissociation. Compared to summary-based ones, auditory-evoked responses based on local information were selectively greater in magnitude in short sounds. Opposite patterns emerged for longer sounds. Neural oscillations revealed that local features and summary statistics rely on neural activity occurring at different temporal scales, faster (beta) or slower (theta-alpha). These dissociations emerged automatically without explicit engagement in a discrimination task. Overall, this study demonstrates that the auditory system developed distinct coding mechanisms to discriminate changes in the acoustic environment based on fine structure and summary representations.\\n\\nSIGNIFICANCE STATEMENTPrior to this study, it was unknown whether we could measure auditory discrimination based on local temporal features or spectrotemporal statistics properties of sounds from brain responses. Results show that the two auditory modes of sound discrimination (local and summary statistics) are automatically attuned to the temporal resolution (high or low) at which a change has occurred. In line with the temporal resolutions of auditory statistics, faster or slower neural oscillations (temporal scales) code sound changes based on local or summary representations. These findings expand our knowledge of some fundamental mechanisms underlying the function of the auditory system.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.03.22.485341',\n",
       "   'title': '3D Reconstruction of Murine Mitochondria Exhibits Changes in Structure Across Aging Linked to the MICOS Complex',\n",
       "   'authors': 'Vue, Z.; Garza-Lopez, E.; Neikirk, K.; Katti, P.; Vang, L.; Beasley, H. K.; Shao, J.-q.; Marshall, A. G.; Crabtree, A.; Murphy, A. C.; Jenkins, B. C.; Prasad, P.; Evans, C. S.; Taylor, B. T.; Mungai, M.; Killion, M.; Stephens, D. C.; Christensen, T. A.; Lam, J.; Rodriguez, B.; Phillips, M. A.; Daneshgar, N.; Koh, H.-J.; Koh, A.; Davis, J.; Devine, N.; Muhammod, S.; Scudese, E.; Arnold, K.; Vanessa Chavarin, V.; Daniel Robinson, R.; Chakraborty, M.; Gaddy, J. A.; Wilson, G.; Zaganjor, E.; Kezos, J.; Dondi, C.; Reddy, A. K.; Glancy, B.; Kirabo, A.; Quintana, A. M.; Dai, D.-F.; Oncorr, K.; Murray',\n",
       "   'author_corresponding': 'Antentor  Hinton',\n",
       "   'author_corresponding_institution': 'Vanderbilt University',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '10',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.03.22.485341.source.xml',\n",
       "   'abstract': 'BackgroundDuring aging, muscle gradually undergoes loss of function including sarcopenia, losing mass, strength, endurance, and oxidative capacity. While mitochondrial aging is associated with decreased mitochondrial capacity, the genes associated with morphological changes in mitochondria during aging still require further elucidation. Furthermore, it is not completely understood how 3D mitochondrial structures are altered during aging in skeletal muscle and cardiac tissues.\\n\\nMethodsWe measured changes in mitochondrial morphology and mitochondrial complexity during the aging of murine gastrocnemius, soleus, and cardiac tissues using serial block face- scanning electron microscopy and 3D reconstruction. Lipidomic and metabolomic analysis elucidated concomitant changes associated with aging. We also used qPCR, transmission electron microscopy quantification, Seahorse Analyzer, and metabolomics to evaluate changes in mitochondria morphology and function upon loss of the MICOS complex.\\n\\nResultsWe identified significant changes in 3D mitochondrial size and network configuration in murine gastrocnemius, soleus, and cardiac tissue during aging. These changes were concomitant with loss of mitochondria contact site and cristae organizing system (MICOS) gene expression during aging. Mitochondrial morphology was similar between aged mice and young mice. We show an age-related loss of the MICOS complex (Chchd3, chchd6, and Mitofilin) while their knockout results in alterations in mitochondrial morphology. Given the critical role of mitochondria in maintaining cellular metabolism, we perform cellular metabolic profiling of young and aged tissues. Metabolomics and lipidomics showed profound alterations, including in membrane integrity, that support our observations of age-related changes in these muscle tissues.\\n\\nDiscussionIn tandem, our data suggest a relationship between the MICOS complex and aging, which could be linked to disease states with further 3D reconstruction studies. Our study highlights the importance of understanding tissue-dependent 3D mitochondrial phenotypical changes which occur across aging with evolutionary conservation between Drosophila and murine models.\\n\\nGraphical Abstract',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.04.14.488421',\n",
       "   'title': 'First clinical validation of whole genome screening on standard trophectoderm biopsies of preimplantation embryos',\n",
       "   'authors': 'Xia, Y.; Chertman, W.; Chandramohan, D.; Katz, M.; Bechor, E.; Podgursky, B.; Hoxie, M.; Zhang, Q.; Kang, J.; Blue, E.; Chen, J.; Schleede, J.; Slotnick, N.; Du, X.; Kort, J.; Boostanfar, R.; Urcia, E.; Behr, B.; Siddiqui, N.',\n",
       "   'author_corresponding': 'Yuntao  Xia',\n",
       "   'author_corresponding_institution': 'Orchid',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'genomics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.04.14.488421.source.xml',\n",
       "   'abstract': 'Whole Genome Sequencing (WGS) is used in healthcare and in the clinic, with the notable exception of preimplantation genetic testing (PGT). In PGT, only a few cells are available for sequencing, requiring DNA amplification which reduces data quality, sequence fidelity and sharply limits subsequent clinical impact. Here we demonstrate the first clinical validation of WGS on embryo biopsies using our lab development protocol, opening the door to broad use of WGS in fertility. We find that amplified DNA with comparable sensitivity and specificity to genomic DNA when performing whole genome sequencing assays. DNA amplification on cell lines and donated human embryos had an amplification success rate of >99.9% and 98.2% respectively and accuracy on both was >99.9% on aneuploidy status. GIAB samples (Genome in the Bottle reference NA12878) showed that our amplified DNA was broadly comparable to genomic DNA (99.99% accuracy, 99.99% specificity, 98.0% sensitivity and 98.1% precision). Using our assay, we were able to call variants, detect mitochondrial heteroplasmy, perform high precision screening without access to parental genomes, detect compound heterozygous variants, and score microdups/dels and uniparental disomies (to reduce risk of diseases such as DiGeorge syndrome and Prader-Willi syndrome). Our clinical study suggests that the full spectrum of traditional clinical genome bioinformatics, so far reserved to large samples, can now be performed on embryos before implantation.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.04.18.488634',\n",
       "   'title': 'Neural instructive signals for associative cerebellar learning',\n",
       "   'authors': 'Silva, N. T.; Ramirez-Buritica, J.; Pritchett, D. L.; Carey, M. R.',\n",
       "   'author_corresponding': 'Megan R. Carey',\n",
       "   'author_corresponding_institution': 'Champalimaud Foundation',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.04.18.488634.source.xml',\n",
       "   'abstract': 'Supervised learning depends on instructive signals that shape the output of neural circuits to support learned changes in behavior. Climbing fiber inputs to the cerebellar cortex represent one of the strongest candidates in the vertebrate brain for conveying neural instructive signals. However, recent studies have shown that Purkinje cell stimulation can also drive cerebellar learning, and the relative importance of these two neuron types in providing instructive signals for cerebellum-dependent behaviors remains unresolved. Here we used cell-type specific perturbations of climbing fibers, Purkinje cells, and other cerebellar circuit elements to systematically evaluate their contributions to delay eyeblink conditioning. Our findings reveal that while optogenetic stimulation of either climbing fibers or Purkinje cells are capable of driving learning under some conditions, even subtle reductions in climbing fiber signaling completely block learning to natural conditioning stimuli. We conclude that climbing fibers and corresponding Purkinje cell complex-spike events provide essential instructive signals for associative cerebellar learning.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.04.24.489283',\n",
       "   'title': 'Molecular and functional characterization of the Drosophila melanogaster conserved smORFome',\n",
       "   'authors': 'Bosch, J. A.; Keith, N.; Escobedo, F.; Fisher, W. W.; LaGraff, J. T.; Rabasco, J.; Wan, K.; Weiszmann, R.; Hu, Y.; Kondo, S.; Brown, J. B.; Perrimon, N.; Celniker, S. E.',\n",
       "   'author_corresponding': 'Justin A Bosch',\n",
       "   'author_corresponding_institution': 'Harvard Medical School',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'genomics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.04.24.489283.source.xml',\n",
       "   'abstract': 'Short polypeptides encoded by small open reading frames (smORFs) are ubiquitously found in eukaryotic genomes and are important regulators of physiology, development, and mitochondrial processes. Here, we focus on a subset of 298 smORFs that are evolutionarily conserved between Drosophila melanogaster and humans. Many of these smORFs are conserved broadly in the bilaterian lineage, with [~]182 conserved in plants. Within these conserved smORFs, we observed remarkably heterogenous spatial and temporal expression patterns - indicating wide-spread tissue-specific and stage-specific mitochondrial architectures. In addition, an analysis of annotated functional domains revealed a predicted enrichment of smORF polypeptides localizing to mitochondria. We conducted an embryonic ribosome profiling experiment finding support for translation of 137 of these smORFs during embryogenesis. We further embarked on functional characterization using CRISPR knockout/activation, RNAi knockdown, and cDNA overexpression, revealing diverse phenotypes. This study underscores the importance of identifying smORF function in disease and phenotypic diversity.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.04.30.490180',\n",
       "   'title': 'Transcription-induced active forces suppress chromatin motion',\n",
       "   'authors': 'Shin, S.; Shi, G.; Cho, H. W.; Thirumalai, D.',\n",
       "   'author_corresponding': 'Dave  Thirumalai',\n",
       "   'author_corresponding_institution': 'The University of Texas at Austin',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.04.30.490180.source.xml',\n",
       "   'abstract': 'The organization of interphase chromosomes in a number of species is starting to emerge thanks to advances in a variety of experimental techniques. However, much less is known about the dynamics, especially in the functional states of chromatin. Recent experiments have shown that the mobility of individual loci in human interphase chromosome decreases during transcription, and increases upon inhibiting transcription. This is a counter-intuitive finding because it is thought that the active mechanical force (F) on the order of ten pico-newtons, generated by RNA polymerase II (RNAPII) that is presumably transmitted to the gene-rich region of the chromatin, would render it more open, thus enhancing the mobility. Inspired by these observations, we developed a minimal active copolymer model for interphase chromosomes to investigate how F affects the dynamical properties of chromatin. The movements of the loci in the gene-rich region are suppressed in an intermediate range of F, and are enhanced at small and large F values. In the intermediate F, the bond length between consecutive loci increases, becoming commensurate with the distance at the minimum of the attractive interaction between non-bonded loci. This results in a transient disorder-to-order transition, leading to the decreased mobility during transcription. Strikingly, the F -dependent change in the locus dynamics preserves the organization of the chromosome at F = 0. Transient ordering of the loci, which is not found in the polymers with random epigenetic profiles, in the gene-rich region might be a plausible mechanism for nucleating a dynamic network involving transcription factors, RNAPII, and chromatin.\\n\\nSignificance StatementIn order to explain a physically counter-intuitive experimental finding that chromatin mobility is reduced during transcription, we introduced a polymer model for interphase chromosome that accounts for RNA polymerase (RNAP) induced active force. The simulations show that, over a range of active forces, the mobility of the gene-rich loci is suppressed. Outside this range, chromosomes are compact and exhibit glass-like dynamics. Our study, which accounts for the experimental observations, leads to a novel and testable mechanism of how transcription could shape the coexistence of fluid- and solid-like properties within chromosomes.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.05.11.491519',\n",
       "   'title': 'A dynamic in vitro model of Down Syndrome neurogenesis with Trisomy 21 gene dosage correction',\n",
       "   'authors': 'Bansal, P.; Banda, E. C.; Glatt-Deeley, H. R.; Stoddard, C.; Linsley, J. W.; Arora, N.; Ahern, D. T.; Kondaveeti, Y.; Nicouleau, M.; Sabariego-Navarro, M.; Dierssen, M.; Finkbeiner, S.; Pinter, S. F.',\n",
       "   'author_corresponding': 'Stefan F Pinter',\n",
       "   'author_corresponding_institution': 'University of Connecticut, UConn Health',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.05.11.491519.source.xml',\n",
       "   'abstract': 'ABSTRACT/SUMMARYExcess gene dosage from human chromosome 21 (chr21) causes Down syndrome (DS), spanning developmental as well as acute phenotypes in terminal cell types. Which phenotypes remain amenable to intervention after development is unknown. To address this question in a model of DS neurogenesis, we generated trisomy 21 (T21) human induced pluripotent stem cells (hiPSCs) alongside otherwise isogenic euploid controls from mosaic DS fibroblasts, and integrated an inducible XIST transgene on one chr21 copy. Monoallelic chr21 silencing by XIST was near-complete and irreversible in hiPSCs. Differential expression reveals T21 neural lineages and T21 hiPSCs suppress similar translation and mitochondrial pathways, and activate cellular stress responses. When XIST is induced before the neural progenitor stage, T21 dosage correction mitigates a pronounced skew towards astrogenesis in differentiation. Because our transgene remained inducible in post-mitotic T21 neurons and astrocytes, we demonstrate XIST efficiently represses genes even after terminal differentiation, which will empower exploration of cell type-specific T21 phenotypes that remain responsive to chr21 dosage.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.06.02.494552',\n",
       "   'title': 'Engineered Wnt7a ligands rescue blood brain barrier and neurobehavioral deficits in a mouse model of COVID-19',\n",
       "   'authors': 'Trevino, T. N.; Fogel, A. B.; Class, J.; Sanborn, M. A.; Vanhollebeke, B.; Rehman, J.; Tai, L. M.; Richner, J. M.; Lutz, S. E.',\n",
       "   'author_corresponding': 'Sarah E Lutz',\n",
       "   'author_corresponding_institution': 'University of Illinois at Chicago',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.06.02.494552.source.xml',\n",
       "   'abstract': 'Respiratory infection with SARS-CoV-2 causes systemic vascular inflammation and cognitive impairment. We sought to identify the underlying mechanisms mediating vascular dysfunction and inflammation following mild respiratory SARS-CoV-2 infection. To this end, we conduced unbiased transcriptional analysis to identify brain endothelial cell signaling pathways dysregulated by SARS-CoV-2 in vivo. This analysis revealed significant suppression of Wnt/{beta}-catenin signaling, a critical regulator of blood brain barrier integrity. We therefore hypothesized that enhancing cerebrovascular Wnt/{beta}-catenin activity would offer protection against BBB permeability, neuroinflammation, and neurological signs in acute infection. Indeed, we found that delivery of cerebrovascular-targeted, engineered Wnt7a ligands protected blood brain barrier integrity, reduced T cell infiltration of the brain, and reduced microglial activation in SARS-CoV-2 infection. Importantly, this therapeutic strategy also mitigated SARS-CoV-2 induced deficits in the novel object recognition assay for learning and memory and the pole descent task for bradykinesia. These observations suggest that enhancement of Wnt/{beta}-catenin signaling or its downstream effectors could be potential interventional strategies for restoring cognitive health following acute viral infections.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.06.02.494595',\n",
       "   'title': 'Understanding spatiotemporal effects of food supplementation on host-parasite interactions using community-based science',\n",
       "   'authors': 'Knutie, S. A.; Bahouth, R.; Bertone, M. A.; Webb, C.; Mehta, M.; Nahom, M.; Barta, R. M.; Ghai, S.; Love, A. C.; Horan, S.; Soldo, A.; Cochrane, E.; Bartholomew, J.; Cowan, E.; Bjerke, H.; Balenger, S. L.; Butler, M. W.; Cornell, A.; Kennedy, A. C.; Rolland, V.; Schultz, E. M.; Stanback, M.; Taff, C. C.; Albery, G. F.',\n",
       "   'author_corresponding': 'Sarah A Knutie',\n",
       "   'author_corresponding_institution': 'University of Connecticut',\n",
       "   'date': '2023-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'ecology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/23/2022.06.02.494595.source.xml',\n",
       "   'abstract': 'O_LISupplemental feeding can increase the overall health of animals but also can have variable effects on how animals defend themselves against parasites. However, the spatiotemporal effects of food supplementation on host-parasite interactions remain poorly understood, likely because large-scale, coordinated efforts are difficult.\\nC_LIO_LIHere, we introduce the Nest Parasite Community Science Project, which is a community-based science project that coordinates studies with bird nest box \"stewards\" from the public and scientific community. This project was established to understand broad ecological patterns between hosts and their parasites.\\nC_LIO_LIThe goal of this study was to determine the effect of food supplementation on eastern bluebirds (Sialia sialis) and their nest parasite community across the geographic range of the bluebirds from 2018-2021. We received 646 nests from 68 stewards in 26 states in the eastern United States. Nest box stewards reported whether or not they fed their bluebirds mealworms or suet, then followed the nesting success of the birds (number of eggs laid and hatched, percent hatched, number and percent fledged). We then identified and quantified parasites in the nests.\\nC_LIO_LIWe found that food supplementation increased fledgling numbers and proportional fledging success. The main nest parasite taxa were parasitic blow flies (Protocalliphora sialia), but a few nests contained fleas (Ceratophyllus idius, C. gallinae, Orchopeas leucopus) and mites (Dermanyssus spp. and Ornithonyssus spp.). Blow flies were primarily found at northern latitudes, where food supplementation affected blow fly abundance. However, the direction of this effect varied substantially in direction and magnitude across years. More stewards fed bluebirds at southern latitudes than at northern latitudes, which contradicted the findings of other community-based science projects.\\nC_LIO_LIOverall, food supplementation of birds was associated with increased host fitness but did not appear to play a consistent role in defense against these parasites across all years. Our study demonstrates the importance of coordinated studies across years and locations to understand the effects of environmental heterogeneity, including human-based food supplementation, on host-parasite dynamics. Studies during a single year or considering only a single population might not provide the necessary data to develop management strategies for species.\\nC_LI',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.06.02.494595',\n",
       "   'title': 'Understanding spatiotemporal effects of food supplementation on host-parasite interactions using community-based science',\n",
       "   'authors': 'Knutie, S. A.; Bahouth, R.; Bertone, M. A.; Webb, C.; Mehta, M.; Nahom, M.; Barta, R. M.; Ghai, S.; Love, A. C.; Horan, S.; Soldo, A.; Cochrane, E.; Bartholomew, J.; Cowan, E.; Bjerke, H.; Balenger, S. L.; Butler, M. W.; Cornell, A.; Kennedy, A. C.; Rolland, V.; Schultz, E. M.; Stanback, M.; Taff, C. C.; Albery, G. F.',\n",
       "   'author_corresponding': 'Sarah A Knutie',\n",
       "   'author_corresponding_institution': 'University of Connecticut',\n",
       "   'date': '2023-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'ecology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/23/2022.06.02.494595.source.xml',\n",
       "   'abstract': 'O_LISupplemental feeding can increase the overall health of animals but also can have variable effects on how animals defend themselves against parasites. However, the spatiotemporal effects of food supplementation on host-parasite interactions remain poorly understood, likely because large-scale, coordinated efforts are difficult.\\nC_LIO_LIHere, we introduce the Nest Parasite Community Science Project, which is a community-based science project that coordinates studies with bird nest box \"stewards\" from the public and scientific community. This project was established to understand broad ecological patterns between hosts and their parasites.\\nC_LIO_LIThe goal of this study was to determine the effect of food supplementation on eastern bluebirds (Sialia sialis) and their nest parasite community across the geographic range of the bluebirds from 2018-2021. We received 646 nests from 68 stewards in 26 states in the eastern United States. Nest box stewards reported whether or not they fed their bluebirds mealworms or suet, then followed the nesting success of the birds (number of eggs laid and hatched, percent hatched, number and percent fledged). We then identified and quantified parasites in the nests.\\nC_LIO_LIWe found that food supplementation increased fledgling numbers and proportional fledging success. The main nest parasite taxa were parasitic blow flies (Protocalliphora sialia), but a few nests contained fleas (Ceratophyllus idius, C. gallinae, Orchopeas leucopus) and mites (Dermanyssus spp. and Ornithonyssus spp.). Blow flies were primarily found at northern latitudes, where food supplementation affected blow fly abundance. However, the direction of this effect varied substantially in direction and magnitude across years. More stewards fed bluebirds at southern latitudes than at northern latitudes, which contradicted the findings of other community-based science projects.\\nC_LIO_LIOverall, food supplementation of birds was associated with increased host fitness but did not appear to play a consistent role in defense against these parasites across all years. Our study demonstrates the importance of coordinated studies across years and locations to understand the effects of environmental heterogeneity, including human-based food supplementation, on host-parasite dynamics. Studies during a single year or considering only a single population might not provide the necessary data to develop management strategies for species.\\nC_LI',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.06.05.494796',\n",
       "   'title': 'Development and application of an uncapped mRNA platform',\n",
       "   'authors': 'Zheng, X.; Liu, B.; Ni, P.; Cai, L.; Shi, X.; Ke, Z.; Zhang, S.; Hu, B.; Yang, B.; Xu, Y.; Long, W.; Fang, Z.; Wang, Y.; Zhang, W.; Xu, Y.; Wang, Z.; Pan, K.; Zhou, K.; Wang, H.; Geng, H.; Hu, H.; Liu, B.',\n",
       "   'author_corresponding': 'Binlei  Liu',\n",
       "   'author_corresponding_institution': 'National \"111\" Centre for Cellular Regulation and Molecular Pharmaceutics,Hubei Provincial Cooperative Innovation Centre of Industrial Fermentation, College of ',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'bioengineering',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.06.05.494796.source.xml',\n",
       "   'abstract': 'A novel uncapped mRNA platform was developed. Five lipid nanoparticle (LNP)-encapsulated mRNA constructs were made to evaluate several aspects of our platform, including transfection efficiency and durability in vitro and in vivo and the activation of humoral and cellular immunity in several animal models. The constructs were eGFP-mRNA-LNP (for enhanced green fluorescence mRNA), Fluc-mRNA-LNP (for firefly luciferase mRNA), S{delta}T-mRNA-LNP (for Delta strain SARS-CoV-2 spike protein trimer mRNA), gDED-mRNA-LNP (for truncated glycoprotein D mRNA coding ectodomain from herpes simplex virus type 2 (HSV2)) and gDFR-mRNA-LNP (for truncated HSV2 glycoprotein D mRNA coding amino acids 1[~]400). Quantifiable target protein expression was achieved in vitro and in vivo with eGFP-and Fluc-mRNA-LNP. S{delta}T-mRNA-LNP, gDED-mRNA-LNP and gDFR-mRNA-LNP induced both humoral and cellular immune responses comparable to those obtained by previously reported capped mRNA-LNP constructs. Notably, S{delta}T-mRNA-LNP elicited neutralizing antibodies in hamsters against the Omicron and Delta strains. Additionally, gDED-mRNA-LNP and gDFR-mRNA-LNP induced potent neutralizing antibodies in rabbits and mice. The mRNA constructs with uridine triphosphate (UTP) outperformed those with N1-methylpseudouridine triphosphate (N1m{psi}TP) in the induction of antibodies via S{delta}T-mRNA-LNP. Our uncapped, process-simplified, and economical mRNA platform may have broad utility in vaccines and protein replacement drugs.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.06.18.496668',\n",
       "   'title': 'Alterations of auditory sensory gating in mice with noise-induced tinnitus treated with nicotine and cannabis extract',\n",
       "   'authors': 'Ciralli, B.; Malfatti, T.; Lima, T. Z.; Silva, S. R. B.; Cederroth, C. R.; Leao, K. E.',\n",
       "   'author_corresponding': 'Katarina E Leao',\n",
       "   'author_corresponding_institution': 'Federal University of Rio Grande do Norte, Brazil',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.06.18.496668.source.xml',\n",
       "   'abstract': 'Tinnitus is a phantom sound perception affecting both auditory and limbic structures. The mechanisms of tinnitus remain unclear and it is debatable whether tinnitus alters attention to sound and the ability to inhibit repetitive sounds, a phenomenon also known as auditory gating. Here we investigate if noise exposure interferes with auditory gating and whether natural extracts of cannabis or nicotine could improve auditory pre-attentional processing in noise-exposed mice. We used 22 male C57BL/6J mice divided into noise-exposed (exposed to a 9-11 kHz narrow band noise for 1 hour) and sham (no sound during noise exposure) groups. Hearing thresholds were measured using auditory brainstem responses, and tinnitus-like behavior was assessed using Gap prepulse inhibition of acoustic startle. After noise exposure, mice were implanted with multi-electrodes in the dorsal hippocampus to assess auditory event-related potentials in response to paired clicks. The results showed that mice with tinnitus-like behavior displayed auditory gating of repetitive clicks, but with larger amplitudes and longer latencies of the N40 component of the aERP waveform. The combination of cannabis extract and nicotine improved auditory gating ratio in noise-exposed mice without permanent hearing threshold shifts. Lastly, the longer latency of the N40 component appears due to an increased sensitivity to cannabis extract in noise-exposed mice compared to sham mice. The study suggests that the altered central plasticity in tinnitus is more sensitive to the combined actions on the cholinergic and the endocannabinoid systems. Overall, the findings contribute to a better understanding of pharmacological modulation of auditory sensory gating.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.06.24.497542',\n",
       "   'title': 'Spatial replication can best advance our understanding of population responses to climate',\n",
       "   'authors': 'Compagnoni, A.; Evers, S.; Knight, T.',\n",
       "   'author_corresponding': 'Aldo  Compagnoni',\n",
       "   'author_corresponding_institution': 'Martin Luther University Halle Wittenberg; Helmholtz Centre for Environmental Research (UFZ)',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'ecology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.06.24.497542.source.xml',\n",
       "   'abstract': 'Understanding the responses of plant populations dynamics to climatic variability is frustrated by the need for long-term datasets. Here, we advocate for new studies that estimate the effects of climate by sampling replicate populations in locations with similar climate. We first use data analysis on spatial locations in the conterminous USA to assess how far apart spatial replicates should be from each other to minimize temporal correlations in climate. We find that on average spatial locations separated by 316 Km (SD = 149Km) have moderate (0.5) correlations in annual precipitation. Second, we use simulations to demonstrate that spatial replication can lead to substantial gains in the range of climates sampled during a given set of years so long as the climate correlations between the populations are at low to moderate levels. Third, we use simulations to quantify how many spatial replicates would be necessary to achieve the same statistical power of a single-population, long-term data set under different strengths and directions of spatial correlations in climate between spatial replicates. Our results indicate that spatial replication is an untapped opportunity to study the effects of climate on demography and to rapidly fill important knowledge gaps in the field of population ecology.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.06.24.497542',\n",
       "   'title': 'Spatial replication can best advance our understanding of population responses to climate',\n",
       "   'authors': 'Compagnoni, A.; Evers, S.; Knight, T.',\n",
       "   'author_corresponding': 'Aldo  Compagnoni',\n",
       "   'author_corresponding_institution': 'Martin Luther University Halle Wittenberg; Helmholtz Centre for Environmental Research (UFZ)',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'ecology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.06.24.497542.source.xml',\n",
       "   'abstract': 'Understanding the responses of plant populations dynamics to climatic variability is frustrated by the need for long-term datasets. Here, we advocate for new studies that estimate the effects of climate by sampling replicate populations in locations with similar climate. We first use data analysis on spatial locations in the conterminous USA to assess how far apart spatial replicates should be from each other to minimize temporal correlations in climate. We find that on average spatial locations separated by 316 Km (SD = 149Km) have moderate (0.5) correlations in annual precipitation. Second, we use simulations to demonstrate that spatial replication can lead to substantial gains in the range of climates sampled during a given set of years so long as the climate correlations between the populations are at low to moderate levels. Third, we use simulations to quantify how many spatial replicates would be necessary to achieve the same statistical power of a single-population, long-term data set under different strengths and directions of spatial correlations in climate between spatial replicates. Our results indicate that spatial replication is an untapped opportunity to study the effects of climate on demography and to rapidly fill important knowledge gaps in the field of population ecology.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.07.06.498926',\n",
       "   'title': 'Tracing back primed resistance in cancer via sister cells',\n",
       "   'authors': 'Dai, J.; Zheng, S.; Falco, M. M.; Bao, J.; Eriksson, J.; Pikkusaari, S.; Forsten, S.; Jiang, J.; Wang, W.; Gao, L.; Perez-Villatoro, F.; Dufva, O.; Saeed, K.; Wang, Y.; Amiryousefi, A.; Färkkilä, A.; Mustjoki, S.; Kauppi, L.; Tang, J.; Vähärautio, A.',\n",
       "   'author_corresponding': 'Anna  Vähärautio',\n",
       "   'author_corresponding_institution': 'University of Helsinki',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'cancer biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2022.07.06.498926.source.xml',\n",
       "   'abstract': 'Exploring non-genetic evolution of cell states during cancer treatments has become attainable by recent advances in lineage-tracing methods. However, transcriptional changes that drive cells into resistant fates may be subtle, necessitating high resolution analysis. We developed ReSisTrace that uses shared transcriptomic features of synchronised sister cells to predict the states that prime treatment resistance. We applied ReSisTrace in ovarian cancer cells perturbed with olaparib, carboplatin or natural killer (NK) cells. The pre-resistant phenotypes were defined by cell cycle and proteostatic features, reflecting the traits enriched in the upcoming subclonal selection. Furthermore, DNA repair deficiency rendered cells susceptible to both DNA damaging agents and NK killing in a context-dependent manner. Finally, we leveraged the pre-resistance profiles to predict and validate small molecules driving cells to sensitive states prior to treatment. In summary, ReSisTrace resolves pre-existing transcriptional features of treatment vulnerability, facilitating both molecular patient stratification and discovery of synergistic pre-sensitizing therapies.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.07.13.499857',\n",
       "   'title': 'Multi-omics analyses identify transcription factor interplay in corneal epithelial fate determination and disease',\n",
       "   'authors': 'Smits, J.; Cunha, D. L.; Qu, J.; Owen, N.; Latta, L.; Szentmary, N.; Seitz, B.; Roux, L. N.; Moosajee, M.; Aberdam, D.; van Heeringen, S. J.; Zhou, H.',\n",
       "   'author_corresponding': 'Huiqing Zhou',\n",
       "   'author_corresponding_institution': 'Department of Molecular Developmental Biology, Faculty of Science, Radboud Institute for Molecular Life Sciences, Radboud University, Nijmegen, The Netherlands',\n",
       "   'date': '2023-08-23',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/23/2022.07.13.499857.source.xml',\n",
       "   'abstract': 'The transparent corneal epithelium in the eye is maintained through the homeostasis regulated by limbal stem cells, while the non-transparent epidermis relies on epidermal keratinocytes for renewal. Despite their cellular similarities, the precise cell fates of these two types of epithelial stem cells, which give rise to functionally distinct epithelia, remain unknown. We performed a multi-omics analysis of human limbal stem cells from the cornea and keratinocytes from the epidermis, and characterized their molecular signatures, highlighting their similarities and differences. Through gene regulatory network analyses, we identified shared and cell type-specific transcription factors that define specific cell fates, and established their regulatory hierarchy. Single-cell RNA-seq analyses of the cornea and the epidermis confirmed these shared and cell type-specific transcription factors. Notably, the shared and limbal stem cell-specific transcription factors can cooperatively target genes associated with corneal opacity. Importantly, we discovered that FOSL2, a direct PAX6 target gene, is a novel candidate associated with corneal opacity, and it regulates genes implicated in corneal diseases. By characterizing molecular signatures, our study unveils the regulatory circuitry governing the limbal stem cell fate and its association with corneal opacity.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.07.21.501061',\n",
       "   'title': 'Climate and Morphology Drive Breeding Periods in Frogs',\n",
       "   'authors': \"Juarez, B. H.; O'Connell, L. A.\",\n",
       "   'author_corresponding': 'Bryan H Juarez',\n",
       "   'author_corresponding_institution': 'Stanford University',\n",
       "   'date': '2023-08-27',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'ecology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/27/2022.07.21.501061.source.xml',\n",
       "   'abstract': 'AimClimate can have profound effects on reproductive behavior and physiology, especially in ectothermic animals. Breeding periods in amphibians have received little attention despite amphibian reliance on climate and water due to their reproductive biology and ecological diversity. The aim of this study is to determine how global climate impacts breeding periods in ectothermic animals through physiology, body size, and microhabitat.\\n\\nLocationAll continents, except Antarctica.\\n\\nTime periodBreeding periods and climate both dating as far back as 1970.\\n\\nMajor taxa studied497 (7%) anuran species of 41 (76%) families.\\n\\nMethodsWe used phylogenetic comparative methods to analyze a global dataset of frog breeding periods, climate, body size, and microhabitat for 497 species.\\n\\nResultsWe found support for a global latitudinal gradient of breeding periods which are longer in the warmer, wetter tropics while shorter in the colder, dryer temperate zone. Latitudinal and non-latitudinal global patterns of breeding period were composites of the same patterns in the temperate and tropical zones. However, the effect of climate and body size in each zone is unique. Breeding periods displayed weak phylogenetic signal and were similar across microhabitats.\\n\\nMain conclusionsBreeding periods show a global latitudinal gradient but this concept breaks down within the tropical zone. Our results are consistent with the importance of physiology in driving breeding periods and we describe how breeding period and body size may exhibit trade-offs which make latitudinal gradients context-dependent. Our results set within an ecophysiological framework have broad implications for understanding reproductive diversity in other ectothermic organisms.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.07.22.501082',\n",
       "   'title': 'The causal analysis of operant selection by means of counterfactual models',\n",
       "   'authors': 'Borgstede, M.; Anselme, P.',\n",
       "   'author_corresponding': 'Matthias Borgstede',\n",
       "   'author_corresponding_institution': 'University of Bamberg: Otto-Friedrich-Universitat Bamberg',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'animal behavior and cognition',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.07.22.501082.source.xml',\n",
       "   'abstract': 'We present a new methodology to partition different sources of behavior change within a selectionist framework based on the Price equation - the Multilevel Model of Behavioral Selection (MLBS). The MLBS provides a theoretical and formal background to disentangle behavior changes due to selection effects (e.g., \"reinforcement\") from behavior changes due to non-selection effects (e.g., \"skill acquisition\").\\n\\nThe MLBS allows for the construction of theory-based causal models that can be used to disentangle different sources of behavior change, even if they cannot be isolated experimentally. Using counterfactual analysis, we show how to quantify the causal effects of selection and non-selection sources of behavior change for a given subject. We further demonstrate how these effects may be tested for statistical significance using a simulation-based permutation test framework. The proposed method is especially useful in situations where confounding variables are difficult to control, such as changes in satiation, motivation or skill within individuals, or environmental fluctuations as they frequently occur in naturalistic settings.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.07.27.501804',\n",
       "   'title': 'Global signal topography in the human brain differs systematically across the lifespan',\n",
       "   'authors': 'Nomi, J.; Bzdok, D.; Li, J.; Bolt, T.; Chang, C.; Kornfeld, S.; Goodman, Z.; Yeo, B. T. T.; Spreng, R. N.; Uddin, L.',\n",
       "   'author_corresponding': 'Jason Nomi',\n",
       "   'author_corresponding_institution': 'University of California Los Angeles',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.07.27.501804.source.xml',\n",
       "   'abstract': 'The global signal (GS) in resting-state fMRI, known to contain artifacts and non-neuronal physiological signals, also contains important neural information related to individual state and trait characteristics. Here we show distinct linear and curvilinear lifespan patterns of GS topography in a cross-sectional lifespan sample, demonstrating its importance for consideration in studies of development and aging. Subcortical brain regions such as the thalamus and putamen show linear associations with the GS across the lifespan. The thalamus has stronger coupling in older-age individuals compared with younger-aged individuals, while the putamen has stronger coupling in younger individuals compared with older individuals. The subcortical nucleus basalis shows a u-shaped pattern similar to cortical regions within the lateral frontoparietal network and dorsal attention network, where coupling with the GS is stronger at early and old age, with weaker coupling in middle age. This differentiation in coupling strength between subcortical and cortical brain activity across the lifespan supports a dual-layer model of GS composition, where subcortical aspects of the GS are differentiated from cortical aspects of the GS. We find that these subcortical-cortical contributions to the GS depend strongly on the lifespan stage of individuals. Our findings demonstrate how neurobiological information within the GS differs across development and highlight the need to carefully consider whether or not to remove this signal when investigating age-related functional differences in the brain.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.07.28.501855',\n",
       "   'title': 'A measure of the trigeminal blink reflex to parametric variation of mechanical stimulation',\n",
       "   'authors': 'Kaiser, E. A.; Haggerty, E. B.; Garner, D. P.; Bunya, V. Y.; Aguirre, G. K.',\n",
       "   'author_corresponding': 'Geoffrey Karl Aguirre',\n",
       "   'author_corresponding_institution': 'Department of Neurology, Perelman School of Medicine, University of Pennsylvania',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.07.28.501855.source.xml',\n",
       "   'abstract': 'The primary goal of this study was to develop a parametric model that relates variation in trigeminal stimulation to properties of the blink response. We measured blink responses in 17 healthy, adult participants to air puffs delivered to the cornea and lateral canthus at five different, log-spaced intensities (3.5 - 60 PSI). Lid position over time was decomposed into amplitude, velocity, and shape components. We found that blink amplitude was lawfully related to log stimulus intensity, with the relationship well described by a Weibull cumulative density function. The parameters of the Weibull fit correspond to the slope of the function and the stimulus intensity required to produce half of a maximal blink response (the half-response threshold). There was a reliable decrease in half-response threshold for the contralateral as compared to ipsilateral blink response. This decrease was consistent across participants despite substantial individual differences in the half-response threshold and slope parameters of the overall sensitivity function, suggesting that the laterality effect arises in the neural circuit subsequent to individual differences in sensitivity. Overall, we find that graded mechanical stimulation of the trigeminal afferents elicits a graded response that is well described by a simple parametric model. We discuss the application of parametric measurements of the blink response to the detection of group differences in trigeminal sensitivity.\\n\\nImpact statementO_LIThere has been limited study of how variation in stimulus intensity influences the blink response. We describe how variation in the intensity of an air-puff stimulus relates to the amplitude of the blink response using a two-parameter model. The parameters of this model allow us to measure individual differences in the sensitivity of the trigeminal blink response, which may be used in future studies to identify population differences in this system.\\nC_LI',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.07.29.502042',\n",
       "   'title': 'Neonicotinoids, more than herbicides, land use, and climate, drive recent butterfly declines in the American Midwest',\n",
       "   'authors': 'Van Deynze, B.; Swinton, S. M.; Hennessy, D. A.; Haddad, N.; Ries, L.',\n",
       "   'author_corresponding': 'Braeden Van Deynze',\n",
       "   'author_corresponding_institution': 'University of Washington',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'ecology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.07.29.502042.source.xml',\n",
       "   'abstract': 'Mounting evidence shows insects are in decline globally. Habitat loss, climate change, and pesticides have all been implicated, but their relative effects have never been evaluated in a comprehensive large-scale study. We harmonized 17 years of land use, climate, multiple classes of pesticides and butterfly data across 60 counties in five states in the US Midwest. We find community-wide declines in butterfly abundance to be most strongly associated with pesticides in general and the use of neonicotinoid-treated seeds in particular. This included the abundance of the migratory monarch (Danaus plexippus), whose decline is the focus of intensive debate and public concern. Insect declines cannot be understood without comprehensive data on all putative drivers, and the 2015 cessation of neonicotinoid data releases in the US will impede future research.\\n\\nOne Sentence SummaryShifts in insecticide use towards neonicotinoid-treated seeds are associated with an 8 percent decline in butterfly species diversity across the American Corn Belt.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.02.502476',\n",
       "   'title': 'Nucleation and spreading rejuvenate polycomb domains every cell cycle',\n",
       "   'authors': 'Veronezi, G. M. B.; Ramachandran, S.',\n",
       "   'author_corresponding': 'Srinivas Ramachandran',\n",
       "   'author_corresponding_institution': 'University of Colorado School of Medicine',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'molecular biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.08.02.502476.source.xml',\n",
       "   'abstract': 'Gene repression by the polycomb pathway is essential for proper development in metazoans. Polycomb domains, characterized by trimethylation of histone H3 (H3K27me3), carry the memory of repression through successive cell divisions, and hence, need to be maintained to counter dilution during replication of parental H3K27me3 with unmodified nascent H3. Yet, how locus-specific H3K27me3 is maintained through replication is unknown. To understand H3K27me3 recovery post-replication, we first defined nucleation sites within each polycomb domain in mouse embryonic stem cells. Then, to track H3K27me3 dynamics in unperturbed cells during replication, we developed CUT&Flow, which maps H3K27me3 domains as a function of the cell cycle stage. Using CUT&Flow, we show that post-replication rejuvenation of polycomb domains occurs by nucleation and spreading, using the same nucleation sites used during de novo domain formation. By using subunit-specific inhibitors of the PRC2 complex, we find that PRC2 need not rely on pre-existing H3K27me3 to target nucleation sites post-replication. The timing of nucleation relative to the cell cycle reflects the combination of replication timing and H3K27me3 deposition kinetics derived from de novo domain formation. Thus, competition between H3K27me3 deposition and nucleosome turnover drives both de novo domain formation and maintenance during every cell cycle.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.02.502495',\n",
       "   'title': 'FMRP-Regulated Alternative Splicing is Multifactorial and Resembles Splicing Control by MBNL1',\n",
       "   'authors': 'Jung, S.; Shah, S.; Han, G.; Richter, J. D.',\n",
       "   'author_corresponding': 'Joel D Richter',\n",
       "   'author_corresponding_institution': 'University of Massachusetts Chan Medical School',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc0_ng',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.08.02.502495.source.xml',\n",
       "   'abstract': 'Fragile X Syndrome (FXS) is a neurodevelopmental disorder that is often modeled in Fmr1 knockout mice where the RNA binding protein FMRP is absent. Here we show that in Fmr1-deficient mice, RNA mis-splicing occurs in several brain regions and peripheral tissues. To assess molecular mechanisms of splicing mis-regulation, we employed N2A cells depleted of Fmr1. In the absence of FMRP, RNA-specific exon skipping events are linked to the splicing factors hnRNPF, PTBP1, and MBNL1. FMRP regulates the translation of Mbnl1 mRNA as well as Mbnl1 RNA auto-splicing. Elevated Mbnl1 auto-splicing in FMRP-deficient cells results in the loss of a nuclear localization signal (NLS)-containing exon. This in turn alters the nucleus-to-cytoplasm ratio of MBNL1. This re-distribution of MBNL1 isoforms in Fmr1-deficient cells could result in downstream splicing changes in other RNAs. Indeed, further investigation revealed that splicing distruptions resulting from Fmr1 depletion could be rescued by overexpression of nuclear MBNL1. Altered Mbnl1 auto-splicing also occurs in human FXS post-mortem brain. These data suggest that FMRP-controlled translation and RNA processing may cascade into a general dys-regulation of splicing in Fmr1-deficient cells.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.03.502586',\n",
       "   'title': 'SNP discovery by exome capture and resequencing in a pea genetic resource collection',\n",
       "   'authors': 'Aubert, G.; Kreplak, J.; Leveugle, M.; Duborjal, H.; Klein, A.; Boucherot, K.; Vieille, E.; Chabert-Martinello, M.; Cruaud, C.; Bourion, V.; Lejeune-Henaut, I.; Pilet-Nayel, M.-L.; Bouchenak-Khelladi, Y.; Francillonne, N.; Tayeh, N.; Pichon, J.-P.; Riviere, N.; Burstin, J.',\n",
       "   'author_corresponding': 'Grégoire AUBERT',\n",
       "   'author_corresponding_institution': 'Agroécologie, INRAE, Institut Agro, Univ. Bourgogne, Univ. Bourgogne Franche-Comté, F-21000 Dijon, France',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '5',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'genomics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.08.03.502586.source.xml',\n",
       "   'abstract': 'Background & SummaryIn addition to being the model plant used by Mendel1 to establish genetic laws, pea (Pisum sativum L.) is a major pulse crop cultivated in many temperate regions of the world. In order to face new challenges imposed particularly by global climate change and new regulations targeted at reducing chemical inputs, pea breeders have to take advantage of the genetic diversity present in the Pisum genepool to develop improved, resilient varieties. The aim of this study was to assess the genetic diversity of a pea germplasm collection and allow genome-wide association studies using this collection.\\n\\nTo be able to perform genome-wide association approaches with high resolution, genotyping with a large set of genetic markers such as Single Nucleotide Polymorphism (SNP) markers well-spread over the genome is required. Rapid advances in second-generation sequencing technologies and the development of bioinformatic tools have revolutionized the access to and the characterization of available genetic diversity. High-density, high-throughput genotyping has been possible for a large number of species, including those with large and complex genomes2 such as pea which genome size is estimated to be 4.45 Gb3. In this study, which is part of the PeaMUST project4, we used a target capture technology based on pea transcriptome sequences to generate exome-enriched genomic libraries that were further subjected to Illumina sequencing in paired-end mode. This methodology was chosen because whole-genome resequencing is relatively expensive for species with large genomes and because capturing genetic variations in repeated non-coding regions is difficult to achieve or to interpret5. Whole-exome sequencing represented an interesting alternative that focused on coding regions only6,7. Mapping the obtained reads on the reference pea genome sequence enabled the discovery of an abundant set of SNPs. The development of this resource is a crucial cornerstone in research and breeding projects towards boosting the improvement of pea production and quality.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.10.503505',\n",
       "   'title': \"Membrane remodeling properties of the Parkinson's disease protein LRRK2\",\n",
       "   'authors': 'Wang, X.; Espadas, J.; Wu, Y.; Cai, S.; Ge, J.; Shao, L.; Roux, A.; De Camilli, P.',\n",
       "   'author_corresponding': 'Pietro De Camilli',\n",
       "   'author_corresponding_institution': 'Howard Hughes Medical Institute, Yale University School of Medicine',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.08.10.503505.source.xml',\n",
       "   'abstract': 'Mutations in Leucine-rich repeat kinase 2 (LRRK2) are responsible for late-onset autosomal dominant Parkinsons disease (PD). LRRK2 has been implicated in a wide range of physiological processes including membrane repair in the endolysosomal system. Here, using cell free systems, we report that purified LRRK2 directly binds acidic lipid bilayers with a preference for highly curved bilayers. While this binding is nucleotide independent, LRRK2 can also deform low curvature liposomes into narrow tubules in a guanylnucleotide-dependent but ATP-independent way. Moreover, assembly of LRRK2 into scaffolds at the surface of lipid tubules can constrict them. We suggest that an interplay between the membrane remodeling and signaling properties of LRRK2 may be key to its physiological function. LRRK2, via its kinase activity, may achieve its signaling role at sites where membrane remodeling occurs.\\n\\nSignificance StatementLRRK2 is one of the most frequently mutated gene in familial Parkinsons disease. While much has been learned about its molecular properties, upstream regulators and protein substrates of its kinase activity, its precise function remains unclear. Recent evidence has pointed to a role of LRRK2 in membrane repair in the endo/lysosomal system. Here we show that purified LRRK2 has membrane remodeling properties. We suggest that its ability to sense and induce membrane curvature may be key to its function in membrane dynamics. These properties may help coordinate a direct role of LRRK2 at the membrane interface with its the signaling role of its kinase domain.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.10.503458',\n",
       "   'title': 'LRTK: A platform agnostic toolkit for linked-read analysis of both human genomes and metagenomes',\n",
       "   'authors': 'Yang, C.; Zhang, Z.; Huang, Y.; Liao, X.; Liao, H.; Xiao, J.; Veldsman, W. P.; Yin, K.; Fang, X.; Zhang, L.',\n",
       "   'author_corresponding': 'Lu Zhang',\n",
       "   'author_corresponding_institution': 'Hong Kong Baptist University',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.08.10.503458.source.xml',\n",
       "   'abstract': 'Linked-read sequencing technologies generate high base quality reads that contain extrapolative information on long-range DNA connectedness. These advantages of linked-read technologies are well known and has been demonstrated in many human genomic and metagenomic studies. However, existing linked-read analysis pipelines (e.g., Long Ranger) were primarily developed to process sequencing data from the human genome and are not suited for analyzing metagenomic sequencing data. Moreover, linked-read analysis pipelines are typically limited to one specific sequencing platform. To address these limitations, we present the Linked-Read ToolKit (LRTK), a unified and versatile toolkit for platform agnostic processing of linked-read sequencing data from both human genomes and metagenomes. LRTK provides functions to perform linked-read simulation, barcode error correction, read cloud assembly, barcode-aware read alignment, reconstruction of long DNA fragments, taxonomic classification and quantification, as well as barcode-assisted genomic variant calling and phasing. LRTK has the ability to process multiple samples automatically, and provides the user with the option to generate reproducible reports during processing of raw sequencing data and at multiple checkpoints throughout downstream analysis. We applied LRTK on two benchmarking and three real linked-read data sets from both the human genome and metagenome. We showcase LRTKs ability to generate comparative performance results from the preceding benchmark study and to report these results in publication-ready HTML document plots. LRTK provides comprehensive and flexible modules along with an easy-to-use Python-based workflow for processing linked-read sequencing datasets, thereby filling the current gap in the field caused by platform-centric genome-specific linked-read data analysis tools.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.13.503807',\n",
       "   'title': 'The ChlorON Series: Turn-on Fluorescent Protein Sensors for Imaging Labile Chloride in Living Cells',\n",
       "   'authors': 'Tutol, J. N.; Ong, W. S. Y.; Phelps, S. M.; Peng, W.; Goenawan, H.; Dodani, S. C.',\n",
       "   'author_corresponding': 'Sheel C. Dodani',\n",
       "   'author_corresponding_institution': 'The University of Texas at Dallas',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'biochemistry',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.08.13.503807.source.xml',\n",
       "   'abstract': 'Beyond its role as the queen electrolyte, chloride can also serve as an allosteric regulator or even a signaling ion. To illuminate this essential anion across such a spectrum of biological processes, researchers have relied on fluorescence imaging with genetically encoded sensors. In large part, these have been derived from the green fluorescent protein found in the jellyfish Aequorea victoria. However, a standalone sensor with a turn-on intensiometric response at physiological pH has yet to be reported. Here, we address this technology gap by building on our discovery of mNeonGreen (mNG) derived from lanYFP found in the cephalochordate Branchiostoma lanceolatum. Targeted engineering of two non-coordinating residues in the chloride binding pocket of mNG unlocks the ChlorON series. In vitro spectroscopy reveals that the binding of chloride tunes the chromophore environment to give rise to the turn-on response. We further showcase how this unique sensing mechanism can be exploited for directly imaging labile chloride in living cells with spatial and temporal resolution, accelerating the path forward for fundamental and translational aspects of chloride biology.\\n\\n\\n\\nO_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=61 SRC=\"FIGDIR/small/503807v2_ufig1.gif\" ALT=\"Figure 1\">\\nView larger version (23K):\\norg.highwire.dtl.DTLVardef@36cc4org.highwire.dtl.DTLVardef@d9dccborg.highwire.dtl.DTLVardef@12ec0fforg.highwire.dtl.DTLVardef@80c9db_HPS_FORMAT_FIGEXP  M_FIG C_FIG',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.16.504114',\n",
       "   'title': 'Piezo1 mechanosensing regulates integrin-dependent chemotactic migration in human T cells',\n",
       "   'authors': \"Liu, C. S. C.; Mandal, T.; Biswas, P.; Hoque, M. A.; Bandopadhyay, P.; Sinha, B. P.; Sarif, J.; D'Rozario, R.; Sinha, D. K.; Sinha, B.; Ganguly, D.\",\n",
       "   'author_corresponding': 'Dipyaman Ganguly',\n",
       "   'author_corresponding_institution': 'CSIR-Indian Institute of Chemical Biology, India',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'immunology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.08.16.504114.source.xml',\n",
       "   'abstract': 'T cells are crucial for efficient antigen-specific immune responses and thus their migration within the body, to inflamed tissues from circulating blood or to secondary lymphoid organs, play a very critical role. T cell extravasation in inflamed tissues depends on chemotactic cues and interaction between endothelial adhesion molecules and cellular integrins. A migrating T cell is expected to sense diverse external and membrane-intrinsic mechano-physical cues, but molecular mechanisms of such mechanosensing in cell migration are not established. We explored if the professional mechanosensor Piezo1 play any role during integrin-dependent chemotaxis of human T cells. We found that deficiency of Piezo1 in human T cells interfered with integrin-dependent cellular motility on ICAM-1-coated surface. Piezo1 recruitment at the leading edge of moving T cells is dependent on and follows focal adhesion formation at the leading edge and local increase in membrane tension on chemokine receptor activation. Piezo1 recruitment and activation, followed by calcium influx and calpain activation, in turn are crucial for the integrin LFA-1 recruitment at the leading edge of the chemotactic human T cells. Thus we find that Piezo1 activation in response to local mechanical cues constitutes a membrane-intrinsic component of the  outside-in signaling in human T cells, migrating in response to chemokines, that mediates integrin recruitment to the leading edge.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.16.504146',\n",
       "   'title': 'Polymer brush bilayer under stationary shear: A joint DFT, scaling theory and MD',\n",
       "   'authors': 'Edwards, M. J.',\n",
       "   'author_corresponding': 'Mike John Edwards',\n",
       "   'author_corresponding_institution': 'Leibniz Institute of Polymer Research Dresden',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.08.16.504146.source.xml',\n",
       "   'abstract': 'The problem of polymer brush bilayer under stationary shear is studied by using the DFT, the scaling theory and MD simulations. Both theory and simulations confirm that the shear stress follows the universal power law [Formula] for the brush bilayers with interpenetration and in the absence of the interpenetration, the shear stress scales linearly with the shear rate. It is also revealed that the presence of explicit solvent molecules prevents the brushes to form and interpenetration zone and therefore with explicit solvents the shear stress scales linearly with shear rate. Therefore, this study strongly confirms that there is no sublinear regime in the world of polymer brush bilayer, neither by solvents nor by hydrodynamic effects. As long as there is an interpenetration zone, the superlinear regime dominates and in the absence of the interpenetration zone the linear regime dominates. Therefore, polymer brushes are not a good candidate for lubrication and all works suggesting that this system is a super lubricant are completely wrong.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.16.504030',\n",
       "   'title': 'Centripetal integration of past events by hippocampal astrocytes and its regulation by the locus coeruleus',\n",
       "   'authors': 'Rupprecht, P.; Duss, S. N.; Becker, D.; Lewis, C. M.; Bohacek, J.; Helmchen, F.',\n",
       "   'author_corresponding': 'Peter Rupprecht',\n",
       "   'author_corresponding_institution': 'University of Zurich',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2022.08.16.504030.source.xml',\n",
       "   'abstract': 'An essential feature of neurons is their ability to centrally integrate information from their dendrites. The activity of astrocytes, in contrast, has been described as mostly uncoordinated across cellular compartments without clear central integration. Here, we describe conditional centripetal integration as a principle of how astrocytes integrate calcium signals from their distal processes to induce somatic activation. We found in mouse hippocampus that global astrocytic activity, as recorded with population calcium imaging, is well explained as a leaky integration of past neuronal and behavioral events on a timescale of seconds. Salient past events, indicated by pupil dilations, facilitated propagation of calcium signals from distal processes to the soma on this slow timescale. Centripetal propagation was reproduced by optogenetic activation of the locus coeruleus, a key regulator of arousal, and reduced by pharmacological inhibition of 1-adrenergic receptors. Together, our results establish astrocytes as computational units of the brain that slowly and conditionally integrate calcium signals to activate their somata upon behaviorally relevant events.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.26.505490',\n",
       "   'title': 'Metabolic diversity in commensal protists regulates intestinal immunity and trans-kingdom competition',\n",
       "   'authors': 'Gerrick, E. R.; Zlitni, S.; West, P. T.; Carter, M. M.; Mechler, C. M.; Olm, M. R.; Caffrey, E. B.; Li, J. A.; Higginbottom, S. K.; Severyn, C. J.; Kracke, F.; Spormann, A. M.; Sonnenburg, J. L.; Bhatt, A. S.; Howitt, M. R.',\n",
       "   'author_corresponding': 'Michael R Howitt',\n",
       "   'author_corresponding_institution': 'Stanford Medical School',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.08.26.505490.source.xml',\n",
       "   'abstract': 'The microbiota influences intestinal health and physiology, yet the contributions of commensal protists to the gut environment have been largely overlooked. Here, we identified several new rodent- and human-associated parabasalid protists. Genomic and metabolomic analyses of murine parabasalids from the genus Tritrichomonas revealed species-level differences in the excretion of the metabolite succinate. This metabolic dissimilarity results in distinct small intestinal immune responses during protist colonization. Metabolic differences between Tritrichomonas species also determine their ecological niche within the microbiota. By manipulating dietary fibers and developing in vitro protist culture, we show that different parabasalid species preferentially rely on dietary polysaccharides or mucus glycans. These polysaccharide preferences create trans-kingdom competition with specific commensal bacteria, which affects intestinal immunity in a diet-dependent manner. Our findings reveal unappreciated diversity in commensal parabasalids, elucidate differences in commensal protist metabolism, and suggest how dietary interventions could regulate their impact on gut health.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.08.24.505123',\n",
       "   'title': 'Rewiring cattle movements to limit infection spread',\n",
       "   'authors': 'Morel-Journel, T.; Ezanno, P.; Vergu, E.',\n",
       "   'author_corresponding': 'Thibaut Morel-Journel',\n",
       "   'author_corresponding_institution': 'INRAE',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'zoology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.08.24.505123.source.xml',\n",
       "   'abstract': 'The cattle tracing databases set up over the past decades in Europe have become major resources for representing demographic processes of livestock and assessing potential risk of infections spreading by trade. The herds registered in these databases are nodes of a network of commercial movements, which can be altered to lower the risk of disease transmission. In this study, we develop an algorithm aimed at reducing the number of infected animals and herds, by rewiring specific movements responsible for trade flows from high- to low-prevalence herds. The algorithm is coupled with a generic computational model describing infection spread within and between herds, based on data extracted from the French cattle movement tracing database (BDNI). This model is used to simulate a wide array of infections, with either a recent outbreak (epidemic) or an outbreak that occurred five years earlier (endemic), on which the performances of the rewiring algorithm are explored. Results highlight the effectiveness of rewiring in containing infections to a limited number of herds for all scenarios, but especially if the outbreak is recent and if the estimation of disease prevalence is frequent. Further analysis reveal that the key parameters of the algorithm affecting infection outcome vary with the infection parameters. Allowing any animal movement from high to low-prevalence herds reduces the effectiveness of the algorithm in epidemic settings, while frequent and fine-grained prevalence assessments improve the impact of the algorithm in endemic settings. According to our results, our approach focusing on a few commercial movements is expected to lead to substantial improvements in the control of a targeted disease, although changes in the network structure should be monitored for potential vulnerabilities to other diseases. Due to its generality, the developed rewiring algorithm could be applied to any network of controlled individual movements liable to spread disease.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.09.07.506931',\n",
       "   'title': 'PAMR1 negatively impacts cell proliferation and migration of Human Colon Cancer HT29 Cell Line',\n",
       "   'authors': 'Haymour, L.; Chaunavel, A.; Diab-Assaf, M.; Maftah, A.; Legardinier, S.',\n",
       "   'author_corresponding': 'Sebastien Legardinier Sr.',\n",
       "   'author_corresponding_institution': 'University of Limoges',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'cancer biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.09.07.506931.source.xml',\n",
       "   'abstract': 'Colorectal cancer (CRC) is becoming one of the most prevalent cancers worldwide. Among cancers, it ranks the third place in terms of incidence and the second in terms of mortality. Even though immunological test allows fast and easy diagnostic method, there is no specific and reliable methods for early detection of CRC. Despite different treatments, high risk of re-occurrence is associated with advanced and metastatic CRC stages. An exhaustive knowledge on specific biomarkers or molecular actors involved in CRC could help to eradicate tumors or limit cancer recurrence. In this study, we focused on PAMR1 (Peptidase Domain Containing Associated with Muscle Regeneration 1), which is already considered as a tumor suppressor in breast and cervical cancers. In silico analysis of RNASeq data showed that PAMR1 was significantly downregulated in CRC tissues compared to their adjacent normal ones, as well as in cervical cancer. Our analysis showed that this downregulation, probably due to promoter hypermethylation, such as in breast cancer tissues, appeared in the four cancer stages as early as the first stage. In consistency with in silico analyses, the expression of PAMR1 was found to be lower at the transcript and protein levels in CRC tissue samples compared to normal ones, as well as in different CRC cell lines (HCT116, HT29, and SW620) compared to normal colon cell line (CCD841CoN). To understand the role of PAMR1 in CRC cancer, recombinant purified PAMR1 or concentrated secretome from CHO overexpressing PAMR1 were used to exogenously treat CRC cell lines with a focus on HT-29 cells as well as Hela cervical cancer cell line known to be sensitive to PAMR1. Transient or stable transfections were also performed to determine the impact of PAMR1 overexpression in HT29 and/or HeLa cells. In this study, we finally showed that presence of PAMR1 could reduce both cell proliferation and cell migration with a positive correlation between these biological effects and PAMR1s quantity. This implies that PAMR1 expresses anti-proliferative and anti-migrative effects in CRC. Further studies to be done in order to confirm the tumor suppressive role of PAMR1 in CRC.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.09.13.507772',\n",
       "   'title': 'No need for extensive artifact rejection for ICA - A multi-study evaluation on stationary and mobile EEG datasets',\n",
       "   'authors': 'Klug, M.; Berg, T.; Gramann, K.',\n",
       "   'author_corresponding': 'Marius Klug',\n",
       "   'author_corresponding_institution': 'TU Berlin',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.09.13.507772.source.xml',\n",
       "   'abstract': 'ObjectiveElectroencephalography (EEG) studies increasingly make use of more ecologically valid experimental protocols involving mobile participants who actively engage with their environment (MoBI; Gramann et al., 2011). These mobile paradigms lead to increased artifacts in the recorded data that are often treated using Independent Component Analysis (ICA). When analyzing EEG data, especially in a mobile context, removing samples regarded as artifactual is a common approach before computing ICA. Automatic tools for this exist, such as the automatic sample rejection of the AMICA algorithm (Palmer et al., 2011), but the impact of the two factors movement intensity and the automatic sample rejection has not been systematically evaluated yet.\\n\\nApproachWe computed AMICA decompositions on eight datasets from six open-access studies with varying degrees of movement intensities using increasingly conservative sample rejection criteria. We evaluated the subsequent decomposition quality in terms of the component mutual information, the amount of brain, muscle, and \"other\" components, the residual variance of the brain components, and an exemplary signal-to-noise ratio.\\n\\nMain resultsWe found that increasing movements of participants led to decreasing decomposition quality for individual datasets but not as a general trend across all movement intensities. The cleaning strength had less impact on decomposition results than anticipated, and moderate cleaning of the data resulted in the best decompositions.\\n\\nSignificanceOur results indicate that the AMICA algorithm is very robust even with limited data cleaning. Moderate amounts of cleaning such as 5 to 10 iterations of the AMICA sample rejection with 3 standard deviations as the threshold will likely improve the decomposition of most datasets, irrespective of the movement intensity.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.09.25.509376',\n",
       "   'title': 'Comparison of gene clustering criteria reveals intrinsic uncertainty in pangenome analyses',\n",
       "   'authors': 'Manzano-Morales, S.; Liu, Y.; Gonzalez-Bodi, S.; Huerta-Cepas, J.; Iranzo, J.',\n",
       "   'author_corresponding': 'Jaime Iranzo',\n",
       "   'author_corresponding_institution': 'Centro de Biotecnologia y Genomica de Plantas, Universidad Politecnica de Madrid (UPM) - Instituto Nacional de Investigacion y Tecnologia Agraria y Alimentaria ',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'genomics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.09.25.509376.source.xml',\n",
       "   'abstract': 'BackgroundA key step for comparative genomics is to group open reading frames into functionally and evolutionarily meaningful gene clusters. Gene clustering is complicated by intraspecific duplications and horizontal gene transfers, that are frequent in prokaryotes. In consequence, gene clustering methods must deal with a trade-off between identifying vertically transmitted representatives of multi-copy gene families (recognizable by synteny conservation) and retrieving complete sets of species-level orthologs. We studied the implications of adopting homology, orthology, or synteny conservation as formal criteria for gene clustering by performing comparative analyses of 125 prokaryotic pangenomes.\\n\\nResultsClustering criteria affect pangenome functional characterization, core genome inference, and reconstruction of ancestral gene content to different extents. Species-wise estimates of pangenome and core genome sizes change by the same factor when using different clustering criteria, which allows for robust cross-species comparisons regardless of the clustering criterion. However, cross-species comparisons of genome plasticity and functional profiles are substantially affected by inconsistencies among clustering criteria. Such inconsistencies are driven not only by mobile genetic elements, but also by genes involved in defense, secondary metabolism, and other accessory functions. In some pangenome features, the variability attributed to methodological inconsistencies can even exceed the effect sizes of ecological and phylogenetic variables.\\n\\nConclusionsChoosing an appropriate criterion for gene clustering is critical to conduct unbiased pangenome analyses. We provide practical guidelines to choose the right method depending on the research goals and the quality of genome assemblies, and a benchmarking dataset to assess the robustness and reproducibility of future comparative studies.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.09.26.509574',\n",
       "   'title': 'Fitting individual-based models of spatial population dynamics to long-term monitoring data',\n",
       "   'authors': 'Malchow, A.-K.; Fandos, G.; Kormann, U.; Grüebler, M. U.; Kery, M.; Hartig, F.; Zurell, D.',\n",
       "   'author_corresponding': 'Anne-Kathleen Malchow',\n",
       "   'author_corresponding_institution': 'Institute for Biochemistry and Biology, University of Potsdam, 14469 Potsdam, Germany',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'ecology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.09.26.509574.source.xml',\n",
       "   'abstract': 'Generating spatial predictions of species distribution is a central task for research and policy. Currently, correlative species distribution models (cSDMs) are among the most widely used tools for this purpose. However, cSDMs fundamental assumption of species distributions in equilibrium with their environment is rarely met in real data and limits their applicability for dynamic projections. Process-based, dynamic SDMs (dSDMs) promise to overcome these limitations as they explicitly represent transient dynamics and enhance spatio-temporal transferability. Software tools for implementing dSDMs become increasingly available, yet their parameter estimation can be complex.\\n\\nHere, we test the feasibility of calibrating and validating a dSDM using long-term monitoring data of Swiss red kites (Milvus milvus). This population has shown strong increases in abundance and a progressive range expansion over the last decades, indicating a non-equilibrium situation. We construct an individual-based model with the RangeShiftR modelling platform and use Bayesian inference for model calibration. This allows the integration of heterogeneous data sources, such as parameter estimates from published literature as well as observational data from monitoring schemes, with coherent assessment of parameter uncertainty. Our monitoring data encompass counts of breeding pairs at 267 sites across Switzerland over 22 years. We validate our model using a spatial-block cross-validation scheme and assess predictive performance with a rank-correlation coefficient.\\n\\nOur model showed very good predictive accuracy of spatial projections and represented well the observed population dynamics over the last two decades. Results suggest that reproductive success was a key factor driving the observed range expansion. According to our model, the Swiss red kite population fills large parts of its current range but has potential for further increases in density.\\n\\nWe demonstrate the practicality of data integration and validation for dSDMs using RangeShifteR. This approach can improve predictive performance compared to cSDMs. The workflow presented here can be adopted for any population for which some prior knowledge on demographic and dispersal parameters as well as spatio-temporal observations of abundance or presence/absence are available. The fitted model provides improved quantitative insights into the ecology of a species, which may greatly help conservation and management actions.\\n\\nOpen Research statementThis submission uses novel code which is provided in an external repository. All data and code required to replicate the presented analyses are provided as private-for-peer review via a public GitHub repository under the following link: https://github.com/UP-macroecology/Malchow_IBMcalibration_2023\\n\\nUpon acceptance, this repository will be archived and versioned with Zenodo and its DOI provided.\\n\\nFor this study, a tagged development version of the R package RangeShiftR was used that is available at: https://github.com/RangeShifter/RangeShiftR-package/releases/tag/v.1.1-beta.0',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.09.28.509676',\n",
       "   'title': 'Mis-regulation of Zn and Mn homeostasis is a key phenotype of Cu stress in Streptococcus pyogenes.',\n",
       "   'authors': 'Hong, Y.; Mackenzie, E. S.; Firth, S. J.; Bolton, J. F.; Stewart, L. J.; Waldron, K. J.; Djoko, K. Y.',\n",
       "   'author_corresponding': 'Karrera Y Djoko',\n",
       "   'author_corresponding_institution': 'Department of Biosciences, Durham University, Durham DH1 3LE, United Kingdom',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.09.28.509676.source.xml',\n",
       "   'abstract': 'All bacteria possess homeostastic mechanisms that control the availability of micronutrient metals within the cell. Cross-talks between different metal homeostasis pathways within the same bacterial organism have been reported widely. In addition, there have been previous suggestions that some metal uptake transporters can promote adventitious uptake of the wrong metal. This work describes the cross-talk between Cu and the Zn and Mn homeostasis pathways in Group A Streptococcus (GAS). Using a {Delta}copA mutant strain that lacks the primary Cu efflux pump and thus traps excess Cu in the cytoplasm, we show that growth in the presence of supplemental Cu promotes downregulation of genes that contribute to Zn or Mn uptake. This effect is not associated with changes in cellular Zn or Mn levels. Co-supplementation of the culture medium with Zn or, to a lesser extent, Mn alleviates key Cu stress phenotypes, namely bacterial growth and secretion of the fermentation end-product lactate. However, neither co-supplemental Zn nor Mn influences cellular Cu levels or Cu availability in Cu-stressed cells. In addition, we provide evidence that the Zn or Mn uptake transporters in GAS do not promote Cu uptake. Together, the results from this study strengthen and extend our previous proposal that mis-regulation of Zn and Mn homeostasis is a key phenotype of Cu stress in GAS.\\n\\nGRAPHICAL ABSTRACT\\n\\nO_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=73 SRC=\"FIGDIR/small/509676v2_ufig1.gif\" ALT=\"Figure 1\">\\nView larger version (32K):\\norg.highwire.dtl.DTLVardef@b33b23org.highwire.dtl.DTLVardef@efaa7borg.highwire.dtl.DTLVardef@a18b5dorg.highwire.dtl.DTLVardef@1c4913f_HPS_FORMAT_FIGEXP  M_FIG C_FIG',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.09.27.509635',\n",
       "   'title': 'Caveolin-1 protects endothelial cells from extensive expansion of transcellular tunnel by stiffening the plasma membrane',\n",
       "   'authors': 'Morel, C.; Lemerle, E.; Tsai, F.-C.; Obadia, T.; Srivastava, N.; Marechal, M.; Salles, A.; Albert, M.; Stefani, C.; Benito, Y.; Vandenesch, F.; Lamaze, C.; Vassilopoulos, S.; Piel, M.; Bassereau, P.; Gonzalez-Rodriguez, D.; Leduc, C.; Lemichez, E.',\n",
       "   'author_corresponding': 'Emmanuel Lemichez',\n",
       "   'author_corresponding_institution': 'Institut Pasteur',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.09.27.509635.source.xml',\n",
       "   'abstract': 'Large transcellular pores elicited by bacterial mono-ADP-ribosyltransferase (mART) exotoxins inhibiting the small RhoA GTPase compromise the endothelial barrier. Recent advances in biophysical modeling point towards membrane tension and bending rigidity as the minimal set of mechanical parameters determining the nucleation and maximal size of transendothelial cell macroaperture (TEM) tunnels induced by bacterial RhoA-targeting mART exotoxins. We report that cellular depletion of caveolin-1, the membrane-embedded building block of caveolae, and depletion of cavin-1, the master regulator of caveolae invaginations, increase the number of TEM per cell. The enhanced nucleation of TEM correlates with a reduction of cell height, due to the increase of cell spreading and decrease of cell volume, which, together with the disruption of RhoA-driven F-actin meshwork, favor membrane apposition for TEM nucleation. Strikingly, caveolin-1 specifically controls the opening speed of TEMs leading to their dramatic 5.4-fold larger widening. Consistent with the increase of TEM density and width in siCAV1 cells, we record a higher lethality in caveolin-1-deficient mice subjected to a catalytically active mART exotoxin targeting RhoA during staphylococcal bloodstream infection. Combined theoretical modeling with independent biophysical measurements of plasma membrane bending rigidity point toward a specific contribution of caveolin-1 to membrane stiffening in addition to the role of cavin-1/caveolin-1-dependent caveolae in the control of membrane tension homeostasis.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.09.30.510274',\n",
       "   'title': 'Spatial summation of pain is associated with pain expectations: Results from home-based paradigm',\n",
       "   'authors': 'Nastaj, J.; Skalski, J.; Budzisz, A.; Szikszay, T. M.; Swoboda, S.; Kowalska, W.; Nowak, D.; Zbroja, E.; Kruszyna, N.; Jakubinska, M.; Grygny, D.; Polczyk, R.; Malecki, A.; Luedtke, K.; Adamczyk, W. M.',\n",
       "   'author_corresponding': 'Waclaw M. Adamczyk',\n",
       "   'author_corresponding_institution': 'The Jerzy Kukuczka Academy of Physical Education',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'physiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2022.09.30.510274.source.xml',\n",
       "   'abstract': 'The purpose of this study was to reproduce the previously observed spatial summation of pain effect (SSp) using non-laboratory procedures and commercial equipment. An additional aim was to explore the association between expectations and SSp. The Cold Pressor Task (CPT) was used to induce SSp. Healthy participants (N=68) immersed their non-dominant hands (divided into 5 segments) into cold water (Cold Pressor Task). Two conditions were used 1) gradual hand immersion (ascending condition) and 2) gradual hand withdrawal (descending condition). Pain intensity was measured on a Visual Analogue Scale (VAS). The influence of psychological factors, such as the volunteers expectations of pain intensity, on the actual perception of pain were also measured on a VAS. Results showed significant SSp ({chi}2(4) = 116.90, p < 0.001), reproduced with non-laboratory equipment in a home-based set-up. Furthermore, two novel findings were observed: i) there was a significant correlation between expectations and perceived pain, indicating a link between pain expectations and SSp, ii) spatial summation increased with the increase in duration exposure to the noxious stimulus (Wald {chi}2(2) = 157.5, p < 0.001). This study suggests that SSp is shaped by a mixture of excitatory and inhibitory mechanisms and can be influenced by the temporal summation of the nociceptive system. Moreover, this study proposes a new feasible way to induce SSp using a home-based set-up.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.03.510672',\n",
       "   'title': 'Human hippocampal ripples signal encoding of episodic memories',\n",
       "   'authors': 'Sakon, J. J.; Halpern, D. J.; Schonhaut, D. R.; Kahana, M. J.',\n",
       "   'author_corresponding': 'John J Sakon',\n",
       "   'author_corresponding_institution': 'University of Pennsylvania',\n",
       "   'date': '2023-08-27',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/27/2022.10.03.510672.source.xml',\n",
       "   'abstract': 'Recent human electrophysiology work has uncovered the presence of high frequency oscillatory events, termed ripples, during awake behavior. This prior work focuses on ripples in the medial temporal lobe (MTL) during memory retrieval. Few studies, however, investigate ripples during item encoding. Many studies have found neural activity during encoding that predicts later recall, termed subsequent memory effects (SMEs), but it is unclear if ripples during encoding also predict subsequent recall. Detecting ripples in 116 neurosurgical participants (n = 61 male) performing an episodic memory task, we find insignificant ripple SMEs in any MTL region, even as these regions exhibit robust high frequency activity (HFA) SMEs. Instead, hippocampal ripples increase during encoding of items leading to recall of temporally or semantically associated items, a phenomenon known as clustering. This subsequent clustering effect (SCE) arises specifically when hippocampal ripples occur during both encoding and retrieval, suggesting that ripples mediate the encoding and future reinstatement of episodic memories.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.04.510799',\n",
       "   'title': 'Alignment, calibration, and validation of an adaptive optics scanning laser ophthalmoscope for high-resolution human foveal imaging',\n",
       "   'authors': 'Moon, B.; Poletti, M.; Roorda, A.; Tiruveedhula, P.; Liu, S. H.; Linebach, G.; Rucci, M.; Rolland, J. P.',\n",
       "   'author_corresponding': 'Benjamin Moon',\n",
       "   'author_corresponding_institution': 'University of Rochester',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'bioengineering',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.10.04.510799.source.xml',\n",
       "   'abstract': 'Advances in adaptive optics scanning laser ophthalmoscope (AOSLO) technology have enabled cones in the human fovea to be reliably resolved, providing new insight into human foveal anatomy, visual perception, and retinal degenerative diseases. These high-resolution ophthalmoscopes require careful alignment of each optical subsystem to ensure diffraction-limited imaging performance, which is necessary for resolving the smallest foveal cones. This paper presents a systematic and rigorous methodology for building, aligning, calibrating, and testing an AOSLO designed for imaging the cone mosaic of the central fovea in humans with cellular resolution. This methodology uses a two-stage alignment procedure and thorough system testing to achieve diffraction-limited performance. Results from retinal imaging of healthy human subjects show that the system can resolve cones at the very center of the fovea, the region where the cones are smallest and most densely packed.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.10.511559',\n",
       "   'title': 'Causal models of human growth and their estimation using temporally-sparse data',\n",
       "   'authors': 'Bunce, J. A.; Fernandez, C. I.; Revilla Minaya, C.',\n",
       "   'author_corresponding': 'John A Bunce',\n",
       "   'author_corresponding_institution': 'Max Planck Institute for Evolutionary Anthropology',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.10.10.511559.source.xml',\n",
       "   'abstract': 'AO_SCPLOWBSTRACTC_SCPLOWExisting models of human growth provide little insight into the mechanisms responsible for inter-individual and inter-population variation in childrens growth trajectories. Building on general theories linking growth to metabolic rates, we develop causal parametric models of height and weight growth incorporating a novel representation of human body allometry and a phase-partitioned representation of ontogeny. These models permit separation of metabolic causes of growth variation, potentially influenced by diet and disease, from allometric factors, potentially under strong genetic control. We estimate model parameters using a Bayesian multilevel statistical design applied to temporally-dense height and weight measurements of U.S. children, and temporally-sparse measurements of Indigenous Amazonian children. This facilitates a comparison of the metabolic and allometric contributions to observed cross-cultural variation in the growth trajectories of the two populations. These theoretical growth models constitute an initial step toward a better understanding of the causes of growth variation in our species, while potentially guiding the development of appropriate, and desired, healthcare interventions in societies confronting growth-related health challenges.\\n\\nShort SummaryNew causal models of human growth facilitate cross-cultural comparisons of metabolism and allometry.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.11.511650',\n",
       "   'title': 'The role of uncertainty and negative feedback loops in the evolution of induced immune defenses',\n",
       "   'authors': 'Asgari, D.; Stewart, A. J.; Meisel, R. P.',\n",
       "   'author_corresponding': 'Richard P Meisel',\n",
       "   'author_corresponding_institution': 'University of Houston',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'evolutionary biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.10.11.511650.source.xml',\n",
       "   'abstract': 'Organisms use constitutive or induced defenses against pathogens, parasites, and herbivores. Constitutive defenses are constantly on, whereas induced defenses are activated upon exposure to an enemy. Constitutive and induced defenses each have costs and benefits, which can affect the type of defense that evolves in response to pathogens. Previous models that compared the two lacked mechanistic details about host defense, did not consider pathogen proliferation rates, or lacked both features. To address this gap, we developed a detailed mechanistic model of the well-characterized Drosophila melanogaster immune signaling network. We evaluated the factors favoring the evolution of constitutive and induced defenses by comparing the fitness of each strategy under stochastic fly-bacteria interactions. We show that an induced defense is favored when bacteria are at low density, heterogeneously distributed, or have fluctuating distributions in ways that depend on the bacterial proliferation rate. Our model also predicts that the specific negative regulators that optimize the induced response depend on the bacterial proliferation rate. We therefore conclude that an induced immune defense is favored in environments where bacterial encounters are uncertain--because of heterogeneity in spatial or temporal distributions--but that benefit depends on the mechanism of induction and pathogen properties.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.13.511989',\n",
       "   'title': 'Decoding Fear or Safety and Approach or Avoidance by Brain-Wide Network Dynamics',\n",
       "   'authors': 'Marques, D. B.; Rossignoli, M. T.; Mesquita, B. d. A.; Prizon, T.; Zacharias, L. R.; Ruggiero, R. N.; Leite, J. P.',\n",
       "   'author_corresponding': 'Rafael Naime Ruggiero',\n",
       "   'author_corresponding_institution': 'Department of Neuroscience and Behavioral Sciences, Ribeirao Preto Medical School, University of Sao Paulo',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.10.13.511989.source.xml',\n",
       "   'abstract': 'Discerning safety from threat and positive or negative outcomes of adversity are fundamental for mental health. Many brain structures have been implicated in both adaptive and maladaptive stress coping, however, how multiple regions function together as a network in the processing of this information is unclear. Here, we recorded local field potentials from seven regions of the mesolimbic-hippocampal-prefrontal cortical network (MLHFC) of male rats during the conditioning of a stimulus (CS) to the absence (safety) and then to the anticipation (fear) of footshocks, and during an approach-avoidance task. We developed a machine learning pipeline to investigate the relevance of specific features of oscillatory activity in the decoding of fear versus safety and approach versus avoidance. We found that decoding performance increased as a function of the number of brain regions included, reaching the best classification if all regions were considered. In addition, the best decoding was obtained from frequencies within the theta range (4-10 Hz). Remarkably, decoder models showed robust generalization within but not between individuals. Nevertheless, we were also able to identify patterns of MLHFC activity that decoded stress coping states from all rats. These patterns were characterized by increased brain-wide theta synchrony during fear and preceding approach. Our results indicate that stress coping information is encoded at the brain-wide level and highlight individual variability in this neural processing. Our findings also suggest that MLHFC network theta activity underlies active stress coping with both aversive and positive motivational valences.\\n\\nSIGNIFICANCE STATEMENTThe appraisal of safety versus threat and positive versus negative valence of adversity are core dimensions of emotional experience and stress coping. We developed a new behavioral protocol that discriminates states of fear, safety, approach, and avoidance in a single subject and a machine learning-based method to investigate how neural oscillations can decode such states. Our work provides evidence that stress coping is processed at multiple regions on a brain-wide level involving network oscillations at the theta frequencies, which especially synchronizes during fear and approach. We highlight the potentials of combining artificial intelligence and multi-site electroencephalography to guide therapeutic decisions in precision psychiatry and theta-boosting stimulation therapies for stress-related disorders, especially related to cognitive and motivational deficits.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.17.512624',\n",
       "   'title': 'The faculty-to-faculty mentorship experience: a survey on challenges and recommendations for improvements',\n",
       "   'authors': 'Sarabipour, S.; Burgess, S. J.; Niemi, N. M.; Smith, C. T.; Bisson Filho, A. W.; Ibrahim, A.; Clark, K.',\n",
       "   'author_corresponding': 'Sarvenaz Sarabipour',\n",
       "   'author_corresponding_institution': 'Institute for Computational Medicine and Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland, United States',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'scientific communication and education',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2022.10.17.512624.source.xml',\n",
       "   'abstract': 'Faculty at research institutions play a central role in advancing knowledge and careers, as well as promoting the well-being of students and colleagues in research environments. Faculty members must balance a host of activities - such as performing research, teaching, sourcing funds, administrative and service duties - with their roles as educators and mentors. Mentorship from experienced peers has been touted as critical for enabling these myriad roles to allow faculty development, career progression, and satisfaction. However, there is little information available on who supports faculty and best ways to structure a faculty mentorship program for early- and mid-career academics. Furthermore, the extent to which mentorship and mentoring programs have been offered to faculty members has not been widely characterized. It is also unclear what challenges faculty receiving mentorship may face and which aspects could be further improved. In the interest of advocating for increased and enhanced faculty mentoring and mentoring programs, we surveyed faculty (i.e., group leaders) around the world to gather data on whether and how they receive mentoring from peers, senior researchers, informal mentoring programs, or formal mentoring programs at their institutions. We received responses from 457 early- and mid-career faculty and found that a substantial portion of respondents either reported having no mentor or a lack of a formal mentoring scheme. Qualitative responses on the quality of mentorship revealed that the most common complaints regarding mentorship included lack of mentor availability, unsatisfactory commitment to mentorship, and non-specific or non-actionable advice. Our findings further identified key mentorship elements desired by faculty mentees. Based on these suggestions, we identify a need for training for faculty mentors as well as strategies for individual mentors, departments, and institutions for funding and design of more intentional and supportive mentorship programs for early- and mid-career faculty.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.18.512728',\n",
       "   'title': 'Spatio-temporal modeling of high-throughput multi-spectral aerial images improves agronomic trait genomic prediction in hybrid maize',\n",
       "   'authors': 'Morales, N.; Anche, M. T.; Kaczmar, N. S.; Lepak, N.; Ni, P.; Romay, M. C.; Santantonio, N.; Buckler, E. S.; Gore, M. A.; Mueller, L. A.; Robbins, K. R.',\n",
       "   'author_corresponding': 'Nicolas Morales',\n",
       "   'author_corresponding_institution': 'Cornell University',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'genetics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.10.18.512728.source.xml',\n",
       "   'abstract': 'Design randomizations and spatial corrections have increased understanding of genotypic, spatial, and residual effects in field experiments, but precisely measuring spatial heterogeneity in the field remains a challenge. To this end, our study evaluated approaches to improve spatial modeling using high-throughput phenotypes (HTP) via unoccupied aerial vehicle (UAV) imagery. The normalized difference vegetation index (NDVI) was measured by a multi-spectral MicaSense camera and ImageBreed. Contrasting to baseline agronomic trait spatial correction and a baseline multi-trait model, a two-stage approach that quantified NDVI local environmental effects (NLEE) was proposed. Firstly, NLEE were separated from additive genetic effects over the growing season using two-dimensional spline (2DSpl), separable autoregressive (AR1) models, or random regression models (RR). Secondly, the NLEE were leveraged within agronomic trait genomic best linear unbiased prediction (GBLUP) either modeling an empirical covariance for random effects, or by modeling fixed effects as an average of NLEE across time or split among three growth phases. Modeling approaches were tested using simulation data and Genomes-to-Fields (G2F) hybrid maize (Zea mays L.) field experiments in 2015, 2017, 2019, and 2020 for grain yield, grain moisture, and ear height. The two-stage approach improved heritability, model fit, and genotypic effect estimation compared to all baseline models. Electrical conductance and elevation from a 2019 soil survey significantly improved model fit, while 2DSpl NLEE were most correlated to the soil parameters and grain yield 2DSpl effects. Simulation of field effects demonstrated improved specificity for RR models. In summary, NLEE increased experimental accuracy and understanding of field spatio-temporal heterogeneity.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.18.512444',\n",
       "   'title': 'Oral Delivery of Kidney Targeting Nanotherapeutics for Polycystic Kidney Disease',\n",
       "   'authors': 'Huang, Y.; Wang, J.; Chin, D.; Mancino, V.; Pham, J.; Li, H.; Jiang, K.; Ram, A.; Poon, C.; Ho, P.-Y.; Peti-Peterdi, J.; Gyarmati, G.; Hallows, K. R.; Chung, E. J.',\n",
       "   'author_corresponding': 'Eun Ji Chung',\n",
       "   'author_corresponding_institution': 'University of Southern California',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'bioengineering',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2022.10.18.512444.source.xml',\n",
       "   'abstract': 'Autosomal dominant polycystic kidney disease (ADPKD) is the most common inherited renal disorder. Although a variety of candidate drugs have been found to modulate cystogenesis in animal studies, results from clinical trials have often been unfavorable due to low renal bioavailability and drug-induced side effects. To mitigate this, nanoparticles can be designed to deliver drugs directly to the target organ to increase effective dose while limiting off-target side effects. Unfortunately, there are no kidney-targeted nanomedicines clinically available, and most of the existing FDA-approved nanoparticles require intravenous administration which is not suitable for ADPKD that require lifelong therapy. To address this, we developed an oral drug delivery system using chitosan nanoparticles (CS-NP) that were loaded with peptide amphiphile micelles carrying metformin (met), an ADPKD drug candidate (CS-KM-met). We previously showed that CS-NP can shield met in the gastrointestinal tract; thus, we hypothesized that CS-NP could also enhance bioavailability of kidney-targeting micelles (KMs) upon oral administration. Specifically, we measured the loading capacity of KM-met in CS-NP, evaluated the stability of CS-KM-met under acidic conditions that mimic the gastric environment, and measured in vitro therapeutic effects. Upon oral administration in C57BL/6J mice, CS-KM-met showed significantly greater bioavailability and accumulation in the kidneys as compared to KM-met without CS-NP or free met for up to 24 hours. As such, CS-KM-met showed enhanced therapeutic efficacy in vivo upon oral administration in PKD mice (Pkd1fl/fl; Pax8-rtTA; Tet-O-Cre) compared to KM-met only. Herein, we demonstrate the potential of an oral delivery nanoformulation for the treatment of chronic kidney diseases such as ADPKD for the first time.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.22.513328',\n",
       "   'title': 'MAVISp: Multi-layered Assessment of VarIants by Structure for proteins',\n",
       "   'authors': 'Arnaudi, M.; Beltrame, L.; Degn, K.; Utichi, M.; Scrima, S.; Sanchez-Izquierdo, P.; Maselli, F.; Krzesinska, K.; Dorcakova, T.; Safer, J.; Meldgard, K.; Becker, P.; Pettenella, A.; Brockhoff, J. B.; Nielsen, A. D.; Vinhas, J.; Sackett, P. W.; Cava, C.; Rohlin, A.; Nilbert, M.; Iqbal, S.; Lambrughi, M.; Tiberti, M.; Papaleo, E.',\n",
       "   'author_corresponding': 'Elena Papaleo',\n",
       "   'author_corresponding_institution': 'Danish Cancer Society Research Center',\n",
       "   'date': '2023-08-25',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/25/2022.10.22.513328.source.xml',\n",
       "   'abstract': 'The role of genomic variants in disease, including cancer, continues to be realized. Thanks to advances in sequencing techniques and their introduction in clinical practice, the number of genomic variants discovered is growing at fast pace. Many variants are classified as Variants of Uncertain Significance (VUS) or conflicting evidence, posing big challenges to their interpretation and usability. We here introduce a modular structure-based framework for the characterization of the impact of protein variants (i.e., MAVISp, Multi-layered Assessment of VarIants by Structure for proteins), accompanied by a Streamlit-based web server (https://services.healthtech.dtu.dk/services/MAVISp-1.0/) to facilitate data access, consultation, and reusability. Currently, MAVISp includes analyses of 167 different proteins and approximately 50 000 variants. New protein targets are routinely analyzed in batches by a team of biocurators through standardized workflows, using high-throughput free energy calculations and simulations. We also illustrated the potential of the approach on a selection of case studies. New variants will be deposited regularly or in connection with future publications where the framework will be applied.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.21.513123',\n",
       "   'title': 'Improving fine-mapping by modeling infinitesimal effects',\n",
       "   'authors': 'Cui, R.; Elzur, R. A.; Kanai, M.; Ulirsch, J. C.; Weissbrod, O.; Daly, M.; Neale, B.; Fan, Z.; Finucane, H. K.',\n",
       "   'author_corresponding': 'Ran Cui',\n",
       "   'author_corresponding_institution': 'Broad Institute of MIT and Harvard',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.10.21.513123.source.xml',\n",
       "   'abstract': 'Fine-mapping aims to identify causal variants for phenotypes. Bayesian fine-mapping algorithms (e.g.: SuSiE, FINEMAP, ABF, and COJO-ABF) are widely used, but assessing posterior probability calibration remains challenging in real data, where model misspecification likely exists, and true causal variants are unknown. We introduce Replication Failure Rate (RFR), a metric to assess fine-mapping consistency by down-sampling. SuSiE, FINEMAP and COJO-ABF show high RFR, indicating potential under-conservative mis-calibration. Simulations reveal that non-sparse genetic architecture can lead to miscalibration, while imputation noise, non-uniform distribution of causal variants, and QC filters have minimal impact. We present SuSiE-inf and FINEMAP-inf, novel fine-mapping methods modeling infinitesimal effects alongside fewer larger causal effects. Our methods exhibit improved calibration, RFR and functional enrichment, competitive recall and computational efficiency. Notably, using our methods posterior effect sizes substantially increases PRS accuracy over SuSiE and FINEMAP. Our work improves causal variants identification for complex traits, a fundamental goal of human genetics.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.24.513547',\n",
       "   'title': 'A computational account of needing and wanting',\n",
       "   'authors': 'Bosulu, J.; Pezzulo, G.; Hetu, S.',\n",
       "   'author_corresponding': 'Juvenal Bosulu',\n",
       "   'author_corresponding_institution': 'Universite de Montreal',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '8',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.10.24.513547.source.xml',\n",
       "   'abstract': 'Our actions and choices sometimes seem driven by what we \"need\", sometimes by what we \"want\", and sometimes by both. Needing is related to deprivation of something biologically significant, and wanting is linked to reward prediction and dopamine and usually has more power on behavioral activation than need states alone. Why sometimes there is an association and sometimes a dissociation between needing and wanting remains largely unknown. This paper aims to clarify the relations between needing and wanting, from the perspective of a normative framework to understand brain and cognition; namely, active inference. In this approach, needing is related to a deviation from preferred states that living things tend to occupy in order to reduce their surprise, while wanting is related to the precision over policies that lead to preferred (e.g., rewarding) states. Also, we hypothesized that needing exerts a pervasive effect that is embodied on the organism. Through a series of simulations, we demonstrate the interplay between needing and wanting systems. Specifically, our simulations show that when need states increase, the tendency to occupy preferred states is enhanced, independently of wanting (or reward prediction), therefore showing a dissociation between needing and wanting. Furthermore, the simulations show that when need states increase, the value of cues that signal reward achievement and the precision of the policies that lead to preferred states increase, suggesting that needs can amplify the value of a reward and its wanting. Taken together, our model and simulations help clarify the directional and underlying influence of need states separately from wanting and reward prediction; and how the same underlying influence of need amplifies wanting, i.e. increases the precision of available reward cues that lead to the preferred state.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.26.513844',\n",
       "   'title': 'Which cues are sexy? The evolution of mate preference in sympatric species reveals the contrasted effect of adaptation and reproductive interference.',\n",
       "   'authors': 'Maisonneuve, L.; Smadi, C.; Llaurens, V.',\n",
       "   'author_corresponding': 'Ludovic Maisonneuve',\n",
       "   'author_corresponding_institution': \"Museum National d'Histoire Naturelle\",\n",
       "   'date': '2023-08-22',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'evolutionary biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.10.26.513844.source.xml',\n",
       "   'abstract': 'Mate preferences may target traits enhancing (1) offspring adaptation and (2) reducing heterospecific matings. Because similar selective pressures are acting on traits shared by different sympatric species, preference enhancing offspring adaptation may increase heterospecific mating, in sharp contrast with the classical case of so-called  magic trait. Using a mathematical model, we study which and how many traits will be used during mate choice, when preferences for locally adapted traits increase heterospecific mating. We especially study the evolution of preference towards an adaptive vs. a neutral trait in sympatric species. We take into account sensory trade-offs which may limit the emergence of preference for several traits. Our model highlights that the evolution of preference towards adaptive vs. neutral traits depends on the selective regimes acting on traits but also on heterospecific interactions. When the costs of heterospecific interactions are high, mate preference is likely to target neutral traits that become a reliable cue limiting heterospecific matings. We show that the evolution of preference towards a neutral trait benefits from a positive feedback loop: the more preference targets the neutral trait, the more it becomes a reliable cue for species recognition. We then reveal the key role of sensory trade-offs and the cost of choosiness favouring the evolution of preferences targeting adaptive traits, rather than traits reducing heterospecific mating. When sensory trade-offs and the cost of choosiness are low, we also show that preferences targeting multiple traits evolve, improving offspring fitness by both transmitting adapted alleles and reducing heterospecific mating. Altogether, our model aims at reconciling  good gene and  magic trait models to provide general predictions on the evolution of mate preferences within natural communities.\\n\\nImpact SummaryMate preferences are widespread throughout the animal kingdom and generate powerful selective forces impacting the diversification of traits and species. The evolution of such preferences has been the focus of multiple theoretical and empirical studies and intense scientific debates. The evolution of mate preference enhancing (1) offspring fitness and (2) reducing heterospecific mating have been mostly studied separately, except in the specific case of preference for so-called  magic trait increasing both offspring survival and species divergence. However, in many cases, the evolution of traits in sympatric species generates conflicting evolutionary forces acting on preferences. On one hand, enhanced offspring survival promotes preference towards locally adaptive traits and may thus lead to convergent evolution of traits among sympatric species. On the other hand, the evolution of similar traits in sympatric species may generate costly heterospecific sexual interactions promoting preference towards traits that diverge between species. Here, we thus build a general mathematical model to investigate the evolutionary factors determining which and how many traits are targeted by mate choice. We especially determine whether preferences will likely target adaptive vs. neutral traits. Our model highlights that the evolution of preferences for adaptive vs. neutral traits in sympatric species depends on within-species mating opportunities but also on the niche overlap between species, tuning the heterospecific interactions. By jointly considering (1) the selection regimes acting on the targeted traits within species, as well as (2) interactions with other species living in sympatry, our theoretical study provides a general framework reconciling these research fields.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.10.28.514163',\n",
       "   'title': 'Synaptic mechanisms for associative learning in the cerebellar nuclei',\n",
       "   'authors': 'Broersen, R.; Albergaria, C.; Carulli, D.; Carey, M. R.; Canto, C. B.; De Zeeuw, C. I.',\n",
       "   'author_corresponding': 'Chris I De Zeeuw',\n",
       "   'author_corresponding_institution': 'Netherlands Institute for Neuroscience; Erasmus Medical Center',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2022.10.28.514163.source.xml',\n",
       "   'abstract': 'Associative learning during delay eyeblink conditioning (EBC) depends on an intact cerebellum. However, the relative contribution of changes in the cerebellar nuclei to learning remains a subject of ongoing debate. In particular, little is known about the changes in synaptic inputs to cerebellar nuclei neurons that take place during EBC and how they shape the membrane potential of these neurons. Here, we probed the ability of these inputs to support associative learning in mice, and investigated structural and cell-physiological changes within the cerebellar nuclei during learning. We find that optogenetic stimulation of mossy fiber afferents to the anterior interposed nucleus (AIP) can substitute for a conditioned stimulus and is sufficient to elicit conditioned responses (CRs) that are adaptively well-timed. Further, EBC induces structural changes in mossy fiber and inhibitory inputs, but not of climbing fiber inputs, and it leads to changes in subthreshold processing of AIP neurons that correlate with conditioned eyelid movements. The changes in synaptic and spiking activity that precede the CRs allow for a decoder to distinguish trials with a CR. Our data reveal how structural and physiological modifications of synaptic inputs to cerebellar nuclei neurons can facilitate learning.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.15.516574',\n",
       "   'title': 'A phylogenetically-conserved axis of thalamocortical connectivity in the human brain',\n",
       "   'authors': 'Oldham, S.; Ball, G.',\n",
       "   'author_corresponding': 'Stuart Oldham',\n",
       "   'author_corresponding_institution': \"Murdoch Children's Research Institute\",\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.11.15.516574.source.xml',\n",
       "   'abstract': 'The thalamus enables key sensory, motor, emotive, and cognitive processes via connections to the cortex. These projection patterns are traditionally considered to originate from discrete thalamic nuclei, however recent work showing gradients of molecular and connectivity features in the thalamus suggests the organisation of thalamocortical connections occurs along a continuous dimension. By performing a joint decomposition of densely sampled gene expression and non-invasive diffusion tractography in the adult human thalamus, we define a principal axis of genetic and connectomic variation along a medial-lateral thalamic gradient. Projections along this axis correspond to an anterior-posterior cortical pattern and are aligned with electrophysiological properties of the cortex. The medial-lateral axis demonstrates phylogenetic conservation, reflects transitions in neuronal subtypes, and shows associations with neurodevelopment and common brain disorders. This study provides evidence for a supra-nuclear axis of thalamocortical organisation characterised by a graded transition in molecular properties and anatomical connectivity.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.16.516735',\n",
       "   'title': 'Pioneer factor Pax7 initiates two-step cell-cycle dependent chromatin opening',\n",
       "   'authors': 'Gouhier, A.; Dumoulin-Gagnon, J.; Lapointe-Roberge, V.; Harris, J.; Balsalobre, A.; DROUIN, J.',\n",
       "   'author_corresponding': 'Jacques DROUIN',\n",
       "   'author_corresponding_institution': 'Institut de recherches cliniques de Montreal (IRCM)',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'molecular biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.11.16.516735.source.xml',\n",
       "   'abstract': 'Pioneer transcription factors direct cell differentiation by deploying new enhancer repertoires through their unique ability to target and initiate remodelling of closed chromatin. The initial steps of their action remain undefined although pioneers were shown to interact with nucleosomal target DNA and with some chromatin remodelling complexes. We now define the sequence of events that enable the pioneer Pax7 with its unique abilities. Chromatin condensation exerted by linker histone H1 is the first constraint on Pax7 recruitment, and this establishes the initial speed of chromatin remodelling. The first step of pioneer action involves recruitment of the KDM1A (LSD1) H3K9me2 demethylase for removal of this repressive mark, as well as recruitment of the MLL complex for deposition of the activating H3K4me1 mark. Further progression of pioneer action requires passage through cell division, and this involves dissociation of pioneer targets from perinuclear lamin B. Only then, the SWI/SNF remodeling complex and the coactivator p300 are recruited, leading to nucleosome displacement and enhancer activation. Thus, the unique features of pioneer actions are those occurring in the lamin-associated compartment of the nucleus. This model is consistent with prior work that showed a dependence on cell division for establishment of new cell fates.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.16.516758',\n",
       "   'title': 'Dynamic traction force measurements of migrating immune cells in 3D matrices',\n",
       "   'authors': 'Boehringer, D.; Condor, M.; Bischof, L.; Czerwinski, T.; Gampl, N.; Ngo, P. A.; Bauer, A.; Lopez-Posadas, R.; Franze, K.; Voskens, C.; Budday, S.; Mark, C.; Fabry, B.; Gerum, R.',\n",
       "   'author_corresponding': 'Ben Fabry',\n",
       "   'author_corresponding_institution': 'Department of Physics, Friedrich-Alexander-Universitaet Erlangen-Nuernberg, Erlangen, Germany',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.11.16.516758.source.xml',\n",
       "   'abstract': 'Immune cells such as natural killer (NK) cells migrate with high speeds of several {micro}m/min through dense tissue, but the traction forces are unknown. We present a method to measure dynamic traction forces of fast migrating cells in non-linear biopolymer matrices. The method accounts for the mechanical non-linearity of the 3D tissue matrix and can be applied to time series of confocal or bright-field image stacks. The method is highly sensitive over a large range of forces and object sizes, from [~]1 nN for axon growth cones up to [~]10 {micro}N for mouse intestinal organoids. We find that NK cells display bursts of large traction forces that increase with matrix stiffness and facilitate migration through tight constrictions.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.17.516891',\n",
       "   'title': 'Quantitative analyses of T cell motion in tissue reveals factors driving T cell search in tissues',\n",
       "   'authors': 'Torres, D. J.; Mrass, P.; Byrum, J.; Gonzales, A.; Martinez, D.; Juarez, M. E.; Thompson, E.; Vezys, V.; Cannon, J. L.',\n",
       "   'author_corresponding': 'Judy L Cannon',\n",
       "   'author_corresponding_institution': 'University of New Mexico',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'immunology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.11.17.516891.source.xml',\n",
       "   'abstract': 'T cells are required to clear infection, moving first in lymph nodes to interact with antigen bearing dendritic cells leading to activation. T cells then move to sites of infection to find and clear infection. T cell motion plays a role in how quickly a T cell finds its target, from initial naive T cell activation by a dendritic cell to interaction with target cells in infected tissue. To better understand how different tissue environments might affect T cell motility, we compared multiple features of T cell motion including speed, persistence, turning angle, directionality, and confinement of motion from T cells moving in multiple tissues using tracks collected with microscopy from murine tissues. We quantitatively analyzed naive T cell motility within the lymph node and compared motility parameters with activated CD8 T cells moving within the villi of small intestine and lung under different activation conditions.\\n\\nOur motility analysis found that while the speeds and the overall displacement of T cells vary within all tissues analyzed, T cells in all tissues tended to persist at the same speed, particularly if the previous speed is very slow (less than 2 {micro}m/min) or very fast (greater than 8 {micro}m/min) with the exception of T cells in the villi for speeds greater than 10 {micro}m/min. Interestingly, we found that turning angles of T cells in the lung show a marked population of T cells turning at close to 180o, while T cells in lymph nodes and villi do not exhibit this \"reversing\" movement. Additionally, T cells in the lung showed significantly decreased meandering ratios and increased confinement compared to T cells in lymph nodes and villi. The combination of these differences in motility patterns led to a decrease in the total volume scanned by T cells in lung compared to T cells in lymph node and villi. These results suggest that the tissue environment in which T cells move can impact the type of motility and ultimately, the efficiency of T cell search for target cells within specialized tissues such as the lung.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.17.516984',\n",
       "   'title': 'Adult consequences of repeated nicotine vapor inhalation in adolescent rats',\n",
       "   'authors': 'Gutierrez, A.; Nguyen, J. D.; Creehan, K. M.; Grant, Y.; Taffe, M. A.',\n",
       "   'author_corresponding': 'Michael A Taffe',\n",
       "   'author_corresponding_institution': 'University of California, San Diego',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'pharmacology and toxicology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.11.17.516984.source.xml',\n",
       "   'abstract': 'IntroductionThere has been a recent resurgence in nicotine inhalation in adolescents due to the popularity and availability of Electronic Nicotine Delivery Systems (ENDS). Almost five times as many US high-school seniors inhale nicotine vapor daily compared with those who smoke tobacco. This study was conducted to determine the impact of repeated adolescent vapor inhalation of nicotine on behavior in adulthood.\\n\\nMethodsMale and female Sprague-Dawley rats were exposed to 30-minute sessions of ENDS vapor inhalation, twice daily, from Post-Natal Day (PND) 31 to PND 40. Conditions included vapor from the propylene glycol (PG) vehicle or nicotine (30 mg/mL in the PG). Animals were assessed for effects of nicotine on open field (PND 74-105) and wheel activity (PND 126-180) and for the self-administration of nicotine vapor (PND 285-395). Plasma levels of nicotine and cotinine were assessed in separate groups of male and female Wistar and Sprague-Dawley rats after a single nicotine inhalation session.\\n\\nResultsGroup mean plasma nicotine ranged from 39 to 59 ng/mL post-session with minimal strain differences detected. Adolescent nicotine exposure modestly enhanced sensitivity to the locomotor stimulating effects of nicotine (0.1-0.8 mg/kg, s.c.) in an open field in female rats, but didnt change effects of nicotine on wheel activity. Female rats exposed to nicotine (30 mg/mL) vapor as adolescents responded more vigorously than PG exposed females for nicotine vapor in a FR5 challenge.\\n\\nConclusionsRepeated adolescent nicotine vapor inhalation leads to enhanced liability for nicotine self-administration in adulthood in female rats, but minimal change in spontaneous locomotor behavior.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.21.517394',\n",
       "   'title': 'Effort Foraging Task reveals positive correlation between individual differences in the cost of cognitive and physical effort in humans',\n",
       "   'authors': 'Bustamante, L. A.; Oshinowo, T.; Lee, J. R.; Tong, E.; Burton, A. R.; Shenhav, A. S.; Cohen, J. D.; Daw, N. D.',\n",
       "   'author_corresponding': 'Laura Ana Bustamante',\n",
       "   'author_corresponding_institution': 'Princeton Neuroscience Institute',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.11.21.517394.source.xml',\n",
       "   'abstract': 'Effort-based decisions, in which people weigh potential future rewards against effort costs required to achieve those rewards, have largely been studied separately for cognitive or physical effort, yet most real-world actions incur both effort costs. What is the relationship between cognitive and physical effort costs? Here we attempt to formalize the mechanisms underlying effort-based decisions and address methodological challenges to isolate and measure these mechanisms. Patch foraging is an ecologically valid reward rate maximization problem with well-developed theoretical tools to understand choices. We developed the Effort Foraging Task, which embedded cognitive or physical effort into a patch foraging sequential decision task, to isolate and quantify the cost of both cognitive and physical effort using a computational model. Participants chose between harvesting a depleting patch, or traveling to a new patch that was costly in time and effort. Participants exit thresholds (reflecting the reward they expected to receive by harvesting when they chose to travel to a new patch) were sensitive to cognitive and physical effort demands, allowing us to quantify the perceived effort cost in monetary terms. Individual differences in cognitive and physical effort costs were positively correlated, suggesting that these are perceived and processed in common terms across different domains. We found patterns of correlation of both cognitive and physical effort costs with self-reported anxiety, cognitive function, behavioral activation, and self-efficacy. This suggests the task captures decision mechanisms associated with real-world motivation and can be used to study individual variation in effort-based decisions across domains of cost.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.22.517560',\n",
       "   'title': 'Neuronal IL-17 controls C. elegans developmental diapause through CEP-1/p53',\n",
       "   'authors': 'Godthi, A.; Min, S.; Srijit, D.; Cruz-Corchado, J.; Deonarine, A.; Misel-Wuchter, K.; Issuree, P. D.; Prahlad, V.',\n",
       "   'author_corresponding': 'Veena Prahlad',\n",
       "   'author_corresponding_institution': 'University of Iowa',\n",
       "   'date': '2023-08-28',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/28/2022.11.22.517560.source.xml',\n",
       "   'abstract': 'How metazoan development is coordinated with nutrient availability remains unclear. Here, we show that the activity of CEP-1, the C. elegans ortholog of the tumor suppressor p53, switches organismal development between reproductive growth and dormancy by responding to nutrient availability signaled by neuronal cytokine, ILC-17.1. Specifically, ILC-17.1 is released from amphid neurons upon food availability and promotes glucose utilization and represses CEP-1/p53: CEP-1/p53 repression promotes growth. In the absence of ILC-17.1, CEP-1/p53 is activated, accumulates in post-embryonically dividing progenitor blast cells, upregulates cell-cycle inhibitors, decreases cytochrome C levels, and causes larvae to arrest as stress-resistant, quiescent dauers. We propose a model whereby ILC-17.1 signaling links nutrient availability and energy metabolism to cell cycle progression and cell fate decisions during development through CEP-1/p53. These findings also suggest that the developmental function of the p53-gene family could have shaped their evolution and are relevant to our understanding of neuroimmune mechanisms in cancer.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.24.516014',\n",
       "   'title': 'Hepatic WDR23 proteostasis mediates insulin clearance by regulating insulin degrading enzyme activity',\n",
       "   'authors': 'Duangjan, C.; Arpawong, T. E.; Spatola, B. N.; Curran, S. P.',\n",
       "   'author_corresponding': 'Sean P Curran',\n",
       "   'author_corresponding_institution': 'University of Southern California',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.11.24.516014.source.xml',\n",
       "   'abstract': 'Clearance of circulating insulin is critical for metabolic homeostasis. In the liver, insulin is degraded by the activity of the insulin-degrading enzyme (IDE). Here we establish a hepatic regulatory axis for IDE through WDR23-proteostasis. Wdr23KO mice have increased IDE expression, reduced circulating insulin, and defective insulin responses. Genetically engineered human cell models lacking WDR23 also increase IDE expression and display dysregulated phosphorylation of insulin signaling cascade proteins, IRS-1, AKT2, MAPK, FoxO, and mTOR, similar to cells treated with insulin, which can be mitigated by chemical inhibition of IDE. Mechanistically, the cytoprotective transcription factor NRF2, a direct target of WDR23-Cul4 proteostasis, mediates the enhanced transcriptional expression of IDE when WDR23 is ablated. Moreover, an analysis of human genetic variation in WDR23 across a large naturally aging human cohort in the US Health and Retirement Study reveals a significant association of WDR23 with altered hemoglobin A1C (HbA1c) levels in older adults, supporting the use of WDR23 as new molecular determinant of metabolic health in humans.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.28.518209',\n",
       "   'title': 'Remembering the \"When\": Hebbian Memory Models for the Time of Past Events',\n",
       "   'authors': 'Brea, J.; Modirshanechi, A.; Iatropoulos, G.; Gerstner, W.',\n",
       "   'author_corresponding': 'Johanni Brea',\n",
       "   'author_corresponding_institution': 'EPFL',\n",
       "   'date': '2023-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/22/2022.11.28.518209.source.xml',\n",
       "   'abstract': 'Humans and animals can remember how long ago specific events happened. In contrast to interval-timing on the order of seconds and minutes, little is known about the neural mechanisms that enable remembering the \"when\" of autobiographical memories stored in the episodic memory system. Based on a systematic exploration of neural coding, association and retrieval schemes, we develop a family of hypotheses about the reconstruction of the time of past events, consistent with Hebbian plasticity in neural networks. We compare several plausible candidate mechanism in simulated experiments and, accordingly, propose how combined behavioral and physiological experiments can be used to pin down the actual neural implementation of the memory for the time of past events.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.30.518510',\n",
       "   'title': 'Identifying Content-Invariant Neural Signatures of Perceptual Vividness',\n",
       "   'authors': 'Barnett, B.; Andersen, L. M.; Fleming, S. M.; Dijkstra, N.',\n",
       "   'author_corresponding': 'Benjamin Barnett',\n",
       "   'author_corresponding_institution': 'University College London',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.11.30.518510.source.xml',\n",
       "   'abstract': 'Some conscious experiences are more vivid than others. Although perceptual vividness is a key component of human consciousness, how variation in this magnitude property is registered by the human brain is unknown. A striking feature of magnitudes in other psychological domains, such as number or reward, is the existence of neural magnitude codes that are independent from sensory contents. To test whether perceptual vividness similarly covaries with neural codes that are invariant to sensory content, we reanalysed existing MEG and fMRI data from two distinct studies, quantifying perceptual vividness via subjective ratings of awareness and visibility. Using representational similarity and decoding analyses, we find evidence for the existence of content-invariant neural codes for perceptual vividness distributed across visual, parietal, and frontal cortices. Our findings are consistent with a hypothesis that the subjective vividness of conscious experience is supported by neural signatures similar to magnitude codes in other cognitive domains.\\n\\nSignificance StatementThe vividness of conscious experience varies across different stimuli and contexts. Despite being a fundamental feature of conscious awareness, exactly how perceptual vividness is encoded in the human brain remains unclear. Neural codes underpinning magnitude in reward and numerosity domains have been shown to be unchanging as stimulus identity varies. Here, we test whether components of neural activity covarying with the magnitude of perceptual vividness are similarly independent of perceptual content in analyses of MEG and fMRI data. We find dynamic, content-invariant neural signatures of vividness in visual, parietal, and frontal cortices. Our findings introduce the surprising notion that neural signatures of conscious experience might follow similar coding principles to those found for magnitude properties of entirely different cognitive domains.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.12.01.518657',\n",
       "   'title': 'JMJD3 Deficiency in Midbrain Dopamine Neurons Disrupts Dopamine Synthesis and Aggravates Chronic Inflammatory Pain.',\n",
       "   'authors': 'He, X.-B.; Guo, F.; Zhang, W.; Le, W.; Zheng, Y.; Ma, Y.; Wu, Y.; Wang, H.; Zhou, Q.',\n",
       "   'author_corresponding': 'Xi-Biao He',\n",
       "   'author_corresponding_institution': 'Shanghai University of Medicine & Health Sciences',\n",
       "   'date': '2023-08-27',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/27/2022.12.01.518657.source.xml',\n",
       "   'abstract': 'Chronic pain is associated with midbrain dopamine levels. The molecules and mechanisms modulating this association remain to be elucidated. By using conditional knockout mice, we report that JMJD3 deficiency in midbrain dopamine neurons causes prolonged mechanical hyperalgesia regardless of sex and age. Both genetic defect and pharmaceutical inhibition of JMJD3 decrease dopamine level in midbrain and striatum, resulting from reduced tyrosine hydroxylase expression in midbrain dopamine neurons. Furthermore, epigenetic experiments reveal that JMJD3 is indispensable for the transcription of tyrosine hydroxylase through direct and indirect manners. These findings suggest JMJD3 and midbrain dopamine neurons as novel players and pharmaceutical targets for chronic pain regulation.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.11.29.518384',\n",
       "   'title': 'Single cell gene set scoring with nearest neighborgraph smoothed data (gssnng).',\n",
       "   'authors': 'Gibbs, D. L.; Strasser, M. K.; Huang, S.',\n",
       "   'author_corresponding': 'David L Gibbs',\n",
       "   'author_corresponding_institution': 'Institute for Systems Biology',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.11.29.518384.source.xml',\n",
       "   'abstract': 'Gene set scoring (or enrichment) is a common dimension reduction task in bioinformatics that can be focused on differences between groups or at the single sample level. Gene sets can represent biological functions, molecular pathways, cell identities, and more. Gene set scores are context dependent values that are useful for interpreting biological changes following experiments or perturbations. Single sample scoring produces a set of scores, one for each member of a group, which can be analyzed with statistical models that can include additional clinically important factors such as gender or age. However, the sparsity and technical noise of single cell expression measures create difficulties for these methods, which were originally designed for bulk expression profiling (microarrays, RNAseq). This can be greatly remedied by first applying a smoothing transformation that shares gene measure information within transcriptomic neighborhoods. In this work, we use the nearest neighbor graph of cells for matrix smoothing to produce high quality gene set scores on a per-cell, per-group, level which is useful for visualization and statistical analysis.\\n\\nAvailability and implementationThe gssnng software is available using the python package index (PyPI) and works with Scanpy AnnData objects. It can be installed using  pip install gssnng. More information and demo notebooks: See https://github.com/IlyaLab/gssnng',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.12.05.519135',\n",
       "   'title': 'DiffDomain enables identification of structurally reorganized topologically associating domains',\n",
       "   'authors': 'Hua, D.; Gu, M.; Zhang, X.; Du, Y.; Xie, H.; Qi, L.; Du, X.; Bai, Z.; Zhu, X.; Tian, D.',\n",
       "   'author_corresponding': 'Dechao Tian',\n",
       "   'author_corresponding_institution': 'Department of Biostatistics and Systems Biology, School of Public Health (Shenzhen), Sun Yat-sen University, Shenzhen, Guangdong 510275, China',\n",
       "   'date': '2023-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/23/2022.12.05.519135.source.xml',\n",
       "   'abstract': 'Topologically associating domains (TADs) are critical structural units in three-dimensional genome organization of mammalian genome. Dynamic reorganizations of TADs between health and disease states are associated with transcription and other essential genome functions. However, computational methods that can identify reorganized TADs are still in the early stages of development. Here, we present DiffDomain, an algorithm leveraging high-dimensional random matrix theory to identify structurally reorganized TADs using chromatin contact maps. Method comparison using multiple real Hi-C datasets reveals that DiffDomain outperforms alternative methods for FPRs, TPRs, and identifying a new subtype of reorganized TADs. The robustness of DiffDomain and its biological applications are demonstrated by applying on Hi-C data from different cell types and disease states. Identified reorganized TADs are associated with structural variations and changes in CTCF binding sites and other epigenomic changes. By applying to a single-cell Hi-C data from mouse neuronal development, DiffDomain can identify reorganized TADs between cell types with reasonable reproducibility using pseudo-bulk Hi-C data from as few as 100 cells per condition. Moreover, DiffDomain reveals that TADs have differential cell-to-population variability and heterogeneous cell-to-cell variability. Therefore, DiffDomain is a statistically sound method for better comparative analysis of TADs using both Hi-C and single-cell Hi-C data.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.12.04.519034',\n",
       "   'title': 'Moderate High Temperature is Beneficial or Detrimental Depending on Carbon Availability in the Green Alga Chlamydomonas reinhardtii',\n",
       "   'authors': 'Zhang, N.; Venn, B.; Bailey, C. E.; Xia, M.; Mattoon, E. M.; Muhlhaus, T.; Zhang, R.',\n",
       "   'author_corresponding': 'Ru Zhang',\n",
       "   'author_corresponding_institution': 'Donald Danforth Plant Science Center',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'plant biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.12.04.519034.source.xml',\n",
       "   'abstract': 'High temperatures impair plant and algal growth and reduce food and biofuel production, but the underlying mechanisms remain elusive. The unicellular green alga Chlamydomonas reinhardtii is a superior model to study heat responses in photosynthetic cells due to its fast growth rate, many similarities in cellular processes to land plants, simple and sequenced genome, and ample genetic and genomics resources. Chlamydomonas grows in light by photosynthesis and/or with the externally supplied organic carbon source, acetate. Most of the published research about Chlamydomonas heat responses used acetate-containing medium. Understanding how organic carbon sources affect heat responses is important for the algal industry but understudied. We cultivated Chlamydomonas wild-type cultures under highly controlled conditions in photobioreactors at control of 25{degrees}C, moderate high temperature of 35{degrees}C, or acute high temperature of 40{degrees}C with and without constant acetate supply for 1- or 4-days. Our results showed that 35{degrees}C increased algal growth with constant acetate supply but reduced algal growth without sufficient acetate. The overlooked and dynamic effects of 35{degrees}C could be explained by induced carbon metabolism, including acetate uptake and assimilation, glyoxylate cycle, gluconeogenesis pathways, and glycolysis. Acute high temperature at 40{degrees}C for more than 2 days was lethal to algal cultures with and without constant acetate supply. Our research provides insights to understand algal heat responses and help improve thermotolerance in photosynthetic cells.\\n\\nHighlightWe revealed the overlooked, dynamic effects of moderate high temperature in algae depending on carbon availability and demonstrated the importance of carbon metabolism in thermotolerance of photosynthetic cells.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.12.07.519272',\n",
       "   'title': 'Brain-wide oscillatory network encodes an aggressive internal state',\n",
       "   'authors': 'Grossman, Y. S.; Talbot, A.; Gallagher, N. M.; Thomas, G.; Fink, A. J.; Walder-Christensen, K. K.; Russo, S.; Carlson, D. E.; Dzirasa, K.',\n",
       "   'author_corresponding': 'Kafui Dzirasa',\n",
       "   'author_corresponding_institution': 'Duke University',\n",
       "   'date': '2023-08-21',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/21/2022.12.07.519272.source.xml',\n",
       "   'abstract': 'Social aggression is an innate behavior that can aid an organism in securing access to resources[1]. Aggression can also disrupt group function and reduce survival under conditions of behavioral pathology[2-4]. Since many brain regions contribute to multiple social behaviors[5-7], expanded knowledge of how the brain distinguishes between social states would enable the development of interventions that suppress aggression, while leaving other social behaviors intact. Here we show that a murine aggressive internal state is encoded by a brain-wide network. This network is organized by prominent theta (4-11Hz) and beta (14-30Hz) oscillations, leading from orbital frontal cortex and medial dorsal thalamus, and converging on ventral medial hypothalamus and medial amygdala. Activity in this network couples to brain-wide cellular firing, and the network is conserved in multiple contexts associated with aggression. Strikingly, network activity during social isolation encodes the trait aggressiveness of mice and causal cellular manipulations known to impact aggression bidirectionally regulate network activity. Finally, we show that the network plays a role in mediating aggressive behavior through causal mediation analysis. Thus, we define a brain-wide network that encodes an aggressive internal state within and across mice.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.12.08.519299',\n",
       "   'title': 'Explaining the primate extinction crisis: predictors of extinction risk and active threats',\n",
       "   'authors': 'Creighton, M. J. A.; Nunn, C. L.',\n",
       "   'author_corresponding': 'Maria J.A. Creighton',\n",
       "   'author_corresponding_institution': 'Duke University',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'evolutionary biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.12.08.519299.source.xml',\n",
       "   'abstract': 'Explaining why some species are disproportionately impacted by the extinction crisis is of critical importance for conservation biology as a science and for proactively protecting species that are likely to become threatened in the future. Using the most current data on threat status, population trends, and threat types for 446 primate species, we advance previous research on the determinants of extinction risk by including a wider array of phenotypic traits as predictors, filling gaps in these trait data using multiple imputation, and investigating the mechanisms that connect organismal traits to extinction risk. Our Bayesian phylogenetically controlled analyses reveal that insular species exhibit higher threat status, while those that are more omnivorous and live in larger groups have lower threat status. The same traits are not linked to risk when repeating our analyses with older IUCN data, which may suggest that the traits influencing species risk are changing as anthropogenic effects continue to transform natural landscapes. We also show that non-insular, larger-bodied, and arboreal species are more susceptible to key threats responsible for primate population declines. Collectively, these results provide new insights to the determinants of primate extinction and identify the mechanisms (i.e., threats) that link traits to extinction risk.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.12.12.519721',\n",
       "   'title': 'PaleoProPhyler: a reproducible pipeline for phylogenetic inference using ancient proteins',\n",
       "   'authors': 'Patramanis, I.; Ramos-Madrigal, J.; Cappellini, E.; Racimo, F.',\n",
       "   'author_corresponding': 'Ioannis Patramanis',\n",
       "   'author_corresponding_institution': 'Section for Molecular Ecology and Evolution , Globe Institute, University of Copenhagen',\n",
       "   'date': '2023-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'confirmatory results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/24/2022.12.12.519721.source.xml',\n",
       "   'abstract': 'Ancient proteins from fossilized or semi-fossilized remains can yield phylogenetic information at broad temporal horizons, in some cases even millions of years into the past. In recent years, peptides extracted from archaic hominins and long-extinct mega-fauna have enabled unprecedented insights into their evolutionary history. In contrast to the field of ancient DNA - where several computational methods exist to process and analyze sequencing data - few tools exist for handling ancient protein sequence data. Instead, most studies rely on loosely combined custom scripts, which makes it difficult to reproduce results or share methodologies across research groups. Here, we present PaleoProPhyler: a new fully reproducible pipeline for aligning ancient peptide data and subsequently performing phylogenetic analyses. The pipeline can not only process various forms of proteomic data, but also easily harness genetic data in different formats (CRAM, BAM, VCF) and translate it, allowing the user to create reference panels for phyloproteomic analyses. We describe the various steps of the pipeline and its many functionalities, and provide some examples of how to use it. PaleoProPhyler allows researchers with little bioinformatics experience to efficiently analyze palaeoproteomic sequences, so as to derive insights from this valuable source of evolutionary data.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.12.20.521208',\n",
       "   'title': 'Cylicins are a structural component of the sperm calyx being indispensable for male fertility in mice and human',\n",
       "   'authors': 'Schneider, S.; Kovacevic, A.; Mayer, M.; Dicke, A.-K.; Arevalo, L.; Koser, S. A.; Hansen, J. N.; Young, S.; Brenker, C.; Kliesch, S.; Wachten, D.; Kirfel, G.; Struenker, T.; Tuettelmann, F.; Schorle, H.',\n",
       "   'author_corresponding': 'Hubert Schorle',\n",
       "   'author_corresponding_institution': 'University Hospital Bonn, Institute of Pathology',\n",
       "   'date': '2023-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/26/2022.12.20.521208.source.xml',\n",
       "   'abstract': 'Cylicins are testis-specific proteins, which are exclusively expressed during spermiogenesis. In mice and humans, two Cylicins, the gonosomal X-linked Cylicin 1 (Cylc1/CYLC1) and the autosomal Cylicin 2 (Cylc2/CYLC2) genes have been identified. Cylicins are cytoskeletal proteins with an overall positive charge due to lysine-rich repeats. While Cylicins have been localized in the acrosomal region of round spermatids, they resemble a major component of the calyx within the perinuclear theca at the posterior part of mature sperm nuclei. However, the role of Cylicins during spermiogenesis has not yet been investigated. Here, we applied CRISPR/Cas9-mediated gene-editing in zygotes to establish Cylc1- and Cylc2-deficient mouse lines as a model to study the function of these proteins. Cylc1 deficiency resulted in male subfertility, whereas Cylc2-/-, Cylc1-/y Cylc2+/-, and Cylc1-/y Cylc2-/- males were infertile. Phenotypical characterization revealed that loss of Cylicins prevents proper calyx assembly during spermiogenesis. This results in decreased epididymal sperm counts, impaired shedding of excess cytoplasm, and severe structural malformations, ultimately resulting in impaired sperm motility. Furthermore, exome sequencing identified an infertile man with a hemizygous variant in CYLC1 and a heterozygous variant in CYLC2, displaying morphological abnormalities of the sperm including the absence of the acrosome. Thus, our study highlights the relevance and importance of Cylicins for spermiogenic remodeling and male fertility in human and mouse, and provides the basis for further studies on unraveling the complex molecular interactions between perinuclear theca proteins required during spermiogenesis.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/2022.12.20.521313',\n",
       "   'title': 'Benchmarking single-cell hashtag oligo demultiplexing methods',\n",
       "   'authors': 'Howitt, G.; Feng, Y.; Tobar, L.; Vassiliadis, D.; Hickey, P.; Dawson, M. A.; Ranganathan, S.; Shanthikumar, S.; Neeland, M.; Maksimovic, J.; Oshlack, A.',\n",
       "   'author_corresponding': 'Jovana Maksimovic',\n",
       "   'author_corresponding_institution': 'Peter MacCallum Cancer Centre, The University of Melbourne',\n",
       "   'date': '2023-08-25',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2023/08/25/2022.12.20.521313.source.xml',\n",
       "   'abstract': 'Sample multiplexing is often used to reduce cost and limit batch effects in single-cell RNA sequencing (scRNA-seq) experiments. A commonly used multiplexing technique involves tagging cells prior to pooling with a hashtag oligo (HTO) that can be sequenced along with the cells RNA to determine their sample of origin. Several tools have been developed to demultiplex HTO sequencing data and assign cells to samples. In this study, we critically assess the performance of seven HTO demultiplexing tools: hashedDrops, HTODemux, GMM-Demux, demuxmix, deMULTIplex, BFF and HashSolo. The comparison uses data sets where each sample has also been demultiplexed using genetic variants from the RNA, enabling comparison of HTO demultiplexing techniques against complementary data from the genetic \"ground truth\". We find that all methods perform similarly where HTO labelling is of high quality, but methods that assume a bimodal counts distribution perform poorly on lower quality data. We also suggest heuristic approaches for assessing the quality of HTO counts in a scRNA-seq experiment.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'}]}"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "response.json()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
